<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
	<head>
		<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
		<meta http-equiv="Content-Style-Type" content="text/css" />
		<meta name="generator" content="MediaWiki 1.14.0" />
		<meta name="keywords" content="TDM Gene Management System 6.0,ICISWiki,ICIS Technical Documentation 6.0,TDM Gene Management System 5.4,TDM Use-Cases Data Models and Schemata 6.0" />
		<link rel="shortcut icon" href="../../../../misc/favicon.ico" />
		<link rel="search" type="application/opensearchdescription+xml" href="../../../../../opensearch_desc.php" title="ICISWiki (en)" />
		<link title="Creative Commons" type="application/rdf+xml" href="../../../../articles/t/d/m/TDM_Gene_Management_System_6.0_7e92.html" rel="alternate meta" />
		<link rel="copyright" href="../../../../../COPYING.html" />
		<link rel="alternate" type="application/rss+xml" title="ICISWiki RSS Feed" href="http://localhost../../../../articles/r/e/c/Special%7ERecentChanges_e0d0.html" />
		<link rel="alternate" type="application/atom+xml" title="ICISWiki Atom Feed" href="http://localhost../../../../articles/r/e/c/Special%7ERecentChanges_e0d0.html" />
		<title>TDM Gene Management System 6.0 - ICISWiki</title>
		<link rel="stylesheet" href="../../../../skins/common/shared.css?195" type="text/css" media="screen" />
		<link rel="stylesheet" href="../../../../skins/common/commonPrint.css?195" type="text/css" media="print" />
		<link rel="stylesheet" href="../../../../skins/monobook/main.css?195" type="text/css" media="screen" />
		<!--[if lt IE 5.5000]><link rel="stylesheet" href="../../../../skins/monobook/IE50Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<!--[if IE 5.5000]><link rel="stylesheet" href="../../../../skins/monobook/IE55Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<!--[if IE 6]><link rel="stylesheet" href="../../../../skins/monobook/IE60Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<!--[if IE 7]><link rel="stylesheet" href="../../../../skins/monobook/IE70Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<link rel="stylesheet" href="../../../../skins/../../../../raw/MediaWiki%7ECommon.css?195" type="text/css" />
		<link rel="stylesheet" href="../../../../skins/../../../../raw/MediaWiki%7EPrint.css?195" type="text/css" media="print" />
		<link rel="stylesheet" href="../../../../skins/../../../../raw/MediaWiki%7EMonobook.css?195" type="text/css" />
		<link rel="stylesheet" href="../../../../skins/../../../../raw/gen.css?195" type="text/css" />
		<!--[if lt IE 7]><script type="text/javascript" src="../../../../skins/common/IEFixes.js?195"></script>
		<meta http-equiv="imagetoolbar" content="no" /><![endif]-->

		<script type= "text/javascript">/*<![CDATA[*/
		var skin = "monobook";
		var stylepath = "../../../../skins";
		var wgArticlePath = "../../../../$1";
		var wgScriptPath = "../../../../..";
		var wgScript = "/icis/index.php";
		var wgVariantArticlePath = false;
		var wgActionPaths = {};
		var wgServer = "http://localhost";
		var wgCanonicalNamespace = "";
		var wgCanonicalSpecialPageName = false;
		var wgNamespaceNumber = 0;
		var wgPageName = "TDM_Gene_Management_System_6.0";
		var wgTitle = "TDM Gene Management System 6.0";
		var wgAction = "view";
		var wgArticleId = "2908";
		var wgIsArticle = true;
		var wgUserName = null;
		var wgUserGroups = null;
		var wgUserLanguage = "en";
		var wgContentLanguage = "en";
		var wgBreakFrames = false;
		var wgCurRevisionId = "15523";
		var wgVersion = "1.14.0";
		var wgEnableAPI = true;
		var wgEnableWriteAPI = true;
		var wgSeparatorTransformTable = ["", ""];
		var wgDigitTransformTable = ["", ""];
		var wgRestrictionEdit = [];
		var wgRestrictionMove = [];
		/*]]>*/</script>

		<script type="text/javascript" src="../../../../skins/common/wikibits.js?195"><!-- wikibits js --></script>
		<!-- Head Scripts -->
		<script type="text/javascript" src="../../../../raw/gen.js"><!-- site js --></script>
	</head>
<body class="mediawiki ltr ns-0 ns-subject page-TDM_Gene_Management_System_6_0 skin-monobook">
	<div id="globalWrapper">
		<div id="column-content">
	<div id="content">
		<a name="top" id="top"></a>
				<h1 id="firstHeading" class="firstHeading">TDM Gene Management System 6.0</h1>
		<div id="bodyContent">
			<h3 id="siteSub">From ICISWiki</h3>
			<div id="contentSub"></div>
									<div id="jump-to-nav">Jump to: <a href="#column-one">navigation</a>, <a href="#searchInput">search</a></div>			<!-- start content -->
			<p align="center"><b><a href="../../../../articles/i/c/i/ICISWiki_eda0.html" title="ICISWiki">Back to Main Page</a> &gt; <a href="../../../../articles/i/c/i/ICIS_Technical_Documentation_6.0_44cc.html" title="ICIS Technical Documentation 6.0">ICIS 6.0 Home Page</a> &gt; <a href="../../../../articles/t/d/m/TDM_Use-Cases_Data_Models_and_Schemata_6.0_0b07.html" title="TDM Use-Cases Data Models and Schemata 6.0">ICIS 6.0 Models &amp; Schemata</a></b></p>
<p><font size="+2" color="blue">MANAGEMENT OF GENETIC DATA</font>
</p>
<table id="toc" class="toc" summary="Contents"><tr><td><div id="toctitle"><h2>Contents</h2></div>
<ul>
<li class="toclevel-1"><a href="#Acknowledgment"><span class="tocnumber">1</span> <span class="toctext">Acknowledgment</span></a></li>
<li class="toclevel-1"><a href="#Introduction"><span class="tocnumber">2</span> <span class="toctext">Introduction</span></a></li>
<li class="toclevel-1"><a href="#Molecular_Data"><span class="tocnumber">3</span> <span class="toctext">Molecular Data</span></a></li>
<li class="toclevel-1"><a href="#Molecular_Variants"><span class="tocnumber">4</span> <span class="toctext">Molecular Variants</span></a></li>
<li class="toclevel-1"><a href="#Derived_Data"><span class="tocnumber">5</span> <span class="toctext">Derived Data</span></a></li>
<li class="toclevel-1"><a href="#The_Gene_Management_System_Table_Structure"><span class="tocnumber">6</span> <span class="toctext">The Gene Management System Table Structure</span></a>
<ul>
<li class="toclevel-2"><a href="#Generation_Challenge_Programme_Domain_Models"><span class="tocnumber">6.1</span> <span class="toctext">Generation Challenge Programme Domain Models</span></a></li>
<li class="toclevel-2"><a href="#Table_Relationships"><span class="tocnumber">6.2</span> <span class="toctext">Table Relationships</span></a></li>
<li class="toclevel-2"><a href="#Table_Definitions"><span class="tocnumber">6.3</span> <span class="toctext">Table Definitions</span></a>
<ul>
<li class="toclevel-3"><a href="#GEMS_ENTITY"><span class="tocnumber">6.3.1</span> <span class="toctext">GEMS_ENTITY</span></a></li>
<li class="toclevel-3"><a href="#GEMS_NAMES"><span class="tocnumber">6.3.2</span> <span class="toctext">GEMS_NAMES</span></a></li>
<li class="toclevel-3"><a href="#GEMS_PUB"><span class="tocnumber">6.3.3</span> <span class="toctext">GEMS_PUB</span></a></li>
<li class="toclevel-3"><a href="#GEMS_ANNOTATION"><span class="tocnumber">6.3.4</span> <span class="toctext">GEMS_ANNOTATION</span></a></li>
<li class="toclevel-3"><a href="#GEMS_FEATURE_METADATA"><span class="tocnumber">6.3.5</span> <span class="toctext">GEMS_FEATURE_METADATA</span></a></li>
<li class="toclevel-3"><a href="#GEMS_EVIDENCE"><span class="tocnumber">6.3.6</span> <span class="toctext">GEMS_EVIDENCE</span></a></li>
<li class="toclevel-3"><a href="#GEMS_GENOMIC_FEATURE"><span class="tocnumber">6.3.7</span> <span class="toctext">GEMS_GENOMIC_FEATURE</span></a></li>
<li class="toclevel-3"><a href="#GEMS_LINKAGE_MAP"><span class="tocnumber">6.3.8</span> <span class="toctext">GEMS_LINKAGE_MAP</span></a></li>
<li class="toclevel-3"><a href="#GEMS_LOCUS"><span class="tocnumber">6.3.9</span> <span class="toctext">GEMS_LOCUS</span></a></li>
<li class="toclevel-3"><a href="#GEMS_GENOMIC_FEATURE_DETECTOR"><span class="tocnumber">6.3.10</span> <span class="toctext">GEMS_GENOMIC_FEATURE_DETECTOR</span></a></li>
<li class="toclevel-3"><a href="#GEMS_GENOMIC_FEATURE_DETECTION_PROTOCOL"><span class="tocnumber">6.3.11</span> <span class="toctext">GEMS_GENOMIC_FEATURE_DETECTION_PROTOCOL</span></a></li>
<li class="toclevel-3"><a href="#GEMS_GENOMIC_FEATURE_ASSAY"><span class="tocnumber">6.3.12</span> <span class="toctext">GEMS_GENOMIC_FEATURE_ASSAY</span></a></li>
<li class="toclevel-3"><a href="#GEMS_DETECTED_VARIANT"><span class="tocnumber">6.3.13</span> <span class="toctext">GEMS_DETECTED_VARIANT</span></a></li>
</ul>
</li>
<li class="toclevel-2"><a href="#Semantic_Correspondence_between_ICIS_GEMS_5.4_and_ICIS_GEMS_6.0"><span class="tocnumber">6.4</span> <span class="toctext">Semantic Correspondence between ICIS GEMS 5.4 and ICIS GEMS 6.0</span></a>
<ul>
<li class="toclevel-3"><a href="#Table_Mappings"><span class="tocnumber">6.4.1</span> <span class="toctext">Table Mappings</span></a></li>
<li class="toclevel-3"><a href="#Field_Mappings"><span class="tocnumber">6.4.2</span> <span class="toctext">Field Mappings</span></a></li>
</ul>
</li>
</ul>
</li>
<li class="toclevel-1"><a href="#Examples_of_Data_Input"><span class="tocnumber">7</span> <span class="toctext">Examples of Data Input</span></a></li>
<li class="toclevel-1"><a href="#Data_Output"><span class="tocnumber">8</span> <span class="toctext">Data Output</span></a></li>
<li class="toclevel-1"><a href="#Summary"><span class="tocnumber">9</span> <span class="toctext">Summary</span></a></li>
</ul>
</td></tr></table><script type="text/javascript"> if (window.showTocToggle) { var tocShowText = "show"; var tocHideText = "hide"; showTocToggle(); } </script>
<a name="Acknowledgment" id="Acknowledgment"></a><h2> <span class="mw-headline"> Acknowledgment </span></h2>
<p>Several sections of this page simply cross reference their counterparts on the <a href="../../../../articles/t/d/m/TDM_Gene_Management_System_5.4_b33d.html" title="TDM Gene Management System 5.4">ICIS 5.4 GEMS narratives</a>.  The ICIS 6.0 GEMS schema, however, is modified and follows more closely the nomenclature and architecture of the <a href="http://www.generationcp.org" class="external text" title="http://www.generationcp.org" rel="nofollow">Generation Challenge Programme</a> <a href="http://pantheon.generationcp.org/index.php?option=com_content&amp;task=view&amp;id=132&amp;Itemid=117" class="external text" title="http://pantheon.generationcp.org/index.php?option=com_content&amp;task=view&amp;id=132&amp;Itemid=117" rel="nofollow">Demeter domain model for genotyping data</a>.
</p>
<a name="Introduction" id="Introduction"></a><h2> <span class="mw-headline"> Introduction </span></h2>
<p>See <a href="../../../../articles/t/d/m/TDM_Gene_Management_System_5.4_b33d.html#Introduction" title="TDM Gene Management System 5.4">ICIS 5.4 GEMS Narrative Introduction</a>.
</p>
<a name="Molecular_Data" id="Molecular_Data"></a><h2> <span class="mw-headline"> Molecular Data </span></h2>
<p>See <a href="../../../../articles/t/d/m/TDM_Gene_Management_System_5.4_b33d.html#Molecular_Data" title="TDM Gene Management System 5.4">ICIS 5.4 GEMS Narrative Molecular Data</a>.
</p>
<a name="Molecular_Variants" id="Molecular_Variants"></a><h2> <span class="mw-headline"> Molecular Variants </span></h2>
<p>See <a href="../../../../articles/t/d/m/TDM_Gene_Management_System_5.4_b33d.html#Molecular_Variants" title="TDM Gene Management System 5.4">ICIS 5.4 GEMS Narrative Molecular Variants</a>.
</p>
<a name="Derived_Data" id="Derived_Data"></a><h2> <span class="mw-headline"> Derived Data </span></h2>
<p>See <a href="../../../../articles/t/d/m/TDM_Gene_Management_System_5.4_b33d.html#Derived_Data" title="TDM Gene Management System 5.4">ICIS 5.4 GEMS Narrative Derived Data</a>.
</p>
<a name="The_Gene_Management_System_Table_Structure" id="The_Gene_Management_System_Table_Structure"></a><h2> <span class="mw-headline"> The Gene Management System Table Structure </span></h2>
<a name="Generation_Challenge_Programme_Domain_Models" id="Generation_Challenge_Programme_Domain_Models"></a><h3> <span class="mw-headline"> Generation Challenge Programme Domain Models</span></h3>
<p>The ICIS 6.0 schema is largely inspired by the structure and class nomenclature of the Generation Challenge Programme <a href="http://pantheon.generationcp.org/index.php?option=com_content&amp;task=view&amp;id=132&amp;Itemid=117" class="external text" title="http://pantheon.generationcp.org/index.php?option=com_content&amp;task=view&amp;id=132&amp;Itemid=117" rel="nofollow">Genotype</a> and <a href="http://pantheon.generationcp.org/index.php?option=com_content&amp;task=view&amp;id=131&amp;Itemid=118" class="external text" title="http://pantheon.generationcp.org/index.php?option=com_content&amp;task=view&amp;id=131&amp;Itemid=118" rel="nofollow">Genomic Mapping</a> domain models. These models (as Unified Modeling Language diagrams, July 2007 Release 1.8) are replicated here below.
</p><p><b>Figure 1. GCP Genotype Model</b>
</p><p><a href="../../../../articles/g/c/p/File%7EGCP_Genotype_Model.png_28e1.html" class="image" title="Image:GCP Genotype Model.png"><img alt="Image:GCP Genotype Model.png" src="../../../../images/7/7c/GCP_Genotype_Model.png" width="1395" height="785" border="0" /></a>
</p><p><b>Figure 2. GCP Genomic Mapping Model</b>
</p><p><a href="../../../../articles/g/c/p/File%7EGCP_LinkageMap_Model.png_30a8.html" class="image" title="Image:GCP LinkageMap Model.png"><img alt="Image:GCP LinkageMap Model.png" src="../../../../images/3/3b/GCP_LinkageMap_Model.png" width="1033" height="635" border="0" /></a>
</p><p>GEMS uses the following table structure to store details on genotype domain model objects (table names and attributes follow the Generation Challenge Programme convention).
</p><p><b>Figure 3. ICIS GEMS 6.0 Schema</b>
<a href="../../../../articles/i/c/i/File%7EICIS_GEMS_6.0_Schema.png_d92f.html" class="image" title="Image:ICIS GEMS 6.0 Schema.png"><img alt="Image:ICIS GEMS 6.0 Schema.png" src="../../../../images/8/88/ICIS_GEMS_6.0_Schema.png" width="1030" height="758" border="0" /></a>
</p><p><b>Notation used:</b>
</p><p><a href="../../../../articles/g/e/m/File%7EGems_schema_notation.png_d2f8.html" class="image" title="Image:Gems schema notation.png"><img alt="Image:Gems schema notation.png" src="../../../../images/2/25/Gems_schema_notation.png" width="554" height="237" border="0" /></a>
</p>
<a name="Table_Relationships" id="Table_Relationships"></a><h3> <span class="mw-headline">Table Relationships</span></h3>
<p>The central table of GEMS 6.0 schema is the Entity table (gems_entity) which is the superclass which records all generic information about any data object in the GEMS. The <b>subclass</b> field of this table is an enumeration of each major class of actual GEMS entity: <em>LinkageMap</em>, <em>GenomicFeature</em>, etc.
</p>
<a name="Table_Definitions" id="Table_Definitions"></a><h3> <span class="mw-headline">Table Definitions</span></h3>
<a name="GEMS_ENTITY" id="GEMS_ENTITY"></a><h4> <span class="mw-headline"> GEMS_ENTITY </span></h4>
<p>The GEMS_ENTITY table contains one entry for every major entity object managed 
by GEMS and keeps track of all generic information about that Entity. In this way
the GEMS_ENTITY table is a direct implementation of the GCP Entity class. The
<b>subclass</b> field of the table partitions contents of the table (via enum values)
into the various non-overlapping classes of GEMS objects, largely described within the 
<a href="http://pantheon.generationcp.org/index.php?option=com_content&amp;task=view&amp;id=132&amp;Itemid=117" class="external text" title="http://pantheon.generationcp.org/index.php?option=com_content&amp;task=view&amp;id=132&amp;Itemid=117" rel="nofollow">GCP Domain Model for Genotyping</a>, 
including the following:
</p>
<ul><li> <b>GenomicFeature:</b> a specific segment (subsequence) of DNA in the genome (specified <i>Germplasm</i>) or generally within a class of organism(i.e. species, e.g. Oryza sativa) of interest. A <i>GenomicFeature</i> may (or may not) have a known location ("locus") on specified maps, a known content ("genotype") in specified <i>Germplasm</i> and/or a specified biological/technical biological (e.g. "gene") or technical (e.g. "SSR marker") function of interest.
</li><li> <b>LinkageMap:</b> a genetic or physical map of genomic loci
</li><li> <b>Locus:</b> experimentally observed assignment ("MapFeature") of a specified <i>GenomicFeature</i> to a specific <i>LinkageMap</i>
</li><li> <b>GenomicFeatureDetector:</b> a specific biological reagent and/or information (e.g. PCR primers) that detect a specific subset of possible <i>GenomicFeatures</i> in a genome.
</li><li> <b>GenomicFeatureDetectionProtocol:</b> non-biologically specific protocol for using a <i>GenomicFeatureDetector</i> in an experimental assay.
</li><li> <b>GenomicFeatureAssay:</b> the potentially many-to-many join of <i>GenomicFeatureDetector</i> with available <i>GenomicFeatureDetectionProtocols</i>
</li><li> <b>DetectedVariant:</b> one specified instance of <i>GenomicFeature</i> content ("allele") detected by a <i>GenomicFeatureDetector</i>. If the variant can be analytically interpreted to belong to a specified <i>GenomicFeature</i>, then it is considered in GCP domain model terms to be a <i>GenomicFeatureVariant</i>
</li></ul>
<p>The important rule to note is that although the GEMS_ENTITY table autoincrements its primary keys, any and all subclassed objects (and corresponding table entries) share the identical primary key to the GEMS_ENTITY to which they correspond (e.g. a given entry in the GEMS_LINKAGE_MAP table will have  the same primary key value as its corresponding entry in the GEMS_ENTITY table).
</p><p>The <b>type</b> field of the GEMS_ENTITY serves to designate more specific subtypes of entity within each subclass, for example, a <i>LinkageMap</i> could be a "GeneticMap" (versus a "PhysicalMap"). This will generally refer to an ontology.  the <b>description</b> field can be any text that meaningfully describes the object.
</p><p>The GEMS_ENTITY table has relations with several other tables that provide for the identification, citations and annotation (see the <a href="http://pantheon.generationcp.org/index.php?option=com_content&amp;task=view&amp;id=116&amp;Itemid=108" class="external text" title="http://pantheon.generationcp.org/index.php?option=com_content&amp;task=view&amp;id=116&amp;Itemid=108" rel="nofollow">GCP domain model for Entity</a>) that describe the given GEMS_ENTITY entry.
</p><p><b>Table 1. GEMS_ENTITY table</b>
</p>
<table border="1" cellpadding="2" width="100%">
<tr>
<th width="20%"> Fieldname
</th><th width="20%"> Description
</th><th width="60%"> Type
</th></tr>
<tr>
<td>gentity_id	</td><td>long, primary key, autoincrement 	</td><td>Unique ID for the Entity
</td></tr>
<tr>
<td>subclass	</td><td>enum (subclasses of entity as noted above) </td><td>Major subclass of GEMS Entity (e.g. LinkageMap, GenomicFeature, etc.)
</td></tr>
<tr>
<td>type</td><td>cvterm_id foreign key to Chado cv module cvterm table</td><td>Minor subclass of object (e.g. DArt marker type)
</td></tr>
<tr>
<td>description</td><td>varchar(255)	</td><td>Description of the specific GEMS Entity
</td></tr></table>
<a name="GEMS_NAMES" id="GEMS_NAMES"></a><h4> <span class="mw-headline"> GEMS_NAMES </span></h4>
<p>GEMS_NAMES table serves as a catalogue of marker detector names and molecular variant names. GOBJTYPE contains the table name where all other information regarding the name specified in the GNVAL field can be found. GNID is the unique identifier of a specific name. GOBJID contains the ID of the name in the table name specified in the GOBJTYPE.
</p><p>Ex.
</p>
<pre>GNNAMES_ID = 2715
GENTITY_ID = 1200
GNVAL = RM105_143
...
this entry indicates that "RM105_143" is a name with identified by 
unique id = 2715 in the GEMS_NAMES table, that is a name of the entity identified 
by the foreign key unique id = 1200 pointing to the GEMS_Entity table. 
</pre>
<p><b>Table 2. GEMS_NAMES table</b>
</p>
<table border="1" cellpadding="2" width="100%">
<tr>
<th width="20%"> Fieldname
</th><th width="20%"> Description
</th><th width="60%"> Type
</th></tr>
<tr>
<td>gnames_id</td><td>Long</td><td> Unique ID of the Name
</td></tr>
<tr>
<td>gentity_id</td><td>Long</td><td>Unique ID of GEMS_ENTITY to which the name belongs
</td></tr>
<tr>
<td>gntype</td><td>Long</td><td>Type of Name
</td></tr>
<tr>
<td>gnstat</td><td>Long</td><td> Status of Name (0- default value 1 - synonyms)
</td></tr>
<tr>
<td>gnval</td><td>Varchar(255)</td><td> Name Value
</td></tr>
<tr>
<td>gnlocn</td><td>Long</td><td>Location where Name was assigned
</td></tr>
<tr>
<td>gevidence_id</td><td>Long</td><td>Unique ID of GEMS_EVIDENCE documenting the support of this name assignment
</td></tr></table>
<a name="GEMS_PUB" id="GEMS_PUB"></a><h4> <span class="mw-headline"> GEMS_PUB </span></h4>
<p>The GEMS_PUB table is the join table between GEMS_ENTITY and the ICIS BIBLIOREFS table. 
</p><p><b>Table 3. GEMS_PUB table</b>
</p>
<table border="1" cellpadding="2" width="100%">
<tr>
<th width="20%"> Fieldname
</th><th width="20%"> Description
</th><th width="60%"> Type
</th></tr>
<tr>
<td>gentity_id</td><td>Long foreign key to the GEMS_ENTITY table</td><td>Unique ID for the GEMS_ENTITY described within the given publication
</td></tr>
<tr>
<td>pub_id</td><td>Long foreign key to the ICIS BIBLIOREFS table</td><td>Unique ID of the publication
</td></tr></table>
<p><br />
</p>
<a name="GEMS_ANNOTATION" id="GEMS_ANNOTATION"></a><h4> <span class="mw-headline"> GEMS_ANNOTATION </span></h4>
<p>The GEMS_ANNOTATION table contains the values of GEMS_ENTITY feature annotation (i.e. GCP Feature data attached to the Entity).  
</p><p><b>Table 4. GEMS_ANNOTATION table</b>
</p>
<table border="1" cellpadding="2" width="100%">
<tr>
<th width="20%"> Fieldname
</th><th width="20%"> Description
</th><th width="60%"> Type
</th></tr>
<tr>
<td>gannot_id	</td><td>Long, primary key 	</td><td>Unique ID for the annotation
</td></tr>
<tr>
<td>gentity_id	</td><td>Long foreign key to the GEMS_ENTITY primary key</td><td>Unique ID for the annotated GEMS_ENTITY
</td></tr>
<tr>
<td>gfmetadata</td><td>Long foreign key to the GEMS_FEATURE_METADATA primary key</td><td>Unique Property ID of the annotation type
</td></tr>
<tr>
<td>data_type</td><td>Long (foreign key to the Chado cvterm table, or just a constant(??))</td><td>Is a controlled vocabulary designating the data type of the feature value
</td></tr>
<tr>
<td>value</td><td>Varchar(255)	</td><td>Value, encoded as a string
</td></tr>
<tr>
<td>value_id</td><td>Long foreign key to a pertinent value table</td><td>May points to a pertinent entry in another pertinent table, based on the value data_type, if the value cannot be adequately represented in a simple string.
</td></tr>
<tr>
<td>rank</td><td>Long 	</td><td>Sequence number for a Feature value series (where the given annotation Feature value is a "vector" of values; rank 0 applies for a single valued feature)
</td></tr></table>
<p><br />
</p>
<a name="GEMS_FEATURE_METADATA" id="GEMS_FEATURE_METADATA"></a><h4> <span class="mw-headline"> GEMS_FEATURE_METADATA </span></h4>
<p>GEMS_FEATURE_METADATA table defines the Feature type (a.k.a. ICIS "property"), unit (a.k.a. ICIS scale) and supportingEvidence (a.k.a. ICIS method) associated with a given type of GEMS_ENTITY annotation (i.e. GEMS_ANNOTATION entry gmetadata_id foreign key).
</p><p><b>Table 5. GEMS_FEATURE_METADATA table</b>
</p>
<table border="1" cellpadding="2" width="100%">
<tr>
<th width="20%"> Fieldname
</th><th width="20%"> Description
</th><th width="60%"> Type
</th></tr>
<tr>
<td>gmetadata_id</td><td>Long primary key</td><td>Unique ID for the combination of annotation Feature type, unit and supportingEvidence
</td></tr>
<tr>
<td>type_id</td><td>Long foreign key into Chado cvterm table	</td><td>Unique ID for the "type" of annotated Feature
</td></tr>
<tr>
<td>unit_id	</td><td>Long foreign key into Chado cvterm table	</td><td>Unique ID for the "unit" of the value of the annotated Feature
</td></tr>
<tr>
<td>supporting_evidence_id</td><td>Long foreign key into GEMS_EVIDENCE table</td><td>Unique ID for the GEMS_EVIDENCE associated with the Feature
</td></tr></table>
<p><br />
</p>
<a name="GEMS_EVIDENCE" id="GEMS_EVIDENCE"></a><h4> <span class="mw-headline"> GEMS_EVIDENCE </span></h4>
<p>The GEMS_EVIDENCE table documents the Evidence supporting a given GEMS_ANNOTATION assertion, including the <b>source</b> of the evidence (e.g. publication, study, analysis program, etc.), <b>observation_date</b>, <b>score</b> and the identity of the <b>curator</b> who recorded the evidence.  
</p><p><b>Table 6. GEMS_EVIDENCE table</b>
</p>
<table border="1" cellpadding="2" width="100%">
<tr>
<th width="20%"> Fieldname
</th><th width="20%"> Description
</th><th width="60%"> Type
</th></tr>
<tr>
<td>gevidence_id</td><td>Long, primary key</td><td>Unique ID for the Evidence entry
</td></tr>
<tr>
<td>curator</td><td>Long foreign key into the ICIS user table 	</td><td>Curator of the specific evidence
</td></tr>
<tr>
<td>source_type</td><td>Long 	</td><td>Specifies the data type of the source_id (i.e. table identification for the source_id foreign key, e.g. BIBLIOREFS, STUDY, etc)
</td></tr>
<tr>
<td>source_id</td><td>Unique Property ID of the pertinent entry in the Evidence source table
</td></tr>
<tr>
<td>observation_date</td><td>date	</td><td>Date when the Feature Evidence was asserted
</td></tr>
<tr>
<td>score</td><td>varchar(64)	</td><td>Some source specific measurement of statistical confidence of the source assertion of the annotation Feature value
</td></tr></table>
<p><br />
</p>
<a name="GEMS_GENOMIC_FEATURE" id="GEMS_GENOMIC_FEATURE"></a><h4> <span class="mw-headline"> GEMS_GENOMIC_FEATURE </span></h4>
<p>GEMS_GENOMIC_FEATURE table documents the <i>LinkageMap</i> location, content and function of a specified DNA segment in a genome. Given that all of the documentation of this object may be represented in the GEMS_ENTITY table and its table relationships, the GEMS_GENOMIC_FEATURE may be represented solely as a <b>subclass</b> indexed set of entries in the GEMS_ENTITY table, hence this table is redundant (i.e. doesn't need to be implemented directly).
</p><p><b>Table 7. GEMS_GENOMIC_FEATURE table</b>
</p>
<table border="1" cellpadding="2" width="100%">
<tr>
<th width="20%"> Fieldname
</th><th width="20%"> Description
</th><th width="60%"> Type
</th></tr>
<tr>
<td>gfeature_id	</td><td>Long 	</td><td>Unique ID for the GenomicFeature
</td></tr></table>
<p><br />
</p>
<a name="GEMS_LINKAGE_MAP" id="GEMS_LINKAGE_MAP"></a><h4> <span class="mw-headline"> GEMS_LINKAGE_MAP </span></h4>
<p>GEMS_LINKAGE_MAP table contains information on genetic, physical, or other kinds of <i>LinkageMap</i> objects 
(see <a href="http://pantheon.generationcp.org/index.php?option=com_content&amp;task=view&amp;id=131&amp;Itemid=118" class="external text" title="http://pantheon.generationcp.org/index.php?option=com_content&amp;task=view&amp;id=131&amp;Itemid=118" rel="nofollow">GCP genomic map domain model description</a>).
</p><p><b>Table 8. GEMS_LINKAGE_MAP table</b>
</p>
<table border="1" cellpadding="2" width="100%">
<tr>
<th width="20%"> Fieldname
</th><th width="20%"> Description
</th><th width="60%"> Type
</th></tr>
<tr>
<td>gmap_id	</td><td>Long 	</td><td>Unique ID for the <i>LinkageMap</i>
</td></tr>
<tr>
<td>parent_map_id	</td><td>Long foreign key pointing reflexively into the GEMS_LINKAGE_MAP</td><td>Unique ID for the <i>LinkageMap</i> within which the given <i>LinkageMap</i> entry resides. For most linkage groups, this is the parent chromosome map. For chromosome maps, the given genome map (set).
</td></tr></table>
<a name="GEMS_LOCUS" id="GEMS_LOCUS"></a><h4> <span class="mw-headline"> GEMS_LOCUS </span></h4>
<p>GEMS_LOCUS table contains information on genetic, physical, or other kinds of <i>Locus</i> objects (GCP <em>MapFeature</em>)
(see <a href="http://pantheon.generationcp.org/index.php?option=com_content&amp;task=view&amp;id=131&amp;Itemid=118" class="external text" title="http://pantheon.generationcp.org/index.php?option=com_content&amp;task=view&amp;id=131&amp;Itemid=118" rel="nofollow">GCP genomic map domain model description</a>).
</p><p><b>Table 9. GEMS_LINKAGE_MAP table</b>
</p>
<table border="1" cellpadding="2" width="100%">
<tr>
<th width="20%"> Fieldname
</th><th width="20%"> Description
</th><th width="60%"> Type
</th></tr>
<tr>
<td>glocus_id	</td><td>Long, primary key	</td><td>Unique ID for the a <i>Locus</i>
</td></tr>
<tr>
<td>glmap_id	</td><td>Long foreign key to the LinkageMap</td><td>Unique ID for the a <i>LinkageMap</i>
</td></tr>
<tr>
<td>gfeature_id	</td><td>Long foreign key to the GenomicFeature </td><td>Unique ID for the a <i>GenomicFeature </i>
</td></tr>
</table>
<a name="GEMS_GENOMIC_FEATURE_DETECTOR" id="GEMS_GENOMIC_FEATURE_DETECTOR"></a><h4> <span class="mw-headline"> GEMS_GENOMIC_FEATURE_DETECTOR </span></h4>
<p>GEMS_GENOMIC_FEATURE_DETECTOR table documents the biologically specific reagents/information required to detect a specified subset of <i>GenomicFeature</i> entities in a genome. Given that all of the documentation of this object may be represented in the GEMS_ENTITY table and its table relationships, the GEMS_GENOMIC_FEATURE_DETECTOR may be represented solely as a <b>subclass</b> indexed set of entries in the GEMS_ENTITY table, hence this table is redundant (i.e. doesn't need to be implemented directly)
</p><p><b>Table 10. GEMS_GENOMIC_FEATURE_DETECTOR table</b>
</p>
<table border="1" cellpadding="2" width="100%">
<tr>
<th width="20%"> Fieldname
</th><th width="20%"> Description
</th><th width="60%"> Type
</th></tr>
<tr>
<td>gfdetector_id</td><td>Long, primary key</td><td>Unique ID of the Genomic Feature Detector
</td></tr></table>
<p><br />
</p>
<a name="GEMS_GENOMIC_FEATURE_DETECTION_PROTOCOL" id="GEMS_GENOMIC_FEATURE_DETECTION_PROTOCOL"></a><h4> <span class="mw-headline"> GEMS_GENOMIC_FEATURE_DETECTION_PROTOCOL </span></h4>
<p>GEMS_GENOMIC_FEATURE_DETECTION_PROTOCOL documents all the generic (non-biological) reagents and methodology for using instances of <i>GenomicFeatureDetector</i> to identify instances of <i>GenomicFeature</i> experimentally. Such reagents and protocols can typically be reused with many distinct <i>GenomicFeatureDetector</i> instances, hence, are reusable across several <i>GenomicFeatureAssays</i>.  Given that all of the documentation of this object may be represented in the GEMS_ENTITY table and its table relationships, the GEMS_GENOMIC_FEATURE_DETECTION_PROTOCOL may be represented solely as a <b>subclass</b> indexed set of entries in the GEMS_ENTITY table, hence this table is redundant (i.e. doesn't need to be implemented directly).
</p><p><b>Table 11. GEMS_GENOMIC_FEATURE_DETECTION_PROTOCOL table</b>
</p>
<table border="1" cellpadding="2" width="100%">
<tr>
<th width="20%"> Fieldname
</th><th width="20%"> Description
</th><th width="60%"> Type
</th></tr>
<tr>
<td>gfdprotocol_id	</td><td>Long, primary key</td><td>Unique identifier of the given protocol
</td></tr></table>
<p><br />
</p>
<a name="GEMS_GENOMIC_FEATURE_ASSAY" id="GEMS_GENOMIC_FEATURE_ASSAY"></a><h4> <span class="mw-headline"> GEMS_GENOMIC_FEATURE_ASSAY </span></h4>
<p>GEMS_GENOMIC_FEATURE_ASSAY (Polymorphic Detector) contains the different combination of <i>GenomicFeatureDetector</i> with <i>GenomicFeatureDetectionProtocol</i>. That is, a <i>GenomicFeatureDetector</i> can have one or more applied conditions (or protocols) for experimental assay. A protocol/condition can also be used for different <i>GenomicFeatureDetector</i> instances. This creates a many-to-many relationship between the markers and protocols, modeled by this join table.
</p><p><b>Table 12. GEMS_GENOMIC_FEATURE_ASSAY table</b>
</p>
<table border="1" cellpadding="2" width="100%">
<tr>
<th width="20%"> Fieldname
</th><th width="20%"> Description
</th><th width="60%"> Type
</th></tr>
<tr>
<td>gfassay_id</td><td>Long, primary key 	</td><td>Unique identifier
</td></tr>
<tr>
<td>gfdetector_id	</td><td>Long foreign key into the GEMS_GENOMIC_FEATURE_DETECTOR table	</td><td>Unique detector identifier
</td></tr>
<tr>
<td>gfdprotocol_id	</td><td>Long foreign key into the GEMS_GENOMIC_FEATURE_DETECTION_PROTOCOL table </td><td>Unique detector identifier
</td></tr></table>
<p><br />
</p>
<a name="GEMS_DETECTED_VARIANT" id="GEMS_DETECTED_VARIANT"></a><h4> <span class="mw-headline"> GEMS_DETECTED_VARIANT </span></h4>
<p>GEMS_DETECTED_VARIANT stores information on molecular variants associated with <i>GenomicFeature</i> instances that are detected by <i>GenomicFeatureDetector</i> systems. Each molecular variant is uniquely identified by the <b>gvariant_id</b> field. Each <b>gvariant_id</b> is associated with the <i>GenomicFeatureDetector</i> that resolves it, using the gfdetector_id foreign key. If analysis can assign the specific molecular variant to a specified <i>GenomicFeature</i>, then the <i>gfeature_id</i> field that links the variant to an entry in the GEMS_GENOMIC_FEATURE table (this makes the entry a full GCP <i>GenomicFeatureVariant</i> instance).   
</p><p><b>Table 13. GEMS_DETECTED_VARIANT table</b>
</p>
<table border="1" cellpadding="2" width="100%">
<tr>
<th width="20%"> Fieldname
</th><th width="20%"> Description
</th><th width="60%"> Type
</th></tr>
<tr>
<td>gvariant_id	</td><td>Long 	</td><td>Unique Molecular Variant ID
</td></tr>
<tr>
<td>gfdetector_id	</td><td>Long foreign key into the GEMS_GENOME_FEATURE_DETECTOR table 	</td><td><i>GenomicFeatureDetector</i> that identified the variant
</td></tr>
<tr>
<td>gfeature_id	</td><td>Long foreign key into the GEMS_GENOME_FEATURE table 		</td><td><i>GenomicFeature</i> to which the variant is specifically assigned (if any)
</td></tr></table>
<p><br />
</p>
<a name="Semantic_Correspondence_between_ICIS_GEMS_5.4_and_ICIS_GEMS_6.0" id="Semantic_Correspondence_between_ICIS_GEMS_5.4_and_ICIS_GEMS_6.0"></a><h3> <span class="mw-headline">Semantic Correspondence between ICIS GEMS 5.4 and ICIS GEMS 6.0 </span></h3>
<p>The table below attempts to map the tables and attributes of the ICIS GEMS 5.4 schema onto the above ICIS GEMS 6.0 schema. 
</p>
<a name="Table_Mappings" id="Table_Mappings"></a><h4> <span class="mw-headline">Table Mappings</span></h4>
<p><b>Table 14. Approximate Semantic Mapping of ICIS GEMS 5.4 Tables onto ICIS GEMS 6.0 Tables</b>
</p>
<table border="1" cellpadding="2" width="100%">
<tr>
<th width="50%"> ICIS 5.4 Table
</th><th width="50%"> ICIS 6.0 Table
</th></tr>
<tr>
<td>no counterpart in 5.4</td><td>GEMS_ENTITY
</td></tr>
<tr>
<td>GEMS_NAMES</td><td>GEMS_NAMES
</td></tr>
<tr>
<td>no counterpart in 5.4</td><td>GEMS_PUB (join table between GEMS_ENTITY and ICIS_BIBREFS)
</td></tr>
<tr>
<td>GEMS_COMP</td><td>GEMS_ANNOTATION used at GEMS_ENTITY level
</td></tr>
<tr>
<td>GEMS_PROP</td><td>GEMS_FEATURE_METADATA + Controlled vocabulary term foreign key for Property itself (Chado cvterm table)
</td></tr>
<tr>
<td>GEMS_SCALE</td><td>Controlled vocabulary term foreign key (Chado cvterm table)
</td></tr>
<tr>
<td>GEMS_METHOD</td><td>GEMS_EVIDENCE + Controlled vocabulary term foreign key for Method itself (Chado cvterm table)
</td></tr>
<tr>
<td>GEMS_MARKER_DETECTOR</td><td>GEMS_GENOMIC_FEATURE_DETECTOR
</td></tr>
<tr>
<td>GEMS_MV</td><td>GEMS_DETECTED_VARIANT
</td></tr>
<tr>
<td>GEMS_PD</td><td>GEMS_GENOMIC_FEATURE_ASSAY
</td></tr>
<tr>
<td>GEMS_PD_COMP</td><td> GEMS_GENOMIC_FEATURE_DETECTION_PROTOCOL
</td></tr>
<tr>
<td>no counterpart in 5.4</td><td>GEMS_LINKAGE_MAP
</td></tr>
<tr>
<td>GEMS_LOCUS</td><td>GEMS_GENOMIC_FEATURE for DNA identity with map location information stored in GEMS_LOCUS
</td></tr></table>
<a name="Field_Mappings" id="Field_Mappings"></a><h4> <span class="mw-headline">Field Mappings</span></h4>
<p><b>Table 6. Semantic Mapping of ICIS GEMS 5.4 onto ICIS GEMS 6.0 </b>
</p>
<table border="1" cellpadding="2" width="100%">
<tr>
<th width="25%"> ICIS 5.4 Table
</th><th width="25%"> ICIS 5.4 Field
</th><th width="25%"> ICIS 6.0 Table
</th><th width="25%"> ICIS 6.0 Field
</th></tr>
<tr>
<td>GEMS_NAMES</td><td>gnid 	</td><td>GEMS_NAMES</td><td>gnames_id
</td></tr>
<tr>
<td>GEMS_NAMES</td><td>gobjid 	</td><td>GEMS_NAMES</td><td>gentity_id
</td></tr>
<tr>
<td>not present</td><td>n/a 	</td><td>GEMS_NAMES</td><td>source_id (foreign key to GEMS_EVIDENCE table)
</td></tr>
<tr>
<td>GEMS_NAMES</td><td>gndate 	</td><td>GEMS_EVIDENCE</td><td>observation_date
</td></tr>
<tr>
<td>GEMS_NAMES</td><td>gnuid 	</td><td>GEMS_EVIDENCE</td><td>curator_id (foreign key to ICIS users table)
</td></tr>
<tr>
<td>not present</td><td>n/a 	</td><td>GEMS_EVIDENCE</td><td>source_type
</td></tr>
<tr>
<td>GEMS_NAMES</td><td>gnref 	</td><td>GEMS_EVIDENCE</td><td>source_id (foreign key to pertinent table, as per source_type)
</td></tr>
<tr>
<td>GEMS_COMP</td><td>cid	</td><td>GEMS_ANNOTATION</td><td>gannot_id
</td></tr>
<tr>
<td>GEMS_COMP</td><td>condid</td><td>GEMS_ANNOTATION</td><td>type_id (foreign key to cvterm table)
</td></tr>
<tr>
<td>GEMS_COMP</td><td>comid</td><td>GEMS_ENTITY</td><td>type
</td></tr>
<tr>
<td>GEMS_COMP</td><td>pid</td><td>GEMS_ANNOTATION</td><td>gmetadata_id
</td></tr>
<tr>
<td>GEMS_COMP</td><td>comval</td><td>GEMS_ANNOTATION</td><td>value
</td></tr>
<tr>
<td>GEMS_COMP</td><td>comuid</td><td>GEMS_EVIDENCE</td><td>curator_id
</td></tr>
<tr>
<td>GEMS_COMP</td><td>comref</td><td>GEMS_EVIDENCE</td><td>source_id (foreign key to ICIS BIBREFS)
</td></tr>
<tr>
<td>GEMS_PROP</td><td>Pid	</td><td>GEMS_FEATURE_METADATA</td><td>gmetadata_id
</td></tr>
<tr>
<td>GEMS_PROP</td><td>propid</td><td>GEMS_FEATURE_METADATA</td><td>type_id (foreign key to cvterm table)
</td></tr>
<tr>
<td>GEMS_PROP</td><td>propname</td><td>Chado cvterm table</td><td>name in cvterm table via type_id
</td></tr>
<tr>
<td>GEMS_PROP</td><td>scaleid</td><td>GEMS_FEATURE_METADATA</td><td>unit_id (foreign key to cvterm table)
</td></tr>
<tr>
<td>GEMS_PROP</td><td>methid</td><td>GEMS_FEATURE_METADATA</td><td>supportingEvidence_id (foreign key to GEMS_EVIDENCE table)
</td></tr>
<tr>
<td>GEMS_METHOD, GEMS_SCALE</td><td>all fields</td><td>Chado cvterm table</td><td>Controlled vocabulary terms with pertinent documentation
</td></tr>
<tr>
<td>GEMS_LOCUS</td><td>locusid</td><td>GEMS_LOCUS</td><td>glocus_id (see also GEMS_GENOMIC_FEATURE below)
</td></tr>
<tr>
<td>GEMS_LOCUS</td><td>chromosome</td><td>GEMS_LOCUS</td><td>glmap_id (foreign key to GEMS_LINKAGE_MAP table - chromosome treated as another map)
</td></tr>
<tr>
<td>GEMS_LOCUS</td><td>position</td><td>GEMS_ANNOTATION</td><td>locus metadata, including position data, is recorded as GEMS_ANNOTATION of the corresponding GEMS_ENTITY (with associated GEMS_FEATURE_METADATA and supporting GEMS_EVIDENCE)
</td></tr>
<tr>
<td>not present</td><td>n/a 	</td><td>GEMS_GENOMIC_FEATURE</td><td>genomic_locus_id foreign key to GEMS_LOCUS
</td></tr>
<tr>
<td>GEMS_MARKER_DETECTOR</td><td>mdid</td><td>GEMS_GENOMIC_FEATURE_DETECTOR</td><td>gfdetector_id
</td></tr>
<tr>
<td>GEMS_MARKER_DETECTOR</td><td>all other fields</td><td>GEMS_ANNOTATION</td><td>detector metadata recorded as GEMS_ANNOTATION of the corresponding GEMS_ENTITY (with associated GEMS_FEATURE_METADATA and supporting GEMS_EVIDENCE)
</td></tr>
<tr>
<td>GEMS_MV</td><td>mvid</td><td>GEMS_DETECTED_VARIANT</td><td>gvariant_id
</td></tr>
<tr>
<td>GEMS_MV</td><td>locusid</td><td>GEMS_DETECTED_VARIANT</td><td>gfeature_id (foreign key to GEMS_GENOMIC_FEATURE table)
</td></tr>
<tr>
<td>GEMS_MV</td><td>mvtype</td><td>GEMS_ENTITY</td><td>type
</td></tr>
<tr>
<td>GEMS_MV</td><td>mdid</td><td>GEMS_DETECTED_VARIANT</td><td>gfdetector_id (foreign key to GEMS_GENOMIC_FEATURE_DETECTOR table)
</td></tr>
<tr>
<td>GEMS_MV</td><td>all other fields</td><td>GEMS_ANNOTATION</td><td>GEMS_DETECTED_VARIANT metadata recorded as GEMS_ANNOTATION of the corresponding GEMS_ENTITY (with associated GEMS_FEATURE_METADATA and supporting GEMS_EVIDENCE)
</td></tr>
<tr>
<td>GEMS_PD</td><td>pdid</td><td>GEMS_GENOMIC_FEATURE_ASSAY</td><td>gfassay_id
</td></tr>
<tr>
<td>GEMS_PD</td><td>condid</td><td>GEMS_GENOMIC_FEATURE_ASSAY</td><td>gfdprotocol_id (foreign key to GEMS_GENOMIC_FEATURE_DETECTION_PROTOCOL table)
</td></tr>
<tr>
<td>GEMS_PD</td><td>mdid</td><td>GEMS_GENOMIC_FEATURE_ASSAY</td><td>gfdetector_id (foreign key to GEMS_GENOMIC_FEATURE_DETECTOR table)
</td></tr>
<tr>
<td>GEMS_PD</td><td>all other fields</td><td>GEMS_ANNOTATION</td><td>GEMS_GENOMIC_FEATURE_ASSAY metadata recorded as GEMS_ANNOTATION of the corresponding GEMS_ENTITY (with associated GEMS_FEATURE_METADATA and supporting GEMS_EVIDENCE)
</td></tr>
<tr>
<td>GEMS_PD_COMP</td><td>pdcomp</td><td>GEMS_GENOMIC_FEATURE_DETECTION_PROTOCOL</td><td>gfdprotocol_id
</td></tr>
<tr>
<td>GEMS_PD_COMP</td><td>pdid</td><td>not present</td><td>not needed, since GEMS_GENOMIC_FEATURE_ASSAY points to the protocol, not vice-versa
</td></tr>
<tr>
<td>GEMS_PD_COMP</td><td>all other fields</td><td>GEMS_ANNOTATION</td><td>GEMS_GENOMIC_FEATURE_DETECTION_PROTOCOL metadata recorded as GEMS_ANNOTATION of the corresponding GEMS_ENTITY (with associated GEMS_FEATURE_METADATA and supporting GEMS_EVIDENCE)
</td></tr>
<tr>
<td>not present</td><td>n/a</td><td>GEMS_PUB</td><td>gpub_id is a foreign key into the ICIS BIBREFS table
</td></tr>
<tr>
<td>not present</td><td>n/a</td><td>GEMS_PUB</td><td>gentity_id is a foreign key into the GEMS_ENTITY
</td></tr></table>
<a name="Examples_of_Data_Input" id="Examples_of_Data_Input"></a><h2> <span class="mw-headline">Examples of Data Input</span></h2>
<p>See <a href="../../../../articles/t/d/m/TDM_Gene_Management_System_5.4_b33d.html#Examples_of_Data_Input" title="TDM Gene Management System 5.4">ICIS 5.4 GEMS Narrative Examples of Data Input</a>.
</p>
<a name="Data_Output" id="Data_Output"></a><h2> <span class="mw-headline"> Data Output </span></h2>
<p>See <a href="../../../../articles/t/d/m/TDM_Gene_Management_System_5.4_b33d.html#Examples_of_Data_Output" title="TDM Gene Management System 5.4">ICIS 5.4 GEMS Narrative Examples of Data Output</a>.
</p>
<a name="Summary" id="Summary"></a><h2> <span class="mw-headline"> Summary </span></h2>
<p>See also <a href="../../../../articles/t/d/m/TDM_Gene_Management_System_5.4_b33d.html#Summary" title="TDM Gene Management System 5.4">ICIS 5.4 GEMS Narrative Summary</a>.
</p>
<!-- 
NewPP limit report
Preprocessor node count: 39/1000000
Post-expand include size: 0/2097152 bytes
Template argument size: 0/2097152 bytes
Expensive parser function count: 0/100
-->
<div class="printfooter">
Retrieved from "<a href="http://localhost../../../../articles/t/d/m/TDM_Gene_Management_System_6.0_7e92.html">http://localhost../../../../articles/t/d/m/TDM_Gene_Management_System_6.0_7e92.html</a>"</div>
						<!-- end content -->
						<div class="visualClear"></div>
		</div>
	</div>
		</div>
		<div id="column-one">
	<div id="p-cactions" class="portlet">
		<h5>Views</h5>
		<div class="pBody">
			<ul>
	
				 <li id="ca-nstab-main" class="selected"><a href="../../../../articles/t/d/m/TDM_Gene_Management_System_6.0_7e92.html" title="View the content page [c]" accesskey="c">Page</a></li>
				 <li id="ca-talk" class="new"><a href="../../../../articles/t/d/m/Talk%7ETDM_Gene_Management_System_6.0_8a10.html" title="Discussion about the content page [t]" accesskey="t">Discussion</a></li>
				 <li id="ca-viewsource"><a href="../../../../articles/t/d/m/TDM_Gene_Management_System_6.0_7e92.html" title="This page is protected.&#10;You can view its source [e]" accesskey="e">View source</a></li>
				 <li id="ca-history"><a href="../../../../articles/t/d/m/TDM_Gene_Management_System_6.0_7e92.html" title="Past versions of this page [h]" accesskey="h">History</a></li>			</ul>
		</div>
	</div>
	<div class="portlet" id="p-personal">
		<h5>Personal tools</h5>
		<div class="pBody">
			<ul>
				<li id="pt-login"><a href="../../../../articles/u/s/e/Special%7EUserLogin_94cd.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</a></li>
			</ul>
		</div>
	</div>
	<div class="portlet" id="p-logo">
		<a style="background-image: url(../../../../misc/icis-wiki3.png);" href="../../../../index.html" title="Visit the main page [z]" accesskey="z"></a>
	</div>
	<script type="text/javascript"> if (window.isMSIE55) fixalpha(); </script>
	<div class='generated-sidebar portlet' id='p-navigation'>
		<h5>Navigation</h5>
		<div class='pBody'>
			<ul>
				<li id="n-mainpage"><a href="../../../../index.html" title="Visit the main page [z]" accesskey="z">Main Page</a></li>
				<li id="n-portal"><a href="../../../../articles/c/o/m/ICISWiki%7ECommunity_Portal_7449.html" title="About the project, what you can do, where to find things">Community portal</a></li>
				<li id="n-recentchanges"><a href="../../../../articles/r/e/c/Special%7ERecentChanges_e0d0.html" title="The list of recent changes in the wiki [r]" accesskey="r">Recent changes</a></li>
				<li id="n-help"><a href="../../../../articles/c/o/n/Help%7EContents_22de.html" title="The place to find out">Help</a></li>
			</ul>
		</div>
	</div>
	<div id="p-search" class="portlet">
		<h5><label for="searchInput">Search</label></h5>
		<div id="searchBody" class="pBody">
			<form action="../../../../articles/s/e/a/Special%7ESearch_a8e5.html" id="searchform"><div>
				<input id="searchInput" name="search" type="text" title="Search ICISWiki [f]" accesskey="f" value="" />
				<input type='submit' name="go" class="searchButton" id="searchGoButton"	value="Go" title="Go to a page with this exact name if exists" />&nbsp;
				<input type='submit' name="fulltext" class="searchButton" id="mw-searchButton" value="Search" title="Search the pages for this text" />
			</div></form>
		</div>
	</div>
	<div class="portlet" id="p-tb">
		<h5>Toolbox</h5>
		<div class="pBody">
			<ul>
				<li id="t-whatlinkshere"><a href="../../../../articles/w/h/a/Special%7EWhatLinksHere_TDM_Gene_Management_System_6.0_6bd5.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</a></li>
				<li id="t-recentchangeslinked"><a href="../../../../articles/r/e/c/Special%7ERecentChangesLinked_TDM_Gene_Management_System_6.0_97be.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</a></li>
<li id="t-specialpages"><a href="../../../../articles/s/p/e/Special%7ESpecialPages_f12b.html" title="List of all special pages [q]" accesskey="q">Special pages</a></li>
				<li id="t-print"><a href="../../../../articles/t/d/m/TDM_Gene_Management_System_6.0_7e92.html" title="Printable version of this page [p]" accesskey="p">Printable version</a></li>				<li id="t-permalink"><a href="../../../../articles/t/d/m/TDM_Gene_Management_System_6.0_7e92.html" title="Permanent link to this version of the page">Permanent link</a></li>			</ul>
		</div>
	</div>
		</div><!-- end of the left (by default at least) column -->
			<div class="visualClear"></div>
			<div id="footer">
				<div id="f-poweredbyico"><a href="http://www.mediawiki.org/"><img src="../../../../skins/common/images/poweredby_mediawiki_88x31.png" alt="Powered by MediaWiki" /></a></div>
				<div id="f-copyrightico"><a href="../../../../../COPYING.html"><img src="/icis/skins/common/images/gnu-fdl.png" alt='GNU Free Documentation License 1.2' /></a></div>
			<ul id="f-list">
					<li id="credits">This page was last modified 01:53, 19 February 2009 by <a href="../../../../articles/m/a/n/User%7EManacleto_937a.html" title="User:Manacleto">Mylah Rystie U. Anacleto</a>. Based on work by <a href="../../../../articles/r/v/a/User%7ERvalerio_c9de.html" title="User:Rvalerio">Rowena Valerio</a>&#32;and&#32;<a href="../../../../articles/r/b/r/User%7ERbruskiewich_a89c.html" title="User:Rbruskiewich">Richard Bruskiewich</a>.</li>
					<li id="copyright">Content is available under <a href="../../../../../COPYING.html" class="external " title="../../../../../COPYING.html">GNU Free Documentation License 1.2</a>.</li>
					<li id="privacy"><a href="../../../../articles/p/r/i/ICISWiki%7EPrivacy_policy_0cbf.html" title="ICISWiki:Privacy policy">Privacy policy</a></li>
					<li id="about"><a href="../../../../articles/a/b/o/ICISWiki%7EAbout_53aa.html" title="ICISWiki:About">About ICISWiki</a></li>
					<li id="disclaimer"><a href="../../../../articles/g/e/n/ICISWiki%7EGeneral_disclaimer_9129.html" title="ICISWiki:General disclaimer">Disclaimers</a></li>
			</ul>
		</div>
</div>

		<script type="text/javascript">if (window.runOnloadHook) runOnloadHook();</script>
<!-- Served in 205.815 secs. --></body></html>
