<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
	<head>
		<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
		<meta http-equiv="Content-Style-Type" content="text/css" />
		<meta name="generator" content="MediaWiki 1.14.0" />
		<meta name="keywords" content="TDM GMS Overview 5.5,TDM GMS Browse" />
		<link rel="shortcut icon" href="../../../../misc/favicon.ico" />
		<link rel="search" type="application/opensearchdescription+xml" href="../../../../../opensearch_desc.php" title="ICISWiki (en)" />
		<link title="Creative Commons" type="application/rdf+xml" href="../../../../articles/t/d/m/TDM_GMS_Overview_5.5_1a1a.html" rel="alternate meta" />
		<link rel="copyright" href="../../../../../COPYING.html" />
		<link rel="alternate" type="application/rss+xml" title="ICISWiki RSS Feed" href="http://localhost../../../../articles/r/e/c/Special%7ERecentChanges_e0d0.html" />
		<link rel="alternate" type="application/atom+xml" title="ICISWiki Atom Feed" href="http://localhost../../../../articles/r/e/c/Special%7ERecentChanges_e0d0.html" />
		<title>TDM GMS Overview 5.5 - ICISWiki</title>
		<link rel="stylesheet" href="../../../../skins/common/shared.css?195" type="text/css" media="screen" />
		<link rel="stylesheet" href="../../../../skins/common/commonPrint.css?195" type="text/css" media="print" />
		<link rel="stylesheet" href="../../../../skins/monobook/main.css?195" type="text/css" media="screen" />
		<!--[if lt IE 5.5000]><link rel="stylesheet" href="../../../../skins/monobook/IE50Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<!--[if IE 5.5000]><link rel="stylesheet" href="../../../../skins/monobook/IE55Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<!--[if IE 6]><link rel="stylesheet" href="../../../../skins/monobook/IE60Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<!--[if IE 7]><link rel="stylesheet" href="../../../../skins/monobook/IE70Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<link rel="stylesheet" href="../../../../skins/../../../../raw/MediaWiki%7ECommon.css?195" type="text/css" />
		<link rel="stylesheet" href="../../../../skins/../../../../raw/MediaWiki%7EPrint.css?195" type="text/css" media="print" />
		<link rel="stylesheet" href="../../../../skins/../../../../raw/MediaWiki%7EMonobook.css?195" type="text/css" />
		<link rel="stylesheet" href="../../../../skins/../../../../raw/gen.css?195" type="text/css" />
		<!--[if lt IE 7]><script type="text/javascript" src="../../../../skins/common/IEFixes.js?195"></script>
		<meta http-equiv="imagetoolbar" content="no" /><![endif]-->

		<script type= "text/javascript">/*<![CDATA[*/
		var skin = "monobook";
		var stylepath = "../../../../skins";
		var wgArticlePath = "../../../../$1";
		var wgScriptPath = "../../../../..";
		var wgScript = "/icis/index.php";
		var wgVariantArticlePath = false;
		var wgActionPaths = {};
		var wgServer = "http://localhost";
		var wgCanonicalNamespace = "";
		var wgCanonicalSpecialPageName = false;
		var wgNamespaceNumber = 0;
		var wgPageName = "TDM_GMS_Overview_5.5";
		var wgTitle = "TDM GMS Overview 5.5";
		var wgAction = "view";
		var wgArticleId = "3884";
		var wgIsArticle = true;
		var wgUserName = null;
		var wgUserGroups = null;
		var wgUserLanguage = "en";
		var wgContentLanguage = "en";
		var wgBreakFrames = false;
		var wgCurRevisionId = "15743";
		var wgVersion = "1.14.0";
		var wgEnableAPI = true;
		var wgEnableWriteAPI = true;
		var wgSeparatorTransformTable = ["", ""];
		var wgDigitTransformTable = ["", ""];
		var wgRestrictionEdit = [];
		var wgRestrictionMove = [];
		/*]]>*/</script>

		<script type="text/javascript" src="../../../../skins/common/wikibits.js?195"><!-- wikibits js --></script>
		<!-- Head Scripts -->
		<script type="text/javascript" src="../../../../raw/gen.js"><!-- site js --></script>
	</head>
<body class="mediawiki ltr ns-0 ns-subject page-TDM_GMS_Overview_5_5 skin-monobook">
	<div id="globalWrapper">
		<div id="column-content">
	<div id="content">
		<a name="top" id="top"></a>
				<h1 id="firstHeading" class="firstHeading">TDM GMS Overview 5.5</h1>
		<div id="bodyContent">
			<h3 id="siteSub">From ICISWiki</h3>
			<div id="contentSub"></div>
									<div id="jump-to-nav">Jump to: <a href="#column-one">navigation</a>, <a href="#searchInput">search</a></div>			<!-- start content -->
			<table id="toc" class="toc" summary="Contents"><tr><td><div id="toctitle"><h2>Contents</h2></div>
<ul>
<li class="toclevel-1"><a href="#THE_GENEALOGY_MANAGEMENT_SYSTEM"><span class="tocnumber">1</span> <span class="toctext">THE GENEALOGY MANAGEMENT SYSTEM</span></a>
<ul>
<li class="toclevel-2"><a href="#INTRODUCTION"><span class="tocnumber">1.1</span> <span class="toctext">INTRODUCTION</span></a>
<ul>
<li class="toclevel-3"><a href="#Unique_Identification_of_Germplasm"><span class="tocnumber">1.1.1</span> <span class="toctext">Unique Identification of Germplasm</span></a></li>
</ul>
</li>
<li class="toclevel-2"><a href="#LOGICAL_STRUCTURE_OF_THE_GENEALOGY_MANAGEMENT_SYSTEM"><span class="tocnumber">1.2</span> <span class="toctext">LOGICAL STRUCTURE OF THE GENEALOGY MANAGEMENT SYSTEM</span></a>
<ul>
<li class="toclevel-3"><a href="#Germplasm_Information"><span class="tocnumber">1.2.1</span> <span class="toctext">Germplasm Information</span></a></li>
<li class="toclevel-3"><a href="#Link_Information"><span class="tocnumber">1.2.2</span> <span class="toctext">Link Information</span></a></li>
<li class="toclevel-3"><a href="#Name_Information"><span class="tocnumber">1.2.3</span> <span class="toctext">Name Information</span></a></li>
<li class="toclevel-3"><a href="#Attribute_Information"><span class="tocnumber">1.2.4</span> <span class="toctext">Attribute Information</span></a></li>
</ul>
</li>
<li class="toclevel-2"><a href="#EXAMPLES_OF_STORAGE_OF_GENEALOGY_INFORMATION"><span class="tocnumber">1.3</span> <span class="toctext">EXAMPLES OF STORAGE OF GENEALOGY INFORMATION</span></a>
<ul>
<li class="toclevel-3"><a href="#Genealogy_Management_for_a_Modified_Pedigree_Breeding_Program"><span class="tocnumber">1.3.1</span> <span class="toctext">Genealogy Management for a Modified Pedigree Breeding Program</span></a>
<ul>
<li class="toclevel-4"><a href="#Generative_methods"><span class="tocnumber">1.3.1.1</span> <span class="toctext">Generative methods</span></a></li>
<li class="toclevel-4"><a href="#Derivative_methods"><span class="tocnumber">1.3.1.2</span> <span class="toctext">Derivative methods</span></a></li>
<li class="toclevel-4"><a href="#Maintenance_methods"><span class="tocnumber">1.3.1.3</span> <span class="toctext">Maintenance methods</span></a></li>
</ul>
</li>
<li class="toclevel-3"><a href="#Genealogy_Management_for_An_Open_Recurrent_Selection_Method"><span class="tocnumber">1.3.2</span> <span class="toctext">Genealogy Management for An Open Recurrent Selection Method</span></a></li>
<li class="toclevel-3"><a href="#Genealogy_Management_for_a_Reciprocal_Half-Sib_Recurrent_Selection_Method"><span class="tocnumber">1.3.3</span> <span class="toctext">Genealogy Management for a Reciprocal Half-Sib Recurrent Selection Method</span></a></li>
<li class="toclevel-3"><a href="#Genealogy_for_a_polycross"><span class="tocnumber">1.3.4</span> <span class="toctext">Genealogy for a polycross</span></a></li>
<li class="toclevel-3"><a href="#Genealogy_for_a_half_diallel"><span class="tocnumber">1.3.5</span> <span class="toctext">Genealogy for a half diallel</span></a></li>
<li class="toclevel-3"><a href="#Genealogy_of_Rice_Variety_IR64"><span class="tocnumber">1.3.6</span> <span class="toctext">Genealogy of Rice Variety IR64</span></a></li>
</ul>
</li>
<li class="toclevel-2"><a href="#REMOTE_ALLOCATION_OF_GERMPLASM_IDENTIFIERS"><span class="tocnumber">1.4</span> <span class="toctext">REMOTE ALLOCATION OF GERMPLASM IDENTIFIERS</span></a></li>
<li class="toclevel-2"><a href="#UPDATING_THE_CENTRAL_GMS"><span class="tocnumber">1.5</span> <span class="toctext">UPDATING THE CENTRAL GMS</span></a></li>
<li class="toclevel-2"><a href="#REFERENCES"><span class="tocnumber">1.6</span> <span class="toctext">REFERENCES</span></a></li>
</ul>
</li>
</ul>
</td></tr></table><script type="text/javascript"> if (window.showTocToggle) { var tocShowText = "show"; var tocHideText = "hide"; showTocToggle(); } </script>
<a name="THE_GENEALOGY_MANAGEMENT_SYSTEM" id="THE_GENEALOGY_MANAGEMENT_SYSTEM"></a><h1> <span class="mw-headline"> THE GENEALOGY MANAGEMENT SYSTEM </span></h1>
<a name="INTRODUCTION" id="INTRODUCTION"></a><h2> <span class="mw-headline">INTRODUCTION</span></h2>
<p>The ICIS project seeks to develop a single data management model which can be applied to a wide variety of crops. This model manages information on genealogy, nomenclature, evaluation and characterization of germplasm for a specific crop. The central idea behind the model is that of unique identification of germplasm and management of the homonyms and synonyms which arise naturally in the germplasm development and utilization process. ICIS comprises several modules dedicated to different aspects of the data management problem.
</p><p>The core of ICIS is a common genealogical data model called the Genealogy Management System (GMS) which must be sufficiently well designed and universal to accommodate a wide range of crops.
</p><p>The functions of GMS are to:
</p>
<table border="0" cellspacing="2" width="100%">
<tr>
<td>a)
</td><td>assign and maintain unique germplasm identification,
</td></tr>
<tr>
<td>b)
</td><td>retain and manage information on genealogy and to
</td></tr>
<tr>
<td>c)
</td><td>manage nomenclature and chronology of germplasm development.
</td><td>&nbsp;
</td></tr></table>
<p>The genealogical core links to various applications required by individual users. Some of the applications will add genealogical data to the GMS, while others will be analytical, for example, displaying genealogies or calculating <a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html#Coefficients_of_Parentage" title="TDM GMS Browse">coefficients of parentage</a>.
</p>
<a name="Unique_Identification_of_Germplasm" id="Unique_Identification_of_Germplasm"></a><h3> <span class="mw-headline">Unique Identification of Germplasm</span></h3>
<p>Since ICIS is an international information system, applications which build or capture genealogical data need to be able to operate independently from site to site and remotely from the central GMS. This presents problems of uniqueness of germplasm identifiers: two remote users must not assign the same identifiers to different germplasm. This problem is resolved by defining a private, or user, GMS for each remote user. The unique identifier is composed of a crop code, followed by a curated user identification number and then by a locally assigned germplasm identifier. For example the rice variety APO has unique identifier IRIS-12-1426 because it was the 1426th germplasm in the local rice database of user number 12. APO now has central GID 204538, but the unique identifier is always kept and never re-used. The central GID has a one-to-one relationship with the unique identifier and so is also unique.
</p>
<a name="LOGICAL_STRUCTURE_OF_THE_GENEALOGY_MANAGEMENT_SYSTEM" id="LOGICAL_STRUCTURE_OF_THE_GENEALOGY_MANAGEMENT_SYSTEM"></a><h2> <span class="mw-headline">LOGICAL STRUCTURE OF THE GENEALOGY MANAGEMENT SYSTEM</span></h2>
<p>In GMS, all specific instances of germplasm are identified by a unique number called the GERMPLASM_ID (GID). Specific instances of germplasm can be thought of as equivalent to samples of seed which exist or existed at some time, produced by some process or germplasm genesis. In GMS these processes are defined as methods and pertinent information on the genesis of the specific instances are linked to the names via the GID. These methods are classified as those that increase, maintain or reduce genetic variability and are termed generative, maintenance or derivative methods, respectively. The genetic consequences of these methods depend on the reproductive behavior of the species (Table 2.2.a), the genetic structure of the populations which are being manipulated and the breeding strategies being used to achieve genetic change (Annex 2.2.a).
</p><p>The most common generative processes are the results of crossing individuals or populations and all methods can be included in a general model for crossing practices in plants (Table 2.2.b). The bases for the classification contained in this model is that the source of the female gametes is determined by the source of the seed, which can be seed from a single plant, a selected set of plants or a random set of plants. The source of the pollen needs to be determined but can be from the same plant, another plant, a selected set of plants or random set of plants. The generated germplasm frequently combines material from different sources and as such tends to increase genetic variation or diversity. Information linking these instances of germplasm to their progenitors is retained in the GMS, as well as information on the method of generation of the germplasm.<br /><br />
</p>
<table border="1" cellspacing="0" cellpadding="2" width="60%" align="left">
<tr>
<td colspan="2"><b>Table 2.2.a: Reproductive Systems in Crop Plants</b>
</td></tr>
<tr>
<td>Self Pollinated<sup>1</sup>
</td><td>Inbreeding, self pollinated or self fertilised species.<br />Examples: Wheat, Rice, Barley
</td></tr>
<tr>
<td>Predominantly Self Pollinated<sup>2</sup>
</td><td>Examples: Cotton, Pigeonpea, Sorghum
</td></tr>
<tr>
<td>Open Pollinated<sup>1</sup>
</td><td>Outbreeding, cross pollinated or cross fertilised species.<br />Examples: Maize, Pearl Millet, Cucurbits
</td></tr>
<tr>
<td>Self Incompatible<sup>3</sup>
</td><td>Self incompatible species.<br />Examples: Rye, White Clover, Papaya
</td></tr>
<tr>
<td>Vegetatively Propagated<sup>4</sup>
</td><td>Vegetatively or clonally propagated species.<br />Examples: Potatoes, Casava, Yam, Taro, Sugar Cane, Pineapple, Strawberries
</td></tr>
<tr>
<td>Apomictic<sup>5</sup>
</td><td>Apomitic (produce seed by asexual means) species.<br />Examples: Green Panic, Buffel grass
</td></tr>
</table><br clear="all" />
<p><sup>1</sup> Catered for in the ICIS Methods Table.<br /><sup>2</sup> Probably catered for as breeding methods usually follow those for Self Pollinated crops.<br /><sup>3</sup> Probably catered for by methods suitable for Open Pollinated crops.<br /><sup>4</sup> Generative and derivative and most maintenance methods catered for as these crops will be either self or cross fertilised in their mode of sexual reproduction.<br /><sup>5</sup> Probably catered for by methods suitable for Vegetatively Propagated crops<br /><br />
</p><p>Other items of germplasm arise by derivative processes from a single source. Derivative methods are divided into two categories, derivative germplasm development methods (DER) which tend to refine and reduce genetic variation and germplasm management methods (MAN) which attempt to preserve variation at the level of the source. Common derivative processes are selection and seed increase. Seed increase attempts to maintain genetic variability whereas selection tends to refine and reduce genetic variation. Both lead to instances of germplasm which are closely related both to the source and to different derivations from a common source. All instances of germplasm derived from a single source form a related group (or “family”) of germplasm. Information on the method of derivation of such germplasm is also stored. Management methods attempt to maintain genetic variation and leads to instances of germplasm which are closely related, preferably identical genetically, to the common source.<br />
</p><p>A full list of methods classified by breeding method of the crops are given in Annex 2.2.a and a full description and listing of the METHODS TABLE are given in Annex 2.2.b. The methods supplied in GMS are a compromise between giving a parsimonious set derived strictly from the model given in Table 2.2.b and an attempt to define every possible method that has or could be used. The first option does not provide enough information to crop coordinators, local coordinators or users. The second swamps users with too much information and a bewildering array of choices. Providing users with the full menu of methods provided in Annex 2.2.b would be counter productive (there are too many producing confusion) and ICIS will enable crop and local coordinators to select a subset for the crop or project that the service. A warning screen should be displayed asking users to explore already defined protocols and adding these to their menu before defining their own.<br /><br />
</p>
<table border="1" cellspacing="0" cellpadding="2" width="90%">
<tr>
<td colspan="4"><b>Table 2.2.b: Schematic outline of a general model for crossing practices in plants.</b>
</td></tr>
<tr>
<td colspan="4">The bases for the classification contained in this model is that the source of the female gametes is determined by the source of the seed, which can be a single seed or seed bulked from a single plant, a selected set of plants or a random set of plants. The source of the pollen needs to be determined but can be from the same plant, another plant, a selected set of plants or a random set of plants. The genetic consequences of these crosses or selections depends on the reproductive behaviour of the species.
</td></tr>
<tr>
<td align="center">Female Source
</td><td align="center">Male Source
</td><td align="center">Generative Methods
</td><td align="center">Derivative Methods
</td></tr>
<tr>
<td align="center" rowspan="5">Single Plant
</td></tr>
<tr>
<td>Same Single Plant
</td><td>40(G) Selfing
</td><td>208(S), 505(O) Single Seed Descent<br />60(G) Plant Identification<br />205(S), 504(O) Single Plant Selection<br />209(S), 506(O) Restorer Selection<br />210(S), Maintainer Selection
</td></tr>
<tr>
<td>Different Single Plant
</td><td>101(S),401(O) Single Cross<br />102(S),402(O) Three-way Cross<br />103(I),403(O) Double Cross<br />404(O) Full diallel cross<br />405(O) Full diallel cross bulked<br />406(O) Half diallel cross<br />407(O) Half diallel cross bulked<br />408(O) Partial diallel cross<br />409(O) Partial diallel cross bulked<br />424(O) Convergent Cross<br />425(O) Partial Plus Diallel Cross<br />107(S) Backcross<br />108(S) Backcross Recessive<br />109(S) Interspecific Cross<br />416(O) Narrow based tester, line CF
</td><td>&nbsp;
</td></tr>
<tr>
<td>Selected Bulk
</td><td>104(S) Female Complex Top Cross<br />110(S) Selected Pollen Cross SF<br />416(O) Narrow based tester, line CF<br />418(O) Broad based tester, Line CF<br />417(O) Narrow Based Tester, POP CF
</td><td>&nbsp;
</td></tr>
<tr>
<td>Bulk
</td><td>104(S) Female Complex Top Cross<br />111(S) Random Pollen Cross SF<br />426(O) Random Half Sib Families<br />427(O) Selected Half Sib Families
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center" rowspan="5">Selected Bulk
</td></tr>
<tr>
<td>Single Plant
</td><td>105(S) Male Complex Top Cross<br />108(S) Backcross Recessive<br />416(O) Narrow based tester line CF
</td><td>&nbsp;
</td></tr>
<tr>
<td>Same Selected Bulk
</td><td>420(O) Polycross CF<br />421(O) Random Mating CF
</td><td>203(S) Purification<br />204(S), 503(O) Rouging<br />206(S) Selected Bulk<br />508(O) Full Mass Selection
</td></tr>
<tr>
<td>Different Selected Bulk
</td><td>106(S) Complex Cross<br />112(S) Open Pollenated SF<br />404(O) Full diallel cross<br />405(O) Full diallel cross bulked<br />406(O) Half diallel cross<br />407(O) Half diallel cross bulked<br />408(O) Partial diallel cross<br />409(O) Partial diallel cross bulked<br />410(O) Subset Cross<br />411(O) Population backcross CF<br />412(O) Selected Pollen Cross CF<br />413(O) Interspecific cross CF<br />417(O) Narrow Based Tester, POP CF<br />419(O) Broad Based Tester, POP CF<br />423(O) Population Cross CF
</td><td>&nbsp;
</td></tr>
<tr>
<td>Bulk
</td><td>415(O) Random Pollen Cross CF
</td><td>509(O) Half Mass Selection
</td></tr>
<tr>
<td align="center" rowspan="5">Bulk
</td></tr>
<tr>
<td>Single Plant
</td><td>105(S) Male Complex Top Cross<br />411(O) Population Backcross CF<br />412(O) Population Backcross Recessive CF
</td><td>&nbsp;
</td></tr>
<tr>
<td>Selected Bulk
</td><td>411(O) Population Backcross CF<br />412(O) Population Backcross Recessive CF<br />414(O) Selected Pollen Cross CF<br />419(O) Broad Based Tester, POP CF<br />423(O) Population Cross CF
</td><td>&nbsp;
</td></tr>
<tr>
<td>Same Bulk
</td><td>422(O) Open pollenation CF
</td><td>207(S), 507(O) Random Bulk
</td></tr>
<tr>
<td>Different Bulk
</td><td>106(S) Complex Cross<br />112(S) Open Pollenation SF<br />413(O) Interspecific Cross CF<br />415(O) Random Pollen Cross CF<br />422(O) Open pollenation CF<br />423(O) Population Cross CF
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center" colspan="4">I: Inbreeding = S: Self Breeding = Self Fertilisation<br />O: Outbreeding = C: Cross Breeding = C: Cross Fertilisation
</td></tr>
</table><br clear="all" />
<p>An example of the compromise occurs between the export and import protocols and the acquisition, seed increase and cultivar generation protocols. Only a generic method is given for the import and export methods (methods 62 and 63) but a series of detailed methods specifying the type of material acquired, increased or released as cultivars are given for the later protocols. It is expected that these decisions will be tested as the number of crops included in ICIS increases. The import and export protocols will connect germplasm to its genealogy if the source is already in ICIS. If not a problem will arise if the source is entered either through another local user or by entry from a program that is independent from ICIS. Never the less, ICIS provides a mechanisms for users to define new methods for germplasm generation, management or derivation.
</p><p>There are no conventions which are universally applied to the naming of breeding methods, strategies, or genetical material used in plant breeding programs. For example, released germplasm from plant breeding programs are referred to as ‘varieties’ or ‘cultivars’ or rarely as ‘cultigens’. Both the first and last are incorrect as ‘variety’ is a taxonomic term and no released genetic material consists of a single genotype. ICIS uses ‘cultivar’ as the generic definition for released genetic material for agricultural use.
</p><p>The genetic consequences of these crosses or selections depends on the reproductive behavior of the species. For example re selection or collection from a landrace population or transgenic generation could link back to one or no progenitor, while complicated crosses involving, for example, multiple pollen parents could link to many. Similar methods for different crops have different genetic consequences depending on the reproductive system of the crop, self-pollinated, cross-pollinated or clonally propagated. These are distinguished in GMS as separate method groups (MGRP) in the methods table (Annex 2.2.b) as S, O, C respectively or G for general methods which apply to all crops.
</p>
<a name="Germplasm_Information" id="Germplasm_Information"></a><h4> <span class="mw-headline"> Germplasm Information </span></h4>
<p>Information of the generation or derivation of any instance of germplasm is stored in the following variables:<br /><br />
</p>
<table border="1" cellspacing="0" cellpadding="2" width="80%">
<tr>
<td colspan="2"><b>Table 2.2.1.a: Germplasm Information stored in GMS.</b>
</td></tr>
<tr>
<td>GERMPLASM_ID
</td><td>Germplasm Identification Code. This is unique for any germplasm recorded. The general rule is: if you wouldn’t mix the packets of seed that exist or did exist you give the GEN an entry in GMS.
</td></tr>
<tr>
<td>METHOD_NO
</td><td>This is the method of genesis of germplasm and details are stored in the method table (<b>Annex 2.2.b</b>). The method indicates whether the germplasm was produced by a generative, derivative or preservative process, and then how the germplasm was produced. Details of the method which change with each application are stored as attribute information (<b>Section 2.2.4</b>).
</td></tr>
<tr>
<td>GROUP_ID
</td><td>Identification code for last instance of germplasm produced by a generative process from which the current instance was derived. GROUP_ID = GERMPLASM_ID if the current instance is produced by a generative process.
</td></tr>
<tr>
<td>SOURCE_ID
</td><td>Identification code for the immediate source of a derived or preserved germplasm. (Not valid for generated germplasm.)
</td></tr>
<tr>
<td>GERMPLASM_USER_ID
</td><td>Identification code of the user who submitted data on the genesis of the germplasm.
</td></tr>
<tr>
<td>LOCATION_NO
</td><td>Identifier of breeding or origin location.
</td></tr>
<tr>
<td>GERMPLASM_DATE
</td><td>Date of generation, derivation or preservation of the germplasm.
</td></tr>
<tr>
<td>GERMPLASM_REFERENCE
</td><td>Identifier of bibliographic reference from where the germplasm data was retrieved.
</td></tr>
</table><br clear="all" />
<a name="Link_Information" id="Link_Information"></a><h4> <span class="mw-headline"> Link Information </span></h4>
<p>Link information is information on the links between an instance of generated germplasm and its progenitors. The number of progenitors and the role of each, such as male or female, is defined by the METHOD in the germplasm information. The link variables required for each progenitor are:<br /><br />
</p>
<table border="2" cellspacing="0" cellpadding="4" width="80%">
<tr>
<td colspan="2"><b>Table 2.2.2.a: Link information stored in GMS.</b>
</td></tr>
<tr>
<td>GERMPLASM_ID
</td><td>Germplasm Identification Code.
</td></tr>
<tr>
<td>NO_PROGENITORS
</td><td>Number of progenitors.
</td></tr>
<tr>
<td>PROGENITOR_ID
</td><td>Progenitor GERMPLASM_ID.
</td></tr>
</table><br clear="all" />
<a name="Name_Information" id="Name_Information"></a><h4> <span class="mw-headline"> Name Information </span></h4>
<p>A third form of information retained by GMS manages nomenclature and chronology of names of germplasm. Names are any designation used to identify germplasm and this includes breeders’ IDs, release names, abbreviations and accession numbers. The required variables are:<br /><br />
</p>
<table border="2" cellspacing="0" cellpadding="4" width="80%">
<tr>
<td colspan="2"><b>Table 2.2.3.a: Name information stored in GMS.</b>
</td></tr>
<tr>
<td>GERMPLASM_ID
</td><td>Germplasm Identification Code.
</td></tr>
<tr>
<td>NAME_TYPE
</td><td>Some names can belong to several instances of germplasm, some to only one, and some instances can have several names of the same type. These relations are indicated by NAME-TYPE. For example some types would be:<br />ACCNO - Accession No in a germplasm bank (USER_ID points to the germplasm bank as a user of the system)<br />BCID - Breeder’s Cross ID<br />CVNAM - Release or Cultivar Name<br />BRNAMBreeder’s Name, a cross name and selection history in the breeder’s own notation<br />ALNAM - Breeder’s advanced line name. Usually assigned when BRNAM gets too long.<br />NTNAM - National testing name<br />PONAM - Population name<br />CNNAM - Cycle name for recurrent selection schemes<br />SYNAM - Synthetic cultivar name<br />HYNAM - Hybrid cultivar name<br />ADVNM - Alternative derivative name<br />ACVNM - Alternative cultivar name<br />AABBR - Alternative abbreviation<br />Note: Provision is made for a user to add to these names.
</td></tr>
<tr>
<td>NAME_STATUS
</td><td>Number indicating the storage type and status of the name:<br /> 0 - ASCII names which are alternative names.<br /> 1 - ASCII names which are preferred names.<br /> 2 - ASCII names which are preferred abbreviations.<br /> 3 - Chinese-GBK (GD) DBCS names.<br /> 4 - Chinese Big 5.<br /> 5 - Japanese.<br /> 6 - Korean.<br /> 8 - ASCII names which are preferred IDs.<br />10 - UNICODE names which are not preferred.<br />Note: The preferred name must be in ASCII.
</td></tr>
<tr>
<td>NAME_USER_ID
</td><td>The source of name information. This is important for integrity, referencing and elimination of homonyms. The user registration number is sufficient since only registered users can provide information to the system. This deals with the problem of homonyms; by incorporating the USER_ID as part of the germplasm name, since a single user would not assign the same name to more than one piece of germplasm in the local user’s system
</td></tr>
<tr>
<td>NAME_VALUE
</td><td>Name value assigned by a user to a specific instance of germplasm.
</td></tr>
<tr>
<td>NAME_LOCATION_NO
</td><td>Location ID where the name was assigned
</td></tr>
<tr>
<td>NAME_DATE
</td><td>Date of name recording or assigning
</td></tr>
<tr>
<td>NAME_REFERENCE
</td><td>Identifier of bibliographic reference from where the name data was retrieved.
</td></tr>
</table><br clear="all" />
<a name="Attribute_Information" id="Attribute_Information"></a><h4> <span class="mw-headline"> Attribute Information </span></h4>
<p>A fourth form of information retained by GMS records any attributes of the germplasm the user would like to retain. These attributes should be controlled by the central database manager so that only attributes relating to genealogy and nomenclature are stored in GMS. The required variables have an exactly similar structure to the names variables as follows:<br /><br />
</p>
<table border="2" cellspacing="0" cellpadding="4" width="80%">
<tr>
<td colspan="2"><b>Table 2.2.4.a: Attribute Information stored in GMS.</b>
</td></tr>
<tr>
<td>GERMPLASM_ID
</td><td>Germplasm Identification Code.
</td></tr>
<tr>
<td>ATTRIBUTE_TYPE
</td><td>Some attributes can belong to several instances of germplasm, some to only one, and some can have several values for a single instance of germplasm. These relations are indicated by ATTRIBUTE_TYPE. For example some types would be:<br />BSTRBreeder’s Selection String (These BSIDs note the specific selection step in a selection history in the breeders own notation. The full selection history may also be stored as a line name in the NAMES table.<br />SPECSpecies<br />OSETOriginating set. The germplasm set from the set generation module where the germplasm was first derived. (See set generation later.)
</td></tr>
<tr>
<td>ATTRIBUTE_USER_ID
</td><td>The source of attribute information.
</td></tr>
<tr>
<td>ATTRIBUTE_VALUE
</td><td>Attribute value assigned by a user to a specific instance of germplasm.
</td></tr>
<tr>
<td>ATTRIBUTE_LOCATION_NO
</td><td>Location of attribute determination.
</td></tr>
<tr>
<td>ATTRIBUTE_DATE
</td><td>Date of attribute determination.
</td></tr>
<tr>
<td>ATTRIBUTE_REFERENCE
</td><td>Identifier of bibliographic reference from where the attribute data was retrieved.
</td></tr>
</table><br clear="all" />
<p>The following are definitions of attributes which add information about the methods are stored in the USER_DEFINED_FIELDS table.<br /><br />
</p>
<table border="2" cellspacing="0" cellpadding="4" width="80%">
<tr>
<td colspan="3"><b>Table 2.2.4.b: Definitions of attributes which add information about methods.</b>
</td></tr>
<tr>
<td>FLDNO
</td><td>FCODE
</td><td>FNAME
</td></tr>
<tr>
<td>103
</td><td>NOTES
</td><td>Notes concerning germplasm origin, development or chronology.
</td></tr>
<tr>
<td>104
</td><td>RELEASE
</td><td>Cultivar type.
</td></tr>
<tr>
<td>208
</td><td>BS
</td><td>Number of plants bulked and target traits or genes.
</td></tr>
<tr>
<td>209
</td><td>IC
</td><td>Target introgression in interspecific cross
</td></tr>
<tr>
<td>210
</td><td>MT
</td><td>Agent and method of induced mutation.
</td></tr>
<tr>
<td>211
</td><td>MATT
</td><td>A description of the method or agent used for a specific instance of the method.
</td></tr>
<tr>
<td>212
</td><td>RMA
</td><td>Attributes for random mating methods. These are:<br />NPSnumber of parents randomly mated.<br />PCTpercent contribution from each parent.<br />MMTHintervention method to
</td></tr>
<tr>
<td>213
</td><td>MCOLL
</td><td>Methods of collection.
</td></tr>
<tr>
<td>214
</td><td>IMPORT
</td><td>Type of germplasm. See Table 2.2.a for lists of types of germplasm.
</td></tr>
<tr>
<td>215
</td><td>BLKM
</td><td>Method of bulking seed, possibilities are:<br />Bulking all seed from plants.<br />Bulking equal volume from each plant.<br />Bulking equal weight from each plant<br />Bulking equal number from each plant
</td></tr>
<tr>
<td>216
</td><td>SUBSETA
</td><td>No of subsets and bulking method. See above for bulking methods.
</td></tr>
<tr>
<td>221
</td><td>ST
</td><td>Target traits or genes.
</td></tr>
<tr>
<td>222
</td><td>SEX
</td><td>Selfs excluded, Yes or No
</td></tr>
</table><br clear="all" />
<a name="EXAMPLES_OF_STORAGE_OF_GENEALOGY_INFORMATION" id="EXAMPLES_OF_STORAGE_OF_GENEALOGY_INFORMATION"></a><h2> <span class="mw-headline"> EXAMPLES OF STORAGE OF GENEALOGY INFORMATION </span></h2>
<p>Several examples of storage of genealogy information are given in this section. Examples of the type of information generated by the system are given and the entry of genealogical information for the rice cultivar IR24 is given as an example of storing historical data. The examples used to illustrate system storage include a modified pedigree plant breeding system in wheat, an open recurrent selection system in wheat, a reciprocal recurrent selection system in maize, a polycross among potato clones and a half diallele using the same four clones.
</p>
<a name="Genealogy_Management_for_a_Modified_Pedigree_Breeding_Program" id="Genealogy_Management_for_a_Modified_Pedigree_Breeding_Program"></a><h4> <span class="mw-headline"> Genealogy Management for a Modified Pedigree Breeding Program </span></h4>
<p>The modified pedigree breeding program illustrated here (Figure 2.3.1.a ) is modified from the program used by the wheat breeding program at the Leslie Research Centre of the Queensland Department of Primary Industries in Australia. This example illustrates the use of the three types of methods used in GMS, generative, derivative and maintenance methods. The system used for the selection history (Figure 2.3.1.a) and used as a breeders name (Table 2.3.1.a) is a modification of the that used by the CIMMYT wheat program and is for illustrative purposes only. It is not the system used by the LRC program.
</p>
<a name="Generative_methods" id="Generative_methods"></a><h5> <span class="mw-headline"> Generative methods </span></h5>
<p>The example provides an illustration of two common generative methods used in pedigree breeding programs, viz. a two way and a three way cross. The three-way cross in this example is a top-cross as the F1 from the first cross is crossed to another adaptive line.
</p><p>For the two way cross assume that line A (GID821; table row 1) is the female parent (FPARNT) and line B (GID822; table row 2) is the male parent (MPARNT) of the single cross (generative method (METH C2W))  98 12 (GID900; table row 4). 98 12 is the 12th cross made in 1998 and has two progenitors (No PROGEN). Note that all lines listed in the table which are derivatives of this two way cross have the pedigree A/B but have different selection histories. 
</p><p>For the three-way cross the F1 (GID900, row 4) from the two-way cross, used as a female, is mated (in the next year) to another adapted parent C (GID823, table row 3) to produce the three-way cross (3WC) (GID950, table row 5) and this is the 13th cross made in 1999. All derivatives of this cross have the pedigree A/B//C. Note that while this pedigree does not indicate whether A/B was an F1 or a fixed line this information is carried in the GERMPLASM TABLE. This clearly indicates that GMS does not store pedigrees, but rather genealogies. Purdy pedigrees are an output (query) on GMS and other forms of a ‘pedigree’ can be programmed as a query.
</p>
<a name="Derivative_methods" id="Derivative_methods"></a><h5> <span class="mw-headline"> Derivative methods </span></h5>
<p>The F1 is a bulk of all the grains produced from the cross (table row 7, GID1001, METH DRB) and is designated 98-12-0R where 0R indicates a bulk and R indicates the place where the F1 was grown (Research Station in this example). Note that the F1 (and all further derivatives) has as a group number the GID (900) of the cross from which they were derived. The source number is the GID of the immediate packet of seed from which each line was derived, in this case the cross GID. The F2 (table row 9, GID2001, METH DRB) is a random bulk of the F1 and is designated 98-12-0R-0R as in this example the F1 was grown on the research station. It has group number GID900 and source number GID1001. The F2 rows are grown at place A. The F2 is segregating and single plant selections are taken and these F3 lines (table rows 11 &amp; 12 are examples) are grown as ‘head rows’ also at place A. As these are single plant selections, the resultant F3 lines are designated 98-12-0R-0R-4A (table row 11) etc where 4A represents the fourth plant selected from this F2 at place A. These F3 lines are again grown as head rows at place A and bulked. The resultant F2 derived F4 bulks are designated 98-12-0R-0R-4A-0A (table row 15) and are used to sow unreplicated preliminary yield trials (designated P in this example) and to grow rows for single plant selections at place A.  
</p><p>In the CIMMYT system for writing selection histories, and in most if not all other systems including IRRI, the two steps indicating the F1 and F2 are omitted in the selection histories for single crosses as the first two -0R-0R- steps are common to all derivatives of a single cross. Using this system the F3 lines would be designated 98-12-4A indicating that this the 4th single plant selection from the F2 of the cross 98-12.
</p><p>The 3WC1 (table row 8, GID1002, METH DRB) designated (98-13)13-0R  is a bulk of all seeds from the three-way cross, GID950 and is grown as a row at place A. It has the group number GID950 of the cross from which it comes and the same GID as its source number. Unlike the F1, which is homogeneous, this population is segregating. Hence single plant selections are often taken from these population types rather than a bulk as is the case for the F1 from a single cross. These ‘F3’ single plant selections are grown as head rows at place A and because they are single plant selections they are designated (93-13)13-0R-7A (row 13) for example. This line has GID950 as its group code and GID2010 as its source code. These lines are treated from this point on in the same way as the single crosses.
</p><p>In the three-way cross only the 1st -0R- step is common to all derivatives and the commonly used (CIMMYT/IRRI) system would designate the ‘F3’ line as (98-13)13-G indicating that this was the 7th single plant selection from the bulk of all the seed derived from the three-way cross (98-13)13.<br /><br />
</p>
<div class="floatleft"><a href="../../../../articles/m/o/d/File%7EModified_pedigree_plant_breeding_program.JPG_1f0c.html" class="image" title="Modified pedigree plant breeding program.JPG"><img alt="" src="../../../../images/8/83/Modified_pedigree_plant_breeding_program.JPG" width="577" height="809" border="0" /></a></div><br clear="all" />
<p><b>Figure 2.3. 1 Diagram of a stylised representation of a modified pedigree plant breeding program.</b>
</p><p><br />
The F4 bulks (table rows 15 &amp; 16) are grown in preliminary multi-environment trials (pMET1) for one year.
</p><p>F5 bulks (<i>eg</i> table row 17, GID2201,  METH DRB) from those selected from pMET1 (50%) are grown for a second year in pMET2 to enable two years of data to be used for selection for yield among the F2 derived F4 bulks.
</p><p>The surviving F4 lines selected using data from the pMETs are grown as rows at site A and single plant selections made.
</p><p>The F5 lines derived (<i>eg</i> table rows 19, GID2301, designated 98-12-0R-0R-4A-0A-2A) are planted as seed multiplication rows at site B.
</p><p>The resultant F6 bulks (<i>eg</i> table row 21, GID3001, METH DRB, designated 98-12-0R-0R-4A-0A-2A-0B) are sown in the first year of a replicated intermediate MET (iMET1), sown at site B for seed multiplication for the first year of regional METs (rMETs) and re sown at site B in the first year of rMETs for seed purification if the line looks promising for release. Once a line is tested in a MET it acquires a breeders name (BN21 and BN22) in these cases.
</p><p>The F7 bulk seed from iMET1 (eg table row 23, GID3101, 98-12-0R-0R-4A-0A-2A-0B-0I) is used for iMET2. The addendum letter I is used here to designate seed produced from an intermediate MET.
</p><p>The F7 bulk seed produced from the multiplication rows (table rows 25 &amp; 26) at site B is used for the first regional MET (rMET1) and for a further multiplication of seed, at site B, to produce the F8 bulk seed for rMET2 (table rows 27 &amp; 28).
</p>
<a name="Maintenance_methods" id="Maintenance_methods"></a><h5> <span class="mw-headline"> Maintenance methods </span></h5>
<p>The example from hear on will follow the history of an F6 line, GID3001, which goes on to be released as a cultivar. This process consists of one derivative method, seed purification, and the maintenance methods, seed increase for yield testing in the final year of the rMETs, a seed multiplication to produce breeders seed for release of a cultivar called in this example Wonder. At this stage if more than one line from the cross is used as a parent the cross may be named, here Wonder Cross. The process of release consists of two production seed increases to obtain enough seed for release and the release process itself.
</p><p>Table row 29 (GID4001, METH DPU) represents the purification at site B of BN21 by derivative method seed purification to produce F<sub>7</sub> pure seed. Note that I have suggested the addendum -PD to indicate purification at site D. The F7 pure seed is used for a seed increase at site B to produce F8 seed for testing in the final rMET3 and for producing breeders seed at site E.
</p><p>Table row 30 (GID4101, METH NBK) represents the maintenance method ’bulk seed increase’ at site B of GID4001 for seed for testing in rMET3.
</p><p>Table row 31 (GID4201, METH VBS) represents the maintenance method ’breeders seed production’ at site E. An addendum -I1E is used to indicate the first seed increase conducted at site E.
</p><p>Table row 32 (GID4201, METH NBK) represents the maintenance method ’bulk seed increase’ at site F. This is the second seed increase, hence the addendum -I2F, representing the second seed increase which was done at site F.
</p><p>Table row 33 (GID4401, METH VPL) represents the maintenance method ’pure line formation’ in preparation for releasing a pure line cultivar. In this case this does not represent a physical process but does in many situations, eg forming and F1 hybrid.
</p><p>Table row 34 (GID4410, METH VCR) represents the maintenance method ’cultivar release’. This process should be maintained separate from the formation odf a cultivar. It can occur more than once for the same material. <br /><br />
</p>
<table border="2" cellspacing="0" cellpadding="4" width="75%">
<tr>
<td colspan="8"><b>Table 2.3.1.a: Germplasm Table for Selection Histories for an example Pedigree Breeding Program.</b>
</td></tr>
<tr>
<td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center" colspan="2">Generative Methods
</td><td>Breeder’s name
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">Row in table
</td><td align="center">GID
</td><td align="center">METH
</td><td align="center">No PROGEN
</td><td align="center">FPARNT
</td><td align="center">MPARNT
</td><td>&nbsp;
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">1
</td><td align="center">821
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td>A
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">2
</td><td align="center">822
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td>B
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">3
</td><td align="center">823
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td>C
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td>.
</td><td>.
</td></tr>
<tr>
<td align="center">4
</td><td align="center">900
</td><td align="center">C2W
</td><td align="center">2
</td><td align="center">821
</td><td align="center">822
</td><td>98-12
</td><td>Single cross
</td></tr>
<tr>
<td align="center">5
</td><td align="center">950
</td><td align="center">C3w
</td><td align="center">2
</td><td align="center">900
</td><td align="center">823
</td><td>(98-12)13
</td><td>Three-way cross
</td></tr>
<tr>
<td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center" colspan="2">Derivative or Management Methods
</td><td>&nbsp;
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">Row in table
</td><td align="center">GID
</td><td align="center">METH
</td><td align="center">&nbsp;
</td><td align="center">GROUP
</td><td align="center">SOURCE
</td><td>&nbsp;
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">7
</td><td align="center">1001
</td><td align="center">DRB
</td><td align="center">&nbsp;
</td><td align="center">900
</td><td align="center">900
</td><td>98-12-0R
</td><td>F1 rows
</td></tr>
<tr>
<td align="center">8
</td><td align="center">1002
</td><td align="center">DRB
</td><td align="center">&nbsp;
</td><td align="center">950
</td><td align="center">950
</td><td>(98-13)13-0R
</td><td>3WC1 rows
</td></tr>
<tr>
<td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td>.
</td><td>.
</td></tr>
<tr>
<td align="center">9
</td><td align="center">2001
</td><td align="center">DRB
</td><td align="center">&nbsp;
</td><td align="center">900
</td><td align="center">1001
</td><td>98-12-0R-0R
</td><td>F2 rows
</td></tr>
<tr>
<td align="center">10
</td><td align="center">2010
</td><td align="center">DRB
</td><td align="center">&nbsp;
</td><td align="center">950
</td><td align="center">1002
</td><td>(98-13)13-0R
</td><td>3WC1 rows
</td></tr>
<tr>
<td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td>.
</td><td>.
</td></tr>
<tr>
<td align="center">11
</td><td align="center">2101
</td><td align="center">DSP
</td><td align="center">&nbsp;
</td><td align="center">900
</td><td align="center">2001
</td><td>98-12-0R-0R-4A
</td><td>F3 rows
</td></tr>
<tr>
<td align="center">12
</td><td align="center">2102
</td><td align="center">DSP
</td><td align="center">&nbsp;
</td><td align="center">900
</td><td align="center">2001
</td><td>09-12-0R-0R-5A
</td><td>F3 rows
</td></tr>
<tr>
<td align="center">13
</td><td align="center">2151
</td><td align="center">DSP
</td><td align="center">&nbsp;
</td><td align="center">950
</td><td align="center">2010
</td><td>(98-13)13-0R-7A
</td><td>’F3’ rows
</td></tr>
<tr>
<td align="center">14
</td><td align="center">2152
</td><td align="center">DSP
</td><td align="center">&nbsp;
</td><td align="center">950
</td><td align="center">2010
</td><td>(98-13)13-0R-8A
</td><td>’F3’ rows
</td></tr>
<tr>
<td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td>.
</td><td>.
</td></tr>
<tr>
<td align="center">15
</td><td align="center">2171
</td><td align="center">DRB
</td><td align="center">&nbsp;
</td><td align="center">950
</td><td align="center">2101
</td><td>98-12-0R-0R-4A-0A
</td><td>F4 bulks
</td></tr>
<tr>
<td align="center">16
</td><td align="center">2172
</td><td align="center">DRB
</td><td align="center">&nbsp;
</td><td align="center">950
</td><td align="center">2102
</td><td>98-12-0R-0R-5A-0A
</td><td>F4 bulks
</td></tr>
<tr>
<td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td>.
</td><td>.
</td></tr>
<tr>
<td align="center">17
</td><td align="center">2201
</td><td align="center">DRB
</td><td align="center">&nbsp;
</td><td align="center">900
</td><td align="center">2171
</td><td>98-12-0R-0R-4A-0A-0P
</td><td>F5 bulks
</td></tr>
<tr>
<td align="center">18
</td><td align="center">2202
</td><td align="center">DRB
</td><td align="center">&nbsp;
</td><td align="center">900
</td><td align="center">2172
</td><td>98-12-0R-0R-5A-0A-0P
</td><td>F5 bulks
</td></tr>
<tr>
<td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td>.
</td><td>.
</td></tr>
<tr>
<td align="center">19
</td><td align="center">2301
</td><td align="center">DSP
</td><td align="center">&nbsp;
</td><td align="center">900
</td><td align="center">2201
</td><td>98-12-0R-0R-4A-0A-2A
</td><td>F5 rows
</td></tr>
<tr>
<td align="center">20
</td><td align="center">2302
</td><td align="center">DSP
</td><td align="center">&nbsp;
</td><td align="center">900
</td><td align="center">2202
</td><td>98-12-0R-0R-4A-0A-3A
</td><td>F5 rows
</td></tr>
<tr>
<td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td>.
</td><td>.
</td></tr>
<tr>
<td align="center">21
</td><td align="center">3001
</td><td align="center">DRB
</td><td align="center">&nbsp;
</td><td align="center">900
</td><td align="center">2301
</td><td>98-12-0R-0R-4A-0A-2A-0B
</td><td>F6 bulks<br />BN21
</td></tr>
<tr>
<td align="center">22
</td><td align="center">3002
</td><td align="center">DRB
</td><td align="center">&nbsp;
</td><td align="center">900
</td><td align="center">2302
</td><td>98-12-0R-0R-4A-0A-3A-0B
</td><td>F6 bulks<br />BN22
</td></tr>
<tr>
<td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td>.
</td><td>.
</td></tr>
<tr>
<td align="center">23
</td><td align="center">3101
</td><td align="center">DRB
</td><td align="center">&nbsp;
</td><td align="center">900
</td><td align="center">3001
</td><td>98-12-0R-0R-4A-0A-2A-0B-0I
</td><td>F7 bulks<br />BN21
</td></tr>
<tr>
<td align="center">24
</td><td align="center">3102
</td><td align="center">DRB
</td><td align="center">&nbsp;
</td><td align="center">900
</td><td align="center">3001
</td><td>98-12-0R-0R-4A-0A-3A-0B-0I
</td><td>F7 bulks<br />BN22
</td></tr>
<tr>
<td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td>.
</td><td>.
</td></tr>
<tr>
<td align="center">25
</td><td align="center">3201
</td><td align="center">DRB
</td><td align="center">&nbsp;
</td><td align="center">900
</td><td align="center">2301
</td><td>98-12-0R-0R-4A-0A-2A-0B-0B
</td><td>F7 bulks<br />BN21
</td></tr>
<tr>
<td align="center">26
</td><td align="center">3202
</td><td align="center">DRB
</td><td align="center">&nbsp;
</td><td align="center">900
</td><td align="center">2302
</td><td>98-12-0R-0R-4A-0A-3A-0B-0B
</td><td>F7 bulks<br />BN22
</td></tr>
<tr>
<td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td>.
</td><td>.
</td></tr>
<tr>
<td align="center">27
</td><td align="center">3301
</td><td align="center">DRB
</td><td align="center">&nbsp;
</td><td align="center">900
</td><td align="center">3201
</td><td>98-12-0R-0R-4A-0A-2A-0B-0B-0B
</td><td>F8 bulks<br />BN21
</td></tr>
<tr>
<td align="center">28
</td><td align="center">3302
</td><td align="center">DRB
</td><td align="center">&nbsp;
</td><td align="center">900
</td><td align="center">3202
</td><td>98-12-0R-0R-4A-0A-3A-0B-0B-0B
</td><td>BN22
</td></tr>
<tr>
<td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td>.
</td><td>.
</td></tr>
<tr>
<td align="center">29
</td><td align="center">4001
</td><td align="center">DPU
</td><td align="center">&nbsp;
</td><td align="center">900
</td><td align="center">3001
</td><td>98-12-0R-0R-4A-0A-2A-0B-PD
</td><td>F7 pure line<br />Wonder cross
</td></tr>
<tr>
<td align="center">30
</td><td align="center">4101
</td><td align="center">NBK
</td><td align="center">&nbsp;
</td><td align="center">900
</td><td align="center">4001
</td><td>98-12-0R-0R-4A-0A-2A-0B-PD-I1D
</td><td>Wonder cross
</td></tr>
<tr>
<td align="center">31
</td><td align="center">4201
</td><td align="center">VBS
</td><td align="center">&nbsp;
</td><td align="center">900
</td><td align="center">4101
</td><td>98-12-0R-0R-4A-0A-2A-0B-PD-I1E
</td><td>Wonder cross
</td></tr>
<tr>
<td align="center">32
</td><td align="center">4301
</td><td align="center">NBK
</td><td align="center">&nbsp;
</td><td align="center">900
</td><td align="center">4201
</td><td>98-12-0R-0R-4A-0A-2A-0B-PD-I1E-I2F
</td><td>Wonder cross
</td></tr>
<tr>
<td align="center">33
</td><td align="center">4401
</td><td align="center">VPL
</td><td align="center">&nbsp;
</td><td align="center">900
</td><td align="center">4301
</td><td>98-12-0R-0R-4A-0A-2A-0B-PD-I1E-I2F-V
</td><td>Wonder cross
</td></tr>
<tr>
<td align="center">34
</td><td align="center">4410
</td><td align="center">VCR
</td><td align="center">&nbsp;
</td><td align="center">900
</td><td align="center">4401
</td><td>98-12-0R-0R-4A-0A-2A-0B-PD-I1E-I2F-V-0AU
</td><td>cv Wonder
</td></tr>
<tr>
<td align="center">35
</td><td align="center">4501
</td><td align="center">ISE
</td><td align="center">&nbsp;
</td><td align="center">900
</td><td align="center">4401
</td><td>98-12-0R-0R-4A-0A-2A-0B-PD-I1E-I2F-0AU-AC
</td><td>cv Wonder<br />WA28
</td></tr>
<tr>
<td colspan="8">ID = is unique germplasm identifier.<br />METH = is the breeding method, either generative, derivative or management.<br />No PROGEN = is the number of progenitors of a derivative method.<br />FPARNT = ID of female parent.<br />MPARNT = ID of male parent.<br />GROUP = ID of last generative process for the derivative or management process.<br />SOURCE = ID of immediate predecessor for the derivative or management process.
</td></tr>
</table><br clear="all" />
<p>Finally, in this example, the seed is placed in a germplasm bank. This is represented by table row 36 (GID4501, METHH ISE), the ‘seed import’ method and acquires another name WA28.
</p><p>In this example many of the seed increase operations reflected in the selection histories are carried out at different sites. If these operations are carried out at the same site some modification of the system for naming selection histories may be required to reflect these differences should this be required.
</p>
<a name="Genealogy_Management_for_An_Open_Recurrent_Selection_Method" id="Genealogy_Management_for_An_Open_Recurrent_Selection_Method"></a><h4> <span class="mw-headline"> Genealogy Management for An Open Recurrent Selection Method </span></h4>
<p>The recurrent selection method for wheat improvement being used by the University of Queensland in their Germplasm Enhancement Program conducted at the Leslie Research Institute in Australia is described by Fabrizius <i>et al</i>. (1996). This note indicates how pedigrees from that system can be recorded in the GMS structure of ICIS.
</p>
<dl><dd>(A) The open reciprocal selection method starts with the selection of a number of parental lines from which a population is developed by randomly mating the F<sub>1</sub>s from a complete diallel using a gametocyte. Consider a system based on 4 parents with GIDs 871, 927, 933 and 1001 as indicated in rows 1, 2, 5 and 8 of the GERMPLASM TABLE (Table 2.3.2.a.). The 6 F<sub>1</sub>s from the diallel could be simply entered as 1004 to 1009 as shown on rows 12-17 in Table 2.3.2.a.
</dd></dl>
<dl><dd>(B) The first step of the recurrent selection cycle is to randomly mate the F<sub>1</sub>s using a gametocyte. To record this we need to define a method in the METHODS TABLE for this form of random mating. Suppose we use code CGO (Method 112 in Table 2.1.d) for the method which has a variable number of parents (defined for each implementation by the NO_ OF_ PROGENITORS parameter in the GERMPLASM TABLE). Method CGO also has no defined number of female parents since all parents act as male and female parents. In our example there are 6 parents, for the random mating recorded in row 18 of Table 2.3.2.a. The first two parents are recorded in the GERMPLASM TABLE as in row 18, the remaining 4 are recorded in the OTHER PROGENITORS TABLE as in rows 1 to 4 of Table 2.3.2.b. The resulting population is termed the C0 population.
</dd></dl>
<dl><dd>(C) The next step in the process is to screen a large number of S0 plants from C0 seed for semidwarf status and rust resistance. A number of these are kept and proceed through a generation of seed increase and rust testing. In practice the number kept will be 2,000, but for our example we record the pedigree of 10 in rows 19 to 28 of Table 2.3.2.a. These retained S1 families represent single plant selections, DSP, out of the C0 population.
</dd></dl>
<dl><dd>(D) Selections are made on the basis of rust scores from the S1 plants, and a number are kept. In practice this number will be 800, but for our example we will consider 5. Seed from these retained families is split for storage, multi-environment testing for two years and bulk quality testing for the survivors. Suppose our retained S1 families are C0-2, C0-3, C0-7, C0-9 and C0-10 from rows 20, 21 25, 27 and 28 of Table 2.3.2.a. These are bulk selected in the third year, with surviving families recorded as A on rows 29 to 33 of Table 2.3.2.a and again in the fourth year, B, to produce four remaining families C0-2AB, C0-7AB, C0-9AB and C0-10AB recorded on rows 34 to 37 of Table 2.3.2.a.
</dd></dl>
<dl><dd>(E) It is possible, as an off-shoot of the system, that single plant selection can be made from the evaluation year nurseries and be incorporated into traditional breeding programs, for example the third plant from the family C0-2A in row 29, and the sixth from C0-7AB in row 35 could be selected for incorporation in other breeding programs. These are the families recorded in the usual manner as in rows 38 and 39 of Table 2.3.2.a.
</dd></dl>
<dl><dd>(F) Following quality testing of the families surviving two years of evaluation, a number of families will be selected to go into a second round of random mating. In fact about 100 will be selected, but we will consider 3 of our surviving families C0-2AB, C0-7AB and C0-9AB. The seed for this random mating cycle will however come from the S1 retained seed, C0-2, C0-7 and C0-9 in our example. As a variation on simple random mating new parental lines can be introduced at this point by including plants from new lines in the random mating process at specified proportions of the population. This is the step that gives rise to the term Open Random Mating, and as this is not in the METHODS TABLE (Table 2.1.d) we need to define a new method, ORM, with variable number of parents. This emphasises the point made previously about the necessity of defining all possible methods for the methods table. The methods defined are a compromise between a too parsimonious set which does not convey enough information and attempting to define every possible method and swamping the user with too much information. The compromise allows new methods to be defined by users 9with MIDs over 1000 and accepted or not by the ICIS crop coordinator.<br />ORM also has no defined female parents, but a variable number of imported parents, and an incorporation percentage for each. These parameters must be stored as attributes of the breeding method for each implementation of the method. In the METHODS TABLE, the ATTRIBUTES flag would be set for method ORM to indicate that attribute values for each implementation are stored in the ATTRIBUTES TABLE. THE CONSTANTS TABLE would define these values as number of imported parents which are specified last in the list of parents, and percent contribution of each. The CONSTANTS TABLE would also indicate that each value occurred on an extension line of the attribute in a specified format. For our example, suppose we wish to retain three S1 families and include two new parents with IDs 930 and 1000 (as in rows 3 and 7 of Table 2.3.2.a) with proportions 15% and 10% of the random mating population.<br />The new population would be recorded as in row 40 of Table 2.3.2.a with GID 1032. The first two parents, C0-2 and C0-7 are on that line, the remaining three are recorded on the OTHER PARENTS TABLE (rows 5 to 7 in Table 2.3.2.b).  The attribute values would be stored on the ATTRIBUTES TABLE as in Table 2.3.2.c. The positive value of ATTRIBUTE-EXT indicates the number of attribute values, in this case one more than the number of imported parents, and the negative values define the order of the remaining values down to -1, the last value.<br /><br />
</dd></dl>
<table border="2" cellspacing="0" cellpadding="4" width="80%">
<tr>
<td colspan="9"><b>Table 2.3.2.a: Germplasm Table For Open Recurrent Selection Example.</b>
</td></tr>
<tr>
<td align="center">Row in Table
</td><td align="center">GID
</td><td align="center">Method
</td><td align="center">Number of progenitors
</td><td align="center" colspan="2">Generative Process
</td><td align="center" colspan="2">Derivative Process
</td><td align="center">Name
</td></tr>
<tr>
<td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">Female Parent
</td><td align="center">Male Parent
</td><td align="center">Group
</td><td align="center">Source
</td><td align="center">&nbsp;
</td></tr>
<tr>
<td align="center">1
</td><td align="center">871
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">2
</td><td align="center">927
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">3
</td><td align="center">930
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">4
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">5
</td><td align="center">933
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">6
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">7
</td><td align="center">1000
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">8
</td><td align="center">1001
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">9
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">10
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">11
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">12
</td><td align="center">1004
</td><td align="center">C2W
</td><td align="center">2
</td><td align="center">871
</td><td align="center">927
</td><td align="center">.
</td><td align="center">.
</td><td>Fdn 12
</td></tr>
<tr>
<td align="center">13
</td><td align="center">1005
</td><td align="center">C2W
</td><td align="center">2
</td><td align="center">871
</td><td align="center">933
</td><td align="center">.
</td><td align="center">.
</td><td>Fdn 13
</td></tr>
<tr>
<td align="center">14
</td><td align="center">1006
</td><td align="center">C2W
</td><td align="center">2
</td><td align="center">871
</td><td align="center">1001
</td><td align="center">.
</td><td align="center">.
</td><td>Fdn 14
</td></tr>
<tr>
<td align="center">15
</td><td align="center">1007
</td><td align="center">C2W
</td><td align="center">2
</td><td align="center">927
</td><td align="center">933
</td><td align="center">.
</td><td align="center">.
</td><td>Fdn 23
</td></tr>
<tr>
<td align="center">16
</td><td align="center">1008
</td><td align="center">C2W
</td><td align="center">2
</td><td align="center">927
</td><td align="center">1001
</td><td align="center">.
</td><td align="center">.
</td><td>Fdn 24
</td></tr>
<tr>
<td align="center">17
</td><td align="center">1009
</td><td align="center">C2W
</td><td align="center">2
</td><td align="center">933
</td><td align="center">1001
</td><td align="center">.
</td><td align="center">.
</td><td>Fdn 34
</td></tr>
<tr>
<td align="center">18
</td><td align="center">1010
</td><td align="center">CGO
</td><td align="center">6
</td><td align="center">1004
</td><td align="center">1005
</td><td align="center">.
</td><td align="center">.
</td><td>C0
</td></tr>
<tr>
<td align="center">19
</td><td align="center">1011
</td><td align="center">DSP
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1010
</td><td align="center">1010
</td><td>C0-1
</td></tr>
<tr>
<td align="center">20
</td><td align="center">1012
</td><td align="center">DSP
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1010
</td><td align="center">1010
</td><td>C0-2
</td></tr>
<tr>
<td align="center">21
</td><td align="center">1013
</td><td align="center">DSP
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1010
</td><td align="center">1010
</td><td>C0-3
</td></tr>
<tr>
<td align="center">22
</td><td align="center">1014
</td><td align="center">DSP
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1010
</td><td align="center">1010
</td><td>C0-4
</td></tr>
<tr>
<td align="center">23
</td><td align="center">1015
</td><td align="center">DSP
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1010
</td><td align="center">1010
</td><td>C0-5
</td></tr>
<tr>
<td align="center">24
</td><td align="center">1016
</td><td align="center">DSP
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1010
</td><td align="center">1010
</td><td>C0-6
</td></tr>
<tr>
<td align="center">25
</td><td align="center">1017
</td><td align="center">DSP
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1010
</td><td align="center">1010
</td><td>C0-7
</td></tr>
<tr>
<td align="center">26
</td><td align="center">1018
</td><td align="center">DSP
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1010
</td><td align="center">1010
</td><td>C0-8
</td></tr>
<tr>
<td align="center">27
</td><td align="center">1019
</td><td align="center">DSP
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1010
</td><td align="center">1010
</td><td>C0-9
</td></tr>
<tr>
<td align="center">28
</td><td align="center">1020
</td><td align="center">DSP
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1010
</td><td align="center">1010
</td><td>C0-10
</td></tr>
<tr>
<td align="center">29
</td><td align="center">1021
</td><td align="center">DRB
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1010
</td><td align="center">1012
</td><td>C0-2A
</td></tr>
<tr>
<td align="center">30
</td><td align="center">1022
</td><td align="center">DRB
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1010
</td><td align="center">1013
</td><td>C0-3A
</td></tr>
<tr>
<td align="center">31
</td><td align="center">1023
</td><td align="center">DRB
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1010
</td><td align="center">1017
</td><td>C0-7A
</td></tr>
<tr>
<td align="center">32
</td><td align="center">1024
</td><td align="center">DRB
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1010
</td><td align="center">1019
</td><td>C0-9A
</td></tr>
<tr>
<td align="center">33
</td><td align="center">1025
</td><td align="center">DRB
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1010
</td><td align="center">1020
</td><td>C0-10A
</td></tr>
<tr>
<td align="center">34
</td><td align="center">1026
</td><td align="center">DRB
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1010
</td><td align="center">1021
</td><td>C0-2AB
</td></tr>
<tr>
<td align="center">35
</td><td align="center">1027
</td><td align="center">DRB
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1010
</td><td align="center">1023
</td><td>C0-7AB
</td></tr>
<tr>
<td align="center">36
</td><td align="center">1028
</td><td align="center">DRB
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1010
</td><td align="center">1024
</td><td>C0-9AB
</td></tr>
<tr>
<td align="center">37
</td><td align="center">1029
</td><td align="center">DRB
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1010
</td><td align="center">1025
</td><td>C0-10AB
</td></tr>
<tr>
<td align="center">38
</td><td align="center">1030
</td><td align="center">DSP
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1010
</td><td align="center">1021
</td><td>C0-2A-3
</td></tr>
<tr>
<td align="center">39
</td><td align="center">1031
</td><td align="center">DSP
</td><td align="center">-1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1010
</td><td align="center">1027
</td><td>C0-2AB-6
</td></tr>
<tr>
<td align="center">40
</td><td align="center">1032
</td><td align="center">ORM
</td><td align="center">5
</td><td align="center">1012
</td><td align="center">1017
</td><td align="center">.
</td><td align="center">.
</td><td>C1
</td></tr>
<tr>
<td align="center">41
</td><td align="center">1033
</td><td align="center">DSP
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1032
</td><td align="center">1032
</td><td>C1-1
</td></tr>
<tr>
<td align="center">42
</td><td align="center">1034
</td><td align="center">DSP
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1032
</td><td align="center">1032
</td><td>C1-2
</td></tr>
<tr>
<td align="center">43
</td><td align="center">1035
</td><td align="center">DSP
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1032
</td><td align="center">1032
</td><td>C1-3
</td></tr>
<tr>
<td align="center">44
</td><td align="center">1036
</td><td align="center">DSP
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1032
</td><td align="center">1032
</td><td>C1-4
</td></tr>
<tr>
<td align="center">45
</td><td align="center">1037
</td><td align="center">DSP
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1032
</td><td align="center">1032
</td><td>C1-5
</td></tr>
<tr>
<td align="center">46
</td><td align="center">1038
</td><td align="center">DSP
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1032
</td><td align="center">1032
</td><td>C1-6
</td></tr>
<tr>
<td align="center">47
</td><td align="center">1039
</td><td align="center">DSP
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1032
</td><td align="center">1032
</td><td>C1-7
</td></tr>
<tr>
<td align="center">48
</td><td align="center">1040
</td><td align="center">DSP
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1032
</td><td align="center">1032
</td><td>C1-8
</td></tr>
<tr>
<td align="center">49
</td><td align="center">1041
</td><td align="center">DRB
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1032
</td><td align="center">1035
</td><td>C1-3A
</td></tr>
<tr>
<td align="center">50
</td><td align="center">1042
</td><td align="center">DRB
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1032
</td><td align="center">1037
</td><td>C1-SA
</td></tr>
<tr>
<td align="center">51
</td><td align="center">1043
</td><td align="center">DRB
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1032
</td><td align="center">1039
</td><td>C1-7A
</td></tr>
<tr>
<td align="center">52
</td><td align="center">1044
</td><td align="center">DRB
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1032
</td><td align="center">1041
</td><td>C1-3AB
</td></tr>
<tr>
<td align="center">53
</td><td align="center">1045
</td><td align="center">DRB
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1032
</td><td align="center">1043
</td><td>C1-AB
</td></tr>
<tr>
<td align="center">54
</td><td align="center">1046
</td><td align="center">DSP
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1032
</td><td align="center">1035
</td><td>C1-3A-3
</td></tr>
<tr>
<td align="center">55
</td><td align="center">1047
</td><td align="center">DSP
</td><td align="center"> 1
</td><td align="center">.
</td><td align="center">.
</td><td align="center">1032
</td><td align="center">1043
</td><td>C1-7AB-9
</td></tr>
</table>
<p><br clear="all" />
</p>
<dl><dd>(G)Again S1 families are screened for dwarf status and resistance and seed from single plant selection (rows 41-48 of Table 2.3.2) is kept while progeny proceed through two seasons of bulk selection (rows 49-51 for first season and rows 52 and 53 for second season in Table 2.3.2.a).
</dd></dl>
<dl><dd>(H) At any point, either S1 seed identified as in rows 41-48 or single plant selections from the evaluation lines, such as in rows 54 and 55 of Table 2.3.2.a can be picked by traditional breeding lines and follow normal pedigree development (rows 54 ans 55).
</dd></dl>
<dl><dd>(I) Subsequent ORM populations C2, C3 <i>etc</i> are developed and recorded exactly as for C1.<br /><br />
</dd></dl>
<table border="2" cellspacing="0" cellpadding="4" width="75%">
<tr>
<td align="center" colspan="4"><b>Table 2.3.2.b: Entries in Other Progenitors Table for Open Recurrent Selection example.</b>
</td></tr>
<tr>
<td align="center">Row
</td><td align="center">ID
</td><td align="center">PROGENITOR NO.
</td><td align="center">PROGENITOR ID
</td></tr>
<tr>
<td align="center">1
</td><td align="center">1010
</td><td align="center">3
</td><td align="center">1006
</td></tr>
<tr>
<td align="center">2
</td><td align="center">1010
</td><td align="center">4
</td><td align="center">1007
</td></tr>
<tr>
<td align="center">3
</td><td align="center">1010
</td><td align="center">5
</td><td align="center">1008
</td></tr>
<tr>
<td align="center">4
</td><td align="center">1010
</td><td align="center">6
</td><td align="center">1009
</td></tr>
<tr>
<td align="center">5
</td><td align="center">1032
</td><td align="center">3
</td><td align="center">1019
</td></tr>
<tr>
<td align="center">6
</td><td align="center">1032
</td><td align="center">4
</td><td align="center"> 930
</td></tr>
<tr>
<td align="center">7
</td><td align="center">1032
</td><td align="center">5
</td><td align="center">1000
</td></tr>
</table>
<p><br clear="all" />
</p>
<a name="Genealogy_Management_for_a_Reciprocal_Half-Sib_Recurrent_Selection_Method" id="Genealogy_Management_for_a_Reciprocal_Half-Sib_Recurrent_Selection_Method"></a><h4> <span class="mw-headline"> Genealogy Management for a Reciprocal Half-Sib Recurrent Selection Method </span></h4>
<p>This section indicates how the pedigrees from the reciprocal half-sib recurrent selection method described by Fehr (1988) on page 192 can be recorded in GMS. The method is designed to simultaneously improve two populations for their combining ability with each other, hence each population is used as a broad based tester for the other. In the method described individual plants (100 or more) are identified in each population and both (1) selfed and the seed stored and (2) crossed as male to six or more individuals in the other population to form a set of half-sib families. The half-sib families are field evaluated and the individuals from each family that performed best are identified. The selfed seed from these top lines (10 or more) are grown and random mated to produce the next cycle.
</p>
<dl><dd>(A) In the example the pedigree starts with the formation of the 3<sup>rd</sup> cycle (Table 2.3.3.a). Reciprocal Recurrent Population A Cycle 3 (RRPAC3) is formed by population random mating (PRM) method (method 421 in Table 2.2.d) among N parents. The first two are in the GERMPLASM TABLE and the rest are in the OTHER PROGENITORS TABLE. These parents are selfed lines from the cycle 2 population A which have been identified as the top lines in the half-sib test. The GIDs of these lines will be entered not a symbolic name as here. This becomes population RRPAC3 with GID 100 (row 1 in Table 2.3.3.a). The same procedure in Population B gives RRPBC3 with GID 101 (row 2 in Table 2.3.3.a).
</dd></dl>
<dl><dd>(B) One hundred or more lines are identified at random (Method IDN) in each population, GIDs 201and 203 (rows 4 and 6 in Table 2.3.3.a) are examples for population A and 301 and 303 (rows 9 and 11) are examples for population B.
</dd></dl>
<dl><dd>(C) These lines are selfed (Method SLF) and become GIDs 202 (row 5), 204 (row 7) for population A and 302 (row 10) and 304 (row 12) for population B are examples.
</dd></dl>
<dl><dd>(D) Each of the lines is then mated (in this case as a male) to at least six individuals in the other population. The other population acts as a broad based tester and in this case is the female of the cross. The method is BROAD BASED TESTER, POP CF (TBP) and as the tester is the female the attribute of the method is set to reflect this and to indicate how many lines are used as females. The seed from the different females is bulked and the attributes will show how the bulking is achieved.  GIDs 451 and 452 give two examples for population A (rows 14 and 15) and GIDs 551 and 552 (rows 17 and 18) for population B.
</dd></dl>
<dl><dd>(E) Seed from the selfed lines which are identified as superior (10 or more) are then grown in a crossing block and random mated to form cycle 4 of each population. These are indicated by GIDs 600 and 601 (rows 20 and 21 in Table 2.3.3.a).<br /><br />
</dd></dl>
<table border="2" cellspacing="0" cellpadding="4" width="80%">
<tr>
<td colspan="9"><b>Table 2.3.3.a: Germplasm Table for Reciprocal Recurrent Selection example.</b>
</td></tr>
<tr>
<td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center" colspan="3">Generative Methods
</td><td align="center">Derivative or Management Methods
</td><td>Breeder’s name
</td></tr>
<tr>
<td align="center">Row in table
</td><td align="center">GID
</td><td align="center">METH
</td><td align="center">No PROGEN
</td><td align="center">FPARNT
</td><td align="center">MPARNT
</td><td align="center">GROUP
</td><td align="center">SOURCE
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">1
</td><td align="center">101
</td><td align="center">PRM
</td><td align="center">N
</td><td align="center">RRPAC2-Sa
</td><td align="center">RRPAC2-Sb
</td><td align="center">.
</td><td align="center">.
</td><td>RRPAC3
</td></tr>
<tr>
<td align="center">2
</td><td align="center">102
</td><td align="center">PRM
</td><td align="center">M
</td><td align="center">RRPBC2-Sa
</td><td align="center">RRPBC2-Sb
</td><td align="center">.
</td><td align="center">.
</td><td>RRPBC3
</td></tr>
<tr>
<td align="center">3
</td><td align="center">.
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">4
</td><td align="center">201
</td><td align="center">IDN
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">101
</td><td align="center">101
</td><td>RRPAC3-I1
</td></tr>
<tr>
<td align="center">5
</td><td align="center">202
</td><td align="center">SLF
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">101
</td><td align="center">201
</td><td>RRPAC3-S1
</td></tr>
<tr>
<td align="center">6
</td><td align="center">203
</td><td align="center">IDN
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">101
</td><td align="center">101
</td><td>RRPAC3-I2
</td></tr>
<tr>
<td align="center">7
</td><td align="center">204
</td><td align="center">SLF
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">101
</td><td align="center">203
</td><td>RRPAC3-S2
</td></tr>
<tr>
<td align="center">8
</td><td align="center">.
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">9
</td><td align="center">301
</td><td align="center">IDN
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">102
</td><td align="center">102
</td><td>RRPBC3-I1
</td></tr>
<tr>
<td align="center">10
</td><td align="center">302
</td><td align="center">SLF
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">102
</td><td align="center">301
</td><td>RRPBC3-S2
</td></tr>
<tr>
<td align="center">11
</td><td align="center">303
</td><td align="center">IDN
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">102
</td><td align="center">102
</td><td>RRPBC3-I2
</td></tr>
<tr>
<td align="center">12
</td><td align="center">304
</td><td align="center">SLF
</td><td align="center">.
</td><td align="center">.
</td><td align="center">.
</td><td align="center">102
</td><td align="center">303
</td><td>RRPBC3-S2
</td></tr>
<tr>
<td align="center">13
</td><td align="center">.
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">14
</td><td align="center">451
</td><td align="center">TBP
</td><td align="center">2
</td><td align="center">102
</td><td align="center">201
</td><td align="center">.
</td><td align="center">.
</td><td>TRRPAC3-1
</td></tr>
<tr>
<td align="center">15
</td><td align="center">452
</td><td align="center">TBP
</td><td align="center">2
</td><td align="center">102
</td><td align="center">202
</td><td align="center">.
</td><td align="center">.
</td><td>TRRPAC3-2
</td></tr>
<tr>
<td align="center">16
</td><td align="center">.
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">17
</td><td align="center">551
</td><td align="center">TBP
</td><td align="center">2
</td><td align="center">101
</td><td align="center">301
</td><td align="center">.
</td><td align="center">.
</td><td>TRRPBC3-1
</td></tr>
<tr>
<td align="center">18
</td><td align="center">552
</td><td align="center">TBP
</td><td align="center">2
</td><td align="center">101
</td><td align="center">302
</td><td align="center">.
</td><td align="center">.
</td><td>TRRPBCR-1
</td></tr>
<tr>
<td align="center">19
</td><td align="center">.
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">20
</td><td align="center">600
</td><td align="center">PRM
</td><td align="center">N
</td><td align="center">202
</td><td align="center">204
</td><td align="center">.
</td><td align="center">.
</td><td>RRPAC4
</td></tr>
<tr>
<td align="center">21
</td><td align="center">601
</td><td align="center">PRM
</td><td align="center">M
</td><td align="center">302
</td><td align="center">304
</td><td align="center">.
</td><td align="center">.
</td><td>RRPBC4
</td></tr>
<tr>
<td colspan="9">ID = is unique germplasm identifier.<br />METH = is the breeding method, either generative, derivative or management.<br />No PROGEN = is the number of progenitors of a derivative method.<br />FPARNT = ID of female parent.<br />MPARNT = ID of male parent.<br />GROUP = ID of last generative process for the derivative or management process.<br />SOURCE = ID of immediate predecessor for the derivative or management process.
</td></tr>
</table>
<p><br clear="all" />
</p><p><br />
</p>
<a name="Genealogy_for_a_polycross" id="Genealogy_for_a_polycross"></a><h4> <span class="mw-headline"> Genealogy for a polycross </span></h4>
<p>The following is an illustration of how a polycross with four parents is stored in GMS. The artificial example uses clones, but inbred lines or selections from an open pollenated population could equally well have been used instead. In a polycross each line is used as a female and mated to all others and the half sib families are kept separate. In each case the other three clones are used as parents, ie. selfs are excluded. This means that the attribute for method 420 has to be set to NO. Also the other clones used as males have to be entered in the other progenitors table.
</p>
<table border="2" cellspacing="0" cellpadding="4" width="80%">
<tr>
<td align="center" colspan="6"><b>Table 2.2.4.a: Schematic representation of a polycross with four clones as parents.</b>
</td></tr>
<tr>
<td align="center">GID
</td><td align="center">Method
</td><td align="center">Number of progenitors
</td><td align="center">Female parent
</td><td align="center">Male parent
</td><td align="center">Comments
</td></tr>
<tr>
<td align="center">51
</td><td>926 (cultivar release)
</td><td align="center">-1
</td><td align="center">…
</td><td align="center">…
</td><td>Clone 1
</td></tr>
<tr>
<td align="center">52
</td><td>33 (unknown derivative)
</td><td align="center">-1
</td><td align="center">…
</td><td align="center">…
</td><td>Clone 2
</td></tr>
<tr>
<td align="center">53
</td><td>33
</td><td align="center">-1
</td><td align="center">…
</td><td align="center">…
</td><td>Clone 3
</td></tr>
<tr>
<td align="center">54
</td><td>504 (single plant sel cf)
</td><td align="center">-1
</td><td align="center">…
</td><td align="center">…
</td><td>Clone 4
</td></tr>
<tr>
<td align="center">55
</td><td>420 (polycross)
</td><td align="center">4
</td><td align="center">51
</td><td align="center">52
</td><td>Cross with clone 1 as female.
</td></tr>
<tr>
<td align="center">56
</td><td>420 (polycross)
</td><td align="center">4
</td><td align="center">52
</td><td align="center">51
</td><td>Cross with clone 2 as female.
</td></tr>
<tr>
<td align="center">57
</td><td>420 (polycross)
</td><td align="center">4
</td><td align="center">53
</td><td align="center">51
</td><td>Cross with clone 3 as female.
</td></tr>
<tr>
<td align="center">58
</td><td>420 (polycross)
</td><td align="center">4
</td><td align="center">54
</td><td align="center">51
</td><td>Cross with clone 4 as female.
</td></tr>
<tr>
<td align="center">59
</td><td>504
</td><td align="center">-1
</td><td align="center">55
</td><td align="center">55
</td><td>Single plant selection from cross with female as clone 1. Note that group is GID = 55 and immediate source is also GID = 55.
</td></tr>
<tr>
<td align="center">60
</td><td>504
</td><td align="center">-1
</td><td align="center">56
</td><td align="center">56
</td><td>Single plant selection from cross with female as clone 2. Note that group is GID = 56 and immediate source is also GID = 56.
</td></tr>
</table>
<p><br clear="all" />
</p>
<table border="2" cellspacing="0" cellpadding="4" width="60%">
<tr>
<td colspan="4"><b>Table 2.3.4.b: Other progenitors table for the polycross example.</b>
</td></tr>
<tr>
<td>GID
</td><td>Progenitor Number
</td><td>Progenitor GIID
</td><td>Comments
</td></tr>
<tr>
<td>55
</td><td>3
</td><td>53
</td><td>Third clone used as male
</td></tr>
<tr>
<td>55
</td><td>4
</td><td>54
</td><td>Fourth clone used as male
</td></tr>
<tr>
<td>56
</td><td>3
</td><td>53
</td><td>Third clone used as male
</td></tr>
<tr>
<td>56
</td><td>4
</td><td>54
</td><td>Fourth clone used as male
</td></tr>
<tr>
<td>…
</td><td>&nbsp;
</td><td>&nbsp;
</td><td>And so for the other 2 half sib families
</td></tr>
</table>
<p><br clear="all" />
</p>
<a name="Genealogy_for_a_half_diallel" id="Genealogy_for_a_half_diallel"></a><h4> <span class="mw-headline"> Genealogy for a half diallel </span></h4>
<p>The following is an illustration of how a half diallel with four parents is stored in GMS. The artificial example uses the same four clones as in the previous example on polycrosses, but again inbred lines or selections from an open pollenated population could equally well have been used. A half diallel is a series of single crosses with the full sib families kept separate. Each cross could equally well have been stored with the method for a single cross. The reason for recording a method for a diallel is to record extra information in the system. No attempt has been made to record every possible crossing strategy in the methods table as this would proliferate this table. Instead a compromise has been attempted between providing a minimal table (see Table 2.2.b) and information on the purposes of the matings.<br /><br />
</p>
<table border="2" cellspacing="0" cellpadding="4" width="80%">
<tr>
<td colspan="6"><b>Table 2.3.5.a: Schematic germplasm table for a half diallel using the same four clones as in the example for the polycross.</b>
</td></tr>
<tr>
<td align="center">GID
</td><td align="center">Method
</td><td align="center">Number of progenitors
</td><td align="center">Female Parent
</td><td align="center">Male Parent
</td><td align="center">Comments
</td></tr>
<tr>
<td align="center">51
</td><td>926 (cultivar release)
</td><td align="center">-1
</td><td align="center">…
</td><td align="center">…
</td><td>Clone 1
</td></tr>
<tr>
<td align="center">52
</td><td>33 (unknown derivative)
</td><td align="center">-1
</td><td align="center">…
</td><td align="center">…
</td><td>Clone 2
</td></tr>
<tr>
<td align="center">53
</td><td>33
</td><td align="center">-1
</td><td align="center">…
</td><td align="center">…
</td><td>Clone 3
</td></tr>
<tr>
<td align="center">54
</td><td>504 (single plant sel cf.)
</td><td align="center">-1
</td><td align="center">…
</td><td align="center">…
</td><td>Clone 4
</td></tr>
<tr>
<td align="center">55
</td><td>406 (half diallel)
</td><td align="center">2
</td><td align="center">51
</td><td align="center">52
</td><td>Cross with clone 1 as female and clone 2 as male.
</td></tr>
<tr>
<td align="center">56
</td><td>406 (half diallel)
</td><td align="center">2
</td><td align="center">51
</td><td align="center">53
</td><td>Cross with clone 1 as female and clone 3 as male.
</td></tr>
<tr>
<td align="center">57
</td><td>406 (half diallel)
</td><td align="center">2
</td><td align="center">51
</td><td align="center">54
</td><td>Cross with clone 1 as female and clone 4 as male.
</td></tr>
<tr>
<td align="center">58
</td><td>406 (half diallel)
</td><td align="center">2
</td><td align="center">52
</td><td align="center">54
</td><td>Cross with clone 2 as female and clone 3 as male.
</td></tr>
<tr>
<td align="center">59
</td><td>406 (half diallel)
</td><td align="center">2
</td><td align="center">53
</td><td align="center">54
</td><td>Cross with clone 2 as female and clone 4 as male.
</td></tr>
<tr>
<td align="center">60
</td><td>406 (half diallel)
</td><td align="center">2
</td><td align="center">54
</td><td align="center">52
</td><td>Cross with clone 3 as female and clone 4 as male.
</td></tr>
<tr>
<td align="center">62
</td><td>504
</td><td align="center">-1
</td><td align="center">55
</td><td align="center">55
</td><td>Single plant selection from cross with female as clone 1 and male clone 2. Note that group is GID = 55 and immediate source is also GID = 55.
</td></tr>
<tr>
<td align="center">63
</td><td>504
</td><td align="center">-1
</td><td align="center">56
</td><td align="center">56
</td><td>Single plant selection from cross with female as clone 1 and male clone 3.  Note that group is GID = 56 and immediate source is also GID = 56.
</td></tr>
</table>
<p><br clear="all" />
</p>
<a name="Genealogy_of_Rice_Variety_IR64" id="Genealogy_of_Rice_Variety_IR64"></a><h4> <span class="mw-headline"> Genealogy of Rice Variety IR64 </span></h4>
<p>The following steps recorded in the IRRI pedigree database (Table 2.3.1.a) lead to the development of variety IR64. BCID is the breeder’s cross ID, FPARENT is the breeder’s notation for the female parent. FPSRC is the source of the female parent, either a previous BCID (defining the cross that produced the female parent, possibly by selection) or an accession number from the germplasm bank. MPARENT and MPSCR are equivalent fields for the male parent. The following illustrate many of the steps needed to enter historical genealogies.<br /><br />
</p>
<table border="2" cellspacing="0" cellpadding="4" width="75%">
<tr>
<td colspan="5"><b>Table 2.3.6.a: The steps recorded in the IRRI pedigree database which led to the cultivar IR64.</b>
</td></tr>
<tr>
<td>BCID
</td><td>FPARENT
</td><td>FPSRC
</td><td>MPARENT
</td><td>MPSRC
</td></tr>
<tr>
<td>IR2040
</td><td>IR1561-149-1
</td><td>IR1561
</td><td>IR661-1-140-3*4/O.NIVARA
</td><td>IR1737
</td></tr>
<tr>
<td>IR2055
</td><td>BPI 121-407
</td><td>15762
</td><td>IR1416-131-5/IR576-160-2
</td><td>IR1833
</td></tr>
<tr>
<td>IR2061
</td><td>IR833-6-2-1-1
</td><td>IR833
</td><td>IR1561-149-1//IR661-1-140-3*4/O.NIVARA
</td><td>IR2040
</td></tr>
<tr>
<td>IR2146
</td><td>IR773A1-36-2-1
</td><td>IR773
</td><td>IR773A1-36-2-1*3/O.NIVARA
</td><td>IR1916
</td></tr>
<tr>
<td>IR5236
</td><td>IR2006-P3-31-3
</td><td>IR2006
</td><td>IR2146-68-1
</td><td>IR2146
</td></tr>
<tr>
<td>IR5338
</td><td>IR2061-465-1-4
</td><td>IR2061
</td><td>IR2055-475-2
</td><td>IR2055
</td></tr>
<tr>
<td>IR5657
</td><td>IR2006-P3-31-3/IR2146-68-1
</td><td>IR5236
</td><td>IR2061-465-1-4/IR2055-475-2
</td><td>IR5338
</td></tr>
<tr>
<td>IR18348
</td><td>IR5657-33-2-1
</td><td>IR5657
</td><td>IR2061-465-1-5-5
</td><td>IR2061
</td></tr>
<tr>
<td colspan="5">IR18348-36-3-3 = IR64
</td></tr>
<tr>
<td colspan="5">BCID = is the breeder’s cross ID.<br />FPARENT = is the breeder’s notation for the female parent.<br />FPSCR = is the source of the female parent, either a previous BCID (defining the cross that produced the female parent, most likely by selection) or an accession number from the germplasm bank.<br />MPARENT = equivalent to FPARENT for male.<br />MPSRC = equivalent to FPSRC for male.
</td></tr>
</table><br clear="all" />
<p>We can presume that the following records are already stored in the germplasm table.<br /><br />
</p>
<table border="2" cellspacing="0" cellpadding="4" width="75%">
<tr>
<td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center" colspan="2">Generative Methods
</td><td align="center" colspan="2">Derivative or Management Methods
</td><td>Breeder’s name
</td></tr>
<tr>
<td align="center">ID
</td><td align="center">METH
</td><td align="center">FPARNT
</td><td align="center">MPARNT
</td><td align="center">GROUP
</td><td align="center">SOURCE
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">38
</td><td align="center">C2W
</td><td align="center">36
</td><td align="center">37
</td><td align="center">.
</td><td align="center">.
</td><td>IR833
</td></tr>
<tr>
<td align="center">68
</td><td align="center">BC
</td><td align="center">63
</td><td align="center">67
</td><td align="center">.
</td><td align="center">.
</td><td>IR1737
</td></tr>
<tr>
<td align="center">71
</td><td align="center">C2W
</td><td align="center">70
</td><td align="center">59
</td><td align="center">.
</td><td align="center">.
</td><td>IR1833
</td></tr>
<tr>
<td align="center">85
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">33
</td><td align="center">84
</td><td>IR773A1-36-2-1
</td></tr>
<tr>
<td align="center">90
</td><td align="center">BC
</td><td align="center">86
</td><td align="center">89
</td><td align="center">.
</td><td align="center">.
</td><td>IR1916
</td></tr>
<tr>
<td align="center">91
</td><td align="center">?
</td><td align="center">81
</td><td align="center">60
</td><td align="center">.
</td><td align="center">.
</td><td>IR2006
</td></tr>
<tr>
<td align="center">93
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">57
</td><td align="center">92
</td><td>IR1561-149-1
</td></tr>
<tr>
<td colspan="7">ID = is unique germplasm identifier.<br />METH = is the breeding method, either generative, derivative or management.<br />FPARNT = ID of female parent.<br />MPARNT = ID of male parent.<br />GROUP = ID of last generative process for the derivative or management process.<br />SOURCE = ID of immediate predecessor for the derivative or management process.
</td></tr>
</table>
<p><br clear="all" />
</p><p>The new pedigrees are entered in the germplasm table as follows.
</p>
<dl><dd>(A) IR2040<br />FPARENT IR1561-149-1 is a derivative of BCID IR1561 and is already in the table with ID 93. MPARENT is a third backcross of O.NIVARA onto IR661-1-140-3 (which is incidently cultivar IR24). This backcross is IR1737, as indicated by MPSRC, and is in the table with ID 68. Hence the new cross, with ID 94, has parents 93 and 68 and BCID IR2040. From the methods table this is a single cross.
</dd></dl>
<table border="2" cellspacing="0" cellpadding="4" width="75%">
<tr>
<td align="center">94
</td><td align="center">C2W
</td><td align="center">93
</td><td align="center">68
</td><td align="center">.
</td><td align="center">.
</td><td>IR2040
</td></tr></table>
<dl><dd>(B) IR2055<br />This has a foreign female parent BPI 121-407, from a gene bank accession 15762. This is not in the germplasm table, and as nothing of its pedigree is known it can be entered as an import of an unknown source with ID 95 as follows:
</dd></dl>
<table border="2" cellspacing="0" cellpadding="4" width="75%">
<tr>
<td align="center">95
</td><td align="center">IUS
</td><td align="center">0
</td><td align="center">0
</td><td align="center">.
</td><td align="center">.
</td><td>BPI 121-407
</td></tr></table>
<dl><dd>The male parent is an F<sub>1</sub> from BCID IR1833 which is in the table as with ID 71, so the new cross IR2055 can be entered as a three-way cross with immediate parents 95 and 71 as follows:
</dd></dl>
<table border="2" cellspacing="0" cellpadding="4" width="75%">
<tr>
<td align="center">96
</td><td align="center">C3W
</td><td align="center">95
</td><td align="center">71
</td><td align="center">.
</td><td align="center">.
</td><td>IR2055
</td></tr></table>
<dl><dd>(C) IR2061<br />This has female parent IR833-6-2-1-1 which is a selection out of cross IR833. The cross is in the table as ID 38, but the selection is not, so it must be added. The first step adds ID 97 for IR833-6 as a single plant selection from 38 (the first derivative in GROUP 38). The second step adds ID 98 for IR833-6-2 as a single plant selection in GROUP 38 from SOURCE 97. IDs 99 and 100 lead to IR833-6-2-1-1.
</dd></dl>
<table border="2" cellspacing="0" cellpadding="4" width="75%">
<tr>
<td align="center">97
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">38
</td><td align="center">38
</td><td>IR833-6
</td></tr>
<tr>
<td align="center">98
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">38
</td><td align="center">97
</td><td>IR833-6-2
</td></tr>
<tr>
<td align="center">99
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">38
</td><td align="center">98
</td><td>IR833-6-2-1
</td></tr>
<tr>
<td align="center">100
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">38
</td><td align="center">99
</td><td>IR833-6-2-1-1
</td></tr></table>
<dl><dd>The male parent is the F<sub>1</sub> of the cross with BCID IR2040 which was entered as ID 94 so the process is a three-way cross. Hence we can now enter IR2061 as:
</dd></dl>
<table border="2" cellspacing="0" cellpadding="4" width="75%">
<tr>
<td align="center">101
</td><td align="center">C3W
</td><td align="center">100
</td><td align="center">94
</td><td align="center">.
</td><td align="center">.
</td><td>IR2061
</td></tr></table>
<dl><dd>(D) IR2146<br />The is the cross of a third backcross of wild species O. NIVARA onto IR773A1-36-2-1 (ID 85) which is the recurrent parent. The third backcross is identified by MPSRC as BCID IR1916 which is in the table as ID 90, so IR2146 is entered as follows:
</dd></dl>
<table border="2" cellspacing="0" cellpadding="4" width="75%">
<tr>
<td align="center">102
</td><td align="center">BC
</td><td align="center">85
</td><td align="center">90
</td><td align="center">.
</td><td align="center">.
</td><td>IR2146
</td></tr></table>
<dl><dd>(E) IR5236<br />This is a single cross between two lines derived from crosses recorded as ID 91 and ID 102. Since the derivatives are not in the table, they must be added:
</dd></dl>
<table border="2" cellspacing="0" cellpadding="4" width="75%">
<tr>
<td align="center">103
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">91
</td><td align="center">91
</td><td>IR2006-P3
</td></tr>
<tr>
<td align="center">104
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">91
</td><td align="center">103
</td><td>IR2006-P3-31
</td></tr>
<tr>
<td align="center">105
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">91
</td><td align="center">104
</td><td>IR2006-P3-31-3
</td></tr>
<tr>
<td align="center">106
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">102
</td><td align="center">102
</td><td>IR2146-68
</td></tr>
<tr>
<td align="center">107
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">102
</td><td align="center">102
</td><td>IR2146-68-1
</td></tr></table>
<dl><dd>The cross is then added as:
</dd></dl>
<table border="2" cellspacing="0" cellpadding="4" width="75%">
<tr>
<td align="center">108
</td><td align="center">C2W
</td><td align="center">111
</td><td align="center">113
</td><td align="center">.
</td><td align="center">.
</td><td>IR5236
</td></tr></table>
<dl><dd>(F) IR5338 is exactly similar to IR5236.
</dd></dl>
<table border="2" cellspacing="0" cellpadding="4" width="75%">
<tr>
<td align="center">109
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">101
</td><td align="center">101
</td><td>IR2061-465
</td></tr>
<tr>
<td align="center">110
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">101
</td><td align="center">109
</td><td>IR2061-465-1
</td></tr>
<tr>
<td align="center">111
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">101
</td><td align="center">110
</td><td>IR2061-465-1-4
</td></tr>
<tr>
<td align="center">112
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">96
</td><td align="center">96
</td><td>IR2055-475
</td></tr>
<tr>
<td align="center">113
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">96
</td><td align="center">112
</td><td>IR2055-475-2
</td></tr>
<tr>
<td align="center">114
</td><td align="center">C2W
</td><td align="center">111
</td><td align="center">113
</td><td align="center">.
</td><td align="center">.
</td><td>IR5338
</td></tr>
<tr>
<td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td>&nbsp;
</td></tr></table>
<dl><dd>(G) Next a double cross (IR5657) is made between the last two F<sub>1</sub>s:
</dd></dl>
<table border="2" cellspacing="0" cellpadding="4" width="75%">
<tr>
<td align="center">115
</td><td align="center">C4W
</td><td align="center">108
</td><td align="center">114
</td><td align="center">.
</td><td align="center">&nbsp;
</td><td align="center">IR5657
</td></tr></table>
<dl><dd>(H) IR18348<br />This is a single cross between derivatives from the double cross just entered (IR5657, ID 115) and the cross IR2061 ( ID 101). The derivatives of IR5657 have to be added. The male parent already has some derivatives added, IDs 109, 110 and 111. This parent is a derivative of ID 110.
</dd></dl>
<table border="2" cellspacing="0" cellpadding="4" width="75%">
<tr>
<td align="center">116
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">115
</td><td align="center">115
</td><td>IR5657-33
</td></tr>
<tr>
<td align="center">117
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">115
</td><td align="center">116
</td><td>IR5657-33-2
</td></tr>
<tr>
<td align="center">118
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">115
</td><td align="center">117
</td><td>IR5657-33-2-1
</td></tr>
<tr>
<td align="center">119
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">101
</td><td align="center">110
</td><td>IR2061-465-1-5
</td></tr>
<tr>
<td align="center">120
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">101
</td><td align="center">119
</td><td>IR2061-465-1-5-5
</td></tr>
<tr>
<td align="center">121
</td><td align="center">C2W
</td><td align="center">118
</td><td align="center">120
</td><td align="center">.
</td><td align="center">.
</td><td>IR18348
</td></tr></table>
<dl><dd>(I)<br />The last step in the development of variety IR64 consists of four generations of selection from IR18348 (ID 121):
</dd></dl>
<table border="2" cellspacing="0" cellpadding="4" width="75%">
<tr>
<td>122
</td><td>DSP
</td><td>.
</td><td>.
</td><td>121
</td><td>121
</td><td>IR18348-36
</td></tr>
<tr>
<td>123
</td><td>DSP
</td><td>.
</td><td>.
</td><td>121
</td><td>122
</td><td>IR18348-36-3
</td></tr>
<tr>
<td>124
</td><td>DSP
</td><td>.
</td><td>.
</td><td>121
</td><td>123
</td><td>IR18348-36-3-3 = IR64
</td></tr>
</table><br clear="all" />
<table border="2" cellspacing="0" cellpadding="4" width="75%">
<tr>
<td colspan="7"><b>Table 2.3.6.b: Germplasm Table for the genealogy of IR64.</b>
</td></tr>
<tr>
<td align="center">&nbsp;
</td><td align="center">&nbsp;
</td><td align="center" colspan="2">Generative Methods
</td><td align="center" colspan="2">Derivative or Management Methods
</td><td>Breeder’s name
</td></tr>
<tr>
<td align="center">ID
</td><td align="center">METH
</td><td align="center">FPARNT
</td><td align="center">MPARNT
</td><td align="center">GROUP
</td><td align="center">SOURCE
</td><td>&nbsp;
</td></tr>
<tr>
<td align="center">38
</td><td align="center">C2W
</td><td align="center">36
</td><td align="center">37
</td><td align="center">.
</td><td align="center">.
</td><td>IR833
</td></tr>
<tr>
<td align="center">68
</td><td align="center">BC
</td><td align="center">63
</td><td align="center">67
</td><td align="center">.
</td><td align="center">.
</td><td>IR1737
</td></tr>
<tr>
<td align="center">71
</td><td align="center">C2W
</td><td align="center">70
</td><td align="center">59
</td><td align="center">.
</td><td align="center">.
</td><td>IR1833
</td></tr>
<tr>
<td align="center">85
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">33
</td><td align="center">84
</td><td>IR773A1-36-2-1
</td></tr>
<tr>
<td align="center">90
</td><td align="center">BC
</td><td align="center">86
</td><td align="center">89
</td><td align="center">.
</td><td align="center">.
</td><td>IR1916
</td></tr>
<tr>
<td align="center">91
</td><td align="center">?
</td><td align="center">81
</td><td align="center">60
</td><td align="center">.
</td><td align="center">.
</td><td>IR2006
</td></tr>
<tr>
<td align="center">93
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">57
</td><td align="center">92
</td><td>IR1561-149-1
</td></tr>
<tr>
<td align="center">94
</td><td align="center">C2W
</td><td align="center">93
</td><td align="center">68
</td><td align="center">.
</td><td align="center">.
</td><td>IR2040
</td></tr>
<tr>
<td align="center">95
</td><td align="center">IUS
</td><td align="center">0
</td><td align="center">0
</td><td align="center">.
</td><td align="center">.
</td><td>BPI 121-407
</td></tr>
<tr>
<td align="center">96
</td><td align="center">C3W
</td><td align="center">95
</td><td align="center">71
</td><td align="center">.
</td><td align="center">.
</td><td>IR2055
</td></tr>
<tr>
<td align="center">97
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">38
</td><td align="center">38
</td><td>IR833-6
</td></tr>
<tr>
<td align="center">98
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">38
</td><td align="center">97
</td><td>IR833-6-2
</td></tr>
<tr>
<td align="center">99
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">38
</td><td align="center">98
</td><td>IR833-6-2-1
</td></tr>
<tr>
<td align="center">100
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">38
</td><td align="center">99
</td><td>IR833-6-2-1-1
</td></tr>
<tr>
<td align="center">101
</td><td align="center">C3W
</td><td align="center">100
</td><td align="center">94
</td><td align="center">.
</td><td align="center">.
</td><td>IR2061
</td></tr>
<tr>
<td align="center">102
</td><td align="center">BC
</td><td align="center">85
</td><td align="center">90
</td><td align="center">.
</td><td align="center">.
</td><td>IR2146
</td></tr>
<tr>
<td align="center">103
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">91
</td><td align="center">91
</td><td>IR2006-P3
</td></tr>
<tr>
<td align="center">104
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">91
</td><td align="center">103
</td><td>IR2006-P3-31
</td></tr>
<tr>
<td align="center">105
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">91
</td><td align="center">104
</td><td>IR2006-P3-31-3
</td></tr>
<tr>
<td align="center">106
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">102
</td><td align="center">102
</td><td>IR2146-68
</td></tr>
<tr>
<td align="center">107
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">102
</td><td align="center">102
</td><td>IR2146-68-1
</td></tr>
<tr>
<td align="center">108
</td><td align="center">C2W
</td><td align="center">111
</td><td align="center">113
</td><td align="center">.
</td><td align="center">.
</td><td>IR5236
</td></tr>
<tr>
<td align="center">109
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">101
</td><td align="center">101
</td><td>IR2061-465
</td></tr>
<tr>
<td align="center">110
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">101
</td><td align="center">109
</td><td>IR2061-465-1
</td></tr>
<tr>
<td align="center">111
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">101
</td><td align="center">110
</td><td>IR2061-465-1-4
</td></tr>
<tr>
<td align="center">112
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">96
</td><td align="center">96
</td><td>IR2055-475
</td></tr>
<tr>
<td align="center">113
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">96
</td><td align="center">112
</td><td>IR2055-475-2
</td></tr>
<tr>
<td align="center">114
</td><td align="center">C2W
</td><td align="center">111
</td><td align="center">113
</td><td align="center">.
</td><td align="center">.
</td><td>IR5338
</td></tr>
<tr>
<td align="center">115
</td><td align="center">C4W
</td><td align="center">108
</td><td align="center">114
</td><td align="center">.
</td><td align="center">.
</td><td>IR5657
</td></tr>
<tr>
<td align="center">116
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">115
</td><td align="center">115
</td><td>IR5657-33
</td></tr>
<tr>
<td align="center">117
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">115
</td><td align="center">116
</td><td>IR5657-33-2
</td></tr>
<tr>
<td align="center">118
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">115
</td><td align="center">117
</td><td>IR5657-33-2-1
</td></tr>
<tr>
<td align="center">119
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">101
</td><td align="center">110
</td><td>IR2061-465-1-5
</td></tr>
<tr>
<td align="center">120
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">101
</td><td align="center">119
</td><td>IR2061-465-1-5-5
</td></tr>
<tr>
<td align="center">121
</td><td align="center">C2W
</td><td align="center">118
</td><td align="center">120
</td><td align="center">.
</td><td align="center">.
</td><td>IR18348
</td></tr>
<tr>
<td align="center">122
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">121
</td><td align="center">121
</td><td>IR18348-36
</td></tr>
<tr>
<td align="center">123
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">121
</td><td align="center">122
</td><td>IR18348-36-3
</td></tr>
<tr>
<td align="center">124
</td><td align="center">DSP
</td><td align="center">.
</td><td align="center">.
</td><td align="center">121
</td><td align="center">123
</td><td>IR18348-36-3-3 = IR64
</td></tr>
</table><br clear="all" />
<a name="REMOTE_ALLOCATION_OF_GERMPLASM_IDENTIFIERS" id="REMOTE_ALLOCATION_OF_GERMPLASM_IDENTIFIERS"></a><h2> <span class="mw-headline"> REMOTE ALLOCATION OF GERMPLASM IDENTIFIERS </span></h2>
<p>The objective of ICIS GMS is to provide a tool for the management of pedigrees to a wide audience of users, many of whom do not have access to a dynamic central database. These users are often generating new entries, so the problem of the remote allocation of unique germplasm identifiers is important. Thus remote users require a copy of all GMS records up to a certain date as well as the maintenance of a local GMS, the LGMS. Each user of GMS will have a unique user ID (USER_ID) with user details stored in a user table maintained by the central ICIS administrator.
</p><p>The LGMS has exactly the same structure as the GMS with internal pointers as well as pointers into the central GMS. Internal or local pointers are referred to as LOCAL IDs, TYPEs or NOs depending on the type of pointer. For example LGMS will have LOCAL_GERMPLASM_IDs, LOCAL_NAME_TYPEs, LOCAL_LOCATION_NOs, LOCAL_ATTRIBUTE_TYPEs and LOCAL_METHOD_NOs. They are only unique within the LGMS; system wide identifiers are assigned only when the central GMS is updated by each user. However the combination of USER_ID and LOCAL ID will be unique and provides the link to old records once the central GMS is updated.
</p><p>The LGMS, therefore, uses LOCAL IDs, but links to progenitors or source germplasm can be IDs in the central GMS or LOCAL IDs in the LGMS. Clearly no links in the central GMS can point to LOCL IDs. LOCAL IDs must be distinguishable from central ID codes since applications must know which GMS is being used for a particular transaction otherwise the uniqueness of identifiers can become compromised through partial updates of the GMS by a user. One method of distinguishing between LOCAL IDs and central IDs would be to make all the former ones negative.
</p><p>Users would periodically send subsets of LGMS to the GMS administrator for update. These subsets could be defined as all records up to a specified date. The GMS administrator would add the LGMS records for all tables to GMS assigning new ID codes, and adding USER_ID and LOCAL_GERMPLASM_ID to germplasm records so that entries in other private databases such as the SGM or LDMS (local data management system) are automatically linked to GMS through a search on these fields and do not need to be explicitly updated. This implies that USER_IDs and LOCAL IDs must never be reused even after updating. After an update the user gets a new version of GMS, and the LGMS is scrubbed of all updated records. Thereafter, any search of LGMS for a LOCAL_GERMPLASM_ID which has already been updated triggers a search in the updated GMS for USER_ID and LOCAL_ID.
</p>
<a name="UPDATING_THE_CENTRAL_GMS" id="UPDATING_THE_CENTRAL_GMS"></a><h2> <span class="mw-headline"> UPDATING THE CENTRAL GMS </span></h2>
<p>Presumably local users could update as soon as set generation is complete. Would Set Records then be updated before going to FBM? Clearly this cannot be a requirement because we must cater for remote users. What about users of the central ICIS DMS? Do they have to update before data entry? Presumably yes. Then any LOCAL ID encountered is known to be an attribute and its ID can be found by an attribute search at data entry. Most remote users will probably not send data when they update GMS.
</p><p>Is it necessary to keep Set Records? Yes probably but not centrally, because if a user does not use ICIS FBM and DMS then set records are the only ICIS format data available for specifying progenitors or selections by choosing from previous trials. If they are kept, is it necessary to update them when GMS is updated? Probably not because any lists dated prior to the last update of the GMS are known to contain LIDs as attributes, while later ones still contain ID pointers.
</p>
<a name="REFERENCES" id="REFERENCES"></a><h2> <span class="mw-headline"> REFERENCES </span></h2>
<p>Fabrizius, M. A., Cooper, M. Podlich, D., Brennan, P.S., Ellison, F.W. and DeLacy, I.H. 1996. Design and simulation of a recurrent selection program to improve yield and protein in spring wheat. University of Queensland, Dept. of Agriculture, Brisbane, QLD 4072 Australia.
</p><p>Fehr, W. R. 1987. Principles of cultivar development. Vol 1. Theory and Technique. Macmillan Publishing Company, New York.
</p>
<!-- 
NewPP limit report
Preprocessor node count: 367/1000000
Post-expand include size: 0/2097152 bytes
Template argument size: 0/2097152 bytes
Expensive parser function count: 0/100
-->
<div class="printfooter">
Retrieved from "<a href="http://localhost../../../../articles/t/d/m/TDM_GMS_Overview_5.5_1a1a.html">http://localhost../../../../articles/t/d/m/TDM_GMS_Overview_5.5_1a1a.html</a>"</div>
						<!-- end content -->
						<div class="visualClear"></div>
		</div>
	</div>
		</div>
		<div id="column-one">
	<div id="p-cactions" class="portlet">
		<h5>Views</h5>
		<div class="pBody">
			<ul>
	
				 <li id="ca-nstab-main" class="selected"><a href="../../../../articles/t/d/m/TDM_GMS_Overview_5.5_1a1a.html" title="View the content page [c]" accesskey="c">Page</a></li>
				 <li id="ca-talk" class="new"><a href="../../../../articles/t/d/m/Talk%7ETDM_GMS_Overview_5.5_64e2.html" title="Discussion about the content page [t]" accesskey="t">Discussion</a></li>
				 <li id="ca-viewsource"><a href="../../../../articles/t/d/m/TDM_GMS_Overview_5.5_1a1a.html" title="This page is protected.&#10;You can view its source [e]" accesskey="e">View source</a></li>
				 <li id="ca-history"><a href="../../../../articles/t/d/m/TDM_GMS_Overview_5.5_1a1a.html" title="Past versions of this page [h]" accesskey="h">History</a></li>			</ul>
		</div>
	</div>
	<div class="portlet" id="p-personal">
		<h5>Personal tools</h5>
		<div class="pBody">
			<ul>
				<li id="pt-login"><a href="../../../../articles/u/s/e/Special%7EUserLogin_94cd.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</a></li>
			</ul>
		</div>
	</div>
	<div class="portlet" id="p-logo">
		<a style="background-image: url(../../../../misc/icis-wiki3.png);" href="../../../../index.html" title="Visit the main page [z]" accesskey="z"></a>
	</div>
	<script type="text/javascript"> if (window.isMSIE55) fixalpha(); </script>
	<div class='generated-sidebar portlet' id='p-navigation'>
		<h5>Navigation</h5>
		<div class='pBody'>
			<ul>
				<li id="n-mainpage"><a href="../../../../index.html" title="Visit the main page [z]" accesskey="z">Main Page</a></li>
				<li id="n-portal"><a href="../../../../articles/c/o/m/ICISWiki%7ECommunity_Portal_7449.html" title="About the project, what you can do, where to find things">Community portal</a></li>
				<li id="n-recentchanges"><a href="../../../../articles/r/e/c/Special%7ERecentChanges_e0d0.html" title="The list of recent changes in the wiki [r]" accesskey="r">Recent changes</a></li>
				<li id="n-help"><a href="../../../../articles/c/o/n/Help%7EContents_22de.html" title="The place to find out">Help</a></li>
			</ul>
		</div>
	</div>
	<div id="p-search" class="portlet">
		<h5><label for="searchInput">Search</label></h5>
		<div id="searchBody" class="pBody">
			<form action="../../../../articles/s/e/a/Special%7ESearch_a8e5.html" id="searchform"><div>
				<input id="searchInput" name="search" type="text" title="Search ICISWiki [f]" accesskey="f" value="" />
				<input type='submit' name="go" class="searchButton" id="searchGoButton"	value="Go" title="Go to a page with this exact name if exists" />&nbsp;
				<input type='submit' name="fulltext" class="searchButton" id="mw-searchButton" value="Search" title="Search the pages for this text" />
			</div></form>
		</div>
	</div>
	<div class="portlet" id="p-tb">
		<h5>Toolbox</h5>
		<div class="pBody">
			<ul>
				<li id="t-whatlinkshere"><a href="../../../../articles/w/h/a/Special%7EWhatLinksHere_TDM_GMS_Overview_5.5_3fb8.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</a></li>
				<li id="t-recentchangeslinked"><a href="../../../../articles/r/e/c/Special%7ERecentChangesLinked_TDM_GMS_Overview_5.5_6c01.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</a></li>
<li id="t-specialpages"><a href="../../../../articles/s/p/e/Special%7ESpecialPages_f12b.html" title="List of all special pages [q]" accesskey="q">Special pages</a></li>
				<li id="t-print"><a href="../../../../articles/t/d/m/TDM_GMS_Overview_5.5_1a1a.html" title="Printable version of this page [p]" accesskey="p">Printable version</a></li>				<li id="t-permalink"><a href="../../../../articles/t/d/m/TDM_GMS_Overview_5.5_1a1a.html" title="Permanent link to this version of the page">Permanent link</a></li>			</ul>
		</div>
	</div>
		</div><!-- end of the left (by default at least) column -->
			<div class="visualClear"></div>
			<div id="footer">
				<div id="f-poweredbyico"><a href="http://www.mediawiki.org/"><img src="../../../../skins/common/images/poweredby_mediawiki_88x31.png" alt="Powered by MediaWiki" /></a></div>
				<div id="f-copyrightico"><a href="../../../../../COPYING.html"><img src="/icis/skins/common/images/gnu-fdl.png" alt='GNU Free Documentation License 1.2' /></a></div>
			<ul id="f-list">
					<li id="credits">This page was last modified 07:02, 27 February 2009 by <a href="../../../../articles/m/h/a/User%7EMhabito_276f.html" title="User:Mhabito">Maria Corina de Luna Habito</a>. </li>
					<li id="copyright">Content is available under <a href="../../../../../COPYING.html" class="external " title="../../../../../COPYING.html">GNU Free Documentation License 1.2</a>.</li>
					<li id="privacy"><a href="../../../../articles/p/r/i/ICISWiki%7EPrivacy_policy_0cbf.html" title="ICISWiki:Privacy policy">Privacy policy</a></li>
					<li id="about"><a href="../../../../articles/a/b/o/ICISWiki%7EAbout_53aa.html" title="ICISWiki:About">About ICISWiki</a></li>
					<li id="disclaimer"><a href="../../../../articles/g/e/n/ICISWiki%7EGeneral_disclaimer_9129.html" title="ICISWiki:General disclaimer">Disclaimers</a></li>
			</ul>
		</div>
</div>

		<script type="text/javascript">if (window.runOnloadHook) runOnloadHook();</script>
<!-- Served in 371.708 secs. --></body></html>
