<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
	<head>
		<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
		<meta http-equiv="Content-Style-Type" content="text/css" />
		<meta name="generator" content="MediaWiki 1.14.0" />
		<meta name="keywords" content="TDM Genealogy Management System 5.4" />
		<link rel="shortcut icon" href="../../../../misc/favicon.ico" />
		<link rel="search" type="application/opensearchdescription+xml" href="../../../../../opensearch_desc.php" title="ICISWiki (en)" />
		<link title="Creative Commons" type="application/rdf+xml" href="../../../../articles/t/d/m/TDM_Genealogy_Management_System_5.4_7420.html" rel="alternate meta" />
		<link rel="copyright" href="../../../../../COPYING.html" />
		<link rel="alternate" type="application/rss+xml" title="ICISWiki RSS Feed" href="http://localhost../../../../articles/r/e/c/Special%7ERecentChanges_e0d0.html" />
		<link rel="alternate" type="application/atom+xml" title="ICISWiki Atom Feed" href="http://localhost../../../../articles/r/e/c/Special%7ERecentChanges_e0d0.html" />
		<title>TDM Genealogy Management System 5.4 - ICISWiki</title>
		<link rel="stylesheet" href="../../../../skins/common/shared.css?195" type="text/css" media="screen" />
		<link rel="stylesheet" href="../../../../skins/common/commonPrint.css?195" type="text/css" media="print" />
		<link rel="stylesheet" href="../../../../skins/monobook/main.css?195" type="text/css" media="screen" />
		<!--[if lt IE 5.5000]><link rel="stylesheet" href="../../../../skins/monobook/IE50Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<!--[if IE 5.5000]><link rel="stylesheet" href="../../../../skins/monobook/IE55Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<!--[if IE 6]><link rel="stylesheet" href="../../../../skins/monobook/IE60Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<!--[if IE 7]><link rel="stylesheet" href="../../../../skins/monobook/IE70Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<link rel="stylesheet" href="../../../../skins/../../../../raw/MediaWiki%7ECommon.css?195" type="text/css" />
		<link rel="stylesheet" href="../../../../skins/../../../../raw/MediaWiki%7EPrint.css?195" type="text/css" media="print" />
		<link rel="stylesheet" href="../../../../skins/../../../../raw/MediaWiki%7EMonobook.css?195" type="text/css" />
		<link rel="stylesheet" href="../../../../skins/../../../../raw/gen.css?195" type="text/css" />
		<!--[if lt IE 7]><script type="text/javascript" src="../../../../skins/common/IEFixes.js?195"></script>
		<meta http-equiv="imagetoolbar" content="no" /><![endif]-->

		<script type= "text/javascript">/*<![CDATA[*/
		var skin = "monobook";
		var stylepath = "../../../../skins";
		var wgArticlePath = "../../../../$1";
		var wgScriptPath = "../../../../..";
		var wgScript = "/icis/index.php";
		var wgVariantArticlePath = false;
		var wgActionPaths = {};
		var wgServer = "http://localhost";
		var wgCanonicalNamespace = "";
		var wgCanonicalSpecialPageName = false;
		var wgNamespaceNumber = 0;
		var wgPageName = "TDM_Genealogy_Management_System_5.4";
		var wgTitle = "TDM Genealogy Management System 5.4";
		var wgAction = "view";
		var wgArticleId = "2819";
		var wgIsArticle = true;
		var wgUserName = null;
		var wgUserGroups = null;
		var wgUserLanguage = "en";
		var wgContentLanguage = "en";
		var wgBreakFrames = false;
		var wgCurRevisionId = "14459";
		var wgVersion = "1.14.0";
		var wgEnableAPI = true;
		var wgEnableWriteAPI = true;
		var wgSeparatorTransformTable = ["", ""];
		var wgDigitTransformTable = ["", ""];
		var wgRestrictionEdit = [];
		var wgRestrictionMove = [];
		/*]]>*/</script>

		<script type="text/javascript" src="../../../../skins/common/wikibits.js?195"><!-- wikibits js --></script>
		<!-- Head Scripts -->
		<script type="text/javascript" src="../../../../raw/gen.js"><!-- site js --></script>
	</head>
<body class="mediawiki ltr ns-0 ns-subject page-TDM_Genealogy_Management_System_5_4 skin-monobook">
	<div id="globalWrapper">
		<div id="column-content">
	<div id="content">
		<a name="top" id="top"></a>
				<h1 id="firstHeading" class="firstHeading">TDM Genealogy Management System 5.4</h1>
		<div id="bodyContent">
			<h3 id="siteSub">From ICISWiki</h3>
			<div id="contentSub"></div>
									<div id="jump-to-nav">Jump to: <a href="#column-one">navigation</a>, <a href="#searchInput">search</a></div>			<!-- start content -->
			<table id="toc" class="toc" summary="Contents"><tr><td><div id="toctitle"><h2>Contents</h2></div>
<ul>
<li class="toclevel-1"><a href="#PHYSICAL_STRUCTURE_OF_THE_ICIS_GMS_DATABASE"><span class="tocnumber">1</span> <span class="toctext">PHYSICAL STRUCTURE OF THE ICIS GMS DATABASE</span></a>
<ul>
<li class="toclevel-2"><a href="#INTRODUCTION"><span class="tocnumber">1.1</span> <span class="toctext">INTRODUCTION</span></a>
<ul>
<li class="toclevel-3"><a href="#The_Central_GMS"><span class="tocnumber">1.1.1</span> <span class="toctext">The Central GMS</span></a></li>
<li class="toclevel-3"><a href="#The_Local_GMS"><span class="tocnumber">1.1.2</span> <span class="toctext">The Local GMS</span></a></li>
<li class="toclevel-3"><a href="#Discrimination_between_central_GMS_and_local_GMS"><span class="tocnumber">1.1.3</span> <span class="toctext">Discrimination between central GMS and local GMS</span></a></li>
<li class="toclevel-3"><a href="#Allocation_of_USER_IDs"><span class="tocnumber">1.1.4</span> <span class="toctext">Allocation of USER IDs</span></a></li>
<li class="toclevel-3"><a href="#Data_Types"><span class="tocnumber">1.1.5</span> <span class="toctext">Data Types</span></a></li>
<li class="toclevel-3"><a href="#Missing_Values"><span class="tocnumber">1.1.6</span> <span class="toctext">Missing Values</span></a></li>
<li class="toclevel-3"><a href="#User_Definition_of_Constants_and_Data_Fields"><span class="tocnumber">1.1.7</span> <span class="toctext">User Definition of Constants and Data Fields</span></a></li>
</ul>
</li>
<li class="toclevel-2"><a href="#GENEALOGY_MANAGEMENT_SYSTEM"><span class="tocnumber">1.2</span> <span class="toctext">GENEALOGY MANAGEMENT SYSTEM</span></a>
<ul>
<li class="toclevel-3"><a href="#GERMPLASM_TABLE_.28GERMPLSM.29"><span class="tocnumber">1.2.1</span> <span class="toctext">GERMPLASM TABLE (GERMPLSM)</span></a></li>
<li class="toclevel-3"><a href="#OTHER_PROGENITORS_TABLE_.28PROGNTRS.29"><span class="tocnumber">1.2.2</span> <span class="toctext">OTHER_PROGENITORS TABLE (PROGNTRS)</span></a></li>
<li class="toclevel-3"><a href="#DEFINITION_OF_METHODS"><span class="tocnumber">1.2.3</span> <span class="toctext">DEFINITION OF METHODS</span></a>
<ul>
<li class="toclevel-4"><a href="#The_METHODS_TABLE_.28METHODS.29"><span class="tocnumber">1.2.3.1</span> <span class="toctext">The METHODS TABLE (METHODS)</span></a></li>
</ul>
</li>
<li class="toclevel-3"><a href="#NAME_DATA"><span class="tocnumber">1.2.4</span> <span class="toctext">NAME DATA</span></a>
<ul>
<li class="toclevel-4"><a href="#The_Names_Table_.28NAMES.29"><span class="tocnumber">1.2.4.1</span> <span class="toctext">The Names Table (NAMES)</span></a></li>
<li class="toclevel-4"><a href="#Name_Standardization"><span class="tocnumber">1.2.4.2</span> <span class="toctext">Name Standardization</span></a></li>
</ul>
</li>
<li class="toclevel-3"><a href="#GERMPLASM_ATTRIBUTES"><span class="tocnumber">1.2.5</span> <span class="toctext">GERMPLASM ATTRIBUTES</span></a>
<ul>
<li class="toclevel-4"><a href="#The_ATTRIBUTES_TABLE_.28ATRIBUTS.29"><span class="tocnumber">1.2.5.1</span> <span class="toctext">The ATTRIBUTES TABLE (ATRIBUTS)</span></a></li>
</ul>
</li>
</ul>
</li>
<li class="toclevel-2"><a href="#USER_DEFINED_CONSTANTS_AND_DATA_FIELDS"><span class="tocnumber">1.3</span> <span class="toctext">USER DEFINED CONSTANTS AND DATA FIELDS</span></a>
<ul>
<li class="toclevel-3"><a href="#The_USER_DEFINED_FIELDS_TABLE_.28UDFLDS.29"><span class="tocnumber">1.3.1</span> <span class="toctext">The USER_DEFINED_FIELDS TABLE (UDFLDS)</span></a></li>
<li class="toclevel-3"><a href="#CONSTANT_Values"><span class="tocnumber">1.3.2</span> <span class="toctext">CONSTANT Values</span></a></li>
<li class="toclevel-3"><a href="#CONSTANT_Field_Values"><span class="tocnumber">1.3.3</span> <span class="toctext">CONSTANT Field Values</span></a></li>
<li class="toclevel-3"><a href="#Name_and_Attribute_Descriptor_Formats"><span class="tocnumber">1.3.4</span> <span class="toctext">Name and Attribute Descriptor Formats</span></a>
<ul>
<li class="toclevel-4"><a href="#Example_1"><span class="tocnumber">1.3.4.1</span> <span class="toctext">Example 1</span></a></li>
<li class="toclevel-4"><a href="#Example_2"><span class="tocnumber">1.3.4.2</span> <span class="toctext">Example 2</span></a></li>
</ul>
</li>
</ul>
</li>
<li class="toclevel-2"><a href="#CORRECTING_RECORDS_IN_THE_GMS_DATABASE"><span class="tocnumber">1.4</span> <span class="toctext">CORRECTING RECORDS IN THE GMS DATABASE</span></a>
<ul>
<li class="toclevel-3"><a href="#The_CHANGES_Table"><span class="tocnumber">1.4.1</span> <span class="toctext">The CHANGES Table</span></a></li>
<li class="toclevel-3"><a href="#Change_location.2C_date_or_reference_of_germplasm"><span class="tocnumber">1.4.2</span> <span class="toctext">Change location, date or reference of germplasm</span></a></li>
<li class="toclevel-3"><a href="#Change_a_progenitor_ID_for_generative_germplasm"><span class="tocnumber">1.4.3</span> <span class="toctext">Change a progenitor ID for generative germplasm</span></a></li>
<li class="toclevel-3"><a href="#Add_a_progenitor_ID_for_a_generative_germplasm"><span class="tocnumber">1.4.4</span> <span class="toctext">Add a progenitor ID for a generative germplasm</span></a></li>
<li class="toclevel-3"><a href="#Change_the_group_ID_for_a_derivative_germplasm"><span class="tocnumber">1.4.5</span> <span class="toctext">Change the group ID for a derivative germplasm</span></a></li>
<li class="toclevel-3"><a href="#Change_a_source_ID_for_a_derivative_germplasm"><span class="tocnumber">1.4.6</span> <span class="toctext">Change a source ID for a derivative germplasm</span></a></li>
<li class="toclevel-3"><a href="#Change_a_germplasm_method"><span class="tocnumber">1.4.7</span> <span class="toctext">Change a germplasm method</span></a></li>
<li class="toclevel-3"><a href="#Replace_an_existing_germplasm_record_with_another_one"><span class="tocnumber">1.4.8</span> <span class="toctext">Replace an existing germplasm record with another one</span></a></li>
<li class="toclevel-3"><a href="#Delete_a_Germplasm_Record"><span class="tocnumber">1.4.9</span> <span class="toctext">Delete a Germplasm Record</span></a></li>
</ul>
</li>
</ul>
</li>
</ul>
</td></tr></table><script type="text/javascript"> if (window.showTocToggle) { var tocShowText = "show"; var tocHideText = "hide"; showTocToggle(); } </script>
<a name="PHYSICAL_STRUCTURE_OF_THE_ICIS_GMS_DATABASE" id="PHYSICAL_STRUCTURE_OF_THE_ICIS_GMS_DATABASE"></a><h1> <span class="mw-headline"> PHYSICAL STRUCTURE OF THE ICIS GMS DATABASE </span></h1>
<a name="INTRODUCTION" id="INTRODUCTION"></a><h3> <span class="mw-headline"> INTRODUCTION </span></h3>
<p>The ICIS GMS database stores and manages information on genesis, genealogy, nomenclature and chronology of germplasm for a particular crop.  It operates on the principle of unique identification of germplasm through system assigned Germplasm IDs (GIDs).  It also manages the ambiguities of synonyms and homonyms through name searching facilities, and it allows remote allocation of unique GIDs by managing a user list and operating through a twin database system comprising a copy of the Central GMS and a Local GMS with identical structure.
Any DBMS program, which supports Open Database Connectivity (ODBC), can be used to run GMS, and each implementation of local GMS can use a different ODBC compliant DBMS.  The DBMS for the central GMS can be chosen independently for each crop, and read-only distribution copies for access by local installations need not use the same DBMS as the actual central GMS.
All user access to the databases is through a suite of access routines provided in the form of a Windows 95 32-bit Dynamic Link Library (DLL).  Basic applications for querying, updating and reporting will be written in the most convenient language for the application, and users will be able to write their own applications by calling the DLL.  Any such programs will operate identically on any GMS installation of any ICIS implementation.
</p>
<a name="The_Central_GMS" id="The_Central_GMS"></a><h4> <span class="mw-headline"> The Central GMS </span></h4>
<p>The ICIS administrator for a particular crop sets up the central GMS, it comprises sixteen tables: four data tables indexed by GIDs, three look-up tables to manage references, methods constant codes and user defined fields, three tables to manage users, installations and structural changes, and six to manage locations.  Only the administrator has write access to the central GMS, users can read it, but all user updates are recorded on an installation of local GMS and must be passed through the administrator to be entered on the central GMS and hence be accessible to other users.
</p>
<a name="The_Local_GMS" id="The_Local_GMS"></a><h4> <span class="mw-headline"> The Local GMS </span></h4>
<p>Each installation of GMS has a number of local users with read-only access to central GMS, or to a copy of it, and write access to a local GMS.  The local administrator has complete write access to every table of local GMS and other users have different levels of access assigned by the local administrator (3.8.3).  The local GMS has the same structure as central GMS except that three implementation specific tables are not included and list management tables are added (3.10).
</p>
<a name="Discrimination_between_central_GMS_and_local_GMS" id="Discrimination_between_central_GMS_and_local_GMS"></a><h4> <span class="mw-headline"> Discrimination between central GMS and local GMS </span></h4>
<p>The GMS user will be completely unaware that there are two databases operating.  However the internal system must be able to distinguish between the two, and this is achieved by the local GMS always using negative key fields, and the central GMS always using positive key fields.  Links to these key fields from any tables in local GMS can be positive or negative depending on whether they refer to a record in the central GMS or the local GMS.  Links in the central GMS are always positive since the central GMS is 'unaware' of records in individual installations until they have been updated.
</p>
<a name="Allocation_of_USER_IDs" id="Allocation_of_USER_IDs"></a><h4> <span class="mw-headline"> Allocation of USER IDs </span></h4>
<p>The ICIS central administrator will allocate a set of unique USER IDs for each installation.  This ensures the uniqueness of remotely assigned Germplasm IDs.  Each installation of GMS will have a local administrator who can assign the USER IDs to specific users by specifying information in the USERS table (3.8.2).  Once filled, the central administrator can only change these fields.  All users of a local installation will have read access to all records of the local GMS.
</p>
<a name="Data_Types" id="Data_Types"></a><h4> <span class="mw-headline"> Data Types </span></h4>
<p>The only data types used in GMS are long and short signed integers and character strings except for latitude (LAT), longitude (LON) and altitude (ALT) in the GEOREF table (3.7.6) which are eight byte or 4 byte (ALT) floating point real fields. Dates are stored as long integer (4 bytes) as follows: YYYYMMDD.  If the day or month and day are unknown then DD or MM  are set to 0.  If day is known neither month nor year should be missing.  Also if month is known then year should not be missing.  User defined data structures always store ASCII text but users may specify a format to convert these strings to data structures with elements of integer, or text types (3.6.3).  Specialist application programs can then use this data as required.  Time can be stored in user defined data structures and should be stored as HHMMSS in twenty-four hour format.
</p>
<a name="Missing_Values" id="Missing_Values"></a><h4> <span class="mw-headline"> Missing Values </span></h4>
<p>Valid values must be provided for all fields unless the admissibility of null values is specifically described in the SCHEMA.  In some cases, valid data for fields includes values like 'UNKNOWN', which are not missing. Missing values referred to in the SCHEMA depending on the data type of the field.  Missing values for integer fields, when permissible, are zero.  Missing values for text fields are null strings and missing values for float fields are -1.0E36.
</p>
<a name="User_Definition_of_Constants_and_Data_Fields" id="User_Definition_of_Constants_and_Data_Fields"></a><h4> <span class="mw-headline"> User Definition of Constants and Data Fields </span></h4>
<p>The database structure allows users to define new methods, data fields and add data types at any time through a system of field definitions.  This means that new methodology in germplasm development can be accommodated, and new types of information can be stored when required.
</p>
<a name="GENEALOGY_MANAGEMENT_SYSTEM" id="GENEALOGY_MANAGEMENT_SYSTEM"></a><h3> <span class="mw-headline"> GENEALOGY MANAGEMENT SYSTEM </span></h3>
<p><a href="../../../../articles/i/r/i/File%7EIRIS_2005_Figure1.JPG_be59.html" class="image" title="Image:IRIS_2005_Figure1.JPG"><img alt="Image:IRIS_2005_Figure1.JPG" src="../../../../images/6/65/IRIS_2005_Figure1.JPG" width="768" height="768" border="0" /></a>
</p>
<a name="GERMPLASM_TABLE_.28GERMPLSM.29" id="GERMPLASM_TABLE_.28GERMPLSM.29"></a><h4> <span class="mw-headline"> GERMPLASM TABLE (GERMPLSM) </span></h4>
<p>Main details of germplasm genesis, origin and preferred name
</p>
<table border="1" cellpadding="2">
<tr>
<th width="30%"> Columns - Long Name (Name)
</th><th width="40%"> Description
</th><th width="15%"> Type
</th><th width="15%"> Size (bytes)
</th></tr>
<tr>
<td> GERMPLSM_ID (GID)
</td><td> Unique germplasm identifier.  GID is automatically generated by the system each time a new record is added. Links to GID in PROGNTRS, NAMES and ATRIBUTS tables
</td><td>Number (Long)
</td><td>4
</td></tr>
<tr>
<td> NO_PROGENITORS (GNPGS)
</td><td> Defines type of genesis and number of progenitors.
<p>For a derivative process GNPGS =-1 and then GPID1 contains the germplasm group ID and GPID2 the source germplasm ID. For a generative process GNPGS contains the number of specified parents. (The number of parents required by a method is recorded by NPRGN on the METHODS TABLE). If GNPGS=1 or 2 then the IDs of the progenitors are contained in the GPID1 and GPID2 fields on the GERMPLSM table. If GNPGS&gt;2 then further IDs are stored on the PROGNTRS table. GNPGS = 0 for land race or wild species collections or if none of the parents is known. GNPGS &lt;= NPRGN, but some of the GNPGS specified parents may be unknown in which case the corresponding GPIDs are MISSING (0). For example in a simple cross with only male parent known, GNPGS would have to be 2 with GPID1=0 and GPD2 set to GID of the known male parent.
</p>
</td><td>Number (Integer)
</td><td>2
</td></tr>
<tr>
<td> METHOD_NO (METHN)
</td><td> Number that identifies the method of genesis for the germplasm.  Details of the method are on the METHODS table and specific parameters may be stored on the ATTRIBUTES table (see MATTR on the METHODS table). Links to MID in METHODS table.
</td><td> Number (Integer).
</td><td>2
</td></tr>
<tr>
<td> PROGENITOR_ID1(GPID1)
</td><td> If GNPGS is -1 then GPID1 is the group ID, which is the GID of last source germplasm, produced by a generative process or the GID of the last source of unknown origin in the development path of the current germplasm.  This ID defines a group of germplasm in which all members are derived from the same generic parent. If GNPGS &gt;0 then GPID1 points to the first progenitor in a generative process (usually the female parent). GPID1=0 if unknown.
</td><td> Number (Long)
</td><td>4
</td></tr>
<tr>
<td> PROGENITOR_ID2 (GPID2)
</td><td> If GNPGS is -1 then GPID2 points to the immediate source from which the current germplasm is derived otherwise it points to the  second progenitor of the germplasm in a generative process (usually the male parent). GPID2=0 if unknown.
</td><td> Number (Long)
</td><td>4
</td></tr>
<tr>
<td> MANAGEMENT_GID (MGID)
</td><td> If the current germplasm is a managed sample then MGID contains the GID of the germplasm at the root of the management tree, else 0.
</td><td> Number (Long)
</td><td>4
</td></tr>
<tr>
<td> GERMPLASM_USER_ID (GERMUID)
</td><td> ID of the user entering details of the current germplasm. UID in the USERS table
</td><td> Number (Integer)
</td><td>2
</td></tr>
<tr>
<td> LOCAL_GERMPLASM_ID (LGID)
</td><td> LGID contains the original, negative GID used in the Local GMS prior to transfer of data records to the central GMS. LGID = 0 if the record was not entered through an update process
</td><td> Number (Long)
</td><td>
</td></tr>
<tr>
<td> LOCATION_NO (GLOCN)
</td><td> Identifier of the location where the germplasm was created as a distinct unit of management with a new GID. LOCN in LOCATION table. GLOCN = MISSING or 0 if location is unknown
<ul><li> For germplasm that acquires a GID because it is a newly-harvested seed sample (for example, as part of a breeding programme, or genebank multiplication programme, or harvested direct from a farmer’s field), this is the location of the plot where the seed were harvested.
</li><li> For a sample of germplasm that is newly collected from a farm store, market place etc, this is the location from which the sample was collected. 
</li><li> For germplasm that acquires a GID because it is transferred from one organization or organizational unit to another (for example, germplasm transfer between genebanks, or from farm or market place to genebank, so that it enters a new management regime and becomes the founding sample of a new management neighbourhood), this is the location of the receiving organizational unit.
<ul><li> All GIDs created on transfer of germplasm to a new management regime should also have a GID representing the "parental" germplasm sample held by the donating organization. They should have method type import, and GPID2 pointing to the donor's sample.
</li></ul>
</li></ul>
<p>N.B. GIDs derived only by maintenance methods from a sample collected from a farm, market place etc, should have their GPID1 point to the GID of the original collected sample, and GLOCN of the collected sample is the collecting location.
</p>
</td><td> Number (Long)
</td><td>4
</td></tr>
<tr>
<td> GERMPLASM_DATE (GDATE)
</td><td> Date of germplasm genesis, i.e. the date on which the germplasm was created as a distinct unit of management with a new GID. (YYYYMMDD). GDATE = 0 if unknown.
<ul><li>	For germplasm that acquires a GID because it is a newly-harvested batch of seed (for example, as part of a breeding programme, or genebank multiplication programme, or harvested direct from a farmer’s field), this is the date the seed were harvested. 
</li></ul>
<ul><li>	For a sample of germplasm that is newly collected from a farm store, market place etc, this is the date on which the sample was collected. 
</li></ul>
<ul><li>	For germplasm that acquires a GID because it is transferred from one organizational unit to another (for example, germplasm transfer between genebanks, or from farm or market place to genebank), this is the date the receiving organization acquired the germplasm and so started managing the germplasm.
</li></ul>
<ul><li>	N.B. GIDs derived only by maintenance methods from a sample collected from a farm, market place etc, should also have an attribute COLLDATE which is the collection date. GPID1 should point to the GID of the original collected sample, and COLLDATE of the accession should equal the GDATE of the collected sample.
</li></ul>
</td><td> Number (Long)
</td><td>4
</td></tr>
<tr>
<td> GERMPLASM_REFERENCE (GREF)
</td><td> A number that identifies a bibliographic reference from where the germplasm data was retrieved. GREF is missing if the source is unpublished or unknown
</td><td> Number (Long)
</td><td>4
</td></tr>
<tr>
<td> GERMPLASM_REPLACE (GRPLCE)
</td><td> Records deletion or replacement for the current GERMPLASM record. 0 for  unchanged,  own GID for deleted, and replacement GID for replaced
</td><td> Number (Integer)
</td><td>4
</td></tr></table>
<a name="OTHER_PROGENITORS_TABLE_.28PROGNTRS.29" id="OTHER_PROGENITORS_TABLE_.28PROGNTRS.29"></a><h4> <span class="mw-headline"> OTHER_PROGENITORS TABLE (PROGNTRS) </span></h4>
<p>Stores IDs of progenitors for generative methods that require more than two parents
</p>
<table border="1" cellpadding="2">
<tr>
<th width="30%"> Columns - Short Name (Name)
</th><th width="40%"> Description
</th><th width="15%"> Type
</th><th width="15%"> Size (bytes)
</th></tr>
<tr>
<td> GERMPLASM_ID (GID)
</td><td> Identifier for the germplasm.  GID in the GERMPLSM table
</td><td> Number (Long)
</td><td>4
</td></tr>
<tr>
<td> PROGENITOR_NO (PNO)
</td><td> Progenitor number 3, 4, ...
</td><td> Number (Integer)
</td><td>2
</td></tr>
<tr>
<td> PROGENITOR_ID (PID)
</td><td> Germplasm identifier for progenitor number PNO.  GID in the GERMPLSM table. PID = 0 if the progenitor is unknown
</td><td> Number (Long)
</td><td>4
</td></tr></table>
<a name="DEFINITION_OF_METHODS" id="DEFINITION_OF_METHODS"></a><h4> <span class="mw-headline"> DEFINITION OF METHODS </span></h4>
<p>Method definitions are stored in the METHODS table, and complete documentation, including bibliographic references, is expected.  One anticipated use of the methods table is to provide information to applications which calculate the degree of relationship between germplasm, or which trace gene flows through germplasm development. 
Some methods depend on parameters, which may vary each time the method is used, such as the number and mixing proportions of parents in the generation of a population.  These parameters can be defined by assigning an attribute to the method with the METHOD_ATTRIBUTE field. Values for each instance of the associated method are recorded on an attribute record linked to the GID of the germplasm being produced.  Attributes are flexible user definable data fields (3.5).
</p>
<a name="The_METHODS_TABLE_.28METHODS.29" id="The_METHODS_TABLE_.28METHODS.29"></a><h5> <span class="mw-headline"> The METHODS TABLE (METHODS) </span></h5>
<p>Details of methods of germplasm genesis
</p>
<table border="1" cellpadding="2">
<tr>
<th width="30%"> Columns - Short Name (Name)
</th><th width="40%"> Description
</th><th width="15%"> Type
</th><th width="15%"> Size (bytes)
</th></tr>
<tr>
<td> METHOD_ID (MID)
</td><td> Identifier for the method of germplasm development. METHN on the GERMPLSM table.
</td><td> Number (Integer)
</td><td>2
</td></tr>
<tr>
<td> METHOD_TYPE (MTYPE)
</td><td> Method type: GEN for generative, DER for derivative, MAN for maintenance methods
</td><td> Text
</td><td>3
</td></tr>
<tr>
<td> METHOD_GROUP (MGRP)
</td><td> Defines the breeding system to which the method applies: S – self fertilizing,   O – cross pollinating, C clonally propagating and G – all systems
</td><td> Text
</td><td>3
</td></tr>
<tr>
<td> METHOD_CODE (MCODE)
</td><td> Mnemonic description of method
</td><td> Text
</td><td>8
</td></tr>
<tr>
<td> METHOD_NAME (MNAME)
</td><td> Name of method
</td><td> Text
</td><td>50
</td></tr>
<tr>
<td> METHOD_REFERENCE (MREF)
</td><td> Number of a reference to the method. Links to the Reference number in the Bibliography Table. May be missing
</td><td> Number (Long)
</td><td>4
</td></tr>
<tr>
<td> METHOD_DESCRIPTION (MDESC)
</td><td> Description of the method.
</td><td> Text
</td><td>255
</td></tr>
<tr>
<td> NO_PROGENITORS (MPRGN)
</td><td> MPRGN =-1 if the method is derivative or management. Otherwise MPRGN is the number of progenitors required for the generative process. MISSING (0) if the number of progenitors is variable
</td><td> Number (Integer)
</td><td>2
</td></tr>
<tr>
<td> NO_FEMALE_PARENTS (MFPRG)
</td><td> Number of female parents for a generative process, these are specified first in the list of parents. MPRGN = MISSING if the method is derivative or the number of female parents is variable (if variable it can be recorded as an attribute for each instance, see MATTR).
</td><td> Number (Integer)
</td><td>2
</td></tr>
<tr>
<td> METHOD_ATTRIBUTE (MATTR)
</td><td> Identification number of an attribute associated with this method.  The attribute is defined in the UDFLDS table. FLDNO in UDFLDS table and ATYPE in ATRIBUTS table.  MISSING if no attribute is associated with the current method.
</td><td> Number (Integer)
</td><td>2
</td></tr>
<tr>
<td> GENETIC_EQUIVALENCE (GENEQ)
</td><td> METHN of a basic method which has equivalent genetic relationship between progenitors and offspring for the purpose of computing coefficients of parentage
</td><td> Number(Integer)
</td><td>2
</td></tr>
<tr>
<td> METHOD_USER (MUID)
</td><td> ID of the user defining the method
</td><td> Number (Integer)
</td><td>2
</td></tr>
<tr>
<td> LOCAL_METHOD_ID (LMID)
</td><td> Local method number from the installation where it was defined
</td><td> Number (Integer)
</td><td>2
</td></tr>
<tr>
<td> METHOD_DATE (MDATE)
</td><td> Date of method definition
</td><td> Number (Long)
</td><td>4
</td></tr></table>
<a name="NAME_DATA" id="NAME_DATA"></a><h4> <span class="mw-headline"> NAME DATA </span></h4>
<p>Germplasm collects a multitude of labels during the development and release process.  These are all tracked through the NAMES table of GMS.  Names are classified by name types which are defined in the USER_DEFINED_FIELDS table (3.6).  An abbreviation may be associated with each name type, they have a maximum length of eight characters and are stored in the NAME field of the NAMES table like any other name.  They are defined in the USER_DEFINED_FIELDS table with USER_FIELD_TYPE = ABBREVIATION but with the same name type as the name Abbreviations inherit the NAME_TYPE, NAME_USER, NAME_LOCATION and NAME_REFERENCE values from the name being abbreviated.  This allows abbreviations to be matched to names.  Unique matching requires the restriction that different names cannot have all these fields identical.  Abbreviations take on their own NAME_DATE so that multiple abbreviations of the same name can be distinguished. 
One name must be defined as the preferred name, one abbreviation may also be declared as preferred (this need not be an abbreviation of the preferred name). Names may contain imbedded information, and this can be made accessible to application programs for specific name types by specifying a format for the imbedded information in the USER_DEFINED_FIELDS table (3.6.3).
All names for local germplasm must appear in the local names table, but some names for germplasm in the central database can appear in the local names table (with positive GID values). One name in the central must be preferred, but central germplasm may also have one name set as preferred in the local names table. This name takes precedence over the preferred name in the central database whenever the particular local is being used. This allows local users to change preferred names or specify local preferred names.
</p>
<a name="The_Names_Table_.28NAMES.29" id="The_Names_Table_.28NAMES.29"></a><h5> <span class="mw-headline"> The Names Table (NAMES) </span></h5>
<p>Stores all germplasm names, abbreviations and naming details.
</p>
<table border="1" cellpadding="2">
<tr>
<th width="30%"> Columns - Short Name (Name)
</th><th width="40%"> Description
</th><th width="15%"> Type
</th><th width="15%"> Size (bytes)
</th></tr>
<tr>
<td> NAME_ID (NID)
</td><td> Unique identifier for the name
</td><td> Number (Long)
</td><td>4
</td></tr>
<tr>
<td> GERMPLASM_ID (GID)
</td><td> Identifier for the germplasm associated with the name. GID in GERMPLSM table
</td><td> Number (Long)
</td><td>4
</td></tr>
<tr>
<td> NAME_TYPE (NTYPE)
</td><td> Number that identifies the type of name or abbreviation. FLDNO in UDFLDS table
</td><td> Number (Integer)
</td><td>2
</td></tr>
<tr>
<td> NAME_STATUS(NSTAT)
</td><td> Number indicating the storage type and status of the name: 1 – Preferred name (must be ASCII), 8 - Preferred ID, 2 - Preferred abbreviations (must be ASCII), 3 - Chinese-GBK (GD) DBCS names,  4 - Chinese Big 5, 5 - Japanese  6 – Korean, 10 - UNICODE names which are not preferred,  9 - the name is marked as deleted
</td><td> Number (Integer)
</td><td>2
</td></tr>
<tr>
<td> NAME_USER_ID (NUID)
</td><td> ID of the user naming the germplasm. UID in the USERS table
</td><td> Number (Integer)
</td><td>2
</td></tr>
<tr>
<td> NAME_VALUE (NVAL)
</td><td> Germplasm name.
<p>For ASCII names each character occupies one byte, but for UNICODE names (eg Chinese), each character occupies 2 bytes
</p>
</td><td> Text
</td><td>255
</td></tr>
<tr>
<td> NAME_LOCATION_NO (NLOCN)
</td><td> Identifier for the location where the name was first assigned to the maintenance neighbourhood of which this GID is a member. LOCN in LOCATION table. NLOCN = 0 if the location is unknown.
<ul><li> If the name was newly created for this GID, then NLOCN=GLOCN
</li><li> If the name was inherited from the source GID (implying a management method of germplasm creation), then NLOCN is inherited from the same name of the source GID. For example, suppose IRRI’s genebank holds an accession donated to it by USDA, identified in USDA with a PI accession number created by USDA. IRIS will contain two GIDs, one (GID-a) to represent the original accession held by USDA, and one (GID-b) to represent the copy held by IRRI, and GPID2 of GID-b will point to GID-a. IRIS will also hold two instances of the corresponding PI number: one as name type ACCNO associated with GID-a, and one as name type FACCN associated with GID-b. Both instances of the PI number must have the same NLOCN, and it must point to USDA.
</li></ul>
</td><td> Number (Long)
</td><td>4
</td></tr>
<tr>
<td> NAME_DATE (NDATE)
</td><td> Date on which the name was first assigned to the maintenance neighbourhood of which this GID is a member. (YYYYMMDD). NDATE = 0 if the date is unknown
<ul><li> If the name was newly created for this GID, then NDATE=GDATE
</li><li> If the name was inherited from the source GID (implying a management method of germplasm creation), then NDATE is inherited from the same name of the source GID. For example, suppose IRRI’s genebank holds an accession donated to it by USDA, identified in USDA with a PI accession number created by USDA. IRIS will hold two instances of the corresponding PI number (see NLOCN). Both instances of the PI number must have the same NDATE, and that must be the date that USDA assigned that PI number to its accession
</li></ul>
</td><td> Number (Long)
</td><td>4
</td></tr>
<tr>
<td> NAME_REFERENCE(NREF)
</td><td> A number that identifies a bibliographic reference for the name. NREF is missing if the source is unpublished or unknown
</td><td> Number (Long)
</td><td>4
</td></tr></table>
<a name="Name_Standardization" id="Name_Standardization"></a><h5> <span class="mw-headline"> Name Standardization </span></h5>
<p>A major problem with identifying germplasm is the detection of minor variants of a name.  For example some users hyphenate foreign language names while others use spaces or capital letters within a name string of small case letters, some separate a character prefix from a number by a space, others do not.  Another problem is that different ODBC drivers implement the same SQL search differently with respect to case sensitivity. For these reasons, a set of name standardization has been devised with the objective of producing the same standardized name from as wide a range of variants as possible. These rules are applied in order as follows:
</p>
<pre>(L= any letter; ^= space; N= any numeral, S= any of {-,',[,],+,.})
a)	Capitalize all letters			Khao-Dawk-Mali105 becomes 
                                               KHAO-DAWK-MALI105
b)	L( becomes L^( and )L becomes )^L 	IR64(BPH) becomes IR64 (BPH)
c)	N( becomes N^( and )N becomes )^N 	IR64(5A) becomes IR64 (5A)
d)	L. becomes L^			 	IR 63 SEL. becomes IR 64 SEL
e)	LN becomes L^N EXCEPT SLN	 	MALI105 becomes MALI 105
                                               but MALI-F4 IS unchanged
f)	NL becomes N^L EXCEPT SNL		B 533A-1 becomes B 533 A-1
                                               but B 533 A-4B is unchanged
g)	LL-LL becomes LL^LL			KHAO-DAWK-MALI 105 becomes 
                                               KHAO DAWK MALI 105
h)	^0N becomes ^N 				IRTP 00123 becomes IRTP 123
i)	^^ becomes ^
j)	REMOVE LEADING OR TRAILING ^
k)	^) becomes ) and (^ becomes (
l)	L-N becomes L^N when there is only one ‘–‘ in the name and L is not preceded by a space
m)	^/ becomes / and /^ becomes /
</pre>
<p>When a name string is supplied to the ICIS DLL for searching, it is searched as supplied and after standardization. Routines, which write names to the database, do not apply the standardization and users must specifically call a standardization routine if they wish to standardize names before storing them. 
Names in the database are not required to be standardized.  It is hoped that the resulting standardized name is acceptable for presentation, but there will be cases where users insist on having a name, which violates the rules.  For example CIMMYT prefers that abbreviations contain no spaces.  This is allowed but the rule violation should not be used to distinguish genotypes, and the standardized name should be given as a synonym.  Users will always be able to search for names that violate the rules and the correct germplasm should be found.
</p>
<a name="GERMPLASM_ATTRIBUTES" id="GERMPLASM_ATTRIBUTES"></a><h4> <span class="mw-headline"> GERMPLASM ATTRIBUTES </span></h4>
<p>Attributes are text variables used to store information about the genesis, genealogy, nomenclature or chronology of germplasm. Attribute types are defined and described on the USER_DEFINED_FIELDS table (3.6.1).  Like names, attributes may contain imbedded information in the form of sub-fields or variables within the attribute text.  The structure and format of such attributes is defined through the USER_FIELD_FMT of the USER_DEFINED_FIELDS table, as is the format of structured names (3.6.3).
</p>
<a name="The_ATTRIBUTES_TABLE_.28ATRIBUTS.29" id="The_ATTRIBUTES_TABLE_.28ATRIBUTS.29"></a><h5> <span class="mw-headline"> The ATTRIBUTES TABLE (ATRIBUTS) </span></h5>
<p>Table of germplasm attributes used to store information about germplasm genesis, genealogy and nomenclature, which is not stored in other tables.
</p>
<table border="1" cellpadding="2">
<tr>
<th width="30%"> Columns - Short Name (Name)
</th><th width="40%"> Description
</th><th width="15%"> Type
</th><th width="15%"> Size (bytes)
</th></tr>
<tr>
<td> ATTRIBUTE_ID (AID)
</td><td> ID of the Attribute (unique)
</td><td> Number (Long)
</td><td>4
</td></tr>
<tr>
<td> GERMPLASM _ID (GID)
</td><td> ID of germplasm with which the attribute is associated. GID in GERMPLASM table.
</td><td> Number (Long)
</td><td>4
</td></tr>
<tr>
<td> ATTRIBUTE_TYPE (ATYPE)
</td><td> Number that identifies the attribute. FLDNO in UDFLDS table. Use 999 to mark the attribute as deleted.
</td><td> Number (Integer).
</td><td>2
</td></tr>
<tr>
<td> ATTRIBUTE_USER_ID (AUID)
</td><td> ID of the user giving the attribute value. UID in USERS table
</td><td> Number (Integer)
</td><td>2
</td></tr>
<tr>
<td> ATTRIBUTE _VALUE (AVAL)
</td><td> Contains the attribute value.
</td><td> Text
</td><td>255
</td></tr>
<tr>
<td> ATTRIBUTE_LOCATION_NO (ALOCN)
</td><td> Identifier for the location where the attribute value was set. LOCN on the LOCATION table. ALOCN = MISSING if the location is unknown.
</td><td> Number (Long)
</td><td>4
</td></tr>
<tr>
<td> ATTRIBUTE_REFERENCE(AREF)
</td><td> Key to the reference for the attribute data. MISSING if the source is unknown
</td><td> Number (Long)
</td><td>4
</td></tr>
<tr>
<td> ATTRIBUTE_DATE (ADATE)
</td><td> Date of value assignation. (YYYYMMDD)
</td><td> Number (Long)
</td><td>4
</td></tr></table>
<a name="USER_DEFINED_CONSTANTS_AND_DATA_FIELDS" id="USER_DEFINED_CONSTANTS_AND_DATA_FIELDS"></a><h3> <span class="mw-headline"> USER DEFINED CONSTANTS AND DATA FIELDS </span></h3>
<p>The ICIS structure allows considerable flexibility in the type and scope of information that can be stored. It does this by storing structural information (metadata) in the USER DEFINED FIELDS TABLE (UDFLDS).  Some of this structural information is standard across implementations and is pre-loaded, some is specific to particular crops and is loaded by the central database administrator, and some is specific to local installations and can be loaded or modified by individual users.
Types of metadata range from the definition of constants, where only the range or scope of allowable values is defined, to definition of fields where the type, scope and structure are defined. Allowable user or reference types are examples of constants and attribute or location descriptors are fields since they include both description and structure. In some cases both types are possible. For example name types may also have a format describing the structure of information imbedded in a name of a particular type.
</p>
<a name="The_USER_DEFINED_FIELDS_TABLE_.28UDFLDS.29" id="The_USER_DEFINED_FIELDS_TABLE_.28UDFLDS.29"></a><h4> <span class="mw-headline"> The USER_DEFINED_FIELDS TABLE (UDFLDS) </span></h4>
<p>Table of definitions of user defined fields and constants
</p>
<table border="1" cellpadding="2">
<tr>
<th width="30%"> Columns - Short Name (Name)
</th><th width="40%"> Description
</th><th width="15%"> Type
</th><th width="15%"> Size (bytes)
</th></tr>
<tr>
<td> USER_FIELD_NO (FLDNO)
</td><td> Unique identifier for the user defined field. NTYPE in NAMES table, ATYPE in the ATRIBUTS table and MATTR in the METHODS table, DTYPE in the LOCDES table and to the table and field specified in FTABLE and FTYPE for constant definitions
</td><td> Number (Integer)
</td><td>2
</td></tr>
<tr>
<td> USER_FIELD_TABLE (FTABLE)
</td><td> Short Name of the ICIS data table to contain the user defined field:
<p>NAMES, ATRIBUTS or LOCDES or the table containing the constant link
</p>
</td><td> Text
</td><td>24
</td></tr>
<tr>
<td> USER_FIELD_TYPE (FTYPE)
</td><td> Type of defined field: NAME, ATTRIBUTE, ABBREVIATION, or DESCRIPTOR  or field name for definition of constants
</td><td> Text
</td><td>12
</td></tr>
<tr>
<td> USER_FIELD_CODE (FCODE)
</td><td> Short name for the user defined field or a permissible values for a constant
</td><td> Text
</td><td>50
</td></tr>
<tr>
<td> USER_FIELD_NAME (FNAME)
</td><td> Long name for the user defined field  or descriptive value for a constant
</td><td> Text
</td><td>50
</td></tr>
<tr>
<td> USER_FIELD_FMT (FFMT)
</td><td> Definition of sub-fields or variables contained in the user defined field. These variables can have multiple elements, and may contain text, integer or real valued data.  (See section 3.6.3). If FFMT is blank then the user-defined field contains a single, unnamed text variable with variable length such as a name.
</td><td> Text
</td><td>255
</td></tr>
<tr>
<td> USER_FIELD_DESCRIPTION (FDESC)
</td><td> Full description of the user defined field, which must describe all sub-fields specified in FFMT and may contain a bibliographic reference.
</td><td> Text
</td><td>255
</td></tr>
<tr>
<td> LOCAL_FIELD_NO(LFLDNO)
</td><td> Local field number
</td><td> Integer
</td><td>2
</td></tr>
<tr>
<td> FIELD_USER_ID(FUID)
</td><td> ID of the user defining the field
</td><td> Integer
</td><td>2
</td></tr>
<tr>
<td> FIELD_DATE(FDATE)
</td><td> Date of field definition
</td><td> Integer
</td><td>4
</td></tr>
<tr>
<td> SCALE_ID(SCALEID)
</td><td> Identification number of its scale. The valid values can be stored in the SCALEDIS or SCALECON table
</td><td> Integer
</td><td>4
</td></tr></table>
<a name="CONSTANT_Values" id="CONSTANT_Values"></a><h4> <span class="mw-headline"> CONSTANT Values </span></h4>
<p>Many fields in ICIS tables are restricted to a discrete set of possible values.  The values actually stored in the tables may be field-Ids (FLDNO) or field-codes (FCODE) from the USER_DEFINED_FIELDS table.  The full set of possible values for such a field is indexed by FTABLE, indicating the table where they are stored, FTYPE, the name of the field in the table, and FCODE, the actual values of the constant.  The FNAME field contains a longer description of the constant value. Users can extend the range of possible values allowed for a constant by adding records to the local USER_DEFINED FIELDS table.  An example is the set of location types that are recorded in the TYPE field of the LOCATION table.  The associated constants in the UDFLDS table are:
</p>
<a name="CONSTANT_Field_Values" id="CONSTANT_Field_Values"></a><h4> <span class="mw-headline"> CONSTANT Field Values </span></h4>
<table border="1" cellpadding="2">
<tr>
<th width="15%"> Columns - FLDNO
</th><th width="15%"> FTABLE
</th><th width="15%"> FTYPE
</th><th width="15%"> FCODE
</th><th width="40%"> FNAME
</th></tr>
<tr>
<td>401
</td><td> LOCATION
</td><td> LTYPE
</td><td>1
</td><td> CONTINENT
</td></tr>
<tr>
<td>402
</td><td> LOCATION
</td><td> LTYPE
</td><td>2
</td><td> GEOGRAPHICAL REGION
</td></tr>
<tr>
<td>403
</td><td> LOCATION
</td><td> LTYPE
</td><td>3
</td><td> GEOPOLITICAL REGION
</td></tr>
<tr>
<td>404
</td><td> LOCATION
</td><td> LTYPE
</td><td>4
</td><td> ECOLOGICAL REGION
</td></tr>
<tr>
<td>405
</td><td> LOCATION
</td><td> LTYPE
</td><td>5
</td><td> COUNTRY
</td></tr>
<tr>
<td>406
</td><td> LOCATION
</td><td> LTYPE
</td><td>6
</td><td> FIRST SUB-NATIONAL DIVISION
</td></tr>
<tr>
<td>407
</td><td> LOCATION
</td><td> LTYPE
</td><td>7
</td><td> SECOND SUB-NATIONAL DIVISION
</td></tr>
<tr>
<td>408
</td><td> LOCATION
</td><td> LTYPE
</td><td>8
</td><td> THIRD SUB-NATIONAL DIVISION
</td></tr>
<tr>
<td>409
</td><td> LOCATION
</td><td> LTYPE
</td><td>9
</td><td> GERMPLASM COLLECTION SITE
</td></tr>
<tr>
<td>410
</td><td> LOCATION
</td><td> LTYPE
</td><td>10
</td><td> BREEDING LOCATION
</td></tr>
<tr>
<td>1401
</td><td> LOCATION
</td><td> LTYPE
</td><td> INGER
</td><td> INGER TEST SITE
</td></tr></table>
<p>The first ten constant values are standard across implementations, the last is specific to IRIS and hence its ID is greater that 1000.  Further constant values can be added to local UDFLDS tables with negative Ids.
</p>
<a name="Name_and_Attribute_Descriptor_Formats" id="Name_and_Attribute_Descriptor_Formats"></a><h4> <span class="mw-headline"> Name and Attribute Descriptor Formats </span></h4>
<p>Sub-fields or variables contained within user defined names or attributes are specified with the USER_FIELD_FMT (FFMT) field of the USER_DEFINED_FIELDS table. Variables may consist of different data types, and have multiple elements as in arrays. The values of all variables in names or attributes are stored in ASCII representation in the order they are defined. The format specifies how the variables may be extracted from the text string.
Each user-defined variable is described in consecutive statements of FFMT in the order that the values will be stored.  Each statement consists of four items separated by commas.  Semicolons separate statements.  Each variable definition appears as follows: <b>NAME</b>, <b>N</b>, <b>T</b>, <b>COUNT</b>;
where:
</p>
<dl><dd><b>NAME</b> contains a name for the variable,
</dd><dd><b>N</b> defines the text length of each element of the variable,
</dd><dd><b>T</b> defines the data type and
</dd><dd><b>COUNT</b> defines the number of elements in the variable.
</dd></dl>
<ul><li> NAME is an alphanumeric value with first character alphabetic.  It can be blank or absent to indicate unnamed space filler.
</li></ul>
<ul><li> N gives the number of characters occupied by each element of the variable if it is an integer value.  N can also consist of a character contained in double quotes defining the delimiter between elements.  If the last variable is a single element text variable, then N may be zero, blank or absent to indicate variable length for that variable.
</li></ul>
<ul><li> T can have value A for ASCII text data, I for integer data and R for real valued data.  IF T is blank or absent A is assumed.
</li></ul>
<ul><li> COUNT may be a positive integer defining the number of elements or it may be the name of a previously defined, single-element, integer variable whose value will define the number of elements. If COUNT is blank then the variable is assumed to have a single element.
</li></ul>
<a name="Example_1" id="Example_1"></a><h5> <span class="mw-headline"> Example 1 </span></h5>
<p>A selection method may record the number and the names of the stresses under which selection was practised, and a breeder’s comment regarding the selection.
</p><p>The format record would look as follows:
</p>
<dl><dd>NSTRESS,4,I,1;STRESS,12,A,NSTRESS; COMMENT,0,A,1
</dd></dl>
<p>or
</p>
<dl><dd>NSTRESS,4,I;STRESS,12,A,NSTRESS; COMMENT
</dd></dl>
<p>or
</p>
<dl><dd>NSTRESS,4,I;STRESS,”,”,A,NSTRESS;COMMENT
</dd></dl>
<a name="Example_2" id="Example_2"></a><h5> <span class="mw-headline"> Example 2 </span></h5>
<p>A germplasm name-type can be defined to contain sub-fields.  These sub-fields are ignored during all GMS name manipulation but may be useful to specific application programs.  As an example, a name could contain a two-letter program ID, a year code, a five character cross designation and a selection string as follows:  UR96IR123 10 3W 4.
The format line would look as follows:
</p>
<dl><dd>PROGRAM,2,A,1;YEAR,2,I,1;CROSS,5,A,1;SELN,0,A,1
</dd></dl>
<p>or
</p>
<dl><dd>PROGRAM,2;YEAR,2,I;CROSS,5;SELN
</dd></dl>
<a name="CORRECTING_RECORDS_IN_THE_GMS_DATABASE" id="CORRECTING_RECORDS_IN_THE_GMS_DATABASE"></a><h3> <span class="mw-headline">CORRECTING RECORDS IN THE GMS DATABASE</span></h3>
<p>Corrections and changes will inevitably occur in any database.  Authorised users can only make these changes, and all changes are logged so that the sequence of changes can be traced and can be ‘undone’ if required.  The history of changes is tracked by records on the CHANGES table.
One common occurrence of changes is when new information about an existing germplasm record is entered into a local database.  If the existing record is also in the local database then the local user can complete the changes.  But if it is in the central database, then changes cannot be completed until the central database is updated. Verifying and completing requested changes is part of the update process and sufficient information and justification needs to be recorded in the changes table to allow the process to be completed.
Verification and completion of changes to the central database may take some time but local users would like to see their changes reflected immediately.  This is achieved by the DLL which always checks the local CHANGES table for central changes and applies them at run time for the specific installation where they are recorded.
</p>
<a name="The_CHANGES_Table" id="The_CHANGES_Table"></a><h4> <span class="mw-headline"> The CHANGES Table </span></h4>
<table border="1" cellpadding="2">
<tr>
<th width="30%"> Columns - Long Name (Name)
</th><th width="40%"> Description
</th><th width="15%"> Type
</th><th width="15%"> Length (bytes)
</th></tr>
<tr>
<td> CHANGE ID (CID)
</td><td>Unique change identification number
</td><td>Integer
</td><td>4
</td></tr>
<tr>
<td> CHANGE_TABLE (CTABLE)
</td><td> Name of the table where the change is to be effected
</td><td>Text
</td><td>16
</td></tr>
<tr>
<td> CHANGE_FIELD (CFIELD)
</td><td>Name of the field where the change is to be effected
</td><td>Text
</td><td>16
</td></tr>
<tr>
<td> CHANGE_RECORD (CRECORD)
</td><td> Key ID of the record where the change is to be effected
</td><td>Integer
</td><td>4
</td></tr>
<tr>
<td> CHANGE_FROM (CFROM)
</td><td> Previous value of the field
</td><td>Integer
</td><td>4
</td></tr>
<tr>
<td> CHANGE_TO (CTO)
</td><td> New value of the field
</td><td>Integer
</td><td>4
</td></tr>
<tr>
<td> CHANGE_DATE (CDATE)
</td><td> Date of the change (YYYYMMDD)
</td><td>Integer
</td><td>4
</td></tr>
<tr>
<td> CHANGE_TIME (CTIME)
</td><td>Time of the change (HHMMSSMM)
</td><td>Integer
</td><td>4
</td></tr>
<tr>
<td> CHANGE_GROUP
</td><td> Group identification for dependant changes(Eg DATE/TIME string)
</td><td>Text
</td><td>20
</td></tr>
<tr>
<td> CHANGE_USER (CUID)
</td><td> Id of the USER making the change
</td><td> Integer
</td><td>2
</td></tr>
<tr>
<td> CHANGE_REFERENCE (CREF)
</td><td> Chang reference
</td><td>Integer
</td><td>4
</td></tr>
<tr>
<td> CHANGE_STATUS
</td><td> Change status: 0=ACTIVE, 1=UNDONE, 2=UPDATED
</td><td> Integer
</td><td> 4
</td></tr>
<tr>
<td> CHANGE_DESCRIPTION (CDES)
</td><td> Change description or reason
</td><td> Text
</td><td> 255
</td></tr></table>
<a name="Change_location.2C_date_or_reference_of_germplasm" id="Change_location.2C_date_or_reference_of_germplasm"></a><h4> <span class="mw-headline"> Change location, date or reference of germplasm </span></h4>
<p>To change location, date or reference of germplasm with GERMPLASM_ID GIDX.
</p><p><b>Checks:</b>
</p>
<pre>   a) Check that GIDX has not been deleted or replaced. 
</pre>
<p><b>Changes:</b>
</p>
<pre>   a) Add a change record to the local database with the following values: 
      CID - Next available negative integer
      CTABLE - GERMPLSM
      CFIELD - GLOCN, GDATE or GREF as appropriate
      CRECORD - GIDX
      CFROM - Current value of the field
      CTO - New value for the field
      CDATE - Current date
      CTIME - Current time
      CGROUP - Null
      CUID - Current user ID.
      CSTATUS - 0
      CDESC - Reason for change
   b) If the germplasm is local change the required field.
   c) (a)-(b) can be repeated for more than one field.
</pre>
<a name="Change_a_progenitor_ID_for_generative_germplasm" id="Change_a_progenitor_ID_for_generative_germplasm"></a><h4> <span class="mw-headline"> Change a progenitor ID for generative germplasm </span></h4>
<p>Change the ID of progenitor number N of germplasm with GERMPLASM_ID GIDX from GIDY to GIDZ (GIDZ may be MISSING (0) to indicate that progenitor number N is unknown)
</p><p><b>Checks:</b>
</p>
<pre>   a) Check that germplasm GIDX is produced by a generative process and has at least<br />      N progenitors (NO_PROGENITORS &ge; N)
   b) Check that GIDZ precedes GIDX if it is not MISSING.
</pre>
<p><b>Changes:</b>
</p>
<pre>   a) Add a record to the CHANGES table.
      IF N is 1 or 2, the first five field of the CHANGE record will have values:
         CID: Next available negative integer
         CTABLE: GERMPLSM
         CFIELD: GPID1 or GPID2 depending on whether N is 1 or 2
         CRECORD: GIDX
         CFROM: GIDY
         CTO: GIDZ
      If N is &gt; 2 the the fields will have the following values:
         CID: Next available negative integer
         CTABLE: PROGNTRS
         CFIELD: PIDx where x=N
         CRECORD: GIDX
         CFROM: GIDY
         CTO: GIDZ    
   b) If GIDX &lt; 0 change PROGENITOR_ID for progenitor number N on the GERMPLASM table<br />      or the OTHER_PROGENITORS table from GIDY to GIDZ.
   c) If GIDZ is missing and N=NO_PROGENITORS then reduce NO_PROGENITORS by 1 and<br />      if N&gt;2 delete the record from the OTHER_PROGENITORS table.
    
</pre>
<a name="Add_a_progenitor_ID_for_a_generative_germplasm" id="Add_a_progenitor_ID_for_a_generative_germplasm"></a><h4> <span class="mw-headline"> Add a progenitor ID for a generative germplasm </span></h4>
<p>Add the germplasm GIDZ as a PROGENITOR_ID for germplasm GIDX.
</p><p><b>Checks:</b>
</p>
<pre>   a) Check that germplasm GIDX is produced by a generative process and determine<br />      the number, N, of progenitors specified, N=NO_PROGENITORS.
   b) Check that the generative method of GIDX supports N+1 progenitors.
   c) Check that GIDZ precedes GIDX
</pre>
<p><b>Changes:</b>
</p>
<pre>   a) Add a CHANGE record with CTABLE=GERMPLSM, CFIELD=GNPGS, CRECORD=GIDX, CFROM=N,<br />      CTO=N+1. Then add a second CHANGE record as in 3.9.3.a  with GIDY=0.
   b) If GIDX&lt;0 change NO_PROGENITORS to N+1 in the local GERMPLSM table.
   c) If GIDX&lt;0 and N+1 is 1 or 2, change PROGENITOR_ID1 or PROGENITOR_ID2 from MISSING<br />      to GIDZ. IF N+1 &gt; 2 add a record to the OTHER_PROGENITORS table<br />      the GERMPLASM_ID=GIDX, PROGENITOR_NO=N+1 and PROGENITOR_ID=GIDZ.
</pre>
<a name="Change_the_group_ID_for_a_derivative_germplasm" id="Change_the_group_ID_for_a_derivative_germplasm"></a><h4> <span class="mw-headline"> Change the group ID for a derivative germplasm </span></h4>
<p>To change the group ID of GIDX from GIDY to GIDZ take the following steps:
</p><p><b>Checks:</b>
</p>
<pre>   a) Check the germplasm GIDX is derivative. 
   b) Check that GIDZ is generative or a derivative with unknown source (GPID1<br />      and GPID2=0)
   c) Check that the new group precedes the target in chronology.
   d) If GIDZ # 0, the new group of GIDX (GGPX) will be GIDZ. Check the source of<br />      GIDX has group GIDZ. If not, set the source to zero (3.9.6) or change its group<br />      to GIDZ using this algorithm.
   e) If GIDZ = 0, then GIDX will become a derivative with unknown source (GPID1=0).<br /> Check that the source (GPID2) is also zero. 
   f) Check that all dependants of GIDX have group GIDZ.
</pre>
<p><b>Changes:</b>
</p>
<pre>   a) Add a CHANGE record with CTABLE=GERMPLSM, CFIELD=GPID1, CRECORD=GIDX CFROM=GIDY,<br />      CTO=GIDZ and put a unique group identifier<br />      such as the DATE/TIME string in CGROUP.
   b) If GIDX&lt;0 change PROGENITOR_ID1 for germplasm GIDX on the GERMPLSM table<br />      from GIDY to GIDZ.
   c) Check all derivatives with derivative paths tracing through GIDX and change<br />      their groups to the new group of  GIDX, GGPX, using this algorithm. These dependant<br />     changes get the same CGROUP string as the original change.
</pre>
<a name="Change_a_source_ID_for_a_derivative_germplasm" id="Change_a_source_ID_for_a_derivative_germplasm"></a><h4> <span class="mw-headline"> Change a source ID for a derivative germplasm </span></h4>
<p>To change the ID of the source for germplasm GIDX from GIDY to GIDZ
</p><p><b>Checks:</b>
</p>
<pre>   a) Check that a derivative process produces GIDX.
   b) Check that GIDZ precedes GIDX.
   c) Check that GIDZ is from the same germplasm group as GIDX. If it is not,<br />     then the group IDs of all dependants will have to be traced and changed.
   d) Check that the selection name for GIDX does not indicate a source other than GIDZ.
</pre>
<p><b>Changes:</b>
</p>
<pre>   a) Add a CHANGE record with CTABLE=GERMPLSM, CFIELD=GPID2, CRECORD=GIDX,<br />      CFROM=GIDY, CTO=GIDZ and CGROUP=date/time. Other Change fields have appropriate values.
   b) If GIDX&lt;0 change PROGENITOR_ID2 for germplasm GIDX on the local GERMPLASM<br />     table from GIDY to GIDZ.
   c) If GIDZ # 0, check the group of GIDX is the same as the group of GIDZ.<br />     If not, change the group of GIDX to the group ID of GIDZ (3.9.5) and check<br />      the group ID of all dependants of GIDX (direct derivatives and derivatives in the same<br />      group with development paths tracing through GIDX).
   d) If GIDZ = 0, and the group of GIDX # 0 then check whether the<br />      group of GIDX should also be set to 0 (missing).
</pre>
<a name="Change_a_germplasm_method" id="Change_a_germplasm_method"></a><h4> <span class="mw-headline"> Change a germplasm method </span></h4>
<p>To change the method of germplasm GIDX from METH1 to METH2:
</p><p><b>Checks:</b>
</p>
<pre>   a) Check that the new method supports the number of progenitors of GIDX
   b) Check that the progenitors are appropriate for the new method of GIDX.
</pre>
<p><b>Changes:</b>
</p>
<pre>   a) Add a CHANGE record with CTABLE=GERMPLSM, CFIELD=METHN, CRECORD=GIDX,<br />      CFROM=METH1, CTO=METH2, CGROUP=date/time string and other fields have<br />      appropriate values. 
   b) If the method is being changed from derivative to generative then<br />      the group IDs of all derivatives with paths tracing through GIDX need to change<br />     to GIDX (3.9.5).
   c) If the method is being changed from generative to derivative all<br />      derivatives in the group GIDX need to change group ID to the new group of GIDX.<br />      (This may be GIDX if it becomes a derivative group source) (3.9.5).
   d) If the number of progenitors change or the method type changes<br />      add a CHANGE record with CTABLE=GERMPLSM, CFIELD=GNPGS, CRECORD=GIDX, CFROM=old value,<br />      CTO=new value, CGROUP=same string as in a.
   e) If GIDX&lt;0 make the changes in the local germplasm table
</pre>
<a name="Replace_an_existing_germplasm_record_with_another_one" id="Replace_an_existing_germplasm_record_with_another_one"></a><h4> <span class="mw-headline"> Replace an existing germplasm record with another one </span></h4>
<p>This change is required when germplasm GIDX is found to represent the same germplasm as GIDY, or when GIDY contains correct or additional information compared to GIDX.
</p><p><b>Checks:</b>
</p>
<pre>   a) Check for dependencies. If GIDX is a derivative and GIDY is from a<br />      different germplasm group to GIDX then trace all germplasm in the same group as<br />      GIDX with derivative paths which do not pass through GIDX as these will remain<br />      unchanged in the change process and their relevance needs to be examined.
   b) Check that GIDX is later than GIDY, or all dependencies are later than GIDY.
   c) Check what names and attributes of GIDX should be copied to GIDY.
   d) If GIDX is generative and GIDY is derivative then GIDX must not occur<br />      as a group source (GPID1 in derivative germplasm).
</pre>
<p><b>Changes:</b>
</p>
<pre>   a) There are four cases to distinguish:
      Case 1. GIDX and GIDY both produced by generative processes
      Case 2. GIDX and GIDY both derivatives in the SAME GERMPLASM GROUP
      Case 3. GIDX is generative and GIDY is derivative in group GY
      Case 4. GIDX is derivative and GIDY is generative or derivative but<br />      from a different germplasm group to GIDX.
   b) Obtain a date-time stamp for the change from the computer clock to be<br />      added stored in the CGROUP field of change records for dependencies<br />      as described below.
   c) For cases 1 and 2, simply change all occurrences of GIDX in progenitor<br />      fields (of the GERMPLASM table and the OTHER-PROGENITORS table) to GIDY using 3.9.3<br />      and 3.9.6. (These occurrences are called dependencies).
      For case 3, change the group IDs (PROGENITOR_ID1) of all derivatives of<br />      GIDX to GY using 3.9.5 and change all other dependencies of GIDX to GIDY<br />      using 3.9.3 or 3.9.6.
      For case 4, change the group ID of all derivatives in the same group as<br />      GIDX which have derivative paths tracing through GIDX to the group of GIDY<br />      (GIDY itself if it is generative) using 3.9.5. Change all other dependencies of GIDX to GIDY<br />      using 3.9.3 or 3.9.6.
   d) Request reason for change.
   e) Add a CHANGE record the CHANGES table with the following values:
      CID - Next available negative integer
      CTABLE - GERMPLSM
      CFIELD - GID
      CRECORD	- GIDX
      CFROM - GIDX
      CTO - GIDY
      CDATE - Current date
      CTIME - Current time
      CGROUP - date/time stamp for all dependant changes YYYYMMDDHHMMSSMM
      CUID - Current user ID.
      CSTATUS	- 0
      CDESC- Reason for change
   f) If GIDX&lt;0 set the GERMPLASM_REPLACE field of record GIDX to value GIDY.
</pre>
<a name="Delete_a_Germplasm_Record" id="Delete_a_Germplasm_Record"></a><h4> <span class="mw-headline"> Delete a Germplasm Record </span></h4>
<p>This change may occur when it is discovered that a germplasm entry with GIDX is definitely wrong, but no alternative is known. If an alternative is known then this correction should not be used because it destroys all dependencies to the deleted germplasm. If the known alternative is already in the GMS then the replace GERMPLASM procedure should be used (3.9.8), otherwise fields of the current germplasm record should be changed to match the correct values (3.9.2-3.9.7). 
</p><p><b>Checks:</b>
</p>
<pre>   a) Check if GIDX has any dependencies – offspring, group subjects or<br />      derivatives. (Check for GIDX in the PROGENITOR_ID1 and PROGENITOR_ID2 fields of the<br />      GERMPLASM table and in the PROGENITOR_ID field of the OTHER_PROGENITORS table).
</pre>
<p><b>Changes:</b>
</p>
<pre>   a) Request a reason for change and obtain a date-time stamp for the deletion.
   b) Change GIDX to MISSING (0) in all dependencies using 3.9.4 – 3.9.6,<br />      adding the same date-time stamp to the CGROUP field of the change record<br />      for each dependency.
   c) Add a CHANGE record for GIDX exactly as in 3.9.8e except that CTO=GIDX. 
   d) If GIDX&lt;0, set the GERMPLASM_REPLACE field of GIDX to value GIDX.
</pre>
<!-- 
NewPP limit report
Preprocessor node count: 86/1000000
Post-expand include size: 0/2097152 bytes
Template argument size: 0/2097152 bytes
Expensive parser function count: 0/100
-->
<div class="printfooter">
Retrieved from "<a href="http://localhost../../../../articles/t/d/m/TDM_Genealogy_Management_System_5.4_7420.html">http://localhost../../../../articles/t/d/m/TDM_Genealogy_Management_System_5.4_7420.html</a>"</div>
						<!-- end content -->
						<div class="visualClear"></div>
		</div>
	</div>
		</div>
		<div id="column-one">
	<div id="p-cactions" class="portlet">
		<h5>Views</h5>
		<div class="pBody">
			<ul>
	
				 <li id="ca-nstab-main" class="selected"><a href="../../../../articles/t/d/m/TDM_Genealogy_Management_System_5.4_7420.html" title="View the content page [c]" accesskey="c">Page</a></li>
				 <li id="ca-talk" class="new"><a href="../../../../articles/t/d/m/Talk%7ETDM_Genealogy_Management_System_5.4_dcbb.html" title="Discussion about the content page [t]" accesskey="t">Discussion</a></li>
				 <li id="ca-viewsource"><a href="../../../../articles/t/d/m/TDM_Genealogy_Management_System_5.4_7420.html" title="This page is protected.&#10;You can view its source [e]" accesskey="e">View source</a></li>
				 <li id="ca-history"><a href="../../../../articles/t/d/m/TDM_Genealogy_Management_System_5.4_7420.html" title="Past versions of this page [h]" accesskey="h">History</a></li>			</ul>
		</div>
	</div>
	<div class="portlet" id="p-personal">
		<h5>Personal tools</h5>
		<div class="pBody">
			<ul>
				<li id="pt-login"><a href="../../../../articles/u/s/e/Special%7EUserLogin_94cd.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</a></li>
			</ul>
		</div>
	</div>
	<div class="portlet" id="p-logo">
		<a style="background-image: url(../../../../misc/icis-wiki3.png);" href="../../../../index.html" title="Visit the main page [z]" accesskey="z"></a>
	</div>
	<script type="text/javascript"> if (window.isMSIE55) fixalpha(); </script>
	<div class='generated-sidebar portlet' id='p-navigation'>
		<h5>Navigation</h5>
		<div class='pBody'>
			<ul>
				<li id="n-mainpage"><a href="../../../../index.html" title="Visit the main page [z]" accesskey="z">Main Page</a></li>
				<li id="n-portal"><a href="../../../../articles/c/o/m/ICISWiki%7ECommunity_Portal_7449.html" title="About the project, what you can do, where to find things">Community portal</a></li>
				<li id="n-recentchanges"><a href="../../../../articles/r/e/c/Special%7ERecentChanges_e0d0.html" title="The list of recent changes in the wiki [r]" accesskey="r">Recent changes</a></li>
				<li id="n-help"><a href="../../../../articles/c/o/n/Help%7EContents_22de.html" title="The place to find out">Help</a></li>
			</ul>
		</div>
	</div>
	<div id="p-search" class="portlet">
		<h5><label for="searchInput">Search</label></h5>
		<div id="searchBody" class="pBody">
			<form action="../../../../articles/s/e/a/Special%7ESearch_a8e5.html" id="searchform"><div>
				<input id="searchInput" name="search" type="text" title="Search ICISWiki [f]" accesskey="f" value="" />
				<input type='submit' name="go" class="searchButton" id="searchGoButton"	value="Go" title="Go to a page with this exact name if exists" />&nbsp;
				<input type='submit' name="fulltext" class="searchButton" id="mw-searchButton" value="Search" title="Search the pages for this text" />
			</div></form>
		</div>
	</div>
	<div class="portlet" id="p-tb">
		<h5>Toolbox</h5>
		<div class="pBody">
			<ul>
				<li id="t-whatlinkshere"><a href="../../../../articles/w/h/a/Special%7EWhatLinksHere_TDM_Genealogy_Management_System_5.4_82c1.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</a></li>
				<li id="t-recentchangeslinked"><a href="../../../../articles/r/e/c/Special%7ERecentChangesLinked_TDM_Genealogy_Management_System_5.4_798e.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</a></li>
<li id="t-specialpages"><a href="../../../../articles/s/p/e/Special%7ESpecialPages_f12b.html" title="List of all special pages [q]" accesskey="q">Special pages</a></li>
				<li id="t-print"><a href="../../../../articles/t/d/m/TDM_Genealogy_Management_System_5.4_7420.html" title="Printable version of this page [p]" accesskey="p">Printable version</a></li>				<li id="t-permalink"><a href="../../../../articles/t/d/m/TDM_Genealogy_Management_System_5.4_7420.html" title="Permanent link to this version of the page">Permanent link</a></li>			</ul>
		</div>
	</div>
		</div><!-- end of the left (by default at least) column -->
			<div class="visualClear"></div>
			<div id="footer">
				<div id="f-poweredbyico"><a href="http://www.mediawiki.org/"><img src="../../../../skins/common/images/poweredby_mediawiki_88x31.png" alt="Powered by MediaWiki" /></a></div>
				<div id="f-copyrightico"><a href="../../../../../COPYING.html"><img src="/icis/skins/common/images/gnu-fdl.png" alt='GNU Free Documentation License 1.2' /></a></div>
			<ul id="f-list">
					<li id="credits">This page was last modified 08:05, 30 September 2008 by <a href="../../../../articles/a/p/o/User%7EAportugal_745d.html" title="User:Aportugal">Arllet Portugal</a>. Based on work by <a href="../../../../articles/c/p/i/User%7ECpimentel_117e.html" title="User:Cpimentel">Clarissa Pimentel</a>.</li>
					<li id="copyright">Content is available under <a href="../../../../../COPYING.html" class="external " title="../../../../../COPYING.html">GNU Free Documentation License 1.2</a>.</li>
					<li id="privacy"><a href="../../../../articles/p/r/i/ICISWiki%7EPrivacy_policy_0cbf.html" title="ICISWiki:Privacy policy">Privacy policy</a></li>
					<li id="about"><a href="../../../../articles/a/b/o/ICISWiki%7EAbout_53aa.html" title="ICISWiki:About">About ICISWiki</a></li>
					<li id="disclaimer"><a href="../../../../articles/g/e/n/ICISWiki%7EGeneral_disclaimer_9129.html" title="ICISWiki:General disclaimer">Disclaimers</a></li>
			</ul>
		</div>
</div>

		<script type="text/javascript">if (window.runOnloadHook) runOnloadHook();</script>
<!-- Served in 196.669 secs. --></body></html>
