<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
	<head>
		<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
		<meta http-equiv="Content-Style-Type" content="text/css" />
		<meta name="generator" content="MediaWiki 1.14.0" />
		<meta name="keywords" content="TDM GMS Browse,TDM GMS Browse,TDM Genealogy Management System,TDM ICIS Application and Database Installation" />
		<link rel="shortcut icon" href="../../../../misc/favicon.ico" />
		<link rel="search" type="application/opensearchdescription+xml" href="../../../../../opensearch_desc.php" title="ICISWiki (en)" />
		<link title="Creative Commons" type="application/rdf+xml" href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html" rel="alternate meta" />
		<link rel="copyright" href="../../../../../COPYING.html" />
		<link rel="alternate" type="application/rss+xml" title="ICISWiki RSS Feed" href="http://localhost../../../../articles/r/e/c/Special%7ERecentChanges_e0d0.html" />
		<link rel="alternate" type="application/atom+xml" title="ICISWiki Atom Feed" href="http://localhost../../../../articles/r/e/c/Special%7ERecentChanges_e0d0.html" />
		<title>TDM GMS Browse - ICISWiki</title>
		<link rel="stylesheet" href="../../../../skins/common/shared.css?195" type="text/css" media="screen" />
		<link rel="stylesheet" href="../../../../skins/common/commonPrint.css?195" type="text/css" media="print" />
		<link rel="stylesheet" href="../../../../skins/monobook/main.css?195" type="text/css" media="screen" />
		<!--[if lt IE 5.5000]><link rel="stylesheet" href="../../../../skins/monobook/IE50Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<!--[if IE 5.5000]><link rel="stylesheet" href="../../../../skins/monobook/IE55Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<!--[if IE 6]><link rel="stylesheet" href="../../../../skins/monobook/IE60Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<!--[if IE 7]><link rel="stylesheet" href="../../../../skins/monobook/IE70Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<link rel="stylesheet" href="../../../../skins/../../../../raw/MediaWiki%7ECommon.css?195" type="text/css" />
		<link rel="stylesheet" href="../../../../skins/../../../../raw/MediaWiki%7EPrint.css?195" type="text/css" media="print" />
		<link rel="stylesheet" href="../../../../skins/../../../../raw/MediaWiki%7EMonobook.css?195" type="text/css" />
		<link rel="stylesheet" href="../../../../skins/../../../../raw/gen.css?195" type="text/css" />
		<!--[if lt IE 7]><script type="text/javascript" src="../../../../skins/common/IEFixes.js?195"></script>
		<meta http-equiv="imagetoolbar" content="no" /><![endif]-->

		<script type= "text/javascript">/*<![CDATA[*/
		var skin = "monobook";
		var stylepath = "../../../../skins";
		var wgArticlePath = "../../../../$1";
		var wgScriptPath = "../../../../..";
		var wgScript = "/icis/index.php";
		var wgVariantArticlePath = false;
		var wgActionPaths = {};
		var wgServer = "http://localhost";
		var wgCanonicalNamespace = "";
		var wgCanonicalSpecialPageName = false;
		var wgNamespaceNumber = 0;
		var wgPageName = "TDM_GMS_Browse";
		var wgTitle = "TDM GMS Browse";
		var wgAction = "view";
		var wgArticleId = "996";
		var wgIsArticle = true;
		var wgUserName = null;
		var wgUserGroups = null;
		var wgUserLanguage = "en";
		var wgContentLanguage = "en";
		var wgBreakFrames = false;
		var wgCurRevisionId = "10248";
		var wgVersion = "1.14.0";
		var wgEnableAPI = true;
		var wgEnableWriteAPI = true;
		var wgSeparatorTransformTable = ["", ""];
		var wgDigitTransformTable = ["", ""];
		var wgRestrictionEdit = [];
		var wgRestrictionMove = [];
		/*]]>*/</script>

		<script type="text/javascript" src="../../../../skins/common/wikibits.js?195"><!-- wikibits js --></script>
		<!-- Head Scripts -->
		<script type="text/javascript" src="../../../../raw/gen.js"><!-- site js --></script>
	</head>
<body class="mediawiki ltr ns-0 ns-subject page-TDM_GMS_Browse skin-monobook">
	<div id="globalWrapper">
		<div id="column-content">
	<div id="content">
		<a name="top" id="top"></a>
				<h1 id="firstHeading" class="firstHeading">TDM GMS Browse</h1>
		<div id="bodyContent">
			<h3 id="siteSub">From ICISWiki</h3>
			<div id="contentSub"></div>
									<div id="jump-to-nav">Jump to: <a href="#column-one">navigation</a>, <a href="#searchInput">search</a></div>			<!-- start content -->
			<table id="toc" class="toc" summary="Contents"><tr><td><div id="toctitle"><h2>Contents</h2></div>
<ul>
<li class="toclevel-1"><a href="#THE_GMS_BROWSE_APPLICATION"><span class="tocnumber">1</span> <span class="toctext">THE GMS BROWSE APPLICATION</span></a>
<ul>
<li class="toclevel-2"><a href="#Introduction"><span class="tocnumber">1.1</span> <span class="toctext">Introduction</span></a></li>
<li class="toclevel-2"><a href="#Searching_for_Germplasm"><span class="tocnumber">1.2</span> <span class="toctext">Searching for Germplasm</span></a>
<ul>
<li class="toclevel-3"><a href="#Name_searches"><span class="tocnumber">1.2.1</span> <span class="toctext">Name searches</span></a></li>
<li class="toclevel-3"><a href="#Searching_for_crosses"><span class="tocnumber">1.2.2</span> <span class="toctext">Searching for crosses</span></a></li>
</ul>
</li>
<li class="toclevel-2"><a href="#Displaying_Records_by_GID"><span class="tocnumber">1.3</span> <span class="toctext">Displaying Records by GID</span></a></li>
<li class="toclevel-2"><a href="#Listing_Relatives"><span class="tocnumber">1.4</span> <span class="toctext">Listing Relatives</span></a>
<ul>
<li class="toclevel-3"><a href="#Close_Relatives"><span class="tocnumber">1.4.1</span> <span class="toctext">Close Relatives</span></a></li>
<li class="toclevel-3"><a href="#Group_Relatives"><span class="tocnumber">1.4.2</span> <span class="toctext">Group Relatives</span></a></li>
<li class="toclevel-3"><a href="#A_Derivative_Neighborhood"><span class="tocnumber">1.4.3</span> <span class="toctext">A Derivative Neighborhood</span></a></li>
<li class="toclevel-3"><a href="#Showing_Multilateral_System_Ancestors"><span class="tocnumber">1.4.4</span> <span class="toctext">Showing Multilateral System Ancestors</span></a></li>
</ul>
</li>
<li class="toclevel-2"><a href="#Pedigree_Analysis"><span class="tocnumber">1.5</span> <span class="toctext">Pedigree Analysis</span></a>
<ul>
<li class="toclevel-3"><a href="#Presentation_of_Generation_Codes"><span class="tocnumber">1.5.1</span> <span class="toctext">Presentation of Generation Codes</span></a></li>
<li class="toclevel-3"><a href="#Pedigree_Trees_and_Tables"><span class="tocnumber">1.5.2</span> <span class="toctext">Pedigree Trees and Tables</span></a></li>
<li class="toclevel-3"><a href="#Mendelgrams"><span class="tocnumber">1.5.3</span> <span class="toctext">Mendelgrams</span></a></li>
<li class="toclevel-3"><a href="#Traces"><span class="tocnumber">1.5.4</span> <span class="toctext">Traces</span></a></li>
<li class="toclevel-3"><a href="#Coefficients_of_Parentage"><span class="tocnumber">1.5.5</span> <span class="toctext">Coefficients of Parentage</span></a></li>
</ul>
</li>
<li class="toclevel-2"><a href="#Changing_and_Correcting_Germplasm_Records"><span class="tocnumber">1.6</span> <span class="toctext">Changing and Correcting Germplasm Records</span></a>
<ul>
<li class="toclevel-3"><a href="#Deleting_or_Replacing_Germplasm"><span class="tocnumber">1.6.1</span> <span class="toctext">Deleting or Replacing Germplasm</span></a></li>
<li class="toclevel-3"><a href="#Setting_Germplasm_or_Name_Fields"><span class="tocnumber">1.6.2</span> <span class="toctext">Setting Germplasm or Name Fields</span></a></li>
<li class="toclevel-3"><a href="#Adding_Notes_and_Names"><span class="tocnumber">1.6.3</span> <span class="toctext">Adding Notes and Names</span></a></li>
</ul>
</li>
<li class="toclevel-2"><a href="#Working_with_Lists_in_BROWSE"><span class="tocnumber">1.7</span> <span class="toctext">Working with Lists in BROWSE</span></a>
<ul>
<li class="toclevel-3"><a href="#Opening_existing_lists"><span class="tocnumber">1.7.1</span> <span class="toctext">Opening existing lists</span></a></li>
<li class="toclevel-3"><a href="#Adding_entries_to_a_list"><span class="tocnumber">1.7.2</span> <span class="toctext">Adding entries to a list</span></a></li>
<li class="toclevel-3"><a href="#Saving_a_displayed_table_as_a_list"><span class="tocnumber">1.7.3</span> <span class="toctext">Saving a displayed table as a list</span></a></li>
<li class="toclevel-3"><a href="#Displaying_the_entries_in_an_open_list"><span class="tocnumber">1.7.4</span> <span class="toctext">Displaying the entries in an open list</span></a></li>
<li class="toclevel-3"><a href="#Exporting_the_combined_pedigree_tree_for_all_entries_in_a_list"><span class="tocnumber">1.7.5</span> <span class="toctext">Exporting the combined pedigree tree for all entries in a list</span></a></li>
<li class="toclevel-3"><a href="#Searching_lists_for_a_specified_germplasm"><span class="tocnumber">1.7.6</span> <span class="toctext">Searching lists for a specified germplasm</span></a></li>
</ul>
</li>
<li class="toclevel-2"><a href="#Using_BROWSE_in_Batch_Mode"><span class="tocnumber">1.8</span> <span class="toctext">Using BROWSE in Batch Mode</span></a></li>
<li class="toclevel-2"><a href="#Setting_BROWSE_options"><span class="tocnumber">1.9</span> <span class="toctext">Setting BROWSE options</span></a></li>
</ul>
</li>
</ul>
</td></tr></table><script type="text/javascript"> if (window.showTocToggle) { var tocShowText = "show"; var tocHideText = "hide"; showTocToggle(); } </script>
<a name="THE_GMS_BROWSE_APPLICATION" id="THE_GMS_BROWSE_APPLICATION"></a><h1> <span class="mw-headline"> THE GMS BROWSE APPLICATION </span></h1>
<a name="Introduction" id="Introduction"></a><h2> <span class="mw-headline"> Introduction </span></h2>
<p>The BROWSE program is a GMS application for viewing and changing GMS records.  It is a FORTRAN 90 console application with a DOS-style user interface (Figure 1.)
</p>
<pre>Figure 1.  The BROWSE Interface
************************************************************************
ICIS UTILITY FOR BROWSING AND MODIFYING A GMS DATABASE (Version 5.4.2.0)
************************************************************************
INI FILE (E:\TEmp\ICIS.INI)
[CENTRAL GMS] DSN=IRIS-CENTRAL-MYSQL
[LOCAL GMS] DSN=IRIS-ADMIN-MYSQL
[BROWSE] LASTDIR=E:\ICIS5\DATABASE
USER GMCLAREN4 CONNECTED TO the Central Administration Local Database
----------------------------------------------------------------------
BROWSE WILL SEARCH FOR NAME STRINGS OR EXECUTE COMMANDS,
A&nbsp;% AT THE END OF A SEARCH STRING MATCHES ALL ENDINGS,
A _ IN A SEARCH STRING MATCHES ANY CHARACTER AT THAT POSITION
,NLOC=xxx and ,NTYPE=yyy RESTRICS THE SEARCH BY LOCATION OR TYPE
COMMANDS START WITH ".", OTHERWISE A SEARCH STRING IS ASSUMED.
THE FOLLOWING COMMANDS ARE AVAILABLE:
.n     - DISPLAY ENTRY n FROM THE CURRENT LIST
.GID # - DISPLAY GMS DATA FOR GID #, EG .GID 12345
.Pn    - DISPLAY GMS DATA FOR PROGENITOR n OF CURRRENT GID
.FCR &lt;Purdy string&gt; - FINDS CROSSES BETWEEN PARENTS
.TRE #,n - LIST THE PEDIGREE TREE FOR n GENERATIONS
.TCE #,n - TRACE ALL DESCENDANTS FOR n GENERATIONS
.INP &lt;file.ext&gt; - SET THE INPUT STREAM TO FILE
.MGM # - DISPLAY A MENDELGRAM FOR GID #
.COP #1,#2 - COMPUTE COP BETWEEN GID1 AND GID2 OR .COP LST FOR GIDS IN A LIST
.REP #1,#2,&lt;reason&gt; - REPLACE GID1 WITH GID2 (#2=0 TO DELETE GID1)
.SET fld=## IN # - CHANGE GERMPLASM OR NAME DATA (.SET for HELP)
.NAME #,&lt;ntype&gt;,&lt;nlocn&gt;,&lt;ndate&gt;,&lt;nref&gt;,&lt;nvalue&gt; - ADDS A NAME FOR GID #
.NOTE #,&lt;avalue&gt; - ADDS A NOTE ATTRIBUTE FOR GID #
.OPT  - GIVES HELP ON OPTIONS
.MORE - GIVES HELP ON MORE FUNCTIONS
ENTER SEARCH OR COMMAND STRING (.HLP=HELP, ./=END)
</pre>
<p>BROWSE reads the ICIS.INI file from the default temp directory (indicated by the environment variable ‘temp’).  If the [Local GMS] section of ICIS.INI has ‘UID’ and ‘PWD’ keys, BROWSE will try to open GMS with the corresponding USER_ID and PASSWORD (<a href="../../../../articles/t/d/m/TDM_ICIS_Application_and_Database_Installation_0520.html#ICIS_INI_Files" title="TDM ICIS Application and Database Installation">See installation guide</a>).  If this fails, the program asks the user for these parameters.  BROWSE also reads a working directory path from the ‘Lastdir’ key of the [Browse] section of ICIS.INI.  If the LastDir key does not exist, the user is asked for a working directory where a log file will be stored and where batch input files can be accessed.  Other options which set defaults or affect displays can be set through keys in the Browse section of the INI file or through the <a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html#Setting_BROWSE_options" title="TDM GMS Browse">.OPT</a> command.
</p><p>Once the database is connected, BROWSE requests a search or command string as in Figure 1.  Command strings always start with a period and the rest of the command string may contain parameters.  Any input line, which is not recognized as a command, is assumed to be a name string.
</p><p>BROWSE will accept a list of names or commands to process from a file and send output to the console and to a log file, BROWSE.LOG, located in the BROWSE working directory.  This log file can be processed or printed, but is overwritten by each BROWSE session.  Names or commands for batch processing are typed into an ASCII file in the BROWSE working directory exactly as they would be at the command prompt.  The command .INP &lt;filename&gt; will cause BROWSE to read the commands from the named file and process them consecutively.  User interventions to select from lists or confirm changes must still be entered at the console although some selection <a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html#Setting_BROWSE_options" title="TDM GMS Browse">options</a> can be specified.
</p>
<a name="Searching_for_Germplasm" id="Searching_for_Germplasm"></a><h2> <span class="mw-headline"> Searching for Germplasm </span></h2>
<a name="Name_searches" id="Name_searches"></a><h3> <span class="mw-headline"> Name searches </span></h3>
<p>Two types of data can be entered at the BROWSE command line, commands which start with a point followed by three specific letters, and name strings which have first four characters not matching any command. When BROWSE receives a name string, it searches the local and central NAMES tables for matches.
</p><p>Name strings can terminate with a ‘%’ symbol which requests a match to the first part of the string with any termination, and they may contain the underscore character, ‘_’, which requests a match to any character at that position. Name strings are searched as typed, and after transformation to <a href="../../../../articles/t/d/m/TDM_Genealogy_Management_System_d3f7.html#Name_Standardization" title="TDM Genealogy Management System">standard form</a>.
</p><p>All matches are listed in a table of ‘hits’ numbered in the left column, showing all names, which will fit onto one line, with the preferred name first.  At the end of the table, the user may enter ‘.n’ where n is an entry number from the list to view all germplasm information for that entry, or another command or search string.  The display for the selected entry is described in <a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html#Displaying_Records_by_GID" title="TDM GMS Browse">below</a>.  The list is retained in memory until another one is formed and the command ‘.n’ will always display germplasm information for the nth entry in the last list.
</p>
<a name="Searching_for_crosses" id="Searching_for_crosses"></a><h3> <span class="mw-headline"> Searching for crosses </span></h3>
<p>The command .FCR &lt;string&gt; will search for crosses according to progenitor combinations rather than simple string matching. The string should be a valid Purdy designation which may contain wild cards. The string is recursively split into female and male parents and lists of gids for the parents are compiled by automatic name searches. BROWSE then checks the database for any cross combinations between parents from the male and female lists. The&nbsp;% and _ wild cards can be used, and also the character ‘^’ which indicates ‘any line from the specified cross name’.
</p><p>If the option <a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html#Setting_BROWSE_options" title="TDM GMS Browse">FCPS=1</a> is set and the string entered in an ordinary name search is a valid Purdy cross string without wild characters, then it will be searched for string matches and it will also be parsed and crosses matching the string will be searched according to the GIDs of the progenitors.
</p>
<a name="Displaying_Records_by_GID" id="Displaying_Records_by_GID"></a><h2> <span class="mw-headline"> Displaying Records by GID </span></h2>
<p>The command ‘.GID’ followed by a GID value displays all GMS records for that GID in the same way as selecting an item from a name search list.  If the specified GID has been replaced, the records for the replacement are displayed. If no GID value follows the command, the last selected GID will be displayed.
</p><p>The display consists of the preferred name, the source, the one-level cross expansion (if it exists) and all alternative names.  All fields in the germplasm record, all attributes, and the full location record follow this, if the germplasm location is not missing. The Generation Number of the associated germplasm is also displayed as described below.
</p>
<a name="Listing_Relatives" id="Listing_Relatives"></a><h2> <span class="mw-headline"> Listing Relatives </span></h2>
<a name="Close_Relatives" id="Close_Relatives"></a><h3> <span class="mw-headline">Close Relatives</span></h3>
<p>The command .REL &lt;gid&gt; requests BROWSE to list all the close relatives of the specified germplasm.  These include the following relatives:
a) The source and group if the germplasm is derivative
b) Its sister lines - those having the same source
c) Immediate derivatives (germplasm having the target germplasm as source).
</p>
<a name="Group_Relatives" id="Group_Relatives"></a><h3> <span class="mw-headline">Group Relatives</span></h3>
<p>The command .GRP &lt;gid&gt; requests BROWSE to list all the group relatives of the specified germplasm, ie all derivative germplasm with the same group ID as &lt;gid&gt;.
</p>
<a name="A_Derivative_Neighborhood" id="A_Derivative_Neighborhood"></a><h3> <span class="mw-headline">A Derivative Neighborhood</span></h3>
<p>The command .DNB &lt;gid&gt;,n will list all derivative relatives of germplasm &lt;gid&gt; within n derivative steps. If &lt;gid&gt; is omitted the current GID is used, if n is omitted then 3 is used. 
</p>
<a name="Showing_Multilateral_System_Ancestors" id="Showing_Multilateral_System_Ancestors"></a><h3> <span class="mw-headline"> Showing Multilateral System Ancestors </span></h3>
<p>The command .MLS &lt;gid&gt; &lt;ngen&gt; will list all MLS ancestors in the pedigree of GID=&lt;gid&gt; up to &lt;ngen&gt;  generations. If &lt;gid&gt; is missing the the current GID is used. If &lt;ngen&gt; is missing the maximum generations is used. MLS lines have an MLS_DATE attribute (UDFLD No. 223) which indicates that they have been designated as part of the Multi-lateral system for germplasm exchange.
</p>
<a name="Pedigree_Analysis" id="Pedigree_Analysis"></a><h2> <span class="mw-headline"> Pedigree Analysis </span></h2>
<a name="Presentation_of_Generation_Codes" id="Presentation_of_Generation_Codes"></a><h3> <span class="mw-headline"> Presentation of Generation Codes </span></h3>
<p>When BROWSE displays germplasm records or when the command .GCD &lt;gid&gt; is entered, the DLL function GMS_ComputeGenerationNo (6.9.4) is called and the results are formatted as follows:
</p>
<pre>&lt;CrossCode&gt;(Fn,Xj,Mk)&lt;MCode&gt;
</pre>
<p>Where
</p>
<ul><li> &lt;CrossCode&gt; is the method abbreviation (MCODE) for the last generative method or ‘Ukn’ for unknown derivative germplasm. If the last generative method is a backcross the &lt;CrossCode&gt; is followed by the backcross dose (NBC).
</li><li> Fn is ‘F’ followed by the number of derivative generations (NGEN). n=1 for generative germplasm, and n=? if the source is unknown in any generation so that the number of generations cannot be counted.
</li><li> X is ‘S’ or ‘D’ followed by a generation number.  Sj indicates that the line was last selected from a single plant in generation j. Dj indicates that chromosome doubling occurred in generation j. If neither event occurred the Xj is omitted.
</li><li> Mk indicates that the last k generations were produced by management methods designed to preserve the genetic status of the line.
</li><li> If the germplasm is derivative then &lt;MCode&gt; is the method abbreviation (MCODE) for the last generation. Otherwise &lt;MCode&gt; is not presented.
</li></ul>
<p>Examples:
</p>
<pre>GENERATION CODE: C2W(F7,S6)DSP         FOR  -49064: CROSS 100-B-1-2-3-B-1
GENERATION CODE: C2W(F6,S4)DRB         FOR  -49063: CROSS 100-B-1-2-3-B
GENERATION CODE: C2W(F4,S3)DSP         FOR  -49067: IR 80000-B-B-1
GENERATION CODE: C2W(F9,S3)DSP         FOR  -49072: IR 80000-B-B-1-B-B-B-B-B
GENERATION CODE: C2W(F1)               FOR  -49046: CROSS 100
GENERATION CODE: C2W(F2)DSB            FOR  -49047: CROSS 100-B
GENERATION CODE: C2W(F3,S2,M1)IDN      FOR  -49050: CROSS 100-B-1-25 P
GENERATION CODE: C2W(F6,D5)DDH         FOR  -49058: CROSS 100-B-1-12-4-DH 23
GENERATION CODE: C2W(F7,D5)DSP         FOR  -49059: CROSS 100-B-1-12-4-DH 23-1
GENERATION CODE: Ukn(F0,M1)ISE         FOR  318468: YUEN NAN CHAN TAO,IRGC 1234,PI 160631
GENERATION CODE: C2W(F4,S3,M1)VCR      FOR  569736: PB 64,IR 64
GENERATION CODE: Ukn(F?)UDM            FOR  -48414: KOLOSAIL
</pre>
<a name="Pedigree_Trees_and_Tables" id="Pedigree_Trees_and_Tables"></a><h3> <span class="mw-headline"> Pedigree Trees and Tables </span></h3>
<p>The command ‘.TRE’ followed by a GID and optionally a level number will display a GMS record as described  <a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html#Displaying_Records_by_GID" title="TDM GMS Browse">above</a> and the pedigree tree for the specified germplasm to the termination of the tree or the specified number of levels, whichever is less.  If no level number is specified, 3 is used, if no GID is specified, the last selected GID is displayed.
</p><p>The pedigree tree is displayed vertically (Figure 2).  Lines with the symbol ‘&gt;’ before the name indicate that the germplasm above the symbol is derived from the germplasm on the current line.  Lines starting with “+” indicate that the germplasm above the “+” is produced by a generative process with the germplasm on the current line as one of its progenitors.  Progenitors occur in the order specified by the germplasm method, so for biparental crosses, female progenitors are above male progenitors. The number of derivative steps from the germplasm on the current line to its group source is displayed to the right of the names followed by the GID and a code symbol. The symbol “*” indicates germplasm which is expanded elsewhere in the tree, ‘&lt;’ indicates germplasm which could be further expanded than the current number of levels and ‘#’ indicates germplasm with no known source.
</p>
<pre>Figure 2.  The BROWSE  Pedigree Tree
------------------------------------
PREFERRED_NAME (TYPE=RELNM):IR 64
SOURCE: IR 18348-36-3
CROSS: IR 5657-33-2-1/IR 2061-465-1-5-5
OTHER NAMES:
DRVNM   IR 18348-36-3-3
ITEST   IRTP 12158
GERMPLASM_ID  =     50533 NO_PROGENITORS =       -1 METHOD = DSP( 205)
PROGENITOR_ID1 =    24489 PROGENITOR_ID2 =    50532 GERMPLASM_USER =    10
LOCAL_GERMPLASM =  -50533 GERMPLASM_DATE = 19850000 LOCATION =        9000
GERMPLASM_REFERENCE=    1 GERMPLASM_REPLACE =     0 ATTRIBUTES:
INGER   12158 PHL 005   1IRGC 66970
LOCN  9000 IRRI-INTERNATIONAL RICE RESEARCH INSTITUTE, LOS BANOS, PHL
----------------------------------------------------------------------
 1IR 64                                                           3   50533
 5 +-IR 5657-33-2-1                                               3   24486
10 | +-IR 5236                                                    0    8180
18 | | +-IR 2006-P3-31-3                                          3    8082&lt;
19 | | +-IR 2146-68-1                                             2    8111&lt;
11 | +-IR 5338                                                    0    8307
20 |   +-IR 2061-465-1-4                                          3    8301&lt;
21 |   +-IR 2055-475-2                                            2    7326&lt;
 6 +-IR 2061-465-1-5-5                                            4   11072
16   +-IR 833-6-2-1-1                                             4    3707
26   | +-IR 262-43-8-11                                           3    1171&lt;
27   | +-GAM PAI-15                                               2     109
44   |  &gt;GAM PAI                                                 &nbsp;?  253220#
17   +-IR 2040                                                    0    3661
28     +-IR 1561-149-1                                            2    3660&lt;
29     +-IR 1737                                                  0    3006&lt; 
----------------------------------------------------------------------
</pre>
<p>The option NAMP affects the names printed on the tree.  If NAMP=0 only the preferred name is printed, if NAMP=1 all names are printed if space allows, and if NAMP=2 an accession number is printed after the preferred name if one exists (see <a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html#Setting_BROWSE_options" title="TDM GMS Browse">.OPT</a>).
</p><p>The command ‘.TAB’ followed by a GID and optionally a level number will display a GMS record as described <a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html#Displaying_Records_by_GID" title="TDM GMS Browse">above</a> and a table of progenitors for the specified germplasm as far as the termination of the tree or to the specified number of levels, whichever is less.  If no level number is specified, 3 is used, if no GID is specified, the last selected GID is displayed. The  table has columns for Child GID, Parent GID, Parent Name, Parent generation (F No), Cross Method, and an Accession Number if available.
</p>
<a name="Mendelgrams" id="Mendelgrams"></a><h3> <span class="mw-headline"> Mendelgrams </span></h3>
<p>The command “.MGM” displays a mendelgram for the target GID.  This is a table of progenitors of the target with their type, contribution, count and origin.  The TYPE is ‘T’ if the progenitor is a terminal progenitor in the pedigree tree, i.e. a landrace or progenitor with unknown pedigree. TYPE is ‘IG’ if the progenitor is intermediate and generative with known parents and is ‘ID’ if it is intermediate and derivative with known source.
</p><p>The contribution, CONT, is the probability that a randomly chosen, unselected allele in the target genotype comes from that progenitor. It is the Coefficient of Parentage between the line and the ancestor (see below).  The COUNT is the number of times a progenitor occurs in the full pedigree tree of the target and CTY is the country of origin or development of the progenitor.
</p><p>A richness index is computed which is the probability that alleles at two randomly chosen, unselected loci come from the same terminal progenitor. (This is 1.0-Sum of Squares of CONT).
</p><p>By default BROWSE lists only terminal progenitors, but if the flag MGAT is set to 1 (command <a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html#Setting_BROWSE_options" title="TDM GMS Browse">.OPT MGAT=1</a>) then all progenitors are listed. Also Browse completely expands the pedigree by default, but the command .MGM &lt;n&gt; will stop expansion after n generations and calculate the statistics as if it were the full expansion.
</p>
<a name="Traces" id="Traces"></a><h3> <span class="mw-headline"> Traces </span></h3>
<p>The command “.TCE &lt;gid&gt;,n” will trace all descendants of germplasm gid for n generations. (In this case a generation is a single step whether generative or derivative.) If n is omitted then all generations are traced.  A summary of the number and type of descendants at each generation is given, and the descendants that have accession numbers, international testing numbers or release names are listed. Care must be taken in selecting germplasm to trace because the number of descendants of an important ancestor can explode.
</p><p>The full set of descendants is stored in a TRACE list in the LISTDATA table. The ENTRYCODE field contains a string indicating several properties of the current germplasm: The number of derivative steps is in column 1 (degree of inbreeding – 1 for generative, 2 for F2 … and * for unknown).  Column 2 contains ‘A’ if there is an accession number associated with the current germplasm, column 3 contains ‘I’ if there is an International Testing number, column 4 contains ‘R’ if there is a release name. Column 5 contains ‘P’, ‘S’, ‘B’ or ‘T’ to indicate whether the current germplasm is a parent, source, both or neither (terminal) in the next generation. If the role of the current germplasm in the next generation is not determined then column 5 contains ‘?’.  Columns 6-10 contain the generation number of the current germplasm. The SOURCE field of the LISTDATA table contains the country of origin of the current germplasm, and the DESIG field contains the preferred name and the specific names indicated in columns 2-4 of ENTRYCD. The full list can be queried, displayed and sorted in various ways with DBMS specific tools or with SETGEN.
</p>
<a name="Coefficients_of_Parentage" id="Coefficients_of_Parentage"></a><h3> <span class="mw-headline"> Coefficients of Parentage </span></h3>
<p>GMS can trace genotypes back to their parental landraces or to lines of unknown pedigree.  Genealogical features of a set of germplasm can be exploited through statistics such as the Coefficient of Parentage (COP).  Originally developed by Wright (1922), the COP for two genotypes estimates the expected percentage of alleles identical by descent at loci which are polymorphic within a population.  The COP between two lines is the inbreeding coefficient of their offspring. St. Martin (1982) adapted COP analysis to inbred crops by assuming that each genotype is completely homozygous, that genotypes without common parentage are unrelated and that parents contribute equally to the offspring, despite inbreeding and selection.  Highly selected, qualitative loci would not follow the assumption of random transmission of a large number of alleles.  However, in a general sense, a COP is a measure of overall common ancestry of two genotypes and estimates latent genomic diversity that is not obvious until a crop is challenged by the appropriate biotic or abiotic stress.  Breeders can use COPs to increase genetic diversity by selecting lines for crossing with divergent parentage.
</p><p>However, please heed the warning that SPURIOUS LOW COP VALUES THAT REFLECT INCOMPLETE PEDIGREE INFORMATION RATHER THAN A TRUE LOW LEVEL OF COANCESTRY MAY BE GENERATED. If the highest theoretical contribution (HTC) of any ancestor is large (say larger that .1) then this may indicate incomplete pedigree information. The HTC corresponds to the first and largest value of the mendelgram and a value of 0.5 usually reflects that one immediate parent is non-expandable, with 0.25 indicating one unexpandable grandparent.
</p><p>The command '.COP &lt;gid1&gt;, &lt;gid2&gt;' will calculate the coefficient of parentage between germplasm gid1 and gid2.  The algorithm followed is described in Sneller, C.H. (Crop Sci. 34:1679-1680 1994). The <a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html#Setting_BROWSE_options" title="TDM GMS Browse">option</a> BTYPE indicates the breeding method: BTYPE=1 for self fertilizing crops (SF) and 0 for cross pollinating crops (CP). Original sources are assumed unrelated and homozygous in SF crops or F1s in CP crops. The generation where sister lines diverge is retrieved from the database, but where this information is unknown they are assumed to be identical for SF crops or to diverge at F1 for CP crops. <a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html#Using_BROWSE_in_Batch_Mode" title="TDM GMS Browse">Batch</a> mode or a list version as described below can be used to calculate multiple COP values.
</p><p>If a germplasm list is active in BROWSE then the command .COP LST &lt;lvl&gt; extracts and the combined pedigree tree up to &lt;lvl&gt; generative generations for all entries in the open list and calculates COP values for all pairwise combinations of germplasm in the list using the available levels. If &lt;lvl&gt; is absent then the tree for each entry is expanded for all generations and the COP values computed using all generations. A list becomes active in BROWSE after a <a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html#The_.LST_command" title="TDM GMS Browse">.LST</a> command or a <a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html#The_.SAV_command" title="TDM GMS Browse">.SAV</a> command or after the command <a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html#Setting_BROWSE_options" title="TDM GMS Browse">.OPT LIST=n</a>.
</p><p>.COP LST displays and prints the COP values for all pairs in the list ordered as the lower triangular part of the COP matrix by rows. It also prints to BROWSE.LOG the lower triangular part of the COP matrix as well as its inverse, or G-inverse if it is singular, by rows in sections of ten columns. Singularity is declared if eigen values less than 1.0e-9 are encountered. The eigen values are printed to BROWSE.LOG, as well as a GIV serial format of the inverse matrix suitable for use by ASREML software.
</p><p>References and Further Reading on Coefficients of Parentage
</p>
<ul><li> Braun, H.-J., Pfeiffer, W.H., and Pollmer, W.G. 1992. Environments for selecting widely adapted spring wheat. Crop Science 32:1420-1427.
</li><li> Barrett, B.A., Kidwell, K.K. and Fox, P.N. 1998. Comparison of  AFLP and pedigree-based genetic diversity assessment methods using wheat cultivars from the Pacific Northwest. Crop Science 38:1271-1278.
</li><li> Brennan, J.P. and Fox, P.N. 1998. Impact of CIMMYT varieties on the genetic diversity of wheat in Australia. Australian Journal of  Agricultural Research 49:175-178.
</li><li> Fox, P.N. and Skovmand, B. 1996. The International Crop Information System (ICIS) - connects genebank to breeder to farmer's field. In Cooper, M. and  Hammer G.L.  eds. Plant Adaptation and Crop Improvement. CAB International.
</li><li> Jain, K.B.L. 1994. Wheat cultivars in India: names, pedigrees, origins, and adaptations. Research Bulletin No. 2, 72 pp., Directorate of Wheat Research, Karnal, India.
</li><li> Smale, M. and McBride, T. 1996. Understanding global trends in the use of wheat diversity and international flows of genetic resources. Part 1 of CIMMYT 1995/96 World Wheat Facts and Trends: understanding global trends in the use of wheat diversity and international flows of wheat genetic resources. Mexico, D.F.: CIMMYT.
</li><li> Souza, E., Fox, P.N., Byerlee, D., and Skovmand, B. 1994. Spring wheat diversity in irrigated areas of two developing countries. Crop Science 34:774-783.
</li><li> Souza, E., Fox, P.N. and Skovmand, B. 1998. Parentage analysis of International Spring Wheat Yield Nurseries 17 to 27. Crop Science 38:337-341.
</li><li> St. Martin, S.K. 1982. Effective population size for the soybean improvement program in maturity groups 00 to IV. Crop Science 22:151-152.
</li><li> Thomas, N. 1996. Use of IARC germplasm in Canadian crop breeding programs: Spillovers to Canada from the CGIAR and some outflows from Canada. Ottawa: CIDA.
</li><li> van Beuningen, L.T. and Busch, R.H. 1977. Genetic diversity among North American spring wheat cultivars. I: Analysis of the coefficient of parentage matrix. Crop Science 37:564.
</li><li> Wright, S. 1922. Coefficients of inbreeding and relationship. American Naturalist 56:330-338.
</li></ul>
<a name="Changing_and_Correcting_Germplasm_Records" id="Changing_and_Correcting_Germplasm_Records"></a><h2> <span class="mw-headline"> Changing and Correcting Germplasm Records </span></h2>
<a name="Deleting_or_Replacing_Germplasm" id="Deleting_or_Replacing_Germplasm"></a><h3> <span class="mw-headline"> Deleting or Replacing Germplasm </span></h3>
<p>The command ‘.REP’ followed by three parameters, GIDX, GIDY and REASON requests BROWSE to replace germplasm with GIDX by germplasm GIDY and add the REASON to the replace attribute.  REASON may be null, and if GIDY is null or zero, then GIDX is deleted.  The algorithms in 3.9.7 and 3.9.8 are followed.  If GIDX is negative, the dependencies and the CHANGE field are changed immediately and the REPLACE attribute is stored.  Otherwise, local dependencies are changed, central dependencies are listed and the REPLACE attribute is stored on the local ATTRIBUTES table to be processed when the central GMS is updated.
</p>
<a name="Setting_Germplasm_or_Name_Fields" id="Setting_Germplasm_or_Name_Fields"></a><h3> <span class="mw-headline"> Setting Germplasm or Name Fields </span></h3>
<p>Any field of a germplasm record can be set with command .SET &lt;Field&gt;=Value IN y, where &lt;Field&gt; can be GPID1, GPID2, GLOCN, GDATE, METHN or GREFN. Change records are added to the <a href="../../../../articles/t/d/m/TDM_Genealogy_Management_System_d3f7.html#CORRECTING_RECORDS_IN_THE_GMS_DATABASE" title="TDM Genealogy Management System">CHANGES table</a> for the target GID (y) If the germplasm is local, y&lt;0, the actual field is also changed. If the target germplasm is central, only the change records are added to the CHANGES table, but DLL functions check for, and apply changes at run-time so this allows local users to make effective immediate changes to the central database.
</p><p>Progenitor numbers for existing central or local germplasm can be set or changed with the commands .SET GPID1=x IN y and .SET GPID2=x IN y.  x and y are valid central or local GIDs, or x may be zero. The new value, x, may be local even when the target, y, is central. If ‘IN y’ is missing the last selected GID is assumed.  (Note: The syntax and spacing of the SET command is critical.) Other GERPLSM fields and NAME fields can be edited in the same way. The full syntax of the .SET command is:
</p>
<pre>.SET fld=xx IN nn
</pre>
<p>Where:
</p>
<ul><li> ‘fld’ is GPIDn (n is 1, 2...3 etc), GLOCN, GDATE, METHN, GREFN, GNPGS,NDATE, NLOCN, NREFN, NTYPE OR NSTAT,
</li><li> ‘xx’  is the new value for the specified field and
</li><li> ‘nn’  is the GID for GERMPLSM fields or NID for NAME fields.
</li></ul>
<a name="Adding_Notes_and_Names" id="Adding_Notes_and_Names"></a><h3> <span class="mw-headline">Adding Notes and Names</span></h3>
<p>Note attributes can be added with the command: 
</p>
<pre>.NOTE &lt;gid&gt;,&lt;avalue&gt;. 
</pre>
<p>If &lt;gid&gt;, is missing the note is added to the current germplasm (&lt;avalue&gt; may not start with a number in this case).
</p><p>Names can be added for any germplasm with the command:
</p>
<pre>.NAME &lt;gid&gt;,&lt;ntype&gt;,&lt;nlocn&gt;,&lt;ndate&gt;,&lt;nref&gt;,&lt;nvalue&gt;
</pre>
<p>IF &lt;gid&gt; is zero or null, the name is added to the current germplasm.
</p><p><br />
</p>
<a name="Working_with_Lists_in_BROWSE" id="Working_with_Lists_in_BROWSE"></a><h2> <span class="mw-headline"> Working with Lists in BROWSE </span></h2>
<a name="Opening_existing_lists" id="Opening_existing_lists"></a><h3> <span class="mw-headline"> Opening existing lists </span></h3>
<p>Existing germplasm lists in GMS can be opened with the command <a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html#Setting_BROWSE_options" title="TDM GMS Browse">.OPT LIST=n</a> where n is the LISTID of the list to be opened. .OPT LIST displays all the lists available. 
</p>
<a name="Adding_entries_to_a_list" id="Adding_entries_to_a_list"></a><h3> <span class="mw-headline"> Adding entries to a list </span></h3>
<p>The command .LST  &lt;gid&gt;,&lt;code&gt;,&lt;source&gt;,&lt;desig&gt; will add germplasm with GID=&lt;gid&gt; to the currently active list. (If no list is active, a new one with name BROWSEn is opened where n is the next available number). Any of the fields in the .LST command may be null and defaults will be used. The default for &lt;desig&gt; is the preferred name if option NAMP=0 or all names if NAMP=1. If &lt;gid&gt; is null the current GID is saved.
</p>
<a name="Saving_a_displayed_table_as_a_list" id="Saving_a_displayed_table_as_a_list"></a><h3> <span class="mw-headline"> Saving a displayed table as a list </span></h3>
<p>Many processes in Browse such as name searches, searches for relatives or tree displays result in numbered lists appearing on the screen. All entries in the last list displayed at any point can be saved to the active germplasm list by entering the command .SAV. If no list is active, a new one will be created (with name SAVEn). An existing list can be opened with the LIST <a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html#Setting_BROWSE_options" title="TDM GMS Browse">option</a>. If the list being saved is a tree then the tree information is also saved and can be viewed in the Group Name and Entry Code fields of the list (in SETGEN for example). The preferred name is written to the DESIG field of the list unless option NAMP is set to 1 in which case all names are stored.
</p>
<a name="Displaying_the_entries_in_an_open_list" id="Displaying_the_entries_in_an_open_list"></a><h3> <span class="mw-headline"> Displaying the entries in an open list </span></h3>
<p>The .DSL &lt;listid&gt; command displays list entries. If &lt;listid&gt; is null, entries for the current open list are displayed, otherwise list &lt;listid&gt; is opened and entries displayed.
</p>
<a name="Exporting_the_combined_pedigree_tree_for_all_entries_in_a_list" id="Exporting_the_combined_pedigree_tree_for_all_entries_in_a_list"></a><h3> <span class="mw-headline"> Exporting the combined pedigree tree for all entries in a list </span></h3>
<p>The command .EXT LST &lt;lvl&gt; extracts and exports the combined pedigree tree up to &lt;lvl&gt; generative generations for all entries in an open list to the LOG file. If &lt;lvl&gt; is absent then the tree for each entry is expanded for all generations. A list becomes open after a <a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html#The_.LST_command" title="TDM GMS Browse">.LST</a> command or a <a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html#The_.SAV_command" title="TDM GMS Browse">.SAV</a> or the <a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html#Setting_BROWSE_options" title="TDM GMS Browse">.OPT LIST=n</a> command. The structure of the tree is described in the output.
</p>
<a name="Searching_lists_for_a_specified_germplasm" id="Searching_lists_for_a_specified_germplasm"></a><h3> <span class="mw-headline"> Searching lists for a specified germplasm </span></h3>
<p>The command .SLS &lt;gid&gt; searches all lists for entries with GID=&lt;gid&gt; and lists them in a table showing list and entry identification.
</p>
<a name="Using_BROWSE_in_Batch_Mode" id="Using_BROWSE_in_Batch_Mode"></a><h2> <span class="mw-headline"> Using BROWSE in Batch Mode </span></h2>
<p>If you prepare a text file with a list of names, one to a line, and then give the command “.INP &lt;path and file name&gt;"at the BROWSE prompt, it will search the names one after the other, and the user can select the germplasm from the hit list in the usual way (enter .n where n is the number of the desired germplasm in the list).  You can also put any BROWSE commands in an input file such as .TRE &lt;gid&gt; or .GID &lt;gid&gt; or .COP &lt;gid1&gt; &lt;gid2&gt; etc. When searching a long list of names a useful command to put at the top of the list is .OPT IOPT=1 which causes BROWSE to accept a single hit in a name search without being prompted. The user must still select from multiple hits. (the command .OPT IOPT=0 at the end of the file of names re-sets the default).  Other options affecting searching and selecting entries are described with the <a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html#Setting_BROWSE_options" title="TDM GMS Browse">.OPT</a> command.  The most useful output from BROWSE when used in batch mode is the log file BROWSE.LOG which is always produced in the default BROWSE directory.
</p>
<a name="Setting_BROWSE_options" id="Setting_BROWSE_options"></a><h2> <span class="mw-headline"> Setting BROWSE options </span></h2>
<p>Browse also has several options which can be set with keys in the [Browse] section of the INI file or with the .OPT command at run time.  These are REFID, LOCID, IOPT, ISRH, MGAT, NAMP, NTCX and LIST.  REFID and LOCID can be used to set default location and reference identifiers.  If these are not present in the INI file, zero values are assigned. IOPT and ISRH affect which germplasm are shown and selected in search lists. MGAT affects the <a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html#Mendelgrams" title="TDM GMS Browse">Mendlegram</a> display and NAMP the <a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html#Pedigree_Trees_and_Tables" title="TDM GMS Browse">Tree</a> display. NTCX affects the cross expansion, causing it to terminate on any germplasm with a name of the type specified by its value. LIST allows BROWSE to open an existing germplasm list for adding entries or continuing a trace.
</p>
<pre>SET OPTIONS WITH COMMAND: .OPT &lt;option&gt;=x
</pre>
<pre>IF &lt;option&gt; IS REFID, SET DEFAULT REFID TO x, ELSE x=0
       IF &lt;option&gt; IS LOCID, SET DEFAULT LOCATION TO x, ELSE x=0
IF &lt;option&gt; IS IOPT, x=1 TO SELECT SINGLE SEARCH HITS,
    x=2 NO SELECTION, HIT LIST LOGED TO BROWSE.LOG (ELSE 0)
IF &lt;option&gt; IS ISRH, x=1 TO SHOW ONLY GENERATIVE GERMPLASM,
    x=2 TO SHOW ONLY DERIVATIVES AND X=3 TO EXCLUDE ACCESSIONS,
    x=4 TO SHOW ONLY GERMPLSM WITH ACCESSION NAMES (ELSE 0)
IF &lt;option&gt; IS MGAT, x=1 TO SHOW ALL PROGENITORS IN THE MENDLEGRAM (ELSE 0)
IF &lt;option&gt; IS NAMP, x=0 FOR PREFERRED NAME, 1 FOR ALL OR 2 FOR ACCESSION NO
IF &lt;option&gt; IS LDST, x=1 TO DISPLAY DERIVATIVE STEPS IN A TREE, ELSE x=0 
IF &lt;option&gt; IS NTCX, x=n TO STOP CROSS EXPANSION ON NAME TYPE n, ELSE x=0
IF &lt;option&gt; IS LIST, x=0 TO CLOSE LIST OR x=n TO OPEN LIST n
IF &lt;option&gt; IS REFID, SET DEFAULT REFID TO x, ELSE x=0
IF &lt;option&gt; IS LOCID, SET DEFAULT LOCID TO x, ELSE x=0
IF &lt;option&gt; IS FCPS, x=1 TO ALWAYS LOOK FOR CROSSES BY PARENTS, ELSE x=0
IF &lt;option&gt; IS BTYPE, x=1 FOR SELF FERTILIZED CROP, 0 FOR CROSS POLINATED
IF &lt;option&gt; IS IYAL, x=1 to accept changes during edit processes without confirmation, else 0.
</pre>
<!-- 
NewPP limit report
Preprocessor node count: 46/1000000
Post-expand include size: 0/2097152 bytes
Template argument size: 0/2097152 bytes
Expensive parser function count: 0/100
-->
<div class="printfooter">
Retrieved from "<a href="http://localhost../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html">http://localhost../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html</a>"</div>
						<!-- end content -->
						<div class="visualClear"></div>
		</div>
	</div>
		</div>
		<div id="column-one">
	<div id="p-cactions" class="portlet">
		<h5>Views</h5>
		<div class="pBody">
			<ul>
	
				 <li id="ca-nstab-main" class="selected"><a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html" title="View the content page [c]" accesskey="c">Page</a></li>
				 <li id="ca-talk" class="new"><a href="../../../../articles/t/d/m/Talk%7ETDM_GMS_Browse_54e3.html" title="Discussion about the content page [t]" accesskey="t">Discussion</a></li>
				 <li id="ca-viewsource"><a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html" title="This page is protected.&#10;You can view its source [e]" accesskey="e">View source</a></li>
				 <li id="ca-history"><a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html" title="Past versions of this page [h]" accesskey="h">History</a></li>			</ul>
		</div>
	</div>
	<div class="portlet" id="p-personal">
		<h5>Personal tools</h5>
		<div class="pBody">
			<ul>
				<li id="pt-login"><a href="../../../../articles/u/s/e/Special%7EUserLogin_94cd.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</a></li>
			</ul>
		</div>
	</div>
	<div class="portlet" id="p-logo">
		<a style="background-image: url(../../../../misc/icis-wiki3.png);" href="../../../../index.html" title="Visit the main page [z]" accesskey="z"></a>
	</div>
	<script type="text/javascript"> if (window.isMSIE55) fixalpha(); </script>
	<div class='generated-sidebar portlet' id='p-navigation'>
		<h5>Navigation</h5>
		<div class='pBody'>
			<ul>
				<li id="n-mainpage"><a href="../../../../index.html" title="Visit the main page [z]" accesskey="z">Main Page</a></li>
				<li id="n-portal"><a href="../../../../articles/c/o/m/ICISWiki%7ECommunity_Portal_7449.html" title="About the project, what you can do, where to find things">Community portal</a></li>
				<li id="n-recentchanges"><a href="../../../../articles/r/e/c/Special%7ERecentChanges_e0d0.html" title="The list of recent changes in the wiki [r]" accesskey="r">Recent changes</a></li>
				<li id="n-help"><a href="../../../../articles/c/o/n/Help%7EContents_22de.html" title="The place to find out">Help</a></li>
			</ul>
		</div>
	</div>
	<div id="p-search" class="portlet">
		<h5><label for="searchInput">Search</label></h5>
		<div id="searchBody" class="pBody">
			<form action="../../../../articles/s/e/a/Special%7ESearch_a8e5.html" id="searchform"><div>
				<input id="searchInput" name="search" type="text" title="Search ICISWiki [f]" accesskey="f" value="" />
				<input type='submit' name="go" class="searchButton" id="searchGoButton"	value="Go" title="Go to a page with this exact name if exists" />&nbsp;
				<input type='submit' name="fulltext" class="searchButton" id="mw-searchButton" value="Search" title="Search the pages for this text" />
			</div></form>
		</div>
	</div>
	<div class="portlet" id="p-tb">
		<h5>Toolbox</h5>
		<div class="pBody">
			<ul>
				<li id="t-whatlinkshere"><a href="../../../../articles/w/h/a/Special%7EWhatLinksHere_TDM_GMS_Browse_1601.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</a></li>
				<li id="t-recentchangeslinked"><a href="../../../../articles/r/e/c/Special%7ERecentChangesLinked_TDM_GMS_Browse_13b0.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</a></li>
<li id="t-specialpages"><a href="../../../../articles/s/p/e/Special%7ESpecialPages_f12b.html" title="List of all special pages [q]" accesskey="q">Special pages</a></li>
				<li id="t-print"><a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html" title="Printable version of this page [p]" accesskey="p">Printable version</a></li>				<li id="t-permalink"><a href="../../../../articles/t/d/m/TDM_GMS_Browse_53fa.html" title="Permanent link to this version of the page">Permanent link</a></li>			</ul>
		</div>
	</div>
		</div><!-- end of the left (by default at least) column -->
			<div class="visualClear"></div>
			<div id="footer">
				<div id="f-poweredbyico"><a href="http://www.mediawiki.org/"><img src="../../../../skins/common/images/poweredby_mediawiki_88x31.png" alt="Powered by MediaWiki" /></a></div>
				<div id="f-copyrightico"><a href="../../../../../COPYING.html"><img src="/icis/skins/common/images/gnu-fdl.png" alt='GNU Free Documentation License 1.2' /></a></div>
			<ul id="f-list">
					<li id="credits">This page was last modified 08:21, 13 February 2008 by <a href="../../../../articles/g/m/c/User%7EGmclaren_b845.html" title="User:Gmclaren">Graham McLaren</a>. Based on work by <a href="../../../../articles/s/f/r/User%7ESfrade_e1c2.html" title="User:Sfrade">Sebastien Frade</a>&#32;and&#32;<a href="../../../../articles/t/m/e/User%7ETmetz_c741.html" title="User:Tmetz">Thomas Metz</a>.</li>
					<li id="copyright">Content is available under <a href="../../../../../COPYING.html" class="external " title="../../../../../COPYING.html">GNU Free Documentation License 1.2</a>.</li>
					<li id="privacy"><a href="../../../../articles/p/r/i/ICISWiki%7EPrivacy_policy_0cbf.html" title="ICISWiki:Privacy policy">Privacy policy</a></li>
					<li id="about"><a href="../../../../articles/a/b/o/ICISWiki%7EAbout_53aa.html" title="ICISWiki:About">About ICISWiki</a></li>
					<li id="disclaimer"><a href="../../../../articles/g/e/n/ICISWiki%7EGeneral_disclaimer_9129.html" title="ICISWiki:General disclaimer">Disclaimers</a></li>
			</ul>
		</div>
</div>

		<script type="text/javascript">if (window.runOnloadHook) runOnloadHook();</script>
<!-- Served in 34.924 secs. --></body></html>
