<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
	<head>
		<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
		<meta http-equiv="Content-Style-Type" content="text/css" />
		<meta name="generator" content="MediaWiki 1.14.0" />
		<meta name="keywords" content="ICIS Workshop 2007,ICISWiki,Testing of PostgreSQL as database backend for ICIS,Workshops and Meetings,Aportugal,CasperNUN,Cpimentel,Gmclaren,Jalarcon,Jnorgaard,Lsison" />
		<link rel="shortcut icon" href="../../../../misc/favicon.ico" />
		<link rel="search" type="application/opensearchdescription+xml" href="../../../../../opensearch_desc.php" title="ICISWiki (en)" />
		<link title="Creative Commons" type="application/rdf+xml" href="../../../../articles/i/c/i/ICIS_Workshop_2007_82fe.html" rel="alternate meta" />
		<link rel="copyright" href="../../../../../COPYING.html" />
		<link rel="alternate" type="application/rss+xml" title="ICISWiki RSS Feed" href="http://localhost../../../../articles/r/e/c/Special%7ERecentChanges_e0d0.html" />
		<link rel="alternate" type="application/atom+xml" title="ICISWiki Atom Feed" href="http://localhost../../../../articles/r/e/c/Special%7ERecentChanges_e0d0.html" />
		<title>ICIS Workshop 2007 - ICISWiki</title>
		<link rel="stylesheet" href="../../../../skins/common/shared.css?195" type="text/css" media="screen" />
		<link rel="stylesheet" href="../../../../skins/common/commonPrint.css?195" type="text/css" media="print" />
		<link rel="stylesheet" href="../../../../skins/monobook/main.css?195" type="text/css" media="screen" />
		<!--[if lt IE 5.5000]><link rel="stylesheet" href="../../../../skins/monobook/IE50Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<!--[if IE 5.5000]><link rel="stylesheet" href="../../../../skins/monobook/IE55Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<!--[if IE 6]><link rel="stylesheet" href="../../../../skins/monobook/IE60Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<!--[if IE 7]><link rel="stylesheet" href="../../../../skins/monobook/IE70Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<link rel="stylesheet" href="../../../../skins/../../../../raw/MediaWiki%7ECommon.css?195" type="text/css" />
		<link rel="stylesheet" href="../../../../skins/../../../../raw/MediaWiki%7EPrint.css?195" type="text/css" media="print" />
		<link rel="stylesheet" href="../../../../skins/../../../../raw/MediaWiki%7EMonobook.css?195" type="text/css" />
		<link rel="stylesheet" href="../../../../skins/../../../../raw/gen.css?195" type="text/css" />
		<!--[if lt IE 7]><script type="text/javascript" src="../../../../skins/common/IEFixes.js?195"></script>
		<meta http-equiv="imagetoolbar" content="no" /><![endif]-->

		<script type= "text/javascript">/*<![CDATA[*/
		var skin = "monobook";
		var stylepath = "../../../../skins";
		var wgArticlePath = "../../../../$1";
		var wgScriptPath = "../../../../..";
		var wgScript = "/icis/index.php";
		var wgVariantArticlePath = false;
		var wgActionPaths = {};
		var wgServer = "http://localhost";
		var wgCanonicalNamespace = "";
		var wgCanonicalSpecialPageName = false;
		var wgNamespaceNumber = 0;
		var wgPageName = "ICIS_Workshop_2007";
		var wgTitle = "ICIS Workshop 2007";
		var wgAction = "view";
		var wgArticleId = "2202";
		var wgIsArticle = true;
		var wgUserName = null;
		var wgUserGroups = null;
		var wgUserLanguage = "en";
		var wgContentLanguage = "en";
		var wgBreakFrames = false;
		var wgCurRevisionId = "15772";
		var wgVersion = "1.14.0";
		var wgEnableAPI = true;
		var wgEnableWriteAPI = true;
		var wgSeparatorTransformTable = ["", ""];
		var wgDigitTransformTable = ["", ""];
		var wgRestrictionEdit = [];
		var wgRestrictionMove = [];
		/*]]>*/</script>

		<script type="text/javascript" src="../../../../skins/common/wikibits.js?195"><!-- wikibits js --></script>
		<!-- Head Scripts -->
		<script type="text/javascript" src="../../../../raw/gen.js"><!-- site js --></script>
	</head>
<body class="mediawiki ltr ns-0 ns-subject page-ICIS_Workshop_2007 skin-monobook">
	<div id="globalWrapper">
		<div id="column-content">
	<div id="content">
		<a name="top" id="top"></a>
				<h1 id="firstHeading" class="firstHeading">ICIS Workshop 2007</h1>
		<div id="bodyContent">
			<h3 id="siteSub">From ICISWiki</h3>
			<div id="contentSub"></div>
									<div id="jump-to-nav">Jump to: <a href="#column-one">navigation</a>, <a href="#searchInput">search</a></div>			<!-- start content -->
			<p><b><a href="../../../../articles/i/c/i/ICISWiki_eda0.html" title="ICISWiki">Back to Main Page</a> &gt; <a href="../../../../articles/w/o/r/Workshops_and_Meetings_11a5.html" title="Workshops and Meetings">Workshops and Meetings</a></b> 
</p>
<div class="center"><div class="floatnone"><a href="../../../../articles/a/a/f/File%7EAAFC_Logo.jpg_d2b2.html" class="image" title="AAFC Logo.jpg"><img alt="" src="../../../../images/0/07/AAFC_Logo.jpg" width="530" height="47" border="0" /></a></div></div><br /> <div class="center"><div class="floatnone"><a href="../../../../articles/s/p/a/File%7ESPARC.JPG_494f.html" class="image" title="SPARC, Swift Current, Saskatchewan"><img alt="SPARC, Swift Current, Saskatchewan" src="../../../../images/0/00/SPARC.JPG" width="373" height="234" border="0" /></a></div></div> 
<p><b><a href="../../../../articles/i/c/i/ICISWiki_eda0.html" title="ICISWiki">Back to Main Page</a> &gt; <a href="../../../../articles/w/o/r/Workshops_and_Meetings_11a5.html" title="Workshops and Meetings">Workshops and Meetings</a></b> 
</p><p><br /> 
</p>
<table id="toc" class="toc" summary="Contents"><tr><td><div id="toctitle"><h2>Contents</h2></div>
<ul>
<li class="toclevel-1"><a href="#ICIS_Developers.27_Workshop_2007"><span class="tocnumber">1</span> <span class="toctext">ICIS Developers' Workshop 2007</span></a></li>
<li class="toclevel-1"><a href="#Agenda_and_Minutes"><span class="tocnumber">2</span> <span class="toctext">Agenda and Minutes</span></a></li>
<li class="toclevel-1"><a href="#Monday.2C_June_4"><span class="tocnumber">3</span> <span class="toctext">Monday, June 4</span></a></li>
<li class="toclevel-1"><a href="#Tuesday.2C_June_5"><span class="tocnumber">4</span> <span class="toctext">Tuesday, June 5</span></a></li>
<li class="toclevel-1"><a href="#Wednesday.2C_June_6"><span class="tocnumber">5</span> <span class="toctext">Wednesday, June 6</span></a></li>
<li class="toclevel-1"><a href="#Thursday.2C_June_7"><span class="tocnumber">6</span> <span class="toctext">Thursday, June 7</span></a></li>
<li class="toclevel-1"><a href="#Friday.2C_June_8"><span class="tocnumber">7</span> <span class="toctext">Friday, June 8</span></a></li>
<li class="toclevel-1"><a href="#Registered_Participants"><span class="tocnumber">8</span> <span class="toctext">Registered Participants</span></a></li>
</ul>
</td></tr></table><script type="text/javascript"> if (window.showTocToggle) { var tocShowText = "show"; var tocHideText = "hide"; showTocToggle(); } </script>
<a name="ICIS_Developers.27_Workshop_2007" id="ICIS_Developers.27_Workshop_2007"></a><h3> <span class="mw-headline"> <center><b>ICIS Developers' Workshop 2007</b></center>  </span></h3>
<p><br /> 
</p>
<center><b>June 4 - 8, 2007</b><br /> 
<p><b>Semiarid Prairie Agricultural Research Centre (SPARC)</b><br /> <b>Swift Current, Saskatchewan</b><br /> 
</p>
<b>Canada</b></center> 
<p><br /> 
</p>
<center><b>Registration Deadline is April 16, 2007</b></center> <center>Download the Registration Form here:</center> <center><a href="../../../../images/0/00/ICIS_2007_Registration.doc" class="internal" title="ICIS 2007 Registration.doc">ICIS 2007 Registration Form (.doc)</a></center> 
<p><br /> 
</p>
<center>For those who plan to travel either before or after the workshop, here is a list of some tourist attractions in Western Canada:</center> <center><a href="../../../../images/f/f8/Places_to_visit_in_Western_Canada.doc" class="internal" title="Places to visit in Western Canada.doc">Places to visit in Western Canada(.doc)</a></center> 
<a name="Agenda_and_Minutes" id="Agenda_and_Minutes"></a><h3> <span class="mw-headline"> Agenda and Minutes  </span></h3>
<a name="Monday.2C_June_4" id="Monday.2C_June_4"></a><h2> <span class="mw-headline"> <b>Monday, June 4 </b>  </span></h2>
<dl><dd><b>1. Workshop Welcome</b> 8:00 - 8:15 (Fran and Graham)
</dd></dl>
<dl><dd><b>2. Activity Reports of ICIS Partners</b> 8:15 - 9:45 (Chair: Shawn, Reporter: Casper) 
<dl><dd>Clicking on the names below will show a PDF-file with activities per participant over the last year. 
</dd><dd>- <a href="../../../../images/0/07/A_ICIS_2007_-_SPARC_Update.pdf" class="internal" title="A ICIS 2007 - SPARC Update.pdf">SPARC (Shawn)</a> 
</dd><dd>- <a href="../../../../images/4/42/C_ProgressAtCIMMYT07.pdf" class="internal" title="C ProgressAtCIMMYT07.pdf">CRIL-CIMMYT (Graham)</a> 
</dd><dd>- <a href="../../../../images/1/19/B_ICIS_Implementation_in_IRRI.pdf" class="internal" title="B ICIS Implementation in IRRI.pdf">CRIL-IRRI (Arllet)</a> 
</dd><dd>- Nunhems (Casper) 
</dd><dd>- <a href="../../../../images/9/97/E_ICIS_Activities_Bayer.pdf" class="internal" title="E ICIS Activities Bayer.pdf">Bayer (Sebastien)</a> 
</dd><dd>- <a href="../../../../images/a/a3/F_UQ_ActivityReport_ICIS2007.pdf" class="internal" title="F UQ ActivityReport ICIS2007.pdf">University of Queensland (Sandra)</a> 
</dd><dd>- <a href="../../../../images/1/1f/G_ICIS_ATFCC_2007.pdf" class="internal" title="G ICIS ATFCC 2007.pdf">Australian Temperate Field Crops Collection (Selwyn)</a>
</dd></dl>
</dd></dl>
<p><br /> 
</p>
<dl><dd><b>3. Improvements of Existing ICIS modules</b> 10:00 - 4:00 (Chair: Casper, Reporter: Jesper) 
<dl><dd>- <a href="../../../../images/1/1f/Ab_ICIS_5-4_Release.pdf" class="internal" title="Ab ICIS 5-4 Release.pdf">Overview of the ICIS 5.4, ICIS Launcher and ICIS Help (Arllet)</a> 
</dd><dd>- <a href="../../../../images/d/dd/Setgen541.pdf" class="internal" title="Setgen541.pdf">SetGen (Clarissa [Candy])</a> 
</dd><dd>- <a href="../../../../images/5/5b/D_COP_Graham.pdf" class="internal" title="D COP Graham.pdf">Coefficient of Parentage in BROWSE (Graham)</a> 
</dd><dd>- <a href="../../../../images/5/57/GMS_SEARCH_modified.pdf" class="internal" title="GMS SEARCH modified.pdf">GMS Search (Lilibeth)</a> 
</dd><dd>- <a href="../../../../images/e/e9/IMS_modified.pdf" class="internal" title="IMS modified.pdf">InTrack (Lilibeth)</a> 
</dd><dd>- <a href="../../../../images/a/a2/G1_FieldbookMay06.pdf" class="internal" title="G1 FieldbookMay06.pdf">Fieldbook and MaizeFinder (Vivek)</a> 
</dd><dd>- <a href="../../../../images/1/1f/G2_GMSInput4Jun07.pdf" class="internal" title="G2 GMSInput4Jun07.pdf">GMSInput (Vivek)</a> 
</dd><dd>- <a href="../../../../images/f/f5/ICISWorkbook.pdf" class="internal" title="ICISWorkbook.pdf">DMS Workbook (Warren)</a> 
</dd><dd>- DMS Retriever (Arllet) 
</dd><dd>- Nunhems Retriever (Casper) 
</dd><dd>- <a href="../../../../images/4/4a/Data_Validation_Tool_v5.4.pdf" class="internal" title="Data Validation Tool v5.4.pdf">Data Validation Tool (Corina [Ching])</a> 
</dd><dd>- <a href="../../../../images/7/79/Installation_Diagnostic_Tool_v5.4.pdf" class="internal" title="Installation Diagnostic Tool v5.4.pdf">Installation Diagnostic Tool (Corina [Ching])</a> 
</dd><dd>- <a href="../../../../images/4/44/UseOfNSIS_installations_inICIS.pdf" class="internal" title="UseOfNSIS installations inICIS.pdf">ICIS Installer - NSIS (Jesper)</a> 
<dl><dd>See the NSIS script source code <a href="../../../../images/7/7c/IWIS3-2007-05-30.nsi" class="internal" title="IWIS3-2007-05-30.nsi">here</a>
</dd></dl>
</dd></dl>
</dd></dl>
<a name="Tuesday.2C_June_5" id="Tuesday.2C_June_5"></a><h2> <span class="mw-headline"> <b>Tuesday, June 5</b>  </span></h2>
<dl><dd><b>4. Current Status in the management of genetic marker and genotyping data</b> 8:00 - 10:00 (Chair: Thomas, Reporter: Casper) 
<dl><dd>Summary: 
</dd><dd>Clicking on the names below will open the original presentation in PDF format. Also an Excel Template is available with a (draft) comparison over the 3 presentations. 
<ul><li>Nunhems (Paul) 
</li><li><a href="../../../../images/2/2d/B_IRRI_CURRENT_STATUS_genotyping_data.pdf" class="internal" title="B IRRI CURRENT STATUS genotyping data.pdf">CRIL (Rowena [Weng])</a> 
</li><li><a href="../../../../images/e/e1/C_Molecular_Data_at_SPARC.pdf" class="internal" title="C Molecular Data at SPARC.pdf">SPARC (Shawn)</a> 
</li><li><a href="../../../../images/3/36/WS2007-Session_4.xls" class="internal" title="WS2007-Session 4.xls">Excel File comparing Nunhems-IRRI-SPARC</a>
</li></ul>
</dd></dl>
</dd></dl>
<p><br /> 
</p>
<dl><dd><dl><dd>Nunhems uses various tools to gather the data (for details, see presentation Paul), they are storing marker information in their NunGEMS system which is based on a LIMS (Nautilus) system in Oracle. Nunhems is storing their marker data (results) in the ICIS DMS database in Oracle. NunGEMS and ICIS are linked via the MarkerName and UniqueID, in this way queries can be written over both systems. The concept is proved, but more flexible query tools are needed on end-user level. Projects can be up to 1500 markers, normal projects are 200-300 markers and over 1000 individuals to test
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>IRRI has developed a GEMS database that is accessible via the ICISWorkBook and a new interface in MS-Access. For GEMS new functions have been developed in the ICISWorkBook. Some characteristics: 
<dl><dd><ul><li>Markers and Alleles: Per ID multiple names are allowed New Markers/Alleles can be added via the ICISWorkBook and maintained by GEMS 
</li><li>Currently per marker one protocol can be stored GEMS, in future: storage of multiple protocols per markers is desired 
</li><li>Desktop tool: Currently is in MS Access 
</li><li>Web application: to develop with AJAX
</li></ul>
</dd></dl>
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>SPARC has not yet a LIMS system in place. SPARC will have to store in near future large amounts of marker data, decided is to store these data in DMS. Data are mainly retrieved with DataComp tool, this tool will need to be expanded, main goals are expanding number of lines that can be compared and compare from fenotypic and genotypic data. 
</dd><dd>SPARC prefers to develop GEMS system within ICIS community.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Some discussion: 
<dl><dd><ul><li>Ron thinks we need to be careful because the Bp is only an estimation and needs to be shown with the trait to be useful. 
</li><li>Richard said that within a dataset it should be clear what is going on, but across datasets is a problem. 
</li><li>Ron thinks we should make sure it is clear what protocol the Bp was read on. 
</li><li>Where does GEMS end and LIMS start
</li></ul>
</dd></dl>
</dd></dl>
</dd></dl>
<p><br /> 
</p>
<dl><dd><b>5. Developments in ICIS for the management of the genetic marker and genotyping data</b> 10:20 - 2:00 (Chair: Sandra, Reporter: Arllet) 
<dl><dd>- Nunhems (Casper)
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Summary&nbsp;: 
</dd><dd>Nunhems is managing multiple crops and each having multiple markers. They will be managed together in NunGems but the application to retrieve them are specific to each crop. An ICIS View Crop Schema will integrate the databases (ICIS central, icis local databases and the NunGems schema). Two data retrievers will be built (one for breeding and the other one is for Research). Part of the Breeding Retriever are the label and barcoding tools. 
</dd><dd>What is needed is a flexible tool that allows end-users to create their own queries. That tool should also export the data in the file formats that can be used in visualization tools or statistical tools. The Retriever is done in MS Access but is there another tools like Business Objects or Rubik?
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Requirements: 
</dd><dd>Input 
</dd><dd>- Germplasm entries and tree information (including neigbourhoods) 
<dl><dd>- Lists or studies 
</dd><dd>- Projects 
</dd><dd>- Observation results which can be Marker scores and Other properties 
</dd><dd>- Marker information which can be Description, Position , Marker Type
</dd></dl>
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Thomas: You mentioned about a flexible tool. My work in IRRI is developing standard management of data. But for the retrieval of data, I am using a cookbook approach where list of recipes or examples are developed. They are not textbook. I started developing recipes for management of research data like finding duplicate record. We already taught some people about it and there was good response from them. If you have a good recipe, then that can be useful.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Casper: Which type of user? Is it breeding or the people from the lab? The Retriever has tables (Oracle Views) which users can use to create queries. But based on our experience, some users find it too complicated
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Thomas: Field people &amp; technical staff, those who collect data, need immediate feedback based on their data.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Casper: We were able to make people move from storing data from Excel to database.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Thomas: The least that the database can do is minimizing the paste and copy methods
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Shawn: Does Microsoft still support the development of the MS Access?
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Thomas: But whatever principle that people will learn from MS Access can be applied to another database that Microsoft will be supporting later on.
</dd></dl>
</dd></dl>
<p><br /> 
</p>
<dl><dd><dl><dd>- <a href="../../../../images/9/95/B_GEMS_development.pdf" class="internal" title="B GEMS development.pdf">IRRI GEMS (Thomas)</a> 
</dd><dd>Summary 
<ul><li>The presentation is to initiate discussion on the future development of GEMS 
</li><li>Genotyping data can be stored as studies in DMS which can be loaded using an (extended) Workbook or through script for different database platform. We have several experiences in SSR … Genotyping data can be considered as primary data/outputs 
</li><li>Marker Data are stored in ICIS GEMS which can be loaded through Workbook as batch. There is also a prototype standalone interface for marker data (SSR). But we don’t have a complete sets of marker data are available in GEMS. This might be due to the way marker data are collected. They are more often inputs/secondary data in genotyping data. 
</li><li>Maybe that is the reason that people don’t spend considerable time to put complete information about it. Who cares about these data enough that the person will complete the information needed for the marker data? 
</li><li>GEMS – we have schema, data loading tools for genotyping and marker data and data curation tool for marker data. 
</li><li>GEMS Schema 
</li><li>forward and backward primer that defines the marker detector 
</li><li>protocol information which is a kind of experimental set-up where the experiments is being done. We want a system where a protocol is used for a part of time and then refine and use in another experiment for the same marker. But we don’t want to change the marker. So what we did is a combination of marker detector and the protocol which we call the polymorphic detector 
</li><li>Molecular variant which is basically the alleles 
</li><li>Name table which looks like the GMS Names table but more of generic table where there is an object which can me marker detector and allele etc 
</li><li>Components that make up entire condition under which a particular marker detector is done. We break up the conditions into different components which have properties.
</li></ul>
</dd></dl>
</dd></dl>
<p><br /> 
</p>
<dl><dd><dl><dd>Shawn: How about references? 
</dd><dd>Thomas: We have it in Gems_names and gems_mv.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Graham requested for a sample database with the GEMS schema implemented and Weng showed the database and the tables of the GEMS 
</dd><dd>- The component table is linked with protocol which can have many components. 
</dd><dd>- The components have properties stored in gems_properties. 
</dd><dd>- The properties can have scale and method. Those are stored in the gems_scale and gems_method tables. 
</dd><dd>- The Gems_pdcomp which is a link between the component and the polymorphic detector
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Graham asked what happened if we have multiple scales for a property. Weng answered that those are handled in GEMS
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Paul commented that the GEMS being developed is not generic because it only contains information about the marker itself but not the protocol to create the marker. Graham clarified that it is not to create the protocol but the protocol that is applied or used.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Paul: Most lab has already set up their own set of protocols.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Ron: we are still at the early stage in applying marker but as more markers are made available from different labs, at that point when we need markers from another lab, we need information. Right now, they are available in the publication. I think this kind of database will be eventually needed.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Graham: 
<ol><li>It needs to link the protocol with the output data (allele data). 
</li><li>Although this is available in publication but once the lab make adjustment, it is good to document this 
</li><li>I think what Paul implied is that we only show the SSR marker system. Although we only show the SSR marker system, the challenge is desing a system that accommodates different system 
</li><li>My question with Thomas is who will be responsible managing these data. The point is it is only done when people finally find the need to curate it but they become directed to another experiment. By that time they will curate it, the information is already gone.
</li></ol>
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Thomas: The LIMS for different labs maybe differ based on what is their focus. Now what are the different applications available in ICIS that are related with LIMS? For sample tracking, we have GMS (GIDs &amp; germplasm methods) such as the DNA, the leaf it was extracted from and plant it came from. But we don’t have the setup of the wells. But you can use IMS and treat wells as location. Protocols can be stored in GEMS.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Paul: My idea of GEMS is a system where the information that cannot be stored in their current LIMS system (Nautilus). Nunza is storing it in NunGEMS.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Richard: Let us say, I used this marker on this genotype with this particular protocol. Marker detector contains the biological information (primer etc). You are detecting whatever amplifies that genome and it differs from one genome to another. You have to separate the different components of the methodologies- what piece of DNA in what condition and the assay (PD). It is the assay that you apply that gives you the molecular variant.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Graham: Our primary information in the breeding database is the PDID.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Paul: I don’t want the AFLP to be together. In a genetic distance, all these markers created will not have added value. I don’t see the added value … GEMS should just consist subset of markers with enough information. Why do you need to put in a database the information that might not be needed?
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Graham: It is like what we do in GMS. We enter in GMS the different generations of a line when what the breeder only concerns is the stable line. Let us take DART data. We collected massive data for DART lines which the intention of doing association analysis. As we get more information about those lines
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Paul: ...
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Graham – we are thinking of quite diverse set of users. If you are using DART, you will eventually be interested how other labs are using it. But in the case of Nunhems, you will only be interested in a marker when other people in the lab becomes interested with it .
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Thomas: Part of the question is the coverage as whether we want to put published data for a crop in GEMS. Or it is more of self-contained in a particular lab, project etc? Another question is what will be details and how to make it flexible? As Graham mentioned, in GCP we are required to look into more complicated situation.
</dd></dl>
</dd></dl>
<p><br /> 
</p>
<dl><dd><dl><dd>- <a href="../../../../images/9/9e/B_Retrieval_scripts2.pdf" class="internal" title="B Retrieval scripts2.pdf">IRRI Retrieval Scripts (Thomas)</a>
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Summary: Thomas mentioned that at this point, we have batch loading of information in Workbook and a standalone application for marker data curation. One issue is whether to re-implement in Delphi with separated interface for each marker technology or as web-based interface? He mentioned also that there was discussion in Wheat CRC which Sandra added is available in cropwiki.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>It is a challenge to have a one-to-one link between our phenotyping and genotyping data
</dd></dl>
</dd></dl>
<p><br /> 
</p>
<dl><dd><dl><dd>Script for Retrieving information 
<ul><li>the advantage of having the script in the server side is that it is faster t run than in the client side; 
</li><li>The disadvantage is it is database specific 
</li><li>Some tests about the execution of the script were done. The script created where get_dataset(Phenotype study) and get_dataset(genotype study) and create_aray_dataset(genotype table) 
</li><li>To display the dataset in parallel: the alleles found in markers are stored in a text field.
</li></ul>
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Martin: Will this give me a subset of the information in the same speed?
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Thomas: Retrieving the subset from the result of the query is faster than assemble the subset.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Martin: I get it now that you are creating a new storage in the server. But how will you deal with synchronization of those storages?
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Thomas: You can create it everyday.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Thomas: We created it with one study but we can create it for several studies.
</dd></dl>
</dd></dl>
<p><br /> 
</p><p><br /> 
</p>
<dl><dd><dl><dd>- <a href="../../../../images/9/97/MOSEL.pdf" class="internal" title="MOSEL.pdf">MoSel (Graham)</a>
</dd></dl>
</dd></dl>
<p><br /> 
</p>
<dl><dd><b>6. Discussions on how to create Genotyping database for the ICIS Community</b> 2:00 - 4:00 (Chair: Shawn, Reporter: Fran and Weng) 
<dl><dd>Summary of presentation: The problem is there is a lack of a tool to integrate genotype, phenotype and pedigree that can help them them in their breeding program. I had discussions with breeders. They are using excel sheets which is being filled by hand.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Graham showed a sample sheet that Tom Hash is using at ICRISAT. IRRI and some other institutes can still manage to use this kind of “tool”. They are using 10 or less markers. This is not manageable with High throughput technology which handles 1500-2000 markers.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>There must be some way to do it automatically? Can this be done only with excel macros?
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Graham and Guy had some discussions on it and came up with a concept for Mosel.To illustrate Mosel, Graham presented how mosel look like. The purpose of the presentation was to stimulate thoughts from others who might who would want to have the same application they have in mind.
</dd></dl>
</dd></dl>
<p><br /> 
</p>
<dl><dd><dl><dd>Fran: For phenotypic traits, are we thinking in terms of quality?
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Guy: we can show categorical and continuous data... whatever we have
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Graham: we can categorize, put colors...
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Guy: heat maps, etc
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Graham: Vivek is this gonna be useful for maize breeding?
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Vivek: we don't have it right now. We're are just beginning to discuss something on that line
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Graham: Can you still go away with spreadsheets?
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Vivek: yeah...
</dd></dl>
</dd></dl>
<p><br /> 
</p>
<dl><dd><dl><dd>Ron: Mosel is like doing DArT which is looking into the whole genome. we should consider where we're at...
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Graham: I've had discussions with Ian Delacy.In DarT Technology, all markers are anonymous. Making all selections are made using trait values but later you will find new information as you see changes..
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Richard: do we have enough data for this kind of tool?
</dd></dl>
</dd></dl>
<p><br /> 
</p>
<dl><dd><dl><dd>Some discussions on an association tool...
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Graham&nbsp;: The main objective of this tool is select which of this lines you want to keep but association can be a bi-product of this tool. 
</dd><dd>It's hard to identify the target markers in the foreground than in the background. A simulation tool is good for deciding which lines will have the trait we want
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Vivek&nbsp;: the collapsing tool could be very useful
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Casper: Is it one of the output tools for DMS, GEMS?
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Graham: That could be. Mosel analysis tool and specify the data input into Mosel. Queries like: what is the list of your study? And press go.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Nunhems: to be able to sort, you need data into it...
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Paul: normal users were able to collect data and combine with the marker data. dozen of visualization tool over in the internet which has the same output in the slide. it's open source and you can ask for customization from the developer. Effort should not be on creating new visualization tool.We can give a example of a large marker see. Data is collected which can be inputted to the software (phenomap?)
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Graham: the objective of Mosel is different from the software. The question is&nbsp;: what is the coverage? which is already stated. genetically speaking, you can calculate any genetic distance from any of the lines. We'll be interested to know the visualization tools.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Paul: real problem is getting the data out. Does not really matter whether we use GEMS or NunGEMS. We Just need to make sure that we create a generic output.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Graham: Paul can you create a proposal with Jesper regarding this?
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Ron&nbsp;: we have the same situation with others (use excel) but we need concentrate on how to target populations. IF we start small, it will ultimate go to this more sophisticated tool
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Guy&nbsp;: what we need is to create a prototype from GCP data.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>NEW TOPIC&nbsp;: LIMS DISCUSSION
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Shawn&nbsp;: I think we should discuss what the lab people need, a round table discussion. we need a LIMS system for the lab before a GEMS can be filled in.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Ron&nbsp;: can you briefly discuss the different databases?
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Shawn&nbsp;: what's the next step after GEMS?
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Arllet&nbsp;: we are using it and will be using it in for mapping data.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Paul: we are using LIMS system and linking it a little bit to marker data. The real value of this systems is the interface between these database. We have a decision, that the lab guys do their molecular selection.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Shawn: do we start developing our own system? Since we are using different systems. Do we take off from GEMS and worry about the LIMS later?
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Graham: LIMS component is very laboratory specific. we decided to do is to use a LIMS system being developed in ICRISAT. They have created all the quality controls for all the steps. They pop out with a genotyping data which we want it to flow straight into ICIS. We would like capturing of the protocol into GEMS. We haven't got our hands into that at the moment. we have ICIS leaf tracking system in ICIS. We want to upgrade from that situation. 
</dd><dd>Shawn: if we want to get something up and running here, we should know how the technicians create the sheets but they should not know where the date gets in.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Shawn&nbsp;: we need a system where lab people just need to learn how to enter data and not need to learn ICIS. If there's a question from a researcher, they should not have to go back to the spreadsheets. Should we go with IRRIs LIMS? GEMS approach?
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Thomas: LIMS and GEMS is often mentioned together. We have to tear them apart. They are two different things.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Ron&nbsp;: LIMS is basically a lab book. It's a template in a laptop instead of a notebook. In lab book we set up the protocols. You set the the grid for the samples to keep track of where the samples go. Record all unusual or a little different. It's nice to have all those properties in one computer system. Type in marker and all protocols will be available.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Graham: Management of protocol in the laboratory is part of the LIMS. Before they do any genotype is one of the process for LIMS. Description of the protocols is also needed in the database. That's part of the LIMS and GEMS. When something goes out of the process is where the GEMS system is. How far outside the lab will you push the LIMS system?
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Shawn: Casper what sorts of issues do you have?
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Casper: how do we slate data into it? some issues were presented during the presentation
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Paul: did you have a look at a commercial LIMS system?
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Shawn/Rob: no
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Paul: there are hundreds available which can fit into your lab. Evaluate 1 and choose 1. Its not that expensive and you will have it right away than create from scratch.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Shawn: Our goal is to incorporate ICIS into our breeding program.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Graham: whatever LIMS system you choose you decide where the interface into ICIS.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Casper&nbsp;: We need to get a bind to get all the databases.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Graham: I think the databases are represent well in the diagram.some of the data in the DMS in not interpretable if there is no GEMS.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Thomas: same thing for GMS and DMS
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Ron: you will never see the phenotypic data again. Genotypic and phenotypic data go hand and hand. 2 process in breeding data. 1 parental mapping and evaluation data from the cross. Setting up the cross is an extremely important process. I am looking for tool that will us to take the pheno and geno and display it so that we can evaluate it. Basically there is a standard and select a line based on that standard. Basically we do it like these but with spreadsheets. Bring up a series of lines and do you have markers? do you have polymorphism? For the traits, we look on those genes and see if there's a marker associated to that.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Shawn&nbsp;: Another problem is having many locals. How to manage it?
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Graham: you can design a tool that connects to databases.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Shawn&nbsp;: Can we decide on the GEMS, decide on the schema? protocols store in GEMS? For us its getting the phenotypic and Genotypic data together.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Graham: you should see GEMS as like GMS with DMS
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Shawn&nbsp;: Agree on a GEMS like a schema that works? Whether we use all the tables and fields together?
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Graham: we've come away from the LIMS database, right?
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Shawn: yes, my worry is that if 5 years from now, when we need to get back into that...
</dd></dl>
</dd></dl>
<a name="Wednesday.2C_June_6" id="Wednesday.2C_June_6"></a><h2> <span class="mw-headline"> <b>Wednesday, June 6</b>  </span></h2>
<dl><dd><b>Tour of SPARC and biotech facilities</b> 8:00 - 10:00
</dd></dl>
<p><br /> 
</p>
<dl><dd><b>7. Integration of External Tools with ICIS</b> 10:20-12:00 (Chair: Rob, Reporter: Shawn)
</dd></dl>
<dl><dd><dl><dd>- Graphical GenoTyper (Paul) 
<dl><dd>See also <a href="../../../../images/7/7d/AG06_GGT-demo.pdf" class="internal" title="AG06 GGT-demo.pdf">presentation of GGT</a> made by author Ralph van Berloo in 2006 
</dd><dd>For more information about GGT on Internet see: <a href="http://www.dpw.wau.nl/pv/pub/ggt" class="external free" title="http://www.dpw.wau.nl/pv/pub/ggt" rel="nofollow">http://www.dpw.wau.nl/pv/pub/ggt</a>. 
</dd></dl>
</dd><dd><b>Summary:</b> 
<dl><dd>- application is freeware and being used in the Nunhems biotech lab 
</dd><dd>- tabbed browsing of the output 
</dd><dd>- Takes the data loaded into it and creates a genetic map 
</dd><dd>- Colours represent marker scores and those colours can be customized 
</dd><dd>- Each bar represents a plant 
</dd><dd>- Data can be loaded from an Excel spreadsheet 
</dd><dd>- Changes can be made to the data within the application, but those changes are not written to the database 
</dd><dd>- Data can be sorted; individual lines can be looked at 
</dd><dd>- Some statistics can be calculated on the marker dataset ex) Calculate map distance 
</dd><dd>- A pie chart can be created from the dataset, but only useful if the map distance is included 
</dd><dd>- Allows for different views of the genetic maps 
</dd><dd>- Comes with a help manual 
</dd><dd>- More information can be found on the presentation Casper did at CIMMYT in 2006 
</dd><dd>- Fran: Does the data have to be in a precise format? 
</dd><dd>- Paul: Yes. The system uses it's own file to load, but it comes from an Excel sheet which has to be set up properly 
</dd><dd>- Images can be exported 
</dd><dd>- Paul: The lab at Nunhems works differently from your lab here at SPARC. The lab at Nunhems handles the selection process because they are considered the experts. Here at SPARC, your breeders do the selection. 
</dd><dd>- Ron - Yes, the lab here offers advice, but the ultimate decision rests with the breeders. This software would be very useful in the backcrossing programs 
</dd><dd>- Paul: The developer of this software really likes to receive feedback, and would probably be accomodating to changes
</dd></dl>
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>- PediTree (Arllet) 
<dl><dd>For more information about PediTree on Internet see: <a href="http://www.dpw.wageningen-ur.nl/pv/pub/Peditree/index.htm" class="external free" title="http://www.dpw.wageningen-ur.nl/pv/pub/Peditree/index.htm" rel="nofollow">http://www.dpw.wageningen-ur.nl/pv/pub/Peditree/index.htm</a>. 
</dd><dd><b>Summary:</b> 
</dd><dd>- Creates a tree-shaped visual of pedigrees 
</dd><dd>- Will do basic estimates of Coefficient of Parentage 
</dd><dd>- Values can be pulled out of ICIS and loaded into PediTree 
</dd><dd>- The Retriever is used to set up the correct information 
<dl><dd>- Select the list, study and traits that you need 
</dd></dl>
</dd><dd>- Change the databases specified in the Peditree.ini 
<dl><dd>- Set the keys so it connects to the Retriever 
</dd></dl>
</dd><dd>- Put PediTree into your Launcher.txt file 
</dd><dd>- Arllet still has to check on how to get data next to the pedigrees in the tree 
</dd><dd>- Will create an ancestry chart from the pedigree tree, but needs a controlled zoom to make viewing it easier 
</dd><dd>- Graham: Works well for bi-parental crosses, but does not contain any selfing generations. 
</dd><dd>- Casper: The application was made for Potato breeding programs and still needs some work to be adapted to other crops
</dd></dl>
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>- HandHeld Applications (Warren) 
</dd><dd><b>Summary:</b> 
<dl><dd>- Goal is to breakdown the DMS Workbook into modules to deal with data entry from handhelds 
</dd><dd>- There have been many requests for handheld and web-based data entry tools 
</dd><dd>- Handheld Workbook 
<dl><dd>- Tabs for each section (Variate, Scale, Study, Data) 
</dd><dd>- Can only store 1 variate at a time 
</dd><dd>- Casper would like to see an import/export option 
</dd><dd>- Each variate data is stored as text files 
</dd><dd>- Graham: not sure you need to define your variate and scale in the handheld, it should be done prior to loading the workbook in the handheld 
</dd><dd>- Casper: would like to see you step through the variates while at the plot 
</dd><dd>- Warren: it does allow you to choose variates from a combo-box 
</dd><dd>- Selwyn: having to choose different variates while at a plot takes too long. You want to enter your data and move on. He would also like to see previous data from the experiment. 
</dd><dd>- Graham: There needs to be more user-driven development for this tool. Unless someone is willing to sit down and explain exactly what is needed, Warren should stop. 
</dd></dl>
</dd><dd>- Warren also demonstrated a comparison tool to query marker data by marker name 
<dl><dd>- Choose your pedigree list 
</dd><dd>- Select which study you are interested in, and it will display in a new box, the marker names associated with that study 
</dd><dd>- Choose a trait and get the phenotypic data in another box 
</dd><dd>- Casper: We need to decide which approach we are going to take in querying this data.
</dd></dl>
</dd></dl>
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>- Nunhems External Tools (Casper) 
<dl><dd><b>Summary:</b> 
</dd><dd>CodeSoft: 
<dl><dd>- Commercial label-design application which Nunhems links to the Retriever via an ODBC connection 
</dd><dd>- Datasets are created in the Nunhems Retriever, and sent via macros to CodeSoft to create labels 
</dd><dd>- Labels are printed on ToshibaTec printers 
</dd></dl>
</dd><dd>2L: 
<dl><dd>- Commercial form design software for handhelds 
</dd><dd>- Taking a dataset (textfile) created by Excel or the Retriever and forms can be made to enter data or view historical data in the field 
</dd><dd>- One design can contain many datasets 
</dd><dd>- Very flexible, but not cheap to buy 
<dl><dd><b>Note: there is also a LITE version available with some limited functionalities. In this case the PC-Developers module is for free, only a license fee should be paid for the software on your hand-held computer, about 400 US$?, more info at: <a href="http://www.sw4hh.com" class="external free" title="http://www.sw4hh.com" rel="nofollow">http://www.sw4hh.com</a></b> 
</dd></dl>
</dd><dd>- WorkAboutPro handhelds are used at Nunhems 
</dd></dl>
</dd><dd>TOAD: 
<dl><dd>- Used for maintenance of Oracle databases, also for development and maintenance of views (queries) 
</dd></dl>
</dd><dd>Remote Controls: 
<dl><dd>- DameWare 
</dd><dd>- Shadow in CITRIX 
</dd><dd>- WebEx - remote control of applications outside of network 
</dd><dd><i>Just recently found on the web that there is also an opensource solution for remote control, see <a href="http://www.crossloop.com/index.html" class="external free" title="http://www.crossloop.com/index.html" rel="nofollow">http://www.crossloop.com/index.html</a></i> 
</dd></dl>
</dd><dd>Help &amp; Manual: 
<dl><dd>- Commercial software to create help files 
</dd><dd>- Can make both browser help files and .pdf files 
</dd></dl>
</dd><dd>Screen Capture Tools: 
<dl><dd>- SnagIt - Captures and saves video clips to your computer 
</dd><dd>- HauteCapture - Captures images on handhelds in real time
</dd></dl>
</dd></dl>
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>- <a href="../../../../images/1/1d/PedigreeDownloadTool_2007.pdf" class="internal" title="PedigreeDownloadTool 2007.pdf">University of Queensland Pedigree Download Tool (Sandra)</a> 
<dl><dd>- UofQ depends heavily on web interface for users 
</dd><dd>- You can currently see the pedigrees, but cannot download them 
</dd><dd>- SetGen will allow users to export information, but they need to have ICIS installed
</dd></dl>
</dd></dl>
</dd></dl>
<p><br /> 
</p>
<dl><dd><b>8. GRIMS developments</b> 1:00-3:00 (Chair: Selwyn, Reporter: Graham) 
<dl><dd>- Maize Genebank (Juan Carlos) 
</dd><dd>- <a href="../../../../images/f/ff/WheatGenebank2007.pdf" class="internal" title="WheatGenebank2007.pdf">Wheat Genebank (Jesper)</a> 
</dd><dd>- <a href="../../../../images/a/a2/ICIS_2007_-_GRIMS.pdf" class="internal" title="ICIS 2007 - GRIMS.pdf">Rice Genebank</a> (Roniela [Ella]) 
</dd><dd>- <a href="../../../../images/e/e5/ICISWorkShop2007_SMTA.pdf" class="internal" title="ICISWorkShop2007 SMTA.pdf">Standard Material Transfer Agreement (Grace)</a> 
</dd><dd>- <a href="../../../../images/3/36/ICIS_ATFCC_2007-2.pdf" class="internal" title="ICIS ATFCC 2007-2.pdf">Web Access to GRIMS Data (Selwyn)</a>
</dd></dl>
</dd></dl>
<p><br /> 
</p>
<dl><dd><b>9. Database Backend Issues</b> 3:20 - 4:00 (Chair: Arllet, Reporter: Thomas) 
<dl><dd>- <a href="../../../../images/d/da/ICIS_Schema_Maintenance_and_Database_Conversion.pdf" class="internal" title="ICIS Schema Maintenance and Database Conversion.pdf">ICIS Schema Maintenance and Database Conversion (pdf)</a> (Corina [Ching])
</dd></dl>
</dd></dl>
<dl><dd><dl><dd><dl><dd>Casper: For the schema upgrades, and in general, consider a more consistent naming convention of tables and fields that can be maintained in the longterm. 
</dd><dd>Thomas: SQL upgrade patches assume that users have the most current version of the database. Consider making this assumption more explicit in the patch name and/or in the release notes for the patch. 
</dd><dd>Selwyn: Consider supporting partial upgrades by making dependencies between schema and applications more explicit, e.g. release notes. 
</dd><dd>Casper: The entire new ICIS package has good upgrade notes and instructions. 
</dd><dd>Thomas: Investigate the technology and/or approach used by the different cross-backend synchronization tools to see whether they are really more efficient and effective than the current conversion, dump, and reload procedure. 
</dd></dl>
</dd><dd>- <a href="../../../../articles/t/e/s/Testing_of_PostgreSQL_as_database_backend_for_ICIS_4b0b.html" title="Testing of PostgreSQL as database backend for ICIS">IRIS in PostgreSQL</a> (Corina [Ching]) 
<dl><dd>Graham, Jesper: PostgreSQL needs to be evaluated as local ICIS database since that is where data are changed. 
</dd><dd>Graham et al: Differences is execution time of the various tests were noted. PostgreSQL was significantly slower is some of the queries, but not consistently so. PostgreSQL was running on a different server, whereas MS Access was running on the same machine as the client. Several explanations for the observed differences are possible, but the differences don't rule out the use of PostgreSQL as a backend for ICIS. 
</dd></dl>
</dd><dd>- IWIS3 in PostgreSQL (Jesper) =&gt; moved to Thursday 
</dd><dd>- Future Directions for the ICIS Schema (Arllet/Richard) =&gt; moved to Thursday
</dd></dl>
</dd></dl>
<a name="Thursday.2C_June_7" id="Thursday.2C_June_7"></a><h2> <span class="mw-headline"> <b>Thursday, June 7</b>  </span></h2>
<dl><dd><dl><dd>- <a href="../../../../images/d/d0/IWIS3inPostgreSQL.pdf" class="internal" title="IWIS3inPostgreSQL.pdf">IWIS3 in PostgreSQL (pdf)</a> (Jesper) 
<dl><dd>Arllet: The full install of ICIS was running correctly against the IWIS3 central database in PostgreSQL. 
</dd><dd>Jesper: Synchronization is fast, ~ 10 min for the extraction of data from SQL Server, and ~ 20 min for the creation of indices. 
</dd><dd>There are still open issues, especially the use of PostgreSQL for a ICIS local database, since it is in a local database where ICIS applications make changes to the data. A priority should be to test PostgreSQL as a backend for ICIS local databases. 
</dd><dd>PostgreSQL is not as easy to install, use and support as MS Access. It's role is more for shared databases that get accessed via ODBC. However, further testing and user experience of PostgreSQL on a local machine should be gathered.
</dd></dl>
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>- <b>Future Directions for the ICIS Schema</b> (Richard) 
<dl><dd>No PowerPoint presentation available. 
</dd><dd>Richard started with the question of what the ICIS schema 6.0 might look like. 
</dd><dd>It may not be advisable to add an ICIS genomics schema, but to use the Generic Model Organism schema [<a href="http://www.gmod.org" class="external text" title="http://www.gmod.org" rel="nofollow">GMOD</a>] instead. CHADO is modular, e.g. covering sequence data, controlled vocabulary, etc. [<a href="http://www.gmod.org/wiki/index.php/Category:Chado" class="external text" title="http://www.gmod.org/wiki/index.php/Category:Chado" rel="nofollow">CHADO</a>]. The CHADO schema is similar in generality to the ICIS schema and Richard suggested we should try and learn and adopt relevant parts in the further development of the ICIS schema. 
</dd><dd>Graham: Should we change the ICIS metadata schema (property, scale, method) to the CHADO schema? Richard showed some of the similarities between the two systems, e.g. the CHADO names and dbxref tables. 
</dd><dd>Additional information on IP status, uploading status, and access control for germplasm in ICIS is needed. An additional field in the germplasm table vs the use of attributes vs the use of ICIS lists were discussed. It is clear that these are different problems that might need different solutions. At IRRI, IP status is already managed as an attribute. Information on uploading status may be maintained in ICIS lists, and differential access control may require additional fields in the germplasm table (GMS) and the study table (DMS) which would need to be used by the applications when retrieving data. 
</dd><dd>On the locational data management in IRIS, Richard reported on the central concept of location ID and other pertinent aspects. The schema is still under development and needs further refinement. Linking of the locational data curated by Isaiah Mukema during the last 3 years still presents a problem, as the currently used location location ID's in IRIS seem not to link to the curated data. 
</dd><dd>Selwyn: If the scope and complexity of the location data becomes too large, it becomes too expensive to maintain, and the question also arises as to who should do it. 
</dd><dd>Richard proposed to set up a section on the ICISWiki to review the schema and the consequences on schema changes during the next year, and discuss it during the next ICIS annual workshop.
</dd></dl>
</dd></dl>
</dd></dl>
<p><br /> 
</p>
<dl><dd><b>10. Issues With Uploading Local Pedigree Data to the Central Database</b> 8:00 - 10:00 (Chair: Jesper, Reporter: Graham) 
<dl><dd>- <b>Separating Sensitive (private) data from non-sensitive (public) data in local</b> (Shawn, Sandra) 
<dl><dd><ul><li>Discussion: 
</li></ul>
<dl><dd>-Should we have a field in GERMPLSM to indicate which germplasm can be uploaded or should we use an attribute? For the moment lets use an attribute and get agreement from the breeders about the stage of publication of pedigrees. This stage should coincide with the making of a list – such as entry into a yield trial stage.
</dd></dl>
</dd></dl>
</dd></dl>
</dd></dl>
<dl><dd><dl><dd><dl><dd><dl><dd>- Another problem is the ability to unload a study from central. If a study is uploaded in error, how can it be replaced? What we need is a tool which allows the removal of a study or set of studies and allows upload of selected studies. This should be the basis of an administrators’ tool for managing GMS and DMS uploading
</dd></dl>
</dd></dl>
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>- <b>Dealing with duplicate entries</b> (Shawn, Jesper) 
<dl><dd><ul><li>Discussion: 
</li></ul>
<dl><dd>- Often lines are tested in locations before the GIDs are available in central. These lines need to be identified and replaced before uploading. A fuzzy name search would be a useful application in the administrators’ tool. This checking only needs to be done for lines imported to the local location since those developed at the location will likely not be known to the central.
</dd></dl>
</dd></dl>
</dd></dl>
</dd></dl>
<p><br /> 
</p>
<dl><dd><dl><dd>- <b>Keeping the DMS and GMS Central Databases synchronized</b> (Shawn) 
<dl><dd><ul><li>Discussion: 
</li></ul>
<dl><dd>- Dealing with replaced GIDs associated with DMS records. It would be nice to have a tool which synchronized GIDs referenced in DMS with changes in the GMS. A short term solution would be to add a query to the local database to make this update from the changed table periodically.
</dd></dl>
</dd></dl>
</dd></dl>
</dd></dl>
<p><br /> 
</p>
<dl><dd><b>11. ICIS security access/issues</b> 10:20-12:00 (Chair: Shawn, Reporter: Casper) 
<dl><dd><ul><li><a href="../../../../images/2/27/Data_Security_SPARC.pdf" class="internal" title="Data Security SPARC.pdf">Security of ICIS data in AAFC (Shawn)</a> 
</li><li><a href="../../../../images/b/b1/SecurityofICISdatainCAGE.pdf" class="internal" title="SecurityofICISdatainCAGE.pdf">Security of ICIS data in CAGE (Sandra)</a> 
</li><li>Security of ICIS data at Nunhems (Casper)
</li></ul>
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Presentations and discussion are about how to deal with different security access levels in ICIS. AAFC and UQ have the problem that users can work in several projects and per project members may have different access to certain to ICIS data, both GMS and DMS.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>For example AAFC is currently working with separate local databases per project, to avoid users seeing to much data. This is not easy for users working on multiple projects (have to switch local database all the time) and also not for administrators (eg how to deal with duplicate data, duplicate IDs over multiple local etc.). AAFC data is made only public to central GWIS3 database after 5 years.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Via the UQ website wheat data are available at two levels. Without password only public data is available. With password more breeder’s data is available, which is needed already in an earlier stage. Currently this is organized via 2 different databases in the background which is not easy for maintenance.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>In Nunhems it is slightly different organized. A breeding team has full access to all data (Read and Write). Other users have Read-Only access to parts of the database. Differentiation is organized via creating different Oracle Views and creating different versions of the Retriever. The Read-Only users have NO access to SetGen and GMSSearch.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>So the conclusion is that Security/Access Restrictions should be improved in ICIS. There is a need to make a difference between GMS and DMS. After some discussion the conclusion looks that: 
<dl><dd><ul><li>For GMS a security/access level is necessary on GID level, this probably requires an extra field in the Germplasm table <i>(Another solution might be to add another table in which is kept which GIDs are PRIVATE and which SEC_GROUPS may have access to this GID. GIDs that NOT appear in this table are PUBLIC).</i> Only needed in local db? 
</li><li>For DMS a security/access level is necessary on STUDY level, this probably requires an extra field in the SYUDY table <i>(Also here might be an extra table be more flexible)</i>. Only needed in local db? 
</li></ul>
</dd></dl>
</dd><dd>Since users can be member of multiple projects probably a SEC_GROUP table is needed in which can be identified which user is member of which group and which access rights are available on GID and STUDY level. 
</dd><dd><i>(See end of this section with a first rough idea of new table structures)</i>
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>In order to set the access-level there is a need to have extra functionalities available in SetGen and the WorkBook to (un-)mark PRIVATE certain data for other users. The idea is to set a flag to selected GIDs via 
<dl><dd><ul><li>Fill column with flag (Is the flag indicated by available groups in the SEC_GROUP table??) 
</li><li>Save Flag as Lock key 
</li></ul>
</dd></dl>
</dd><dd>This tool can be similar as how several saving of NAMES and ATTRIBUTES is organized. 
</dd><dd>Also an extra functionality will be needed to (un-)mark PRIVATE Studies. 
</dd><dd>Additional to this also tools should become available in the “Administrators” tool that Jesper will start developing.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Item of investigation is still how dll functions should be adapted and what action should be taken is cases eg PRIVATE marked GIDs appear in a tree in SetGen or GMSSearch. First feeling is that changes in DLL for GMS can be easy, for DMS more changes might be needed. Arllet will investigate this.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>Questions that arise are: 
<dl><dd><ul><li>Might we need multiple levels of “PRIVATE”? For example in some cases it will be allowed to show the germplasm only (NOT the parents) and in some cases also the parents might be shown. 
</li><li>Can this solution also solve the problem of BAYER to show only selected Names of a germplasm to certain users. Bayer likes to add multiple names to a Germplasm, but depending on the user only show 1 or a few names.
</li></ul>
</dd></dl>
</dd></dl>
</dd></dl>
<p><br /> 
</p>
<dl><dd><dl><dd><i>Possible idea for implementation this item</i> 
</dd><dd>Add Security Groups to the UDFLDS table in the local database. Adding of these groups should be able ONLY by the administrator’s tool that Jesper will develop.
</dd></dl>
</dd></dl>
<dl><dd><dl><dd><b>New Table 1: SEC_GRP_GMS</b> 
<dl><dd><ul><li>SECGRPID (N): ID for security groups (Links to UDFLDS.FLDNO) 
</li><li>GID (N): GID (Links to GERMPLSM.GID) 
</li><li>USERID (N): Links to USERS.USERID (Only users listed in this table may be able to see all information of this GID) 
</li><li>SECLVL (N): Depending on security level only parents can be set to Private, or whole GID can be set to private 
</li><li>GRPSTAT (N): Status of Security (eg 0 for active, 9 for removed (PUBLIC again). In this way can be traced over the time which GIDs have been private once and for which groups 
</li></ul>
</dd></dl>
</dd><dd>
</dd></dl>
</dd></dl>
<dl><dd><dl><dd><b>New Table 2: SEC_GRP_DMS</b> 
<dl><dd><ul><li>SECGRPID (N): ID for security groups (Links to UDFLDS.FLDNO) 
</li><li>STUDYID (N): STUDYID (Links to STUDY.STUDYID) 
</li><li>USERID (N): Links to USERS table (Only users listed in this table may be able to see all information of this GID 
</li><li>GRPSTAT (N): Status of Security (eg 0 for active, 9 for removed (PUBLIC again) In this way can be traced over the time which STUDIES have been private once and for which groups
</li></ul>
</dd></dl>
</dd></dl>
</dd></dl>
<p><br /> 
</p>
<dl><dd><b>12. Break Out Sessions</b> 1:00 - 4:00 
<dl><dd>- Storing of Passport data 
</dd><dd>- <b>Pedigree Management (Shawn, Sandra, Jesper)</b> 
<dl><dd>- Jesper, Shawn and Sandra met about how to handle updates to IWIS3: 
<dl><dd>- Sandra brought up concerns that her changes from last year were not reflected in the new version of IWIS3. Jesper explained that if the changes are not listed in the CHANGES table in the local GMS, then they are not uploaded to IWIS3. 
</dd><dd>- It was decided that if conflicts in pedigree information arise, Canadian lines will be resolved by Shawn's updates, Australian lines will be resolved by Sandra's updates, and any others Jesper will notify us to double check prior to uploading. 
</dd><dd>- Due to and large addition of American pedigree information and some errors that need to be sorted out, Shawn asked Jesper to discard the previous Canadian update databases from 2005 and he will create a new one and send it by the end of summer. 
</dd><dd>- Jesper explained that not all of the tables from the local GMS are currently being uploaded. Graham later explained the it is because the queries were not yet developed for all tables. He encourgaed Jesper to include queries for all local GMS tables in the new Administration tool he is creating. 
</dd><dd>- Shawn, Sandra and Jesper agreed to work closely on pedigree management to ensure the updates to IWIS3 run smoothly.
</dd></dl>
</dd></dl>
</dd></dl>
</dd></dl>
<p><br /> 
</p>
<dl><dd><dl><dd>- Structure of Genotyping Data (Graham) 
</dd><dd>- Ontology (Richard)
</dd></dl>
</dd></dl>
<p><br /> 
</p>
<dl><dd><b>== Workshop Supper - Chinook Golf Course == 7:00 - 9:00</b>
</dd></dl>
<a name="Friday.2C_June_8" id="Friday.2C_June_8"></a><h2> <span class="mw-headline"> <b>Friday, June 8</b>  </span></h2>
<dl><dd><b>13. ICIS Software development and collaboration tools</b> 8:00 - 10:00 (Chair: Fran, Reporter: Martin) 
<dl><dd>- <a href="../../../../images/9/9d/ICIS_communication.pdf" class="internal" title="ICIS communication.pdf">ICISWiki and CropForge (pdf)</a> (Thomas) 
<ul><li>CropForge: 
</li></ul>
<dl><dd>- Suggestion: to have bug reports (trackers) open for not only members 
</dd><dd>- Suggestion: filtering system for bugs (from which module, etc.) 
</dd><dd>- Conclusion: there are more tracking systems; functionality of Jira and CropForge tracker are similar;CropForge tracker is missing an obligatory version of the system the bug is reported about 
</dd><dd>- Suggestion: add category filter to the bug tracker
</dd></dl>
</dd></dl>
</dd></dl>
<dl><dd><dl><dd><ul><li>ICIS Wiki: 
</li></ul>
<dl><dd>- Suggestion: all user should have a non-empty home page (better for patrolling) 
</dd><dd>- Suggestion: find what are the options for externalization (export) of cropwiki pages
</dd></dl>
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>- <a href="../../../../images/7/77/Software_development_Process_of_ICIS.pdf" class="internal" title="Software development Process of ICIS.pdf">ICIS software development and release management</a> (Arllet) 
<dl><dd>- A bit of uncertainty about versioning system 
</dd><dd>- Suggestion: Release notes should be readily available on the CropForge site (without having to download anything. i.e. zip file containing CHANGES.txt, RELEASENOTES.txt) 
</dd><dd>- Suggestion: As the software development group gets bigger, it would be a good idea to prepare a "proposed changes" document &amp; publish this document for review/comment by the ICIS community before any development takes place.
</dd></dl>
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>- <a href="../../../../images/d/d1/Managing_ICIS_Codes.pdf" class="internal" title="Managing ICIS Codes.pdf">Managing Code</a> (Lilibeth, Clarissa) 
<dl><dd>- Suggestion: Add to best practices: Commit often 
</dd><dd>- Suggestion: Add to best practices: Commit only compilable code 
</dd><dd>- Suggestion: Add to best practices: In-code documentation (document as you go) 
</dd><dd>- Suggestion: Add a more formal way how to test code (Graham: don't expect your users to test everything. We need stricter standards of testing to be done by the developer)
</dd></dl>
</dd></dl>
</dd></dl>
<dl><dd><dl><dd>- <a href="../../../../images/5/51/IP-Licensing2007.pdf" class="internal" title="IP-Licensing2007.pdf">Intellectual Property and Licensing issues (pdf)</a> (Thomas)
</dd></dl>
</dd></dl>
<dl><dd>
</dd></dl>
<dl><dd><b>14. Integration of ICIS with GCP Platform and Progress on the ICIS web interface</b> 10:20 - 1:30 (Chair: Graham, Reporter: Thomas) 
<dl><dd>- <a href="../../../../images/3/34/ICIS_Workshop_2007_GCP.pdf" class="internal" title="ICIS Workshop 2007 GCP.pdf">Generation Challenge Program Platform and ICIS as GCP Data source</a> (Richard) 
<dl><dd>- ICIS Web - The Next Generation 
</dd><dd>- Towards Crop Information Network
</dd></dl>
</dd></dl>
</dd></dl>
<p><br /> 
</p>
<dl><dd>
</dd></dl>
<dl><dd><b>15. Future Planning</b> 1:30 - 3:00 (Chair: Graham, Reporter: Shawn)
</dd></dl>
<ul><li><b>Development and Consolidation of new modules</b> 
<ul><li>(CRIL/GCP) -Linking ICIS database to the GCP domain model (Mylah, Weng, Martin, Richard) 
</li><li>(CRIL)- Genotyping module 
<ul><li>data management (Graham, Weng) 
</li><li>Integration of existing tools (structure,GDPC tools) (Guy) 
</li><li>Develop a prototype visualization tool (Guy) 
</li></ul>
</li><li>(UQ)-storage of multivariate data (evaluation estimates and their precision) (Ian) 
</li><li>(GRC/CRIL) GRIMS 
<ul><li>Backend conversion of Oracle to PostgreSQL (Ella) 
</li><li>managing mission information (Ella) 
</li><li>display of preferred id (Arllet) 
</li></ul>
</li><li>(CIMMYT/CRIL)-look at the transfer of wheat genetic resources data to ICIS following with the experience on rice (Jesper) 
</li><li>(CRIL)- adoption of specific RGMIMS components (Richard) 
</li><li>(CRIL/CIP)-data warehousing, Mondrian/Rubik, Release cultivar warehouse. Crop Finder and views (Edwin, Juan Carlos) 
</li><li>(CRIL/AAFC)-record level access rights to GERMPLASM table, STUDY level access to DMS (Arllet, Ching, Shawn) 
</li><li>(CRIL/Bayer) – Restricted access to Names (Beth, Sebastien) 
</li><li>(CRIL/Bayer) – linkage between Workbook and CROPSTAT (Beth, John)
</li></ul>
</li></ul>
<ul><li><b>Stand-alone Applications</b> 
<ul><li>(CRIL)-cross finding facilty in standalone applications (Beth) 
</li><li>(CRIL)-combining ability analysis in CROPSTAT (Graham) 
</li><li>(CRIL/Bayer)-additional functionality in Workbook for hybrid breeding (Beth, Warren, John) 
</li><li>(CRIL)- administrator’s tool (Jesper) 
<ul><li>Loading/Deleting Central Studies 
</li><li>Partial upload of Germplasm records 
</li><li>Synchronization of GMS and DMS GIDs 
</li><li>Duplicate names search – Validation Tool (Ching) 
</li></ul>
</li><li>Managing user groups 
</li><li>(CRIL/AAFC)-implementation of naming conventions (Candy, Shawn) 
</li><li>(AAFC/CRIL)-development of Data Comparison Tool (Shawn) 
</li><li>(CRIL) – display CID, SID and CIMMYT Cross expansion in GMS Search (Jesper, Beth) 
</li><li>(CRIL/Bayer/Nunhems) – Investigate/implement alternative naming and ordering of list data columns in SetGen (Candy, Casper, Sebastien)
</li></ul>
</li></ul>
<ul><li><b>Web Interface</b> 
<ul><li>(GRC/CRIL/ATFCC/GPG2) - Web interface for Genetic Resources data (Ruaraidh, Ella, Mylah &amp;&nbsp;?) 
</li><li>(CRIL)-integration of Scalable Vector Graphics to produce high quality visualization in web interface (Supat) 
</li><li>(UQ)-make a pedigree download tool, compatibility with Peditree (Sandra) 
</li><li>(UQ/CRIL)-List management, germplasm and trait list management (Richard, Mylah) 
</li><li>(CRIL/PBGB) – Seed request module which connects inventory with evaluation data (Mylah, Grace, Beth) 
</li><li>(CRIL/GCP/UQ) -Web interface for DArT Data (Weng, Sandra, Richard, Mylah) 
</li></ul>
</li><li><b>Utilities</b> 
<ul><li>(CRIL)-Validation of coding and documentation standards (Arllet, Thomas) 
</li><li>(ICIS community)-user documentation and help, wiki/multimedia (Thomas) 
</li><li>(CRIL/CIMMYT)-finalizing version 8 of Fieldbook with links to ICIS (Vivek, Graham)
</li></ul>
</li></ul>
<ul><li><b>ICIS implementations and conversion </b> 
<ul><li>(CRIL)-integration of Maize Finder with ICIS (Juan Carlos) 
</li><li>(UASB)-establishment of Sorghum Information Systems (Sashidah) 
</li><li>(IRRI/CIAT/WARDA)-cross referencing of CG rice germplasm (Ruraidh) 
</li><li>(CIMMYT/ICARDA)-cross referencing of CG wheat/barley germplasm (Jesper) 
</li><li>(CIMMYT/IITA)-cross referencing of CG maize germplasm (William) 
</li><li>(ATFCC) – Construction of Central DB for Lentil, Chickpea and pea (Selwyn) 
</li><li>(ATFCC) – Publication of Genetic Resources Characterization on the web (Selwyn) 
</li><li>(ATFCC) – Loading of the observational data into the Central DMS (Selwyn) 
</li><li>(UQ/AAFC/ICARDA/CRIL) – Uploading pedigree updates into IWIS3 and IBIS (Jesper, Shawn, Sandra, Akin) 
</li></ul>
</li><li><b>ICIS Structure</b> 
<ul><li>(CRIL) – Tree structure for Study Names (Arllet, Warren) 
</li><li>(CRIL) – Changes table for DMS (Arllet) 
</li><li>(CRIL) – Naming of datasets (Warren) 
</li><li>(CRIL) – Make Study Type UD Field (Arllet) 
</li><li>(CRIL) – Identify management methods in the Germplasm table (GNPGS) (Arllet) 
</li><li>(CRIL/Nunhems) – Extra columns in the list data (Candy) 
</li><li>(CRIL) – Full implementation of PostgreSQL as a backend (Ching)
</li></ul>
</li></ul>
<p><br /> 
</p>
<dl><dd>
</dd></dl>
<dl><dd><dl><dd>- Wrap-up 3:20 - 4:00
</dd></dl>
</dd></dl>
<a name="Registered_Participants" id="Registered_Participants"></a><h2> <span class="mw-headline"> Registered Participants  </span></h2>
<div class="center"><div class="floatnone"><a href="../../../../articles/i/c/i/File%7EICIS_2007_Group_Photo_3.jpg_3e7a.html" class="image" title="ICIS 2007 Participants"><img alt="ICIS 2007 Participants" src="../../../../images/8/8b/ICIS_2007_Group_Photo_3.jpg" width="914" height="612" border="0" /></a></div></div> 
<p><br /> 
</p>
<table width="75%" cellspacing="0" cellpadding="5" border="1" align="center" class="wikitable sortable">
<caption> <b>ICIS 2007 Developers Workshop Participants</b>
</caption>
<tr>
<th> Name
</th><th> Affiliation
</th></tr>
<tr>
<td> <a href="../../../../articles/s/y/a/User%7ESYates-SPARC_662e.html" title="User:SYates-SPARC">Shawn Yates</a>
</td><td> Agriculture and Agri-Food Canada
</td></tr>
<tr>
<td> Fran Clarke
</td><td> Agriculture and Agri-Food Canada
</td></tr>
<tr>
<td> John Clarke
</td><td> Agriculture and Agri-Food Canada
</td></tr>
<tr>
<td> Ron Knox
</td><td> Agriculture and Agri-Food Canada
</td></tr>
<tr>
<td> Jay Ross
</td><td> Agriculture and Agri-Food Canada
</td></tr>
<tr>
<td> Devin Dahlman
</td><td> Agriculture and Agri-Food Canada
</td></tr>
<tr>
<td> <a href="../../../../articles/g/m/c/User%7EGmclaren_b845.html" title="User:Gmclaren">Graham McLaren</a>
</td><td> IRRI-CIMMYT
</td></tr>
<tr>
<td> <a href="../../../../articles/r/b/r/User%7ERbruskiewich_a89c.html" title="User:Rbruskiewich">Richard Bruskiewich</a>
</td><td> IRRI
</td></tr>
<tr>
<td> <a href="../../../../articles/t/m/e/User%7ETmetz_c741.html" title="User:Tmetz">Thomas Metz</a>
</td><td> IRRI
</td></tr>
<tr>
<td> <a href="../../../../articles/a/p/o/User%7EAportugal_745d.html" title="User:Aportugal">Arllet Portugal</a>
</td><td> IRRI
</td></tr>
<tr>
<td> <a href="../../../../articles/w/c/o/User%7EWconstantino_4eab.html" title="User:Wconstantino">Warren Vincent Constantino</a>
</td><td> IRRI
</td></tr>
<tr>
<td> <a href="../../../../articles/l/s/i/User%7ELsison_43a8.html" title="User:Lsison">Lilibeth Sison (Beth)</a>
</td><td> IRRI
</td></tr>
<tr>
<td> <a href="../../../../articles/r/v/a/User%7ERvalerio_c9de.html" title="User:Rvalerio">Rowena Valerio (Weng)</a>
</td><td> IRRI
</td></tr>
<tr>
<td> <a href="../../../../articles/c/p/i/User%7ECpimentel_117e.html" title="User:Cpimentel">Clarissa Pimentel (Candy)</a>
</td><td> IRRI
</td></tr>
<tr>
<td> <a href="../../../../articles/m/h/a/User%7EMhabito_276f.html" title="User:Mhabito">Maria Corina Habito (Ching)</a>
</td><td> IRRI
</td></tr>
<tr>
<td> <a href="../../../../articles/m/a/n/User%7EManacleto_937a.html" title="User:Manacleto">Mylah Rystie Anacleto</a>
</td><td> IRRI
</td></tr>
<tr>
<td> <a href="../../../../articles/r/h/e/User%7ERherrera_2c14.html" title="User:Rherrera">Roniela Prantilla (Ella)</a>
</td><td> IRRI
</td></tr>
<tr>
<td> <a href="../../../../articles/g/l/s/User%7EGlscapilit_78e3.html" class="new" title="User:Glscapilit (page does not exist)">Grace Lee Capilit</a>
</td><td> IRRI
</td></tr>
<tr>
<td> <a href="../../../../articles/c/a/s/User%7ECasperNUN_85dd.html" title="User:CasperNUN">Casper aan den Boom</a>
</td><td> Nunhems Netherlands BV
</td></tr>
<tr>
<td> Paul Buddiger
</td><td> Nunhems Netherlands BV
</td></tr>
<tr>
<td> Stefan van Lier
</td><td> Nunhems Netherlands BV
</td></tr>
<tr>
<td> Rob Aerts
</td><td> Nunhems Netherlands BV
</td></tr>
<tr>
<td> <a href="../../../../articles/j/n/o/User%7EJnorgaard_e415.html" title="User:Jnorgaard">Jesper Nørgaard</a>
</td><td> CIMMYT
</td></tr>
<tr>
<td> <a href="../../../../articles/v/i/v/User%7EVivek_b46c.html" class="new" title="User:Vivek (page does not exist)">Bindiganavile Vivek</a>
</td><td> CIMMYT
</td></tr>
<tr>
<td> <a href="../../../../articles/j/a/l/User%7EJalarcon_6ef6.html" title="User:Jalarcon">Juan Carlos Alarcon</a>
</td><td> CIMMYT
</td></tr>
<tr>
<td> <a href="../../../../articles/s/a/n/User%7ESandraM_bee9.html" class="new" title="User:SandraM (page does not exist)">Sandra Micallef</a>
</td><td> University of Queensland
</td></tr>
<tr>
<td> <a href="../../../../articles/s/f/r/User%7ESfrade_e1c2.html" title="User:Sfrade">Sebastien Frade</a>
</td><td> Bayer Cropscience - BioScience
</td></tr>
<tr>
<td> Selwyn Ellis
</td><td> Australian Temperate Field Crops Collection
</td></tr>
<tr>
<td> Martin Senger
</td><td> GCP/EBI
</td></tr></table>

<!-- 
NewPP limit report
Preprocessor node count: 50/1000000
Post-expand include size: 0/2097152 bytes
Template argument size: 0/2097152 bytes
Expensive parser function count: 0/100
-->
<div class="printfooter">
Retrieved from "<a href="http://localhost../../../../articles/i/c/i/ICIS_Workshop_2007_82fe.html">http://localhost../../../../articles/i/c/i/ICIS_Workshop_2007_82fe.html</a>"</div>
						<!-- end content -->
						<div class="visualClear"></div>
		</div>
	</div>
		</div>
		<div id="column-one">
	<div id="p-cactions" class="portlet">
		<h5>Views</h5>
		<div class="pBody">
			<ul>
	
				 <li id="ca-nstab-main" class="selected"><a href="../../../../articles/i/c/i/ICIS_Workshop_2007_82fe.html" title="View the content page [c]" accesskey="c">Page</a></li>
				 <li id="ca-talk" class="new"><a href="../../../../articles/i/c/i/Talk%7EICIS_Workshop_2007_8ee8.html" title="Discussion about the content page [t]" accesskey="t">Discussion</a></li>
				 <li id="ca-viewsource"><a href="../../../../articles/i/c/i/ICIS_Workshop_2007_82fe.html" title="This page is protected.&#10;You can view its source [e]" accesskey="e">View source</a></li>
				 <li id="ca-history"><a href="../../../../articles/i/c/i/ICIS_Workshop_2007_82fe.html" title="Past versions of this page [h]" accesskey="h">History</a></li>			</ul>
		</div>
	</div>
	<div class="portlet" id="p-personal">
		<h5>Personal tools</h5>
		<div class="pBody">
			<ul>
				<li id="pt-login"><a href="../../../../articles/u/s/e/Special%7EUserLogin_94cd.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</a></li>
			</ul>
		</div>
	</div>
	<div class="portlet" id="p-logo">
		<a style="background-image: url(../../../../misc/icis-wiki3.png);" href="../../../../index.html" title="Visit the main page [z]" accesskey="z"></a>
	</div>
	<script type="text/javascript"> if (window.isMSIE55) fixalpha(); </script>
	<div class='generated-sidebar portlet' id='p-navigation'>
		<h5>Navigation</h5>
		<div class='pBody'>
			<ul>
				<li id="n-mainpage"><a href="../../../../index.html" title="Visit the main page [z]" accesskey="z">Main Page</a></li>
				<li id="n-portal"><a href="../../../../articles/c/o/m/ICISWiki%7ECommunity_Portal_7449.html" title="About the project, what you can do, where to find things">Community portal</a></li>
				<li id="n-recentchanges"><a href="../../../../articles/r/e/c/Special%7ERecentChanges_e0d0.html" title="The list of recent changes in the wiki [r]" accesskey="r">Recent changes</a></li>
				<li id="n-help"><a href="../../../../articles/c/o/n/Help%7EContents_22de.html" title="The place to find out">Help</a></li>
			</ul>
		</div>
	</div>
	<div id="p-search" class="portlet">
		<h5><label for="searchInput">Search</label></h5>
		<div id="searchBody" class="pBody">
			<form action="../../../../articles/s/e/a/Special%7ESearch_a8e5.html" id="searchform"><div>
				<input id="searchInput" name="search" type="text" title="Search ICISWiki [f]" accesskey="f" value="" />
				<input type='submit' name="go" class="searchButton" id="searchGoButton"	value="Go" title="Go to a page with this exact name if exists" />&nbsp;
				<input type='submit' name="fulltext" class="searchButton" id="mw-searchButton" value="Search" title="Search the pages for this text" />
			</div></form>
		</div>
	</div>
	<div class="portlet" id="p-tb">
		<h5>Toolbox</h5>
		<div class="pBody">
			<ul>
				<li id="t-whatlinkshere"><a href="../../../../articles/w/h/a/Special%7EWhatLinksHere_ICIS_Workshop_2007_7667.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</a></li>
				<li id="t-recentchangeslinked"><a href="../../../../articles/r/e/c/Special%7ERecentChangesLinked_ICIS_Workshop_2007_a625.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</a></li>
<li id="t-specialpages"><a href="../../../../articles/s/p/e/Special%7ESpecialPages_f12b.html" title="List of all special pages [q]" accesskey="q">Special pages</a></li>
				<li id="t-print"><a href="../../../../articles/i/c/i/ICIS_Workshop_2007_82fe.html" title="Printable version of this page [p]" accesskey="p">Printable version</a></li>				<li id="t-permalink"><a href="../../../../articles/i/c/i/ICIS_Workshop_2007_82fe.html" title="Permanent link to this version of the page">Permanent link</a></li>			</ul>
		</div>
	</div>
		</div><!-- end of the left (by default at least) column -->
			<div class="visualClear"></div>
			<div id="footer">
				<div id="f-poweredbyico"><a href="http://www.mediawiki.org/"><img src="../../../../skins/common/images/poweredby_mediawiki_88x31.png" alt="Powered by MediaWiki" /></a></div>
				<div id="f-copyrightico"><a href="../../../../../COPYING.html"><img src="/icis/skins/common/images/gnu-fdl.png" alt='GNU Free Documentation License 1.2' /></a></div>
			<ul id="f-list">
					<li id="credits">This page was last modified 08:56, 18 March 2009 by <a href="../../../../articles/t/m/e/User%7ETmetz_c741.html" title="User:Tmetz">Thomas Metz</a>. Based on work by <a href="../../../../articles/c/a/s/User%7ECasperNUN_85dd.html" title="User:CasperNUN">Casper aan den Boom</a>,&#32;<a href="../../../../articles/r/b/r/User%7ERbruskiewich_a89c.html" title="User:Rbruskiewich">Richard Bruskiewich</a>,&#32;<a href="../../../../articles/m/h/a/User%7EMhabito_276f.html" title="User:Mhabito">Maria Corina de Luna Habito</a>,&#32;<a href="../../../../articles/j/n/o/User%7EJnorgaard_e415.html" title="User:Jnorgaard">Jesper Nørgaard Welen</a>,&#32;<a href="../../../../articles/s/y/a/User%7ESYates-SPARC_662e.html" title="User:SYates-SPARC">Shawn Yates</a>,&#32;<a href="../../../../articles/a/p/o/User%7EAportugal_745d.html" title="User:Aportugal">Arllet Portugal</a>,&#32;<a href="../../../../articles/r/v/a/User%7ERvalerio_c9de.html" title="User:Rvalerio">Rowena Valerio</a>,&#32;<a href="../../../../articles/r/h/e/User%7ERherrera_2c14.html" title="User:Rherrera">Roniela Herrera Prantilla</a>,&#32;<a href="../../../../articles/s/f/r/User%7ESfrade_e1c2.html" title="User:Sfrade">Sebastien Frade</a>,&#32;<a href="../../../../articles/j/o/h/User%7EJohndeleon_4644.html" class="new" title="User:Johndeleon (page does not exist)">John de Leon</a>&#32;and&#32;<a href="../../../../articles/m/u/l/User%7EMulat_4d7e.html" class="new" title="User:Mulat (page does not exist)">Teri Ulat</a>.</li>
					<li id="copyright">Content is available under <a href="../../../../../COPYING.html" class="external " title="../../../../../COPYING.html">GNU Free Documentation License 1.2</a>.</li>
					<li id="privacy"><a href="../../../../articles/p/r/i/ICISWiki%7EPrivacy_policy_0cbf.html" title="ICISWiki:Privacy policy">Privacy policy</a></li>
					<li id="about"><a href="../../../../articles/a/b/o/ICISWiki%7EAbout_53aa.html" title="ICISWiki:About">About ICISWiki</a></li>
					<li id="disclaimer"><a href="../../../../articles/g/e/n/ICISWiki%7EGeneral_disclaimer_9129.html" title="ICISWiki:General disclaimer">Disclaimers</a></li>
			</ul>
		</div>
</div>

		<script type="text/javascript">if (window.runOnloadHook) runOnloadHook();</script>
<!-- Served in 107.256 secs. --></body></html>
