<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
	<head>
		<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
		<meta http-equiv="Content-Style-Type" content="text/css" />
		<meta name="generator" content="MediaWiki 1.14.0" />
		<meta name="keywords" content="SP3 Use Cases 2008 Document 1" />
		<link rel="shortcut icon" href="../../../../misc/favicon.ico" />
		<link rel="search" type="application/opensearchdescription+xml" href="../../../../../opensearch_desc.php" title="ICISWiki (en)" />
		<link title="Creative Commons" type="application/rdf+xml" href="../../../../articles/s/p/3/SP3_Use_Cases_2008_Document_1_5040.html" rel="alternate meta" />
		<link rel="copyright" href="../../../../../COPYING.html" />
		<link rel="alternate" type="application/rss+xml" title="ICISWiki RSS Feed" href="http://localhost../../../../articles/r/e/c/Special%7ERecentChanges_e0d0.html" />
		<link rel="alternate" type="application/atom+xml" title="ICISWiki Atom Feed" href="http://localhost../../../../articles/r/e/c/Special%7ERecentChanges_e0d0.html" />
		<title>SP3 Use Cases 2008 Document 1 - ICISWiki</title>
		<link rel="stylesheet" href="../../../../skins/common/shared.css?195" type="text/css" media="screen" />
		<link rel="stylesheet" href="../../../../skins/common/commonPrint.css?195" type="text/css" media="print" />
		<link rel="stylesheet" href="../../../../skins/monobook/main.css?195" type="text/css" media="screen" />
		<!--[if lt IE 5.5000]><link rel="stylesheet" href="../../../../skins/monobook/IE50Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<!--[if IE 5.5000]><link rel="stylesheet" href="../../../../skins/monobook/IE55Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<!--[if IE 6]><link rel="stylesheet" href="../../../../skins/monobook/IE60Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<!--[if IE 7]><link rel="stylesheet" href="../../../../skins/monobook/IE70Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<link rel="stylesheet" href="../../../../skins/../../../../raw/MediaWiki%7ECommon.css?195" type="text/css" />
		<link rel="stylesheet" href="../../../../skins/../../../../raw/MediaWiki%7EPrint.css?195" type="text/css" media="print" />
		<link rel="stylesheet" href="../../../../skins/../../../../raw/MediaWiki%7EMonobook.css?195" type="text/css" />
		<link rel="stylesheet" href="../../../../skins/../../../../raw/gen.css?195" type="text/css" />
		<!--[if lt IE 7]><script type="text/javascript" src="../../../../skins/common/IEFixes.js?195"></script>
		<meta http-equiv="imagetoolbar" content="no" /><![endif]-->

		<script type= "text/javascript">/*<![CDATA[*/
		var skin = "monobook";
		var stylepath = "../../../../skins";
		var wgArticlePath = "../../../../$1";
		var wgScriptPath = "../../../../..";
		var wgScript = "/icis/index.php";
		var wgVariantArticlePath = false;
		var wgActionPaths = {};
		var wgServer = "http://localhost";
		var wgCanonicalNamespace = "";
		var wgCanonicalSpecialPageName = false;
		var wgNamespaceNumber = 0;
		var wgPageName = "SP3_Use_Cases_2008_Document_1";
		var wgTitle = "SP3 Use Cases 2008 Document 1";
		var wgAction = "view";
		var wgArticleId = "3381";
		var wgIsArticle = true;
		var wgUserName = null;
		var wgUserGroups = null;
		var wgUserLanguage = "en";
		var wgContentLanguage = "en";
		var wgBreakFrames = false;
		var wgCurRevisionId = "11409";
		var wgVersion = "1.14.0";
		var wgEnableAPI = true;
		var wgEnableWriteAPI = true;
		var wgSeparatorTransformTable = ["", ""];
		var wgDigitTransformTable = ["", ""];
		var wgRestrictionEdit = [];
		var wgRestrictionMove = [];
		/*]]>*/</script>

		<script type="text/javascript" src="../../../../skins/common/wikibits.js?195"><!-- wikibits js --></script>
		<!-- Head Scripts -->
		<script type="text/javascript" src="../../../../raw/gen.js"><!-- site js --></script>
	</head>
<body class="mediawiki ltr ns-0 ns-subject page-SP3_Use_Cases_2008_Document_1 skin-monobook">
	<div id="globalWrapper">
		<div id="column-content">
	<div id="content">
		<a name="top" id="top"></a>
				<h1 id="firstHeading" class="firstHeading">SP3 Use Cases 2008 Document 1</h1>
		<div id="bodyContent">
			<h3 id="siteSub">From ICISWiki</h3>
			<div id="contentSub"></div>
									<div id="jump-to-nav">Jump to: <a href="#column-one">navigation</a>, <a href="#searchInput">search</a></div>			<!-- start content -->
			<table id="toc" class="toc" summary="Contents"><tr><td><div id="toctitle"><h2>Contents</h2></div>
<ul>
<li class="toclevel-1"><a href="#Platform_Development_Meeting_to_discuss_Use_Case_3."><span class="tocnumber">1</span> <span class="toctext">Platform Development Meeting to discuss Use Case 3.</span></a>
<ul>
<li class="toclevel-2"><a href="#Introduction"><span class="tocnumber">1.1</span> <span class="toctext">Introduction</span></a></li>
<li class="toclevel-2"><a href="#Molecular_Breeding_Design_Tool_.28MBDT.29"><span class="tocnumber">1.2</span> <span class="toctext">Molecular Breeding Design Tool (MBDT)</span></a></li>
<li class="toclevel-2"><a href="#Sample_and_marker_tracker_.28Molecular_LIMS.29"><span class="tocnumber">1.3</span> <span class="toctext">Sample and marker tracker (Molecular LIMS)</span></a></li>
<li class="toclevel-2"><a href="#Molecular_Selection_Tool_.28MoSel.29"><span class="tocnumber">1.4</span> <span class="toctext">Molecular Selection Tool (MoSel)</span></a></li>
<li class="toclevel-2"><a href="#Data_Loading_Application"><span class="tocnumber">1.5</span> <span class="toctext">Data Loading Application</span></a></li>
<li class="toclevel-2"><a href="#Development_Decisions"><span class="tocnumber">1.6</span> <span class="toctext">Development Decisions</span></a></li>
</ul>
</li>
</ul>
</td></tr></table><script type="text/javascript"> if (window.showTocToggle) { var tocShowText = "show"; var tocHideText = "hide"; showTocToggle(); } </script>
<a name="Platform_Development_Meeting_to_discuss_Use_Case_3." id="Platform_Development_Meeting_to_discuss_Use_Case_3."></a><h1> <span class="mw-headline">Platform Development Meeting to discuss Use Case 3.</span></h1>
<p>18 and 19 February 2008 at ICRISAT
</p><p>Jayashree, Dave Hoisington, Tom Hash, Graham McLaren
</p>
<a name="Introduction" id="Introduction"></a><h2> <span class="mw-headline">Introduction</span></h2>
<p>An overview of the Molecular Breeding Information System was discussed. It was agreed that there were four components:
</p>
<ul><li>Molecular Breeding Design Tool (MBDT)
</li><li>Sample and marker tracker (Molecular LIMS)
</li><li>Molecular Selection Tool (MoSel)
</li><li>Data loading application
</li></ul>
<p>The first and third of these components are applications which need to be developed by the ICRISAT/CIMMYT team, the second and fourth are information system components mostly concerned with LIMS functions which will be implemented separately by ICRISAT and CIMMYT. The ICRISAT team has started work on the design tool, and the CIMMYT team has spent time considering the design of the selection tool. Hence it was decided that we should continue with that division of labor for the time being and look for opportunities to share components across tools.
</p>
<a name="Molecular_Breeding_Design_Tool_.28MBDT.29" id="Molecular_Breeding_Design_Tool_.28MBDT.29"></a><h2> <span class="mw-headline">Molecular Breeding Design Tool (MBDT)</span></h2>
<p>The design tool has the following functions:
</p>
<ul><li>Assist breeders in selecting parental germplasm based on phenotypic data, known adaptation, economic importance.
<ul><li>What traits are present and which are absent in potential recipients?
</li><li>Find and display known genes and QTL available for supplying the missing traits.
</li><li>List potential donors for missing traits.
</li><li>Develop and display trait maps 
</li></ul>
</li></ul>
<p>This function relies heavily on integration and assimilation of external knowledge and information. Initially the application interface should facilitate appropriate queries, but even then most results are likely to come from external analysis and the interface should enable capture of the information from that external analysis.
</p>
<ul><li>Verify and display genotypic and trait association data
<ul><li>Check availability of genotyping data for potential recipients and donors.
</li><li>If not available, formulate lists of germplasm and required marker analysis and pass the information to the Molecular LIMS for field and lab requests.
</li><li>Display graphical genotypes and known trait associations for potential recipients. 
</li><li>Display genotype and trait association information on potential donors.
</li></ul>
</li></ul>
<ul><li>Analyse compatibility between donors and recipients 
<ul><li>Show donor and recipient regions
</li><li>Choose markers (exact or flanking) for important regions. These must be polymorphic between donors and recipients, as close as possible to genes of interest, and should mark regions for introgression from donors or specific regions for retention of genotype in recipient germplasm.
</li><li>Identify potential linkage drag
</li><li>Analyse background similarity via genetic distances of coefficients of parentage.
</li><li>Select background markers, polymorphic between recipients and donors and suitably spaced over the genome. 
</li><li>Select recipient and donor germplasm, design crosses and pass crossing lists to the Molecular LIMS for making crosses.
</li></ul>
</li></ul>
<ul><li>Specify target genotypes 
</li></ul>
<p>This function will benefit substantially from integration of simulation tools like QULine to pick crossing strategies, population sizes and most compatible parents. However this will not be attempted in year 1 of the use case development.
</p><p>The MBDT has two main display formats, one for trait maps and one for displaying genotyping and trait association data for specific recipients and donors. The first has a mapping flavor and technology could draw heavily on the CMTV, while the second is a graphical genotyping tool which also lies at the heart of the third component - MOSEL.
</p>
<a name="Sample_and_marker_tracker_.28Molecular_LIMS.29" id="Sample_and_marker_tracker_.28Molecular_LIMS.29"></a><h2> <span class="mw-headline">Sample and marker tracker (Molecular LIMS)</span></h2>
<p>The Molecular LIMS has the role of driving and coordination field and lab work to facilitate molecular breeding. It must accommodate the following functions:
</p>
<ul><li>Sample tracking for germplasm - crosses and progeny
</li><li>Field lay-out and field book preparation
</li><li>Capture of field evaluations linked to germplasm samples
</li><li>Sample tracking for DNA extraction and analysis
</li><li>Marker and protocol identification for the molecular lab
</li><li>Specification of Sample by Marker analysis requirements
</li><li>Capture of processed genotyping data linked to germplasm samples.
</li></ul>
<p>This component requires database infrastructure and a LIMS user interfaces which are compatible with user field and laboratory practices. Each implementation of the Molecular Breeding Information System will likely have different requirements and the partners in the platform project for this use case will implement this component separately. ICRISAT has designed the database component and started integration with the laboratory LIMS and ICRIS database. CIMMYT will use the ICIS breeders interface integrated with the IWIS3 and IMIS databases.
</p><p>Future developments will be the deployment of automatic query and field data capture from hand held devices. 
</p>
<a name="Molecular_Selection_Tool_.28MoSel.29" id="Molecular_Selection_Tool_.28MoSel.29"></a><h2> <span class="mw-headline">Molecular Selection Tool (MoSel)</span></h2>
<p>The Molecular Selection Tool is required to facilitate the selection of the most promising lines in terms of closeness to the target genotype and likelihood of reaching the target according to a proposed development strategy. Main functions include:
</p>
<ul><li>Quality assurance
<ul><li>Verification that genotypes of test lines are compatible with parental genotypes.
</li></ul>
</li></ul>
<ul><li>Display genotype and phenotype information
<ul><li>Display graphical genotypes of targets, parents and test lines so that test lines can be compared with parents and targets.
</li><li>Graphical genotypes of test lines can be displayed in terms of segment origin or distance from a target genotype. For the classical diploid MAB situation segment origin is useful:
</li></ul>
</li></ul>
<pre>                M1        M2                   M3  M4
            AAAAAAAAAAHHHHHHHHHHHHHHHHHHHHHHHHHHHHBBBB    
</pre>
<pre>            M1 - last homozygous recurrent marker in a segment
            M2 - first heterozygous marker in a segment
            M3 - last heterozygous marker in a segment
            M4 - first homozygous donor marker (not possible in MAB without a selfing generation)
            A - color 1 indicating homozygous recurrent
            H - color 2 indicating heterozygous regions
            B - color 3 indicating homozygous donor
            Markers should be spaced according to map distance if possible
            Colors should merge between markers of different origins to indicate unknown crossover points
            A fourth color is necessary to display segments of unknown origin
</pre>
<p>A display based on genetic distances to target genotypes will be useful in extending the tool to polyploid species and to more complex breeding strategies such as marker assisted recurrent selection schemes where there may be numerous donors and recipients and target genotypes are complex mixtures. Schemes for computing genetic distances can incorporate probabilities of origin by descent from individual founders. 
</p>
<ul><li><ul><li>Allow partitioned display of foreground and background markers. Problems with the generalized approach include how, or whether, to distinguish foreground and background markers. How to indicate relative importance of different segments (perhaps importance can be coded and displayed on target genotypes by line thickness or color intensity).
</li></ul>
</li></ul>
<ul><li>Order, group and filter test genotypes
<ul><li>Compute ranking scores of test genotypes using weighted averages of distances to target genotypes. A problem is to select weights for different loci. Critical loci to be retained or introgressed should have high weights while background loci should have lower weights. One possibility is to compute separate distances for background and foreground markers then filter test lines on an acceptable threshold of foreground proximity and rank remaining lines on distance to background markers.
</li><li>One could compute pairwise distances between lines (including test lines, target genotypes and parental lines) based on genetic and phenotypic values and show ordination plots or dendrograms.
</li><li>Filter, sort and scroll genotype displays for test lines in proximity to target and parental lines.
</li></ul>
</li></ul>
<ul><li>Choose lines and crossing schemes for further development
<ul><li>If the filtered set of test lines is relatively small these can be inspected individually to select a few for backcrossing, intercrossing or selfing. Many factors are used in the final choice including any available phenotypic data, as well as the number and specificity of crossovers required to approach the target genotype. It will be important to capture these decision rules from users and build them into the analytical tools.
</li><li>Users could mark regions requiring crossover to approach the target and select a crossing scheme then the tool could compute the (map based) probability of observing the required crossover events.
</li><li>Best lines are selected and sent to the Molecular LIMS as crossing lists for the next cycle.
</li><li>Markers fixed in recipient populations should also be identified so that they do not need to be processed for subsequent generations.
</li></ul>
</li></ul>
<a name="Data_Loading_Application" id="Data_Loading_Application"></a><h2> <span class="mw-headline">Data Loading Application</span></h2>
<p>The Data Loading Application is required to allow users to identify data to be published to a central database for future use. At ICRISAT this application is need to feed selected data to ICRIS. At CIMMYT the update procedures from ICIS local to central will be used.
</p>
<a name="Development_Decisions" id="Development_Decisions"></a><h2> <span class="mw-headline">Development Decisions</span></h2>
<ul><li>Development teams at ICRISAT and CIMMYT have agreed to use the Eclipse RCP environment to develop the tools. 
</li><li>Existing or new projects will be set up on CropForge using Subversion for code maintenance and exchange.
</li><li>The GCP Wiki will be used to exchange design and development information until such time as a more formal bug and feature tracking system is required.
</li><li>Development of MBDT and MOSEL can progress under the control of individual teams, but prototypes will be exchanges as soon as possible and each team should act as alpha testers for the other.
</li><li>Common code modules should be identified as soon as possible (during the design phase) and decisions on which team should contribute the module should be taken.
</li><li>The first common code decision involved a graphical heat map display tool. Teams will collaborate on researching available, existing components which allow the display of large matrices using heat strip technology. Sufficient control is required to allow ordering, spacing and filtering of rows and columns.
</li></ul>

<!-- 
NewPP limit report
Preprocessor node count: 8/1000000
Post-expand include size: 0/2097152 bytes
Template argument size: 0/2097152 bytes
Expensive parser function count: 0/100
-->
<div class="printfooter">
Retrieved from "<a href="http://localhost../../../../articles/s/p/3/SP3_Use_Cases_2008_Document_1_5040.html">http://localhost../../../../articles/s/p/3/SP3_Use_Cases_2008_Document_1_5040.html</a>"</div>
						<!-- end content -->
						<div class="visualClear"></div>
		</div>
	</div>
		</div>
		<div id="column-one">
	<div id="p-cactions" class="portlet">
		<h5>Views</h5>
		<div class="pBody">
			<ul>
	
				 <li id="ca-nstab-main" class="selected"><a href="../../../../articles/s/p/3/SP3_Use_Cases_2008_Document_1_5040.html" title="View the content page [c]" accesskey="c">Page</a></li>
				 <li id="ca-talk" class="new"><a href="../../../../articles/s/p/3/Talk%7ESP3_Use_Cases_2008_Document_1_0178.html" title="Discussion about the content page [t]" accesskey="t">Discussion</a></li>
				 <li id="ca-viewsource"><a href="../../../../articles/s/p/3/SP3_Use_Cases_2008_Document_1_5040.html" title="This page is protected.&#10;You can view its source [e]" accesskey="e">View source</a></li>
				 <li id="ca-history"><a href="../../../../articles/s/p/3/SP3_Use_Cases_2008_Document_1_5040.html" title="Past versions of this page [h]" accesskey="h">History</a></li>			</ul>
		</div>
	</div>
	<div class="portlet" id="p-personal">
		<h5>Personal tools</h5>
		<div class="pBody">
			<ul>
				<li id="pt-login"><a href="../../../../articles/u/s/e/Special%7EUserLogin_94cd.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</a></li>
			</ul>
		</div>
	</div>
	<div class="portlet" id="p-logo">
		<a style="background-image: url(../../../../misc/icis-wiki3.png);" href="../../../../index.html" title="Visit the main page [z]" accesskey="z"></a>
	</div>
	<script type="text/javascript"> if (window.isMSIE55) fixalpha(); </script>
	<div class='generated-sidebar portlet' id='p-navigation'>
		<h5>Navigation</h5>
		<div class='pBody'>
			<ul>
				<li id="n-mainpage"><a href="../../../../index.html" title="Visit the main page [z]" accesskey="z">Main Page</a></li>
				<li id="n-portal"><a href="../../../../articles/c/o/m/ICISWiki%7ECommunity_Portal_7449.html" title="About the project, what you can do, where to find things">Community portal</a></li>
				<li id="n-recentchanges"><a href="../../../../articles/r/e/c/Special%7ERecentChanges_e0d0.html" title="The list of recent changes in the wiki [r]" accesskey="r">Recent changes</a></li>
				<li id="n-help"><a href="../../../../articles/c/o/n/Help%7EContents_22de.html" title="The place to find out">Help</a></li>
			</ul>
		</div>
	</div>
	<div id="p-search" class="portlet">
		<h5><label for="searchInput">Search</label></h5>
		<div id="searchBody" class="pBody">
			<form action="../../../../articles/s/e/a/Special%7ESearch_a8e5.html" id="searchform"><div>
				<input id="searchInput" name="search" type="text" title="Search ICISWiki [f]" accesskey="f" value="" />
				<input type='submit' name="go" class="searchButton" id="searchGoButton"	value="Go" title="Go to a page with this exact name if exists" />&nbsp;
				<input type='submit' name="fulltext" class="searchButton" id="mw-searchButton" value="Search" title="Search the pages for this text" />
			</div></form>
		</div>
	</div>
	<div class="portlet" id="p-tb">
		<h5>Toolbox</h5>
		<div class="pBody">
			<ul>
				<li id="t-whatlinkshere"><a href="../../../../articles/w/h/a/Special%7EWhatLinksHere_SP3_Use_Cases_2008_Document_1_6a5e.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</a></li>
				<li id="t-recentchangeslinked"><a href="../../../../articles/r/e/c/Special%7ERecentChangesLinked_SP3_Use_Cases_2008_Document_1_b661.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</a></li>
<li id="t-specialpages"><a href="../../../../articles/s/p/e/Special%7ESpecialPages_f12b.html" title="List of all special pages [q]" accesskey="q">Special pages</a></li>
				<li id="t-print"><a href="../../../../articles/s/p/3/SP3_Use_Cases_2008_Document_1_5040.html" title="Printable version of this page [p]" accesskey="p">Printable version</a></li>				<li id="t-permalink"><a href="../../../../articles/s/p/3/SP3_Use_Cases_2008_Document_1_5040.html" title="Permanent link to this version of the page">Permanent link</a></li>			</ul>
		</div>
	</div>
		</div><!-- end of the left (by default at least) column -->
			<div class="visualClear"></div>
			<div id="footer">
				<div id="f-poweredbyico"><a href="http://www.mediawiki.org/"><img src="../../../../skins/common/images/poweredby_mediawiki_88x31.png" alt="Powered by MediaWiki" /></a></div>
				<div id="f-copyrightico"><a href="../../../../../COPYING.html"><img src="/icis/skins/common/images/gnu-fdl.png" alt='GNU Free Documentation License 1.2' /></a></div>
			<ul id="f-list">
					<li id="credits">This page was last modified 01:45, 5 March 2008 by <a href="../../../../articles/g/m/c/User%7EGmclaren_b845.html" title="User:Gmclaren">Graham McLaren</a>. </li>
					<li id="copyright">Content is available under <a href="../../../../../COPYING.html" class="external " title="../../../../../COPYING.html">GNU Free Documentation License 1.2</a>.</li>
					<li id="privacy"><a href="../../../../articles/p/r/i/ICISWiki%7EPrivacy_policy_0cbf.html" title="ICISWiki:Privacy policy">Privacy policy</a></li>
					<li id="about"><a href="../../../../articles/a/b/o/ICISWiki%7EAbout_53aa.html" title="ICISWiki:About">About ICISWiki</a></li>
					<li id="disclaimer"><a href="../../../../articles/g/e/n/ICISWiki%7EGeneral_disclaimer_9129.html" title="ICISWiki:General disclaimer">Disclaimers</a></li>
			</ul>
		</div>
</div>

		<script type="text/javascript">if (window.runOnloadHook) runOnloadHook();</script>
<!-- Served in 294.925 secs. --></body></html>
