<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
	<head>
		<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
		<meta http-equiv="Content-Style-Type" content="text/css" />
		<meta name="generator" content="MediaWiki 1.14.0" />
		<meta name="keywords" content="01&amp;02 Introduction" />
		<link rel="shortcut icon" href="../../../../misc/favicon.ico" />
		<link rel="search" type="application/opensearchdescription+xml" href="../../../../../opensearch_desc.php" title="ICISWiki (en)" />
		<link title="Creative Commons" type="application/rdf+xml" href="../../../../articles/0/1/%26/01%2602_Introduction_d4c8.html" rel="alternate meta" />
		<link rel="copyright" href="../../../../../COPYING.html" />
		<link rel="alternate" type="application/rss+xml" title="ICISWiki RSS Feed" href="http://localhost../../../../articles/r/e/c/Special%7ERecentChanges_e0d0.html" />
		<link rel="alternate" type="application/atom+xml" title="ICISWiki Atom Feed" href="http://localhost../../../../articles/r/e/c/Special%7ERecentChanges_e0d0.html" />
		<title>01&amp;02 Introduction - ICISWiki</title>
		<link rel="stylesheet" href="../../../../skins/common/shared.css?195" type="text/css" media="screen" />
		<link rel="stylesheet" href="../../../../skins/common/commonPrint.css?195" type="text/css" media="print" />
		<link rel="stylesheet" href="../../../../skins/monobook/main.css?195" type="text/css" media="screen" />
		<!--[if lt IE 5.5000]><link rel="stylesheet" href="../../../../skins/monobook/IE50Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<!--[if IE 5.5000]><link rel="stylesheet" href="../../../../skins/monobook/IE55Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<!--[if IE 6]><link rel="stylesheet" href="../../../../skins/monobook/IE60Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<!--[if IE 7]><link rel="stylesheet" href="../../../../skins/monobook/IE70Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<link rel="stylesheet" href="../../../../skins/../../../../raw/MediaWiki%7ECommon.css?195" type="text/css" />
		<link rel="stylesheet" href="../../../../skins/../../../../raw/MediaWiki%7EPrint.css?195" type="text/css" media="print" />
		<link rel="stylesheet" href="../../../../skins/../../../../raw/MediaWiki%7EMonobook.css?195" type="text/css" />
		<link rel="stylesheet" href="../../../../skins/../../../../raw/gen.css?195" type="text/css" />
		<!--[if lt IE 7]><script type="text/javascript" src="../../../../skins/common/IEFixes.js?195"></script>
		<meta http-equiv="imagetoolbar" content="no" /><![endif]-->

		<script type= "text/javascript">/*<![CDATA[*/
		var skin = "monobook";
		var stylepath = "../../../../skins";
		var wgArticlePath = "../../../../$1";
		var wgScriptPath = "../../../../..";
		var wgScript = "/icis/index.php";
		var wgVariantArticlePath = false;
		var wgActionPaths = {};
		var wgServer = "http://localhost";
		var wgCanonicalNamespace = "";
		var wgCanonicalSpecialPageName = false;
		var wgNamespaceNumber = 0;
		var wgPageName = "01\x2602_Introduction";
		var wgTitle = "01\x2602 Introduction";
		var wgAction = "view";
		var wgArticleId = "1015";
		var wgIsArticle = true;
		var wgUserName = null;
		var wgUserGroups = null;
		var wgUserLanguage = "en";
		var wgContentLanguage = "en";
		var wgBreakFrames = false;
		var wgCurRevisionId = "15684";
		var wgVersion = "1.14.0";
		var wgEnableAPI = true;
		var wgEnableWriteAPI = true;
		var wgSeparatorTransformTable = ["", ""];
		var wgDigitTransformTable = ["", ""];
		var wgRestrictionEdit = [];
		var wgRestrictionMove = [];
		/*]]>*/</script>

		<script type="text/javascript" src="../../../../skins/common/wikibits.js?195"><!-- wikibits js --></script>
		<!-- Head Scripts -->
		<script type="text/javascript" src="../../../../raw/gen.js"><!-- site js --></script>
	</head>
<body class="mediawiki ltr ns-0 ns-subject page-01_02_Introduction skin-monobook">
	<div id="globalWrapper">
		<div id="column-content">
	<div id="content">
		<a name="top" id="top"></a>
				<h1 id="firstHeading" class="firstHeading">01&amp;02 Introduction</h1>
		<div id="bodyContent">
			<h3 id="siteSub">From ICISWiki</h3>
			<div id="contentSub"></div>
									<div id="jump-to-nav">Jump to: <a href="#column-one">navigation</a>, <a href="#searchInput">search</a></div>			<!-- start content -->
			<table id="toc" class="toc" summary="Contents"><tr><td><div id="toctitle"><h2>Contents</h2></div>
<ul>
<li class="toclevel-1"><a href="#USING_ICIS_TO_MANAGE_INFORMATION_IN_BREEDING_PROGRAMS"><span class="tocnumber">1</span> <span class="toctext">USING ICIS TO MANAGE INFORMATION IN BREEDING PROGRAMS</span></a>
<ul>
<li class="toclevel-2"><a href="#INTRODUCTION"><span class="tocnumber">1.1</span> <span class="toctext">INTRODUCTION</span></a></li>
<li class="toclevel-2"><a href="#THE_INTERNATIONAL_CROP_INFORMATION_SYSTEM_.28ICIS.29"><span class="tocnumber">1.2</span> <span class="toctext">THE INTERNATIONAL CROP INFORMATION SYSTEM (ICIS)</span></a>
<ul>
<li class="toclevel-3"><a href="#Data_Model_for_the_Genealogy_Management_System"><span class="tocnumber">1.2.1</span> <span class="toctext">Data Model for the Genealogy Management System</span></a></li>
<li class="toclevel-3"><a href="#Data_Model_for_the_Data_Management_System"><span class="tocnumber">1.2.2</span> <span class="toctext">Data Model for the Data Management System</span></a></li>
</ul>
</li>
<li class="toclevel-2"><a href="#AN_ICIS_EXAMPLE:THE_INTERNATIONAL_RICE_INFORMATION_SYSTEM_.28IRIS.29"><span class="tocnumber">1.3</span> <span class="toctext">AN ICIS EXAMPLE:THE INTERNATIONAL RICE INFORMATION SYSTEM (IRIS)</span></a></li>
<li class="toclevel-2"><a href="#LINKS_TO_GLOBAL_CROP_INFORMATICS_RESOURCES"><span class="tocnumber">1.4</span> <span class="toctext">LINKS TO GLOBAL CROP INFORMATICS RESOURCES</span></a></li>
<li class="toclevel-2"><a href="#ICIS_SOFTWARE_AVAILABILITY"><span class="tocnumber">1.5</span> <span class="toctext">ICIS SOFTWARE AVAILABILITY</span></a></li>
<li class="toclevel-2"><a href="#REFERENCES"><span class="tocnumber">1.6</span> <span class="toctext">REFERENCES</span></a></li>
</ul>
</li>
</ul>
</td></tr></table><script type="text/javascript"> if (window.showTocToggle) { var tocShowText = "show"; var tocHideText = "hide"; showTocToggle(); } </script>
<a name="USING_ICIS_TO_MANAGE_INFORMATION_IN_BREEDING_PROGRAMS" id="USING_ICIS_TO_MANAGE_INFORMATION_IN_BREEDING_PROGRAMS"></a><h1> <span class="mw-headline">USING ICIS TO MANAGE INFORMATION IN BREEDING PROGRAMS</span></h1>
<a name="INTRODUCTION" id="INTRODUCTION"></a><h2> <span class="mw-headline">INTRODUCTION</span></h2>
<p>International germplasm exchange was the engine of the Green Revolution.  In the past, however, much of the important information generated from this exchange was only accessible locally, i.e. in field books or researchers’ files. Although major international initiatives for germplasm collection and conservation followed the Green Revolution, much collected materials are still not used because information related to their characterization remain difficult to access.  As a result, the potential impact upon agriculture has not yet been fully realized.  
</p><p>The free exchange of information, through international crop information systems, should now provide the foundation for a Second Green Revolution that adds value to germplasm by seamlessly uniting its conservation, evaluation, utilization and exchange. Furthermore, new technologies in molecular biology and genomics mean that traditional phenotypic information must be linked to large quantities of sequence and genetic information so that functional genomics and allele mining activities can speed up germplasm enhancement.
</p><p>CIMMYT devised an information strategy and developed software on a mainframe computer during the 1980’s to facilitate the unambiguous identification of wheat germplasm, thereby establishing links between information coming from different sources.  The read-only International Wheat Information System (IWIS) compact disk (Fox et al., 1996) duplicated data querying capabilities and some of the genealogical diagnostics of the mainframe version In 1995, CIMMYT and IRRI surveyed other CGIAR centers to establish a project to develop an International Crop Information System (Fox and Skovmand, 1996) applicable to a wide range of crops. Extensive communication among CGIAR centers highlighted the economies to be gained by collaborating on the development of an information system that could be used for many crops.  
</p>
<a name="THE_INTERNATIONAL_CROP_INFORMATION_SYSTEM_.28ICIS.29" id="THE_INTERNATIONAL_CROP_INFORMATION_SYSTEM_.28ICIS.29"></a><h2> <span class="mw-headline">THE INTERNATIONAL CROP INFORMATION SYSTEM (ICIS)</span></h2>
<p>Several CGIAR centers, National Agricultural Research Systems and Advanced Research Institutes are collaborating to develop ICIS as a generic system that will accommodate all data sources for any crop and breeding system. The vision of ICIS is to integrate different data types in both private and public datasets into a single information system and provide specialist views and applications that operate on the single integrated data platform. After all phases of development are complete, ICIS will support a range of activities – from germplasm conservation, evaluation, functional genomics, allele mining, breeding, testing and release.  Data will be accessible from CD-ROM or the WWW, and users may either adopt the complete system or link only to its innovative genealogical features.  
</p><p>The driving force behind ICIS is accessing and sharing data rather than providing analytical and statistical tools. This is because the major bottleneck to intelligent data integration and utilization is not statistical software, but rather the drudgery of finding, extracting, preparing and managing the data.  ICIS exports managed data in formats designed to make full use of external statistical software.
</p><p><b>ICIS currently comprises:</b>
</p>
<ol><li>	a genealogy management component to capture and process historical genealogies as well as to maintain evolving pedigrees, and to provide the basis for unique identification and internationally accepted nomenclature conventions for each crop;
</li><li>	a data management component for genetic, phenotypic, and environmental data generated by evaluation and testing, as well as for providing links to genomic maps;
</li><li>	links to Geographical Information Systems (GIS) that can manipulate all data associated with latitude and longitude, e.g. international, regional and national testing programs;
</li><li>	applications for maintaining, updating and correcting genealogy records and tracking changes and updates;
</li><li>	applications for producing field books and managing sets of breeding material and for diagnostics such as coefficients of parentage and genetic profiles for planning crosses;
</li><li>	tools to add new breeding methods, new data fields and new traits;
</li><li>	tools for submitting data to crop curators and for distributing data updates via CD-ROM and electronic networking;
</li><li>	project management capabilities including basic experimental designs; and links to program management and monitoring systems;
</li><li>	(RICE VERSION) integrated interfaces to a growing range of structural and functional genomics datasets.
</li></ol>
<p><b>Remote Users</b>
</p><p>One of the innovative features of ICIS is that it permits independent users to integrate their own local data with public central data. ICIS does this by allowing read-only access to the central database for a particular crop and supporting a local copy of the ICIS data model where the local data is stored.  Apart from providing user-friendly access to the data, so that crop scientists can make informed decisions, the system provides a local data management system for the user and captures relevant data for the crop. Periodic updates to the central database by users make their data available to all other users as well as browsers of the central database.
</p><p>The data model and database system of ICIS are designed for maximum flexibility to cater for as wide a range of crops as possible. The model must be independently adopted for a specific crop and data entered to create an independent system for that crop.
</p><p><br />
</p>
<a name="Data_Model_for_the_Genealogy_Management_System" id="Data_Model_for_the_Genealogy_Management_System"></a><h3> <span class="mw-headline">Data Model for the Genealogy Management System</span></h3>
<p>The core of ICIS is a common genealogical data model called the Genealogy Management System (GMS) which is generically designed to accommodate a wide range of crops. The functions of GMS are to:
</p>
<ul><li>assign and maintain unique germplasm identification,
</li><li>retain and manage information on genealogy and 
</li><li>manage nomenclature and chronology of germplasm development.
</li></ul>
<p>Each germplasm entity is identified by a GERMPLASM_ID (GID). The logical connection between a GID and a packet of seeds or other propagating materials is that different packets which germplasm specialists would not mix get different GIDs. Information on method, location, date of genesis and other attributes is managed through the data model shown in Figure 1. Each germplasm record is linked to its progenitors through their GIDs.
</p><p>Germplasm is divided into two categories – generative and derivative. Generative germplasm is produced by methods such as crossing or mutation that tend to increase and combine genetic variation. Derivative germplasm is produced by methods such as selection which tend to refine, target and reduce genetic variation or maintain genetic status through management methods such as seed increase or conservation.  Germplasm produced by generative methods may have any number of progenitors.  Derivative germplasm is derived from a single germplasm source.
</p><p>Each instance of germplasm, whether generative or derivative, falls into a single germplasm group identified by a group source.  For germplasm produced by a generative process (such as an F1 from a cross) or for germplasm of unknown genesis (such as a land race) this group is defined by its own GID. Germplasm produced by derivative or management methods retains the group ID of its source, although the group is often known when the source is not. For example, the cross may be known even when the line source is missing.
</p><p><br />
Figure 1. Data Model for the ICIS Genealogy Management System (GMS)
</p><p><br />
Method definitions are stored with complete documentation, including bibliographic references.  Some methods depend on parameters, which may vary each time the method is used, such as the number and mixing proportions of parents in the generation of a population.  These parameters can be defined by assigning an attribute to the GID of the germplasm being produced.  Attributes are flexible and user-definable data fields.
</p><p>Germplasm gets a multitude of labels during the development and release process.  These are all tracked as NAMES in GMS.  One name must be identified as the preferred name for display purposes. For a given genetic entity, different preferred names can be used in local and central applications. Names may contain imbedded information, and this can be made accessible to application programs for specific name types by specifying a format for the name.
</p><p>Attributes are text variables used to store information about the genesis, genealogy, nomenclature or chronology of germplasm. Attribute types are defined and described as USER_DEFINED_FIELDS.  Like names, attributes may contain imbedded information in the form of sub-fields or variables within the attribute text.
</p><p>Location information is stored to record the origin or destination of germplasm or the location of sites where information or data on germplasm was collected.  Locations can be as precise as fields or plots or as large as countries or even regions. Locations are defined by name and can be associated with latitude and longitude points or polygons to allow spatial analysis.
</p>
<a name="Data_Model_for_the_Data_Management_System" id="Data_Model_for_the_Data_Management_System"></a><h3> <span class="mw-headline">Data Model for the Data Management System </span></h3>
<p>The Functions of the Data Management System (DMS) are to:
</p>
<ul><li>store and manage documented and structured data from genetic resource, variety evaluation and crop improvement studies;
</li><li>link data to specialized data sources such as GMS, soil and climate databases and; 
</li><li>facilitate inquiries, searches and data extraction across studies according to a structured criteria for data selection.
</li></ul>
<p>DMS can accommodate raw observed data, derived data and summary statistics. Data may have continuous or discrete numeric values, text or categorical character values. For example, observations on disease resistance or nutrient efficiency of a genotype can be numerical measurements, scored or calculated indices or text data. More complex forms of data, such as pictures or documents will also be considered. The basic data model of DMS is shown in Figure 2 which shows the linkages between the entities described below.
</p><p>A STUDY is the basic, reportable unit of research. It is synonymous with the notions of experiment, nursery or survey. Since DMS must deal with any of these, we will use the term study. A study is characterized by a set of scientific objectives and testable hypotheses and results in the collection of one or more data sets. The division of data into sets is usually motivated by convenience. For example, data collected from different sampling scales is most conveniently treated in different data sets. Similarly, data collected at different times or from different locations are also often treated as different data sets, although it is feasible to treat these divisions in a single data set. The point is that DMS is flexible enough to manage data in all the ways that researchers require.
</p><p><br />
Figure 2. Data Model for the ICIS Data Management System (DMS)
</p><p><br />
FACTORS are classifying variables in a study that take values from finite sets of discrete LEVELS. These levels are usually labeled in some ways to document the source and context of the data by expressing the conditions under which the data were collected or derived. Examples of this are the names of treatments or design structures applying to the unit or units from which data are recorded, or conditions such as the time and location of measurement. These LABELS are usually listed in columns in the data set. The study itself is treated in the data model as a factor with exactly one level – the study name.  Hence, every study has at least one factor. A single factor is often represented by more than one set of LABELs. 
</p><p>Factors are named and described in each study. They have three main attributes – the PROPERTY of the experimental material or survey units being manipulated or stratified, the METHOD or procedure by which the levels are applied, and the SCALE or measurement units in which the levels are expressed. All levels of a particular factor are expressed in the same scale. Names of factors are consistent within studies and equivalent factors are linked across studies through common PROPERTIES. PROPERTY entries are subject to a controlled vocabulary to facilitate this linkage. 
</p><p>Data sources such as field objects or sampling units are identified by combinations of levels of design or sampling factors. Data values such as treatment means are associated with level combinations of treatment factors which do not correspond to field objects but which can be thought of as data sources. Both types of data sources, field objects, and treatment combinations are referred to as OBSERVATION UNITS. Observation units are conceptually equivalent to rows in a serially structured spreadsheet.
</p><p>Each study involves the recording of data for one or more properties of some observation units. The data being recorded are described by VARIATES and are often represented as data columns in spreadsheets. Each VARIATE has the same three attributes as a FACTOR, i.e. the PROPERTY or trait being measured, the METHOD or procedure by which the value is observed or derived, and the SCALE or measurement units in which the value is expressed. VARIATES are named and described within studies and the name should be consistent throughout a study. The common vocabulary of PROPERTY links VARIATES across studies in the same way that FACTORS are linked across studies.
</p><p>The central entity in the DMS data model is the DATUM which links to exactly one OBSERVATION UNIT and exactly one VARIATE. The DATUM is conceptually equivalent to a cell in a variate column of a spreadsheet or field book. The most important attribute of a datum is its VALUE, i.e. the recorded value of the associated VARIATE for the associated OBSERVATION UNIT. 
</p><p>Effects are sets of observation units in a study that are indexed by levels of subsets of the FACTORS in the study. Effects form natural hierarchies according to the nesting of their index-factor subsets. Data associated with different effects may result from data collection at different sampling scales or on different field objects, but they often arise by statistical amalgamation of data values over related units in lower level effects. 
</p><p><b>Correcting Data</b>
</p><p>Corrections and changes will inevitably occur in any database.  Only authorised users can make these changes, and all changes are logged so that the sequence of changes can be traced and can be ‘undone’ if required. Such logged functions are increasingly necessary to comply with requirements associated with establishing intellectual property rights of over germplasm.
</p><p>Changes commonly occur when new information about an existing germplasm record is entered into a local database.  If the existing record is in the local database, then the local user can complete the changes.  But if it is in the central database, then changes cannot be completed until the central database is updated. Verifying and completing requested changes is part of the update process, and sufficient information and justification needs to be recorded in the changes table to allow the process to be completed.
</p><p>Verification and completion of changes to the central database may take some time, but local users would like to see their changes reflected immediately.  This is achieved by the software that always checks the local CHANGES table for central changes and applies them at run time for the specific installation where they are recorded.
</p><p><b>User defined data</b>
</p><p>The ICIS data model is extremely flexible and can accommodate new data types as required. Users are able to define new relationships between germplasm by specifying new breeding methods. They can specify new attributes of the germplasm to be stored, types of names, location descriptors and traits, scales and methods for characterization and evaluation data.
</p><p><b>Stand Alone Software Modules</b>
</p><p>Components of ICIS include a Genealogy Management System (GMS), Set Generation Module (SETGEN) including the External Pedigree Input Tool (EPIT), Field Book Module (FLDBK), Trait Management System (TMS), Data Management System (DMS), a Work Book (WRKBK) for data input and query and a Data Retriever for cross-study data queries (RTV).  The first four modules focus on germplasm and management of genealogy and nomenclature. The next three handle the management of evaluation and characterization data, and the Retriever provides access to both raw and processed data.
</p>
<a name="AN_ICIS_EXAMPLE:THE_INTERNATIONAL_RICE_INFORMATION_SYSTEM_.28IRIS.29" id="AN_ICIS_EXAMPLE:THE_INTERNATIONAL_RICE_INFORMATION_SYSTEM_.28IRIS.29"></a><h2> <span class="mw-headline">AN ICIS EXAMPLE:THE INTERNATIONAL RICE INFORMATION SYSTEM (IRIS)</span></h2>
<p>The GMS of the rice implementation of ICIS – the International Rice Information System (IRIS) – stores information on more than one million varieties, breeding lines, and accessions of rice. This allows pedigree analysis to trace germplasm flows and relationships between lines that can be used to improve evaluation estimates or plan crop improvement programs. The IRIS DMS contains 5 million data values from over 500 studies from breeding, screening and international testing trials. This allows integrative analysis over different environments.
</p><p><b>Access via the World wide Web</b>
</p><p>A WWW interface for ICIS has been developed and deployed for IRIS (<a href="http://www.iris.irri.org" class="external free" title="http://www.iris.irri.org" rel="nofollow">http://www.iris.irri.org</a>). This will extend the functionality of ICIS into a broader range of biological datasets including molecular genetic (QTL), genome sequence, mutant, gene expression array, proteomics and allele mining data.  Each class of data is accessed through a specialized “view” which queries and displays the target data type in its biological context and cross-references it to related data including traditional germplasm and evaluation data.  
</p><p>Special effort is being made to incorporate controlled vocabularies and ontologies from the Gene Ontology Consortium (www.geneontology.org) and Plant Ontology Consortium (www.plantontology.org), in particular, to capture traits and phenotyping. One particular context in exploiting such ontologies is the “MutantView” with which researchers can specify mutant phenotypes as a means of identifying specific mutant stocks.  Entries in the IRIS GeneView are cross-indexed to GO ontologies where feasible.
</p>
<a name="LINKS_TO_GLOBAL_CROP_INFORMATICS_RESOURCES" id="LINKS_TO_GLOBAL_CROP_INFORMATICS_RESOURCES"></a><h2> <span class="mw-headline">LINKS TO GLOBAL CROP INFORMATICS RESOURCES</span></h2>
<p>Other databases of agricultural information have close links with IRIS (or ICIS in general). The System-wide Information Network for Genetic Resources (SINGER;<a href="http://www.singer.cgiar.org" class="external free" title="http://www.singer.cgiar.org" rel="nofollow">http://www.singer.cgiar.org</a>) aims to provide global access to genetic resources data across all CGIAR mandated crops and commodities. Germplasm records in ICIS that relate to accessions in the CGIAR collections are linked with SINGER so the ICIS can provide information on the utilization and deployment of those genetic resources.  Each individual ICIS database manages data for a particular crop and generally does not share data with other crops. The integrating role for genetic resources information across crops is played by SINGER.  IRIS also has collaboration with the USDA sponsored GrainGenes and Gramene database initiatives; with the International Rice Genome Sequencing Project (RGP, TIGR and other partners), with Oryzabase (Japanese National Institute of Genetics) and with the Beijing Genome Institute.  Linking these implementations to ICIS will provide access to sequence information and molecular maps which will facilitate functional genomics and allele mining and integrate information across crops at the genomic end of the spectrum.
</p>
<a name="ICIS_SOFTWARE_AVAILABILITY" id="ICIS_SOFTWARE_AVAILABILITY"></a><h2> <span class="mw-headline">ICIS SOFTWARE AVAILABILITY</span></h2>
<p>ICIS is freely available and the ICIS System, including WWW site components, is being developed under the open source software model.  Where possible, the IRIS genomic and WWW interface components are being developed by the adoption of public open source database schemata and software components such as those from the GO consortium (e.g. the GO database schemata and the Amigo browser), GMOD (gbrowse and cmap at <a href="http://www.gmod.org" class="external free" title="http://www.gmod.org" rel="nofollow">http://www.gmod.org</a>) and the Open Bioinformatics Foundation (www.open-bio.org).
</p><p>To promote the open and collaborative spirit of the project, a community project site was established for the ICIS project at the bioinformatics.org WWW site, which also provides email list services for discussion groups about ICIS. Community participation is also promoted by open annual ICIS workshops.  The latest information about the project may be accessed at www.icis.cgiar.org.
</p>
<a name="REFERENCES" id="REFERENCES"></a><h2> <span class="mw-headline">REFERENCES</span></h2>
<p>Fox, P.N., Lopez, C., Skovmand, B., Sanchez, H., Herrera, R., White, J.W., Duveiller, E. and van Ginkel, M. 1996. International Wheat Information System (IWIS), Version 1. Mexico, D.F.: CIMMYT. On Compact Disk.
</p><p>Fox, P.N. and Skovmand, B. 1996. The International Crop Information System (ICIS) - Connects Genebank to Breeder to Farmer’s Field. In Plant Adaptation and Crop Improvement (eds M. Cooper and G.L. Hammer), CAB International.
</p><p>McLaren CG, Bruskiewich RM, Portugal AM, Cosico AB. 2005. The International Rice Information System: a platform for meta-analysis of rice crop data. Plant Physio. 139 (2): 637-642
</p>
<!-- 
NewPP limit report
Preprocessor node count: 10/1000000
Post-expand include size: 0/2097152 bytes
Template argument size: 0/2097152 bytes
Expensive parser function count: 0/100
-->
<div class="printfooter">
Retrieved from "<a href="http://localhost../../../../articles/0/1/%26/01%2602_Introduction_d4c8.html">http://localhost../../../../articles/0/1/%26/01%2602_Introduction_d4c8.html</a>"</div>
						<!-- end content -->
						<div class="visualClear"></div>
		</div>
	</div>
		</div>
		<div id="column-one">
	<div id="p-cactions" class="portlet">
		<h5>Views</h5>
		<div class="pBody">
			<ul>
	
				 <li id="ca-nstab-main" class="selected"><a href="../../../../articles/0/1/%26/01%2602_Introduction_d4c8.html" title="View the content page [c]" accesskey="c">Page</a></li>
				 <li id="ca-talk" class="new"><a href="../../../../articles/0/1/%26/Talk%7E01%2602_Introduction_a98d.html" title="Discussion about the content page [t]" accesskey="t">Discussion</a></li>
				 <li id="ca-viewsource"><a href="../../../../articles/0/1/%26/01%2602_Introduction_d4c8.html" title="This page is protected.&#10;You can view its source [e]" accesskey="e">View source</a></li>
				 <li id="ca-history"><a href="../../../../articles/0/1/%26/01%2602_Introduction_d4c8.html" title="Past versions of this page [h]" accesskey="h">History</a></li>			</ul>
		</div>
	</div>
	<div class="portlet" id="p-personal">
		<h5>Personal tools</h5>
		<div class="pBody">
			<ul>
				<li id="pt-login"><a href="../../../../articles/u/s/e/Special%7EUserLogin_94cd.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</a></li>
			</ul>
		</div>
	</div>
	<div class="portlet" id="p-logo">
		<a style="background-image: url(../../../../misc/icis-wiki3.png);" href="../../../../index.html" title="Visit the main page [z]" accesskey="z"></a>
	</div>
	<script type="text/javascript"> if (window.isMSIE55) fixalpha(); </script>
	<div class='generated-sidebar portlet' id='p-navigation'>
		<h5>Navigation</h5>
		<div class='pBody'>
			<ul>
				<li id="n-mainpage"><a href="../../../../index.html" title="Visit the main page [z]" accesskey="z">Main Page</a></li>
				<li id="n-portal"><a href="../../../../articles/c/o/m/ICISWiki%7ECommunity_Portal_7449.html" title="About the project, what you can do, where to find things">Community portal</a></li>
				<li id="n-recentchanges"><a href="../../../../articles/r/e/c/Special%7ERecentChanges_e0d0.html" title="The list of recent changes in the wiki [r]" accesskey="r">Recent changes</a></li>
				<li id="n-help"><a href="../../../../articles/c/o/n/Help%7EContents_22de.html" title="The place to find out">Help</a></li>
			</ul>
		</div>
	</div>
	<div id="p-search" class="portlet">
		<h5><label for="searchInput">Search</label></h5>
		<div id="searchBody" class="pBody">
			<form action="../../../../articles/s/e/a/Special%7ESearch_a8e5.html" id="searchform"><div>
				<input id="searchInput" name="search" type="text" title="Search ICISWiki [f]" accesskey="f" value="" />
				<input type='submit' name="go" class="searchButton" id="searchGoButton"	value="Go" title="Go to a page with this exact name if exists" />&nbsp;
				<input type='submit' name="fulltext" class="searchButton" id="mw-searchButton" value="Search" title="Search the pages for this text" />
			</div></form>
		</div>
	</div>
	<div class="portlet" id="p-tb">
		<h5>Toolbox</h5>
		<div class="pBody">
			<ul>
				<li id="t-whatlinkshere"><a href="../../../../articles/w/h/a/Special%7EWhatLinksHere_01%2602_Introduction_ba72.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</a></li>
				<li id="t-recentchangeslinked"><a href="../../../../articles/r/e/c/Special%7ERecentChangesLinked_01%2602_Introduction_3812.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</a></li>
<li id="t-specialpages"><a href="../../../../articles/s/p/e/Special%7ESpecialPages_f12b.html" title="List of all special pages [q]" accesskey="q">Special pages</a></li>
				<li id="t-print"><a href="../../../../articles/0/1/%26/01%2602_Introduction_d4c8.html" title="Printable version of this page [p]" accesskey="p">Printable version</a></li>				<li id="t-permalink"><a href="../../../../articles/0/1/%26/01%2602_Introduction_d4c8.html" title="Permanent link to this version of the page">Permanent link</a></li>			</ul>
		</div>
	</div>
		</div><!-- end of the left (by default at least) column -->
			<div class="visualClear"></div>
			<div id="footer">
				<div id="f-poweredbyico"><a href="http://www.mediawiki.org/"><img src="../../../../skins/common/images/poweredby_mediawiki_88x31.png" alt="Powered by MediaWiki" /></a></div>
				<div id="f-copyrightico"><a href="../../../../../COPYING.html"><img src="/icis/skins/common/images/gnu-fdl.png" alt='GNU Free Documentation License 1.2' /></a></div>
			<ul id="f-list">
					<li id="credits">This page was last modified 17:36, 25 February 2009 by <a href="../../../../articles/g/m/c/User%7EGmclaren_b845.html" title="User:Gmclaren">Graham McLaren</a>. Based on work by <a href="../../../../articles/t/m/e/User%7ETmetz_c741.html" title="User:Tmetz">Thomas Metz</a>,&#32;<a href="../../../../articles/w/e/u/User%7EWeusebio_4cb2.html" title="User:Weusebio">William Eusebio</a>&#32;and&#32;<a href="../../../../articles/j/o/h/User%7EJohn_ac14.html" class="new" title="User:John (page does not exist)">John</a>.</li>
					<li id="copyright">Content is available under <a href="../../../../../COPYING.html" class="external " title="../../../../../COPYING.html">GNU Free Documentation License 1.2</a>.</li>
					<li id="privacy"><a href="../../../../articles/p/r/i/ICISWiki%7EPrivacy_policy_0cbf.html" title="ICISWiki:Privacy policy">Privacy policy</a></li>
					<li id="about"><a href="../../../../articles/a/b/o/ICISWiki%7EAbout_53aa.html" title="ICISWiki:About">About ICISWiki</a></li>
					<li id="disclaimer"><a href="../../../../articles/g/e/n/ICISWiki%7EGeneral_disclaimer_9129.html" title="ICISWiki:General disclaimer">Disclaimers</a></li>
			</ul>
		</div>
</div>

		<script type="text/javascript">if (window.runOnloadHook) runOnloadHook();</script>
<!-- Served in 36.235 secs. --></body></html>
