<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
	<head>
		<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
		<meta http-equiv="Content-Style-Type" content="text/css" />
		<meta name="generator" content="MediaWiki 1.14.0" />
		<meta name="keywords" content="Mini-workshop 2007,ICISWiki,R-script for preliminary data analysis,Workshops and Meetings" />
		<link rel="shortcut icon" href="../../../../misc/favicon.ico" />
		<link rel="search" type="application/opensearchdescription+xml" href="../../../../../opensearch_desc.php" title="ICISWiki (en)" />
		<link title="Creative Commons" type="application/rdf+xml" href="../../../../articles/m/i/n/Mini-workshop_2007.html" rel="alternate meta" />
		<link rel="copyright" href="../../../../../COPYING.html" />
		<link rel="alternate" type="application/rss+xml" title="ICISWiki RSS Feed" href="http://localhost../../../../articles/r/e/c/Special%7ERecentChanges_e0d0.html" />
		<link rel="alternate" type="application/atom+xml" title="ICISWiki Atom Feed" href="http://localhost../../../../articles/r/e/c/Special%7ERecentChanges_e0d0.html" />
		<title>Mini-workshop 2007 - ICISWiki</title>
		<link rel="stylesheet" href="../../../../skins/common/shared.css?195" type="text/css" media="screen" />
		<link rel="stylesheet" href="../../../../skins/common/commonPrint.css?195" type="text/css" media="print" />
		<link rel="stylesheet" href="../../../../skins/monobook/main.css?195" type="text/css" media="screen" />
		<!--[if lt IE 5.5000]><link rel="stylesheet" href="../../../../skins/monobook/IE50Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<!--[if IE 5.5000]><link rel="stylesheet" href="../../../../skins/monobook/IE55Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<!--[if IE 6]><link rel="stylesheet" href="../../../../skins/monobook/IE60Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<!--[if IE 7]><link rel="stylesheet" href="../../../../skins/monobook/IE70Fixes.css?195" type="text/css" media="screen" /><![endif]-->
		<link rel="stylesheet" href="../../../../skins/../../../../raw/MediaWiki%7ECommon.css?195" type="text/css" />
		<link rel="stylesheet" href="../../../../skins/../../../../raw/MediaWiki%7EPrint.css?195" type="text/css" media="print" />
		<link rel="stylesheet" href="../../../../skins/../../../../raw/MediaWiki%7EMonobook.css?195" type="text/css" />
		<link rel="stylesheet" href="../../../../skins/../../../../raw/gen.css?195" type="text/css" />
		<!--[if lt IE 7]><script type="text/javascript" src="../../../../skins/common/IEFixes.js?195"></script>
		<meta http-equiv="imagetoolbar" content="no" /><![endif]-->

		<script type= "text/javascript">/*<![CDATA[*/
		var skin = "monobook";
		var stylepath = "../../../../skins";
		var wgArticlePath = "../../../../$1";
		var wgScriptPath = "../../../../..";
		var wgScript = "/icis/index.php";
		var wgVariantArticlePath = false;
		var wgActionPaths = {};
		var wgServer = "http://localhost";
		var wgCanonicalNamespace = "";
		var wgCanonicalSpecialPageName = false;
		var wgNamespaceNumber = 0;
		var wgPageName = "Mini-workshop_2007";
		var wgTitle = "Mini-workshop 2007";
		var wgAction = "view";
		var wgArticleId = "2403";
		var wgIsArticle = true;
		var wgUserName = null;
		var wgUserGroups = null;
		var wgUserLanguage = "en";
		var wgContentLanguage = "en";
		var wgBreakFrames = false;
		var wgCurRevisionId = "12760";
		var wgVersion = "1.14.0";
		var wgEnableAPI = true;
		var wgEnableWriteAPI = true;
		var wgSeparatorTransformTable = ["", ""];
		var wgDigitTransformTable = ["", ""];
		var wgRestrictionEdit = [];
		var wgRestrictionMove = [];
		/*]]>*/</script>

		<script type="text/javascript" src="../../../../skins/common/wikibits.js?195"><!-- wikibits js --></script>
		<!-- Head Scripts -->
		<script type="text/javascript" src="../../../../raw/gen.js"><!-- site js --></script>
	</head>
<body class="mediawiki ltr ns-0 ns-subject page-Mini-workshop_2007 skin-monobook">
	<div id="globalWrapper">
		<div id="column-content">
	<div id="content">
		<a name="top" id="top"></a>
				<h1 id="firstHeading" class="firstHeading">Mini-workshop 2007</h1>
		<div id="bodyContent">
			<h3 id="siteSub">From ICISWiki</h3>
			<div id="contentSub"></div>
									<div id="jump-to-nav">Jump to: <a href="#column-one">navigation</a>, <a href="#searchInput">search</a></div>			<!-- start content -->
			<p align="center"><b><a href="../../../../articles/i/c/i/ICISWiki_eda0.html" title="ICISWiki">Back to Main Page</a> &gt; <a href="../../../../articles/w/o/r/Workshops_and_Meetings_11a5.html" title="Workshops and Meetings">Workshops and Meetings</a></b></p>
<table id="toc" class="toc" summary="Contents"><tr><td><div id="toctitle"><h2>Contents</h2></div>
<ul>
<li class="toclevel-1"><a href="#ICIS_mini_-_workshop"><span class="tocnumber">1</span> <span class="toctext">ICIS mini - workshop</span></a>
<ul>
<li class="toclevel-2"><a href="#Participants"><span class="tocnumber">1.1</span> <span class="toctext">Participants</span></a></li>
<li class="toclevel-2"><a href="#Summary_of_Requirements_for_the_Project"><span class="tocnumber">1.2</span> <span class="toctext">Summary of Requirements for the Project</span></a>
<ul>
<li class="toclevel-3"><a href="#Basic_Queries_for_Genotype.2C_Phenotype_and_characterization_data"><span class="tocnumber">1.2.1</span> <span class="toctext">Basic Queries for Genotype, Phenotype and characterization data</span></a></li>
</ul>
</li>
<li class="toclevel-2"><a href="#Monday_19th_March_2007"><span class="tocnumber">1.3</span> <span class="toctext">Monday 19th March 2007</span></a>
<ul>
<li class="toclevel-3"><a href="#Visualisation_tools_for_genotype_and_phenotype_data."><span class="tocnumber">1.3.1</span> <span class="toctext">Visualisation tools for genotype and phenotype data.</span></a></li>
<li class="toclevel-3"><a href="#Use_of_DArT_data_in_selection_and_breeding"><span class="tocnumber">1.3.2</span> <span class="toctext">Use of DArT data in selection and breeding</span></a></li>
</ul>
</li>
<li class="toclevel-2"><a href="#Tuesday_20th_March_2007"><span class="tocnumber">1.4</span> <span class="toctext">Tuesday 20th March 2007</span></a>
<ul>
<li class="toclevel-3"><a href="#Storing_DArT_data_in_ICIS"><span class="tocnumber">1.4.1</span> <span class="toctext">Storing DArT data in ICIS</span></a></li>
<li class="toclevel-3"><a href="#DArT_:_Row_data_production_and_processing"><span class="tocnumber">1.4.2</span> <span class="toctext">DArT&nbsp;: Row data production and processing</span></a></li>
</ul>
</li>
<li class="toclevel-2"><a href="#Wednesday_21st_March_2007"><span class="tocnumber">1.5</span> <span class="toctext">Wednesday 21st March 2007</span></a>
<ul>
<li class="toclevel-3"><a href="#Pre-structuring_of_data_for_data_warehousing_to_increase_performance_on_data_querying"><span class="tocnumber">1.5.1</span> <span class="toctext">Pre-structuring of data for data warehousing to increase performance on data querying</span></a></li>
<li class="toclevel-3"><a href="#Query_Design_and_User_Interface_Refinement"><span class="tocnumber">1.5.2</span> <span class="toctext">Query Design and User Interface Refinement</span></a></li>
<li class="toclevel-3"><a href="#By_trait"><span class="tocnumber">1.5.3</span> <span class="toctext">By trait</span></a></li>
<li class="toclevel-3"><a href="#By_germplasm"><span class="tocnumber">1.5.4</span> <span class="toctext">By germplasm</span></a></li>
<li class="toclevel-3"><a href="#Occurrence_table"><span class="tocnumber">1.5.5</span> <span class="toctext">Occurrence table</span></a></li>
<li class="toclevel-3"><a href="#Features_required"><span class="tocnumber">1.5.6</span> <span class="toctext">Features required</span></a></li>
</ul>
</li>
<li class="toclevel-2"><a href="#Thursday_22nd_March_2007"><span class="tocnumber">1.6</span> <span class="toctext">Thursday 22nd March 2007</span></a>
<ul>
<li class="toclevel-3"><a href="#Recap_of_decisions_taken_so_far:"><span class="tocnumber">1.6.1</span> <span class="toctext">Recap of decisions taken so far:</span></a></li>
<li class="toclevel-3"><a href="#Molecular_selection"><span class="tocnumber">1.6.2</span> <span class="toctext">Molecular selection</span></a></li>
</ul>
</li>
<li class="toclevel-2"><a href="#Friday_23rd_March_2007"><span class="tocnumber">1.7</span> <span class="toctext">Friday 23rd March 2007</span></a></li>
</ul>
</li>
</ul>
</td></tr></table><script type="text/javascript"> if (window.showTocToggle) { var tocShowText = "show"; var tocHideText = "hide"; showTocToggle(); } </script>
<a name="ICIS_mini_-_workshop" id="ICIS_mini_-_workshop"></a><h2> <span class="mw-headline"> ICIS mini - workshop </span></h2>
<p>Date  &nbsp;: March 19-23, 2007<br />
Venue &nbsp;: University of Queensland, Brisbane, Australia<br />
Agenda&nbsp;: <a href="../../../../images/4/4b/ICIS_Workshop_Brisbane_March_2007.pdf" class="internal" title="ICIS Workshop Brisbane March 2007.pdf">pdf</a> 
</p>
<a name="Participants" id="Participants"></a><h3> <span class="mw-headline"> Participants </span></h3>
<ul><li> UQ
<ul><li>Ian DeLacy
</li><li>Sandra Micallef
</li><li>Vivi Arief
</li></ul>
</li><li>IRRI
<ul><li>Graham McLaren
</li><li>Thomas Metz
</li><li>Rowena Valerio
</li></ul>
</li><li>Triticarte
<ul><li>Grzegorz Uszynski
</li></ul>
</li><li>CRCVAW
<ul><li>Clare Johnson
</li></ul>
</li><li>ACPFG
<ul><li>Dave Edwards
</li></ul>
</li></ul>
<a name="Summary_of_Requirements_for_the_Project" id="Summary_of_Requirements_for_the_Project"></a><h3> <span class="mw-headline"> Summary of Requirements for the Project </span></h3>
<a name="Basic_Queries_for_Genotype.2C_Phenotype_and_characterization_data" id="Basic_Queries_for_Genotype.2C_Phenotype_and_characterization_data"></a><h4> <span class="mw-headline"> Basic Queries for Genotype, Phenotype and characterization data </span></h4>
<ol><li> Given germplasm list and clone list, produce genotype data
</li><li> Given germplasm list, trait list and study list, produce phenotypic data
<ol><li> Merge 1 and  2
</li></ol>
</li><li> List Management
<ol><li> Access germplasm list from central
</li><li> Access germplasm list from text file
</li><li> Access germplasm list from local
</li><li> Appending and merging of germplasm list
</li><li> Management of study list (folder system for studies and naming system or description for datasets)
</li><li> Management of trait list
</li></ol>
</li><li> Dataset download/ description of datasets
</li><li> Content Description 
<ol><li>Given a germplasm list, what studies were these germplasms evaluated?
</li><li>Given a germplasm list, what clones have geen genotyped?
</li><li>Given a germplasm list, what traits have been measured/
</li><li>Tabulate co-occurence of genotypes/traits in studies. How many genotypes are evaluated in each pair of studies?
</li></ol>
</li><li> A pedigree view/download tool - will be discussed with the team
</li></ol>
<p><br />
</p>
<hr />
<a name="Monday_19th_March_2007" id="Monday_19th_March_2007"></a><h3> <span class="mw-headline">Monday 19th March 2007</span></h3>
<a name="Visualisation_tools_for_genotype_and_phenotype_data." id="Visualisation_tools_for_genotype_and_phenotype_data."></a><h4> <span class="mw-headline">Visualisation tools for genotype and phenotype data.</span></h4>
<p>Rowena Valerio gave a demonstration of ghe Genotype Visualisation Tool (GVT) for marker data from David Marshall (SCRI, UK).  <a href="http://cropforge.org/projects/genotypevisual/" class="external free" title="http://cropforge.org/projects/genotypevisual/" rel="nofollow">http://cropforge.org/projects/genotypevisual/</a>
Graham McLaren gave an overview of a presentation given by Casper aan den Boom at ICIS 2006, about the Graphical Genotype Tool (GGT), developed by The Netherlands Plant Genomics Network.  This software is freely available but it is not open source.  www.pbr.wwr.nl/uk/resources
IRRI staff are working with the GGT developers to integrate it with ICIS.
</p><p>Dave Edwards, the newly appointed leader for the Brisbane node of the Australian Centre for Plant Functional Genomics (ACPFG), talked about his work with Marker QTL databases at DPI (Department of Primary Industries), and possible future collaborations with Dave Mathews of Grain Genes (<a href="http://wheat.pw.usda.gov" class="external free" title="http://wheat.pw.usda.gov" rel="nofollow">http://wheat.pw.usda.gov</a>)
</p><p>Graham McLaren gave another presentation which Kyle Braak (CIMMYT) presented at ICIS2006, about the Comparative Map and Trait Viewer (CMTV).  This tool uses the ISYS Integrated System (<a href="http://www.ncgr.org/isys/" class="external free" title="http://www.ncgr.org/isys/" rel="nofollow">http://www.ncgr.org/isys/</a>), which was developed by NCGR in collaboration with 4 CGIAR centres and which allows construction of consensus maps.  The CMTV tool uses ISYS to connect to other tools and data sources.
</p><p>Another visualisation tool is the Peditree software, which is also planned to be integrated with ICIS.( <a href="http://www.dpw.wageningen-ur.nl/pv/pub/Peditree/index.htm" class="external free" title="http://www.dpw.wageningen-ur.nl/pv/pub/Peditree/index.htm" rel="nofollow">http://www.dpw.wageningen-ur.nl/pv/pub/Peditree/index.htm</a>)
</p><p>COP (Coefficient of Parentage) Matrices&nbsp;: Ian DeLacy and Graham McLaren discussed some specifications this visualisation tool should have to be useful and efficient when handling large amount of data
</p><p>IRRI recommended lines web interface:  This tool was presented at ICIS2006 by Alex Cosico from IRRI.
Input:  A set of pedigrees which has a large amount of clean phenotypic data (multiple observations for each trait)
Aim: to be able to query the dataset to select best lines for seed.
</p>
<ol><li>select traits of interest
</li><li>specify criteria 
</li></ol>
<p>Values for criteria selection can be selected from histograms and sliders – these give a graphical visualisation of the searching space in the database so users can narrow down what they want out of the database.
(<a href="http://www.iris.irri.org/nursery/views/nursery.jsp" class="external free" title="http://www.iris.irri.org/nursery/views/nursery.jsp" rel="nofollow">http://www.iris.irri.org/nursery/views/nursery.jsp</a>)
</p>
<a name="Use_of_DArT_data_in_selection_and_breeding" id="Use_of_DArT_data_in_selection_and_breeding"></a><h4> <span class="mw-headline">Use of DArT data in selection and breeding</span></h4>
<p>Ian DeLacy gave a talk about Marker Assisted Breeding and integrating pedigree based whole genome analysis.
QTL detection and validation has limited power, is expensive and time consuming.
</p>
<ul><li>Genotypic data: Pedigree data, phenotype data, gene data
</li><li>Genotypic information:  Graphical genotypes, information on relatives
</li><li>What is needed to do the pedigree association analysis?
</li><li>Marker system
</li><li>Information management system
</li><li>Bioinformatics system (analysis)
</li><li>Decision support system for breeding (visualisation decision tools)
</li><li>And most importantly integration of systems
</li></ul>
<p>Overall gain is genetic gain
</p>
<ul><li>IBS:  Identity by State
</li><li>IBD: Identity by descent 
</li></ul>
<p>Association Mapping and Genetic Diversity for wheat improvement:
</p><p>Prerequisites:
</p>
<ol><li>Sufficient markers for structured characterization and for markers in specific genes/regions. Road marker generation: maize and barley are much ahead of wheat. DArT: genome specific. ~1000 markers expected.  It is low cost ($36/sample), but there is no complete map so far.  Lack of D genome markers and a question of reliability.
</li><li>Diverse material
</li><li>Determine ID - LD: Non random association of alleles at different loci.  Physical distance:  Evolutionary forces affecting LD are drift, admixture, population size and selection.  Not much is known about LD in wheat.  LD in other crops&nbsp;: Barley, Arabidopsis, Sorghum and Maize.
</li><li>Access population structure
</li></ol>
<p>Quantification by determining genetic diversity (not pedigree based)
(see paper by Gael Pressoir and Jianming Yu)
</p>
<hr />
<a name="Tuesday_20th_March_2007" id="Tuesday_20th_March_2007"></a><h3> <span class="mw-headline">Tuesday 20th March 2007 </span></h3>
<a name="Storing_DArT_data_in_ICIS" id="Storing_DArT_data_in_ICIS"></a><h4> <span class="mw-headline">Storing DArT data in ICIS</span></h4>
<p>Rowena Valerio gave a demonstration of how to store DArT data in ICIS. A tutorial for this exercise can be found on the ICISWIKI site&nbsp;:  <a href="http://cropwiki.irri.org/icis/index.php/Loading_of_DArT_Data_into_ICIS" class="external free" title="http://cropwiki.irri.org/icis/index.php/Loading_of_DArT_Data_into_ICIS" rel="nofollow">http://cropwiki.irri.org/icis/index.php/Loading_of_DArT_Data_into_ICIS</a>
</p><p>Components of ICIS include
GMS&nbsp;: Geneology Management System
DMS:  Data Management System
IMS:  Inventory Management System
GRIMS:  Genetic Resources Information Management System
GEMS: Gene Management System
</p><p>To load  the DArT data into GEMS and DMS, the workbook is used.
In the workbook, the Factors are GID, Marker ID and Clone ID.  The marker ID is generated by GEMS when loading – if it does not already exist for that GID, it is created – similar to how GIDs are generated in GMS).  The variate in the workbook is Allele.
The first step is to transfer the data in the spreadsheet (given by breeders) from ‘crosstab’ format into serial format.
The second step is to get the marker IDs (look them up in the database and  create them if they do not exist)
(Note for Rowena:  New IDs assigned should be clearly identified from ones already in the database, to make sure you don’t have any typing/spelling errors resulting in duplicate markers.)
The current limitation is that Excel spreadsheets are only limited to 64,000 rows.  Possible solutions to this problem is either having more than one spreadsheet in the workbook or load data straight from the matrix instead of creating the serial format and then loading.
Explanation of how data is stored in DMS and GEMS schema– Graham gave a presentation prepared by Arllet Portugal.
</p><p>Discussion followed on what DArT data values should be eg.
1 or P for presence
0 or A for absence
X or M or NULL for missing or cannot be determined
</p><p><br />
</p>
<a name="DArT_:_Row_data_production_and_processing" id="DArT_:_Row_data_production_and_processing"></a><h4> <span class="mw-headline">DArT&nbsp;: Row data production and processing</span></h4>
<pre>– Grzegorz Uszynski
</pre>
<p>Grzegorz Uszynski from Diversity Array Technologies gave a presentation on how the DArT data is produced.
</p><p>Definition of some terms used in the presentation:
</p>
<ul><li>Library&nbsp;: a collection of clones already in their database
</li><li>Targets:  Clones which are prepared in the same way as the library ones.
</li><li>Spots on image indicate level of hybridization
</li><li>System based on LAMP architecture (Linux, Apache, MySql, PHP)
</li></ul>
<p>LIMS system at Diversity Array Technologies is called DArT.db and is web enabled
</p>
<ul><li>It ensures proper clone and sample tracking
</li><li>Barcodes utilities
</li><li>Covers all lab technologies
</li><li>All major tasks are described as protocols
</li><li>Integrated image analysis component so analysis can be defined and results viewed online
</li><li>Includes support for client/order management
</li></ul>
<p>DArT soft is a toolbox for data analysis
</p>
<ul><li>2 modes of data anlaysis: local and network
</li><li>Local analysis: image analysis and polymorphism analysis
</li><li>A sample is a DNA extract described by plate location
</li><li>Organism/species, genotype/tissuename combination
</li><li>Genotype(name) is a synonym for sample
</li><li>Primary aim is to identify sample NOT to invent new terminology (ontology)
</li></ul>
<p>Marker identification
</p>
<ul><li>Marker is a DNA fragment identified as polymorphic in an analysed group of
</li></ul>
<p>samples
</p>
<ul><li>Always identified by clone name and clone ID
</li><li>Relations between these identifiers are 1:1:[0/1]
</li></ul>
<p>DArTsoft development and future
</p>
<ul><li>Working with Brian Cullis on data detrending and gradients corrections
</li><li>Working with NICTA on machine learning technologies
</li></ul>
<p>Discussion on data output format: XML vs XLS vs CSV
</p>
<ul><li>Excel most popular but limitations exist in terms of number of rows and columns
</li><li>XML – not normally readable (needs converters to read on its own), very large files but perfect for multidimensional approach
</li><li>CSV is easily portable and readable in other software packages such as Excel.  However multidimensional is problematic.  Quite compact in size and expansion means changes in format.
</li></ul>
<p>Discussion followed with Grsegorz about how markers are detected – discussion to be taken further with Andrejz Kiljan tomorrow.
</p>
<hr />
<a name="Wednesday_21st_March_2007" id="Wednesday_21st_March_2007"></a><h3> <span class="mw-headline">Wednesday 21st March 2007 </span></h3>
<a name="Pre-structuring_of_data_for_data_warehousing_to_increase_performance_on_data_querying" id="Pre-structuring_of_data_for_data_warehousing_to_increase_performance_on_data_querying"></a><h4> <span class="mw-headline">Pre-structuring of data for data warehousing to increase performance on data querying </span></h4>
<pre>Thomas Metz and Rowena Valerio
</pre>
<p>PostgreSQL is much faster than MySQL.  Comparing how long it takes for each to covert a spreadsheet from serial format to matrix format – PostgreSQL takes 2.5 seconds while MySQL takes 68 seconds.  The reason for this being that in PostgreSQL a dataset from a query is kept in memory and looped through next query instead of querying the whole database again and again.
</p><p>The phenotype and genotype data are in two separate tables.  These tables need to be joined by linking the GID fields.  
</p><p>Discussion on how to represent a GID which has been genotyped more than once and has different alleles each time.
</p><p>Data structure: 3 separate tables
</p>
<ul><li>List of clones
</li><li>List of GIDs 
</li><li>Phenotype data
</li></ul>
<ol><li>Get genotype data for a specified list of GIDs and clones
<ol><li>Specifying a GID list either 
<ul><li>by filtering or 
</li><li>by choosing a germplasm list
</li></ul>
</li><li>Selecting clones
<ul><li>Filtering by 
<ol><li>chromosome
</li><li>quality value
</li><li>PIC value
</li><li>by non missing&nbsp;%
</li></ol>
</li><li>by input list
</li></ul>
</li></ol>
</li><li>Get phenotype data for a specified list of germplasm and traits
<ul><li>germplasm specified same way as in (A)
</li><li>traits specified mostly through an input list (need to make a list of traits that are available in database, with a check box next to it)
</li></ul>
</li></ol>
<table border="1" cellpadding="0" cellspacing="0">
<tr>
<td colspan="2" valign="top"> <p align="center"><strong> TRAITS </strong></p>
</td><td width="102" valign="top"> <p><strong>&nbsp;%PRESENT </strong></p>
</td><td width="126" valign="top"> <p><strong> VALUES </strong></p>
</td></tr>
<tr>
<td colspan="2" valign="top"> <p>&nbsp; </p>
</td><td width="102" valign="top"> <p>&nbsp; </p>
</td><td width="126" valign="top"> <p>&nbsp; </p>
</td></tr>
<tr>
<td width="23"> <p>&nbsp; </p>
</td><td width="241"> Yield (t/ha)
</td><td width="102"> <p> 70% </p>
</td><td width="126"> <p>&gt; 2.5 &lt; 3.7 t/ha </p>
</td></tr>
<tr>
<td> <p>&nbsp;  </p>
</td><td> Plant height (cm)
</td><td width="102"> <p>63% </p>
</td><td width="126"> <p>&gt; 10 &lt; 20 cm </p>
</td></tr>
<tr>
<td> <p>&nbsp; </p>
</td><td> Disease A
</td><td width="102"> <p>50% </p>
</td><td width="126"> <p>&nbsp; </p>
</td></tr>
<tr>
<td> <p>&nbsp;  </p>
</td><td> Yield (grams/plot)
</td><td width="102"> <p>80% </p>
</td><td width="126"> <p>&nbsp; </p>
</td></tr>
<tr>
<td> <p>&nbsp;  </p>
</td><td> Yield (Kg/ha)
</td><td width="102"> <p>20% </p>
</td><td width="126"> <p>&nbsp; </p>
</td></tr></table>
<p>Values can be specified through an interface similar to the “Recommended lines” web interface designed by Alex Cosico (<a href="http://www.iris.irri.org/nursery/views/nursery.jsp" class="external free" title="http://www.iris.irri.org/nursery/views/nursery.jsp" rel="nofollow">http://www.iris.irri.org/nursery/views/nursery.jsp</a>)
</p>
<a name="Query_Design_and_User_Interface_Refinement" id="Query_Design_and_User_Interface_Refinement"></a><h4> <span class="mw-headline">Query Design and User Interface Refinement</span></h4>
<p>Crosstab queries required to summarise data by germplasm and study/trial
</p>
<a name="By_trait" id="By_trait"></a><h4> <span class="mw-headline">By trait</span></h4>
<table border="1" cellpadding="0" cellspacing="0">
<tr>
<td valign="top"> <p>&nbsp; </p>
</td><td colspan="5" valign="top"> <p>Trial ID </p>
</td></tr>
<tr>
<td valign="top"> <p>Trait name </p>
</td><td width="40" valign="top"> <p>1 </p>
</td><td width="40" valign="top"> <p>2 </p>
</td><td width="40" valign="top"> <p>3 </p>
</td><td width="40" valign="top"> <p>4 </p>
</td><td width="40" valign="top"> <p>5 </p>
</td></tr>
<tr>
<td valign="top"> <p>Grain Yield (t/ha) </p>
</td><td width="40" valign="top"> <p> Y </p>
</td><td width="40" valign="top"> <p> Y </p>
</td><td width="40" valign="top"> <p> Y </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td><td width="40" valign="top"> <p> Y </p>
</td></tr>
<tr>
<td valign="top"> <p>Grain Yield (kg/ha) </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td><td width="40" valign="top"> <p> Y </p>
</td><td width="40" valign="top"> <p> Y </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td></tr>
<tr>
<td valign="top"> <p>Plant height (cm) </p>
</td><td width="40" valign="top"> <p> Y </p>
</td><td width="40" valign="top"> <p> Y</p>
</td><td width="40" valign="top"> <p> Y </p>
</td><td width="40" valign="top"> Y
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td></tr>
<tr>
<td valign="top"> <p>Disease A </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td><td width="40" valign="top"> <p> Y </p>
</td><td width="40" valign="top"> <p> Y </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td><td width="40" valign="top"> <p> Y </p>
</td></tr>
<tr>
<td valign="top"> <p>Disease B </p>
</td><td width="40" valign="top"> <p> Y </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td><td width="40" valign="top"> <p> Y </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td><td width="40" valign="top"> <p> Y </p>
</td></tr></table>
<a name="By_germplasm" id="By_germplasm"></a><h4> <span class="mw-headline">By germplasm</span></h4>
<table border="1" cellpadding="0" cellspacing="0">
<tr>
<td valign="top"> <p>&nbsp; </p>
</td><td colspan="5" valign="top"> <p>Trial ID </p>
</td></tr>
<tr>
<td valign="top"> <p>GID </p>
</td><td width="40" valign="top"> <p>1 </p>
</td><td width="40" valign="top"> <p>2 </p>
</td><td width="40" valign="top"> <p>3 </p>
</td><td width="40" valign="top"> <p>4 </p>
</td><td width="40" valign="top"> <p>5 </p>
</td></tr>
<tr>
<td valign="top"> <p>Hartog&nbsp;: </p>
</td><td width="40" valign="top"> <p> Y </p>
</td><td width="40" valign="top"> <p> Y </p>
</td><td width="40" valign="top"> <p> Y </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td><td width="40" valign="top"> <p> Y </p>
</td></tr>
<tr>
<td valign="top"> <p>Sunco </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td><td width="40" valign="top"> <p> Y </p>
</td><td width="40" valign="top"> <p> Y </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td></tr>
<tr>
<td valign="top"> <p>Baviacora </p>
</td><td width="40" valign="top"> <p> Y </p>
</td><td width="40" valign="top"> <p> Y </p>
</td><td width="40" valign="top"> <p> Y </p>
</td><td width="40" valign="top"> <p> Y </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td></tr>
<tr>
<td valign="top"> <p>Yallaroi </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td><td width="40" valign="top"> <p> Y </p>
</td><td width="40" valign="top"> <p> Y </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td><td width="40" valign="top"> <p> Y </p>
</td></tr>
<tr>
<td valign="top"> <p>Tamaroi </p>
</td><td width="40" valign="top"> <p> Y </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td><td width="40" valign="top"> <p> Y </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td><td width="40" valign="top"> <p> Y </p>
</td></tr></table>
<a name="Occurrence_table" id="Occurrence_table"></a><h4> <span class="mw-headline">Occurrence table</span></h4>
<p>indicates what proportion of one particular study occurs in another study (lines or germplasm)
</p>
<table width="291" border="1" cellpadding="0" cellspacing="0">
<tr>
<td width="97" valign="top"> <p>&nbsp; </p>
</td><td colspan="5" valign="top"> <p>Year or study </p>
</td></tr>
<tr>
<td valign="top"> <p>Year or study </p>
</td><td width="40" valign="top"> <p>1 </p>
</td><td width="40" valign="top"> <p>2 </p>
</td><td width="40" valign="top"> <p>3 </p>
</td><td width="40" valign="top"> <p>4 </p>
</td><td width="40" valign="top"> <p>5 </p>
</td></tr>
<tr>
<td valign="top"> <p>1 </p>
</td><td width="40" valign="top"> <p> -- </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td></tr>
<tr>
<td valign="top"> <p>2 </p>
</td><td width="40" valign="top"> <p>10% </p>
</td><td width="40" valign="top"> <p> -- </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td></tr>
<tr>
<td valign="top"> <p>3 </p>
</td><td width="40" valign="top"> <p>10% </p>
</td><td width="40" valign="top"> <p>14% </p>
</td><td width="40" valign="top"> <p> -- </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td></tr>
<tr>
<td valign="top"> <p>4 </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td><td width="40" valign="top"> <p>7% </p>
</td><td width="40" valign="top"> <p>15% </p>
</td><td width="40" valign="top"> <p> -- </p>
</td><td width="40" valign="top"> <p>&nbsp; </p>
</td></tr>
<tr>
<td valign="top"> <p>5 </p>
</td><td width="40" valign="top"> <p>2% </p>
</td><td width="40" valign="top"> <p>5% </p>
</td><td width="40" valign="top"> <p>21% </p>
</td><td width="40" valign="top"> <p>3% </p>
</td><td width="40" valign="top"> <p> -- </p>
</td></tr></table>
<a name="Features_required" id="Features_required"></a><h4> <span class="mw-headline">Features required</span></h4>
<ul><li><b>SETGEN:</b>  Merge lists
</li><li><b>WORKBOOK:</b>  Download a whole study (or dataset within study)
</li><li><b>WEB INTERFACE:</b> download whole study (or dataset within study)
</li><li><b>DMS:</b>  Give names to datasets for more meaningful descriptions to help downloads.
</li></ul>
<hr />
<a name="Thursday_22nd_March_2007" id="Thursday_22nd_March_2007"></a><h3> <span class="mw-headline">Thursday 22nd March 2007 </span></h3>
<a name="Recap_of_decisions_taken_so_far:" id="Recap_of_decisions_taken_so_far:"></a><h4> <span class="mw-headline">Recap of decisions taken so far:</span></h4>
<p>Queries needed for GP and Characterisation data.
</p>
<ol><li>Given a Germplasm List and Clone list -&gt; produce genotype data
</li><li>Given a germplams list and trait list and study list -&gt; produce phenotype data
</li><li>Merge A and B on GID?
</li><li>&sect; List management
<ol><li>Access germplam list from Central
</li><li>Access germplasm list from text
</li><li>Access germplasm list from local
</li><li>Appending and merging (distinct values only) germplasm lists
</li><li>Management of study lists – introduce folder system for studies
</li><li>Management of trait lists (ontology)
</li></ol>
</li><li>&sect; Data set download and description of datasets
</li><li>Content description
<ol><li>Given a germplasm list –
<ol><li>What studies have evaluated lines?
</li><li>What clones have been genotyped?
</li><li>What traits have been measured?
</li></ol>
</li><li>Tabulate GID vs study
</li><li>Tabulate co-occurrence – how many genotypesa are evaluated in each pair of studies (for a trait or any or all of a set of traits)
</li></ol>
</li><li>A pedigree view and Download Tool
</li></ol>
<p>&sect; These are a priority for CAGE project at UQ.
</p>
<a name="Molecular_selection" id="Molecular_selection"></a><h4> <span class="mw-headline">Molecular selection</span></h4>
<p>Graham McLaren&nbsp;: MOSEL – molecular selection tool to facilitate marker assisted breeding.  It will access pedigree, genotype and phenotype data for test and reference lines (ancestors).
There are 3 types of neighbourhoods
</p>
<ol><li>Progenitors, derivatives and maintenance
</li><li>Founders&nbsp;: earliest genotyped ancestors from which all others ancestors and test lines have been produced
</li><li>Genetic info: about loci provided through a map dataset.  Map dat may specify QTL and gene information.
</li></ol>
<p><b>Allele Index</b>
</p>
<table border="1" cellpadding="0" cellspacing="0">
<tr>
<td width="70" valign="top"> <p>LOCUS </p>
</td><td width="70" valign="top"> <p>ALLELE </p>
</td><td width="70" valign="top"> <p>FOUNDER </p>
</td><td width="70" valign="top"> <p>TARGET </p>
</td></tr>
<tr>
<td width="70" valign="top"> <p>1 </p>
</td><td width="70" valign="top"> <p>A </p>
</td><td width="70" valign="top"> <p>1 </p>
</td><td width="70" valign="top"> <p>Y </p>
</td></tr>
<tr>
<td width="70" valign="top"> <p>1 </p>
</td><td width="70" valign="top"> <p>B </p>
</td><td width="70" valign="top"> <p>1 </p>
</td><td width="70" valign="top"> <p>N </p>
</td></tr>
<tr>
<td width="70" valign="top"> <p>1 </p>
</td><td width="70" valign="top"> <p>A </p>
</td><td width="70" valign="top"> <p>2 </p>
</td><td width="70" valign="top"> <p>Y </p>
</td></tr>
<tr>
<td width="70" valign="top"> <p>1 </p>
</td><td width="70" valign="top"> <p>C </p>
</td><td width="70" valign="top"> <p>2 </p>
</td><td width="70" valign="top"> <p>N </p>
</td></tr>
<tr>
<td width="70" valign="top"> <p>2 </p>
</td><td width="70" valign="top"> <p>M </p>
</td><td width="70" valign="top"> <p>1 </p>
</td><td width="70" valign="top"> <p>N </p>
</td></tr>
<tr>
<td width="70" valign="top"> <p>2 </p>
</td><td width="70" valign="top"> <p>N </p>
</td><td width="70" valign="top"> <p>1 </p>
</td><td width="70" valign="top"> <p>N </p>
</td></tr>
<tr>
<td width="70" valign="top"> <p>2 </p>
</td><td width="70" valign="top"> <p>P </p>
</td><td width="70" valign="top"> <p>2 </p>
</td><td width="70" valign="top"> <p>Y </p>
</td></tr>
<tr>
<td width="70" valign="top"> <p>&hellip;. </p>
</td><td width="70" valign="top"> <p>&nbsp; </p>
</td><td width="70" valign="top"> <p>&nbsp; </p>
</td><td width="70" valign="top"> <p>&nbsp; </p>
</td></tr></table>
<p><br />
</p>
<table border="1" cellpadding="0" cellspacing="0">
<tr>
<td width="100" rowspan="2" valign="top"> <p>GermPID </p><p>GID EntNo </p>
</td><td width="100" rowspan="2" valign="top"> <p>Selection </p>
</td><td colspan="4" valign="top"> <p>Target Loci </p>
</td><td width="150" rowspan="2" valign="top"> <p>Loci in founder 2 </p><p>Which carry the target </p><p>L F2, Lr, Ls, Lt </p>
</td><td rowspan="2" valign="top"> <p>Founder 3 </p>
</td></tr>
<tr>
<td width="35" valign="top"> <p>L1 </p>
</td><td width="35" valign="top"> <p>L2 </p>
</td><td width="35" valign="top"> <p>L3 </p>
</td><td width="35" valign="top"> <p>L4 </p>
</td></tr>
<tr>
<td width="100" valign="top"> <p>&nbsp; </p>
</td><td width="100" valign="top"> <p>&nbsp; </p>
</td><td width="35" valign="top"> <p>&nbsp; </p>
</td><td width="35" valign="top"> <p>&nbsp; </p>
</td><td width="35" valign="top"> <p>&nbsp; </p>
</td><td width="35" valign="top"> <p>&nbsp; </p>
</td><td width="150" valign="top"> <p>&nbsp; </p>
</td><td valign="top"> <p>&nbsp; </p>
</td></tr>
<tr>
<td width="100" valign="top"> <p>Test line </p>
</td><td width="100" valign="top"> <p>&nbsp; </p>
</td><td width="35" valign="top"> <p> &uml; X </p>
</td><td width="35" valign="top"> <p>&nbsp; </p>
</td><td width="35" valign="top"> <p>&nbsp; </p>
</td><td width="35" valign="top"> <p>&nbsp; </p>
</td><td width="150" valign="top"> <p>&nbsp; </p>
</td><td valign="top"> <p>&nbsp; </p>
</td></tr></table>
<p>X= distance of the line to the target
<b>Additional information:</b>
R project:  <a href="http://cran.at.r-project.org/" class="external free" title="http://cran.at.r-project.org/" rel="nofollow">http://cran.at.r-project.org/</a>    Heat plus package
</p><p>Graphviz and GGobi (<a href="http://www.ggobi.org" class="external free" title="http://www.ggobi.org" rel="nofollow">http://www.ggobi.org</a>) – tools for visualisation
</p><p><br />
</p>
<hr />
<a name="Friday_23rd_March_2007" id="Friday_23rd_March_2007"></a><h3> <span class="mw-headline">Friday 23rd March 2007 </span></h3>
<p>Meeting with ITS representative Gavin Fuller.
Gavin Fuller from ITS (Information Technology Services) joined the meeting tp amswer some questions from Thomas about the ITS server environment.
</p><p>ITS prefer UNIX to Windows and MySQL to PostgreSQL.
Gavin to get back to us with the questions raised.
</p><p>Additional discussions about how to synchronise different levels of Global/Central/Provincial Central/Local databases.
We need to have a system where difference between databases is detected and extracted and sent to the administrator for review.
</p><p>More on this discussion to be included in ICIS 2007?
</p>
<hr />
<p><a href="../../../../articles/r/-/s/R-script_for_preliminary_data_analysis.html" title="R-script for preliminary data analysis">R-script for preliminary data analysis</a>
</p>
<!-- 
NewPP limit report
Preprocessor node count: 226/1000000
Post-expand include size: 0/2097152 bytes
Template argument size: 0/2097152 bytes
Expensive parser function count: 0/100
-->
<div class="printfooter">
Retrieved from "<a href="http://localhost../../../../articles/m/i/n/Mini-workshop_2007.html">http://localhost../../../../articles/m/i/n/Mini-workshop_2007.html</a>"</div>
						<!-- end content -->
						<div class="visualClear"></div>
		</div>
	</div>
		</div>
		<div id="column-one">
	<div id="p-cactions" class="portlet">
		<h5>Views</h5>
		<div class="pBody">
			<ul>
	
				 <li id="ca-nstab-main" class="selected"><a href="../../../../articles/m/i/n/Mini-workshop_2007.html" title="View the content page [c]" accesskey="c">Page</a></li>
				 <li id="ca-talk" class="new"><a href="../../../../articles/m/i/n/Talk%7EMini-workshop_2007_708f.html" title="Discussion about the content page [t]" accesskey="t">Discussion</a></li>
				 <li id="ca-viewsource"><a href="../../../../articles/m/i/n/Mini-workshop_2007.html" title="This page is protected.&#10;You can view its source [e]" accesskey="e">View source</a></li>
				 <li id="ca-history"><a href="../../../../articles/m/i/n/Mini-workshop_2007.html" title="Past versions of this page [h]" accesskey="h">History</a></li>			</ul>
		</div>
	</div>
	<div class="portlet" id="p-personal">
		<h5>Personal tools</h5>
		<div class="pBody">
			<ul>
				<li id="pt-login"><a href="../../../../articles/u/s/e/Special%7EUserLogin_94cd.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</a></li>
			</ul>
		</div>
	</div>
	<div class="portlet" id="p-logo">
		<a style="background-image: url(../../../../misc/icis-wiki3.png);" href="../../../../index.html" title="Visit the main page [z]" accesskey="z"></a>
	</div>
	<script type="text/javascript"> if (window.isMSIE55) fixalpha(); </script>
	<div class='generated-sidebar portlet' id='p-navigation'>
		<h5>Navigation</h5>
		<div class='pBody'>
			<ul>
				<li id="n-mainpage"><a href="../../../../index.html" title="Visit the main page [z]" accesskey="z">Main Page</a></li>
				<li id="n-portal"><a href="../../../../articles/c/o/m/ICISWiki%7ECommunity_Portal_7449.html" title="About the project, what you can do, where to find things">Community portal</a></li>
				<li id="n-recentchanges"><a href="../../../../articles/r/e/c/Special%7ERecentChanges_e0d0.html" title="The list of recent changes in the wiki [r]" accesskey="r">Recent changes</a></li>
				<li id="n-help"><a href="../../../../articles/c/o/n/Help%7EContents_22de.html" title="The place to find out">Help</a></li>
			</ul>
		</div>
	</div>
	<div id="p-search" class="portlet">
		<h5><label for="searchInput">Search</label></h5>
		<div id="searchBody" class="pBody">
			<form action="../../../../articles/s/e/a/Special%7ESearch_a8e5.html" id="searchform"><div>
				<input id="searchInput" name="search" type="text" title="Search ICISWiki [f]" accesskey="f" value="" />
				<input type='submit' name="go" class="searchButton" id="searchGoButton"	value="Go" title="Go to a page with this exact name if exists" />&nbsp;
				<input type='submit' name="fulltext" class="searchButton" id="mw-searchButton" value="Search" title="Search the pages for this text" />
			</div></form>
		</div>
	</div>
	<div class="portlet" id="p-tb">
		<h5>Toolbox</h5>
		<div class="pBody">
			<ul>
				<li id="t-whatlinkshere"><a href="../../../../articles/w/h/a/Special%7EWhatLinksHere_Mini-workshop_2007_5ec4.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</a></li>
				<li id="t-recentchangeslinked"><a href="../../../../articles/r/e/c/Special%7ERecentChangesLinked_Mini-workshop_2007_f845.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</a></li>
<li id="t-specialpages"><a href="../../../../articles/s/p/e/Special%7ESpecialPages_f12b.html" title="List of all special pages [q]" accesskey="q">Special pages</a></li>
				<li id="t-print"><a href="../../../../articles/m/i/n/Mini-workshop_2007.html" title="Printable version of this page [p]" accesskey="p">Printable version</a></li>				<li id="t-permalink"><a href="../../../../articles/m/i/n/Mini-workshop_2007.html" title="Permanent link to this version of the page">Permanent link</a></li>			</ul>
		</div>
	</div>
		</div><!-- end of the left (by default at least) column -->
			<div class="visualClear"></div>
			<div id="footer">
				<div id="f-poweredbyico"><a href="http://www.mediawiki.org/"><img src="../../../../skins/common/images/poweredby_mediawiki_88x31.png" alt="Powered by MediaWiki" /></a></div>
				<div id="f-copyrightico"><a href="../../../../../COPYING.html"><img src="/icis/skins/common/images/gnu-fdl.png" alt='GNU Free Documentation License 1.2' /></a></div>
			<ul id="f-list">
					<li id="credits">This page was last modified 20:10, 2 May 2008 by <a href="../../../../articles/c/a/s/User%7ECasperNUN_85dd.html" title="User:CasperNUN">Casper aan den Boom</a>. Based on work by <a href="../../../../articles/r/v/a/User%7ERvalerio_c9de.html" title="User:Rvalerio">Rowena Valerio</a>,&#32;<a href="../../../../articles/v/2E/a/User%7EV.arief_9e2b.html" class="new" title="User:V.arief (page does not exist)">Vivi Arief</a>,&#32;<a href="../../../../articles/s/a/n/User%7ESandraM_bee9.html" class="new" title="User:SandraM (page does not exist)">Sandra Micallef</a>&#32;and&#32;<a href="../../../../articles/t/m/e/User%7ETmetz_c741.html" title="User:Tmetz">Thomas Metz</a>.</li>
					<li id="copyright">Content is available under <a href="../../../../../COPYING.html" class="external " title="../../../../../COPYING.html">GNU Free Documentation License 1.2</a>.</li>
					<li id="privacy"><a href="../../../../articles/p/r/i/ICISWiki%7EPrivacy_policy_0cbf.html" title="ICISWiki:Privacy policy">Privacy policy</a></li>
					<li id="about"><a href="../../../../articles/a/b/o/ICISWiki%7EAbout_53aa.html" title="ICISWiki:About">About ICISWiki</a></li>
					<li id="disclaimer"><a href="../../../../articles/g/e/n/ICISWiki%7EGeneral_disclaimer_9129.html" title="ICISWiki:General disclaimer">Disclaimers</a></li>
			</ul>
		</div>
</div>

		<script type="text/javascript">if (window.runOnloadHook) runOnloadHook();</script>
<!-- Served in 129.650 secs. --></body></html>
